{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Gerando arquivo para NER-NestedClinBr\n",
    "Deixar sempre o arquivo de test para fazer o teste completo (com o pipeline todo)\n",
    "1o. -> passar pelo modelo de NER normal (binario), pegar possiveis entidades\n",
    "2o. -> combinacao de palavras, modelo de span\n",
    "\n",
    "Descontinuas: treinar duplicando as frases, tirando os termos do meio.. na hora da predicao: se duas entidades estão mito proximas (ou no mesmo chunk), tirar o espaço e ver se são (ou tentar achar uma regra, ex sempre q tem tratamento e uma anotmia perto, usar uma janela.. ex duas palavras de distancia)...\n",
    "\n",
    "Para tese: testar tbm sem as descontinuas.. pra pode comparar com outros metodos...\n",
    "\n",
    "v2 - com corpus original Tempclinbr (arq .txt), contendo as quebras de linha.. aqui quebrar cada frase do texto em uma entrada no dicionario.\n",
    "\n",
    "v3 - split de train em 75% para dev.. conforme feito no TempClinBr.. e tbm tirar os 41 primeiros tokens dos textos: \n",
    "'Data de Criação do Documento: 22/04/2014' como codigo Joao"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "from pathlib import Path\n",
    "import re\n",
    "import pickle\n",
    "import random\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "def getTiposEntidade():\n",
    "    return ['Problema','Teste','Tratamento','Anatomia']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "def replaceWhiteSpaces(str):\n",
    "    return re.sub('\\s{2,}',' ',str)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def save_obj(name, obj):\n",
    "    existeDir = os.path.exists('../obj')\n",
    "    if not existeDir:\n",
    "        os.makedirs('../obj')\n",
    "    with open('../obj/'+ name + '.pkl', 'wb') as f:\n",
    "        pickle.dump(obj, f)\n",
    "\n",
    "def load_obj(name):\n",
    "    existeDir = os.path.exists('../obj')\n",
    "    if not existeDir:\n",
    "        os.makedirs('../obj')\n",
    "    try:\n",
    "        with open('../obj/' + name + '.pkl', 'rb') as f:\n",
    "            return pickle.load(f)\n",
    "    except:\n",
    "        return None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 177,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 177,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "greeting = ['hello','my','name','is','bob','how','are','you','my','name','is']\n",
    "\n",
    "def getIndices(l, sl, indiceInicio):\n",
    "    results=[]\n",
    "    sl = sl.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \").split()\n",
    "    l=l.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \").split()\n",
    "    sll=len(sl)\n",
    "    for ind in (i for i,e in enumerate(l) if e==sl[0]):\n",
    "        if l[ind:ind+sll]==sl:\n",
    "            results.append((ind,ind+sll-1))\n",
    "\n",
    "    lista_indices=[]\n",
    "    for result in results:\n",
    "        inicio, fim = result\n",
    "        if inicio>indiceInicio:\n",
    "            for i in range(inicio,fim+1):\n",
    "                lista_indices.append(i)\n",
    "            return lista_indices\n",
    "    return []\n",
    "\n",
    "getIndices('Medicamentos atenolol 100mg, 12-12h,','atenolol 100mg', 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "'''\n",
    "def getDicSentences(pastaCorpus):\n",
    "    devePrintar=True\n",
    "    #devePrintar=False\n",
    "    dic_sentences = {}\n",
    "    numMaxTokensPorFrase=0 # numero tokens da maior frase\n",
    "    num=0\n",
    "    listaEntidades=[]\n",
    "    frasesComDescontinuas=[]\n",
    "    for filename in os.listdir(pastaCorpus):\n",
    "        f = os.path.join(pastaCorpus, filename)\n",
    "        if os.path.isfile(f):\n",
    "            fileNameSemExtensao=os.path.splitext(filename)[0]\n",
    "            if devePrintar:\n",
    "                print('\\n\\n--fileName (sem extensao):--', fileNameSemExtensao)\n",
    "            extension = os.path.splitext(filename)[1][1:]\n",
    "            if extension=='ann':\n",
    "                # tokens da frase\n",
    "                fileTxt = open(os.path.join(pastaCorpus, fileNameSemExtensao)+'.txt', \"r\", encoding='utf-8')\n",
    "                linha=fileTxt.readlines()\n",
    "                fileTxt.close()\n",
    "                if devePrintar:\n",
    "                    print('linha:', linha)\n",
    "                frases=[]\n",
    "                numL=0\n",
    "                for l in linha:\n",
    "                    numL=numL+1\n",
    "                    if numL==1: # descarta primeira frase, que é data de criação do doc\n",
    "                        #print('descartando frase:', l)\n",
    "                        continue\n",
    "                    if l.strip() and l.strip()!='\\n':\n",
    "                        l2 = l.split('.') # quebrando frases\n",
    "                        for l3 in l2:\n",
    "                            if l3.strip() and l3.strip()!='\\n':\n",
    "                                novaFrase = l3.replace('\\n','').strip()+'.'\n",
    "                                frases.append(novaFrase)\n",
    "                #print('frases:', frases)\n",
    "                # para cada frase\n",
    "                # para tokenizar nesses tokens\n",
    "                frasesTokens={}\n",
    "                for frase in frases:\n",
    "                    tokens=[]\n",
    "                    frase2 = frase.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                    frase2 = replaceWhiteSpaces(frase2)\n",
    "                    frase2 = frase2.split()\n",
    "                    for numtoken, token in enumerate(frase2):\n",
    "                        tokens.append([token,numtoken])\n",
    "                        if numMaxTokensPorFrase<len(tokens):\n",
    "                            numMaxTokensPorFrase = len(tokens)\n",
    "                    frasesTokens[frase]=tokens\n",
    "                    #linhaTokens=linha.copy()    \n",
    "                #print('frasesTokens:', frasesTokens)\n",
    "                # agora, as entidades\n",
    "                fileAnn = open(f, \"r\", encoding='utf-8')\n",
    "                linha=fileAnn.readlines()\n",
    "                fileAnn.close()\n",
    "\n",
    "                dicEntidades={}\n",
    "                for entidade_linha in linha:\n",
    "                    if ';' not in entidade_linha: # tem descontinua?\n",
    "                        entidade = entidade_linha.split('\\t')\n",
    "                        tipo_entidade = entidade[1]\n",
    "                        inicio, fim = tipo_entidade.split()[1:3]\n",
    "                        tipo_entidade = tipo_entidade.split()[0]\n",
    "                        termos_entidade = entidade[2].replace('\\n','')\n",
    "                        termos_entidade = termos_entidade.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                        dicEntidades[(int(inicio), int(fim))]=[tipo_entidade, replaceWhiteSpaces(termos_entidade)]\n",
    "                    else:\n",
    "                        frasesComDescontinuas.append(num)\n",
    "                        #print('descontinua')\n",
    "                        entidade = entidade_linha.split('\\t')\n",
    "                        # ex T10\tProblema 244 252;279 306\tdispneia aos mdoeardos-leves esforço\n",
    "                        #Problema 244 252;279 306\n",
    "                        entidade_temp=entidade[1].split(';')\n",
    "                        entidade1=entidade_temp[0]\n",
    "                        tipo_entidade = entidade1\n",
    "                        inicio1, fim1 = tipo_entidade.split()[1:3]\n",
    "                        tipo_entidade_string = tipo_entidade.split()[0]\n",
    "                        # mandar só os termos referentes...\n",
    "                        tamTermo1=int(fim1)-int(inicio1)\n",
    "                        termos_entidade = entidade[2].replace('\\n','')\n",
    "                        termos_entidade=termos_entidade[:tamTermo1]\n",
    "                        termos_entidade = termos_entidade.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                        dicEntidades[(int(inicio1), int(fim1))]=[tipo_entidade_string, replaceWhiteSpaces(termos_entidade)]\n",
    "\n",
    "                        entidade2=entidade_temp[1]\n",
    "                        inicio2, fim2 = entidade2.split()[0:2]\n",
    "                        termos_entidade = entidade[2].replace('\\n','')\n",
    "                        termos_entidade=termos_entidade[tamTermo1:len(termos_entidade)]\n",
    "                        termos_entidade = termos_entidade.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                        dicEntidades[(int(inicio2), int(fim2))]=[tipo_entidade_string, replaceWhiteSpaces(termos_entidade)]\n",
    "\n",
    "\n",
    "                #print('frasesTokens:', frasesTokens)\n",
    "                indicesDic = sorted(dicEntidades.keys(), key = lambda item: item[0])\n",
    "                listaIndicesJaUsados = []\n",
    "                #list_students.sort(key = lambda x: x[1])   #index 1 means second element\n",
    "                #print(indicesDic)\n",
    "                #print('dicEntidades:', dicEntidades)\n",
    "                \n",
    "                for key, value in frasesTokens.items():\n",
    "                    indiceAnterior=-1\n",
    "                    if devePrintar:\n",
    "                        print('key:', key)\n",
    "                        print('indicesDic:', indicesDic)\n",
    "                    for i in indicesDic:\n",
    "                        if devePrintar:\n",
    "                            print('i:',i)\n",
    "                            print('indiceAnterior:', indiceAnterior)\n",
    "                            print('dicEntidades[i]:',dicEntidades[i])\n",
    "                        tipo_entidade,termos_entidade = dicEntidades[i]\n",
    "                        #verificar em todas frases do texto\n",
    "                        novos_indices=getIndices(key, termos_entidade, indiceAnterior)\n",
    "                        if devePrintar:\n",
    "                            print('novos_indices:', novos_indices)\n",
    "                        #if len(novos_indices)==0:\n",
    "                        #    continue # vai pra proxima entidade\n",
    "                        #else:\n",
    "                        #    print('achouuuuuuuuuu')\n",
    "                        if len(novos_indices)!=0:\n",
    "                            if novos_indices[0]==0 and len(novos_indices)>1:\n",
    "                                indiceAnterior=novos_indices[0]+1\n",
    "                            else:\n",
    "                                indiceAnterior=novos_indices[0]\n",
    "                            #idx = sentence.index(\"x\")\n",
    "                            listaEntidades.append([termos_entidade, novos_indices, tipo_entidade])\n",
    "                            listaIndicesJaUsados.append(i)\n",
    "                            #print('removendo iiiiiiiiiii:',i)\n",
    "                            #print('indiceAnterior:', indiceAnterior)\n",
    "                            \n",
    "\n",
    "                    for indicesUsados in listaIndicesJaUsados:\n",
    "                        #print('indicesUsados::::::::::', indicesUsados)\n",
    "                        indicesDic.remove(indicesUsados)\n",
    "                    listaIndicesJaUsados = []\n",
    "                    if len(value)>0:\n",
    "                        #print('incluindo:', key)\n",
    "                        dic_sentences[num]=[value, listaEntidades]\n",
    "                        listaEntidades=[]\n",
    "                        tokens=[]\n",
    "                        num=num+1\n",
    "\n",
    "        #print(num)\n",
    "        #if num>318:\n",
    "        #    break\n",
    "\n",
    "        #if num>3:\n",
    "        #    break\n",
    "    print('numMaxTokensPorFrase:', numMaxTokensPorFrase)\n",
    "    return dic_sentences, frasesComDescontinuas\n",
    "            \n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 212,
   "metadata": {},
   "outputs": [],
   "source": [
    "#pastasCorpus = [r'corpus\\test',r'corpus\\train']\n",
    "#pastasCorpus = [r'corpus\\test']\n",
    "pastasCorpus = [r'corpus\\TESTE']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 321,
   "metadata": {},
   "outputs": [],
   "source": [
    "def getDicSentences(pastaCorpus):\n",
    "    devePrintar=True\n",
    "    #devePrintar=False\n",
    "    dic_sentences = {}\n",
    "    numMaxTokensPorFrase=0 # numero tokens da maior frase\n",
    "    num=0\n",
    "    listaEntidades=[]\n",
    "    frasesComDescontinuas=[]\n",
    "    for filename in os.listdir(pastaCorpus):\n",
    "        f = os.path.join(pastaCorpus, filename)\n",
    "        if os.path.isfile(f):\n",
    "            fileNameSemExtensao=os.path.splitext(filename)[0]\n",
    "            if devePrintar:\n",
    "                #print('\\n\\n--fileName (sem extensao):--', fileNameSemExtensao)\n",
    "                pass\n",
    "            extension = os.path.splitext(filename)[1][1:]\n",
    "            if extension=='ann':\n",
    "                # tokens da frase\n",
    "                fileTxt = open(os.path.join(pastaCorpus, fileNameSemExtensao)+'.txt', \"r\", encoding='utf-8')\n",
    "                linha=fileTxt.readlines()\n",
    "                fileTxt.close()\n",
    "                if devePrintar:\n",
    "                    #print('linha:', linha)\n",
    "                    pass\n",
    "                frases=[]\n",
    "                allFrasesString=''\n",
    "                numL=0\n",
    "                for l in linha:\n",
    "                    numL=numL+1\n",
    "                    allFrasesString = allFrasesString+l\n",
    "                    if numL==1: # descarta primeira frase, que é data de criação do doc\n",
    "                        #print('descartando frase:', l)\n",
    "                        continue\n",
    "                    if l.strip() and l.strip()!='\\n':\n",
    "                        l2 = l.split('.') # quebrando frases\n",
    "                        for l3 in l2:\n",
    "                            if l3.strip() and l3.strip()!='\\n':\n",
    "                                novaFrase = l3.replace('\\n','').strip()+'.'\n",
    "                                frases.append(novaFrase)\n",
    "                #print('frases:', frases)\n",
    "                # para cada frase\n",
    "                # para tokenizar nesses tokens\n",
    "                #print('allFrasesString:', allFrasesString)\n",
    "                frasesTokens={}\n",
    "                #numCaracteresTotal=42 # primeira frase ignorada\n",
    "                numIndiceAnterior=0\n",
    "                for frase in frases:\n",
    "                    tokens=[]\n",
    "                    frase2 = frase.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                    frase2 = replaceWhiteSpaces(frase2)\n",
    "                    frase2 = frase2.split()\n",
    "                    for numtoken, token in enumerate(frase2):\n",
    "                        #numCaracteresTotal=numCaracteresTotal+len(tokens)\n",
    "                        #print('token:', token)\n",
    "                        #print('numIndiceAnterior:', numIndiceAnterior)\n",
    "                        if token!='.':\n",
    "                            numCaracteresTotal = allFrasesString.find(token, numIndiceAnterior, len(allFrasesString))\n",
    "                        else:\n",
    "                            numCaracteresTotal=numIndiceAnterior+1\n",
    "                        #print('numCaracteresTotal:', numCaracteresTotal)\n",
    "                        #tokens.append([token,numtoken])\n",
    "                        tokens.append([token,numtoken, numCaracteresTotal])\n",
    "                        numIndiceAnterior = numCaracteresTotal\n",
    "                        if numMaxTokensPorFrase<len(tokens):\n",
    "                            numMaxTokensPorFrase = len(tokens)\n",
    "                    frasesTokens[frase]=tokens\n",
    "                    #linhaTokens=linha.copy()    \n",
    "                #print('frasesTokens:', frasesTokens)\n",
    "                # agora, as entidades\n",
    "                fileAnn = open(f, \"r\", encoding='utf-8')\n",
    "                linha=fileAnn.readlines()\n",
    "                fileAnn.close()\n",
    "\n",
    "                dicEntidades={}\n",
    "                for entidade_linha in linha:\n",
    "                    if ';' not in entidade_linha: # tem descontinua?\n",
    "                        entidade = entidade_linha.split('\\t')\n",
    "                        tipo_entidade = entidade[1]\n",
    "                        inicio, fim = tipo_entidade.split()[1:3]\n",
    "                        tipo_entidade = tipo_entidade.split()[0]\n",
    "                        termos_entidade = entidade[2].replace('\\n','')\n",
    "                        termos_entidade = termos_entidade.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                        dicEntidades[(int(inicio), int(fim))]=[tipo_entidade, replaceWhiteSpaces(termos_entidade)]\n",
    "                    else:\n",
    "                        frasesComDescontinuas.append(num)\n",
    "                        #print('descontinua')\n",
    "                        entidade = entidade_linha.split('\\t')\n",
    "                        # ex T10\tProblema 244 252;279 306\tdispneia aos mdoeardos-leves esforço\n",
    "                        #Problema 244 252;279 306\n",
    "                        entidade_temp=entidade[1].split(';')\n",
    "                        entidade1=entidade_temp[0]\n",
    "                        tipo_entidade = entidade1\n",
    "                        inicio1, fim1 = tipo_entidade.split()[1:3]\n",
    "                        tipo_entidade_string = tipo_entidade.split()[0]\n",
    "                        # mandar só os termos referentes...\n",
    "                        tamTermo1=int(fim1)-int(inicio1)\n",
    "                        termos_entidade = entidade[2].replace('\\n','')\n",
    "                        termos_entidade=termos_entidade[:tamTermo1]\n",
    "                        termos_entidade = termos_entidade.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                        dicEntidades[(int(inicio1), int(fim1))]=[tipo_entidade_string, replaceWhiteSpaces(termos_entidade)]\n",
    "\n",
    "                        entidade2=entidade_temp[1]\n",
    "                        inicio2, fim2 = entidade2.split()[0:2]\n",
    "                        termos_entidade = entidade[2].replace('\\n','')\n",
    "                        termos_entidade=termos_entidade[tamTermo1:len(termos_entidade)]\n",
    "                        termos_entidade = termos_entidade.strip().replace('/',' / ').replace(')',' ) ').replace('(',' ( ').replace(']',' ] ').replace('[',' [ ').replace(',',' , ').replace('.',' . ').replace(';',' ; ').replace('-',' - ').replace('+',' + ').replace(\"'\",\" ' \").replace(\" = \",\" = \").replace(\"#\",\" # \").replace(\" $ \",\" $ \").replace(\" ! \",\" ! \").replace(\"?\",\" ? \").replace(\"%\",\" % \").replace(\":\",\" : \").replace(\">\",\" > \").replace(\"<\",\" < \")\n",
    "                        dicEntidades[(int(inicio2), int(fim2))]=[tipo_entidade_string, replaceWhiteSpaces(termos_entidade)]\n",
    "\n",
    "\n",
    "                #print('frasesTokens:', frasesTokens)\n",
    "                indicesDic = sorted(dicEntidades.keys(), key = lambda item: item[0])\n",
    "                listaIndicesJaUsados = []\n",
    "                #list_students.sort(key = lambda x: x[1])   #index 1 means second element\n",
    "                #print(indicesDic)\n",
    "                #print('dicEntidades:', dicEntidades)\n",
    "                \n",
    "                for key, value in frasesTokens.items():\n",
    "                    for i in indicesDic:\n",
    "                        tipo_entidade,termos_entidade = dicEntidades[i]\n",
    "                        ##key (i) = indices old a comparar\n",
    "                        #print('key:', key)\n",
    "                        #print('value:', value)\n",
    "                        #print('i:', i)\n",
    "                        for token in value:\n",
    "                            if i[0]==token[2]:\n",
    "                                #print('token:', token)\n",
    "                                novo_inicio, novo_fim = [token[1],token[1]+len(termos_entidade.split())]\n",
    "                                novos_indices=[]\n",
    "                                for k in range(novo_inicio,novo_fim):\n",
    "                                    novos_indices.append(k)\n",
    "                                listaEntidades.append([termos_entidade, novos_indices, tipo_entidade])\n",
    "                        \n",
    "                    if len(value)>0:\n",
    "                        #print('incluindo:', key)\n",
    "                        dic_sentences[num]=[value, listaEntidades]\n",
    "                        listaEntidades=[]\n",
    "                        num=num+1 \n",
    "        #print(num)\n",
    "        #if num>318:\n",
    "        #    break\n",
    "\n",
    "        #if num>3:\n",
    "        #    break\n",
    "    print('numMaxTokensPorFrase:', numMaxTokensPorFrase)\n",
    "    return dic_sentences, frasesComDescontinuas\n",
    "            \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 322,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--Teste:-- corpus\\TESTE\n",
      "numMaxTokensPorFrase: 46\n",
      "len(sentences): 28\n",
      "len(Descontinuas): 0\n",
      "len(frasesComDescontinuas): 0\n"
     ]
    }
   ],
   "source": [
    "#print('--Treinamento:--', pastasCorpus[1])\n",
    "#dic_sentencesTrain, frasesComDescontinuasTrain = getDicSentences(pastasCorpus[1])\n",
    "#print('len(sentences):', len(dic_sentencesTrain))\n",
    "#print('len(Descontinuas):',len(frasesComDescontinuasTrain))\n",
    "#print('len(frasesComDescontinuas):',len(set(frasesComDescontinuasTrain)))\n",
    "\n",
    "print('--Teste:--', pastasCorpus[0])\n",
    "dic_sentencesTest, frasesComDescontinuasTest = getDicSentences(pastasCorpus[0])\n",
    "print('len(sentences):', len(dic_sentencesTest))\n",
    "print('len(Descontinuas):',len(frasesComDescontinuasTest))\n",
    "print('len(frasesComDescontinuas):',len(set(frasesComDescontinuasTest)))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 323,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{0: [[['Lucas', 0, 43],\n",
       "   [',', 1, 48],\n",
       "   ['57', 2, 50],\n",
       "   ['anos', 3, 53],\n",
       "   ['.', 4, 54]],\n",
       "  []],\n",
       " 1: [[['Homoenxerto', 0, 58],\n",
       "   ['aortico', 1, 70],\n",
       "   ['(', 2, 78],\n",
       "   ['por', 3, 80],\n",
       "   ['EAo', 4, 84],\n",
       "   [')', 5, 88],\n",
       "   ['+', 6, 90],\n",
       "   ['MIE', 7, 92],\n",
       "   ['-', 8, 96],\n",
       "   ['DA', 9, 98],\n",
       "   [';', 10, 100],\n",
       "   ['RAD', 11, 102],\n",
       "   ['-', 12, 106],\n",
       "   ['CD', 13, 108],\n",
       "   [';', 14, 110],\n",
       "   ['SAF', 15, 112],\n",
       "   ['-', 16, 116],\n",
       "   ['MG', 17, 118],\n",
       "   ['em', 18, 121],\n",
       "   ['2006', 19, 124],\n",
       "   ['-', 20, 129],\n",
       "   ['Doutor', 21, 131],\n",
       "   ['Vital', 22, 138],\n",
       "   ['Brazil', 23, 144],\n",
       "   ['.', 24, 145]],\n",
       "  [['Homoenxerto aortico', [0, 1, 2], 'Tratamento'],\n",
       "   ['EAo', [4, 5], 'Problema']]],\n",
       " 2: [[['Hipertenso', 0, 152], [',', 1, 162], ['.', 2, 163]],\n",
       "  [['Hipertenso', [0, 1], 'Problema']]],\n",
       " 3: [[['Em', 0, 164],\n",
       "   ['uso', 1, 167],\n",
       "   ['de', 2, 171],\n",
       "   [':', 3, 173],\n",
       "   ['AAS', 4, 175],\n",
       "   ['100', 5, 179],\n",
       "   ['mg', 6, 183],\n",
       "   ['dia', 7, 186],\n",
       "   [',', 8, 189],\n",
       "   ['losartana', 9, 191],\n",
       "   ['100', 10, 201],\n",
       "   ['mg', 11, 205],\n",
       "   ['dia', 12, 208],\n",
       "   [',', 13, 211],\n",
       "   ['atovastatina', 14, 213],\n",
       "   ['20', 15, 226],\n",
       "   ['mg', 16, 229],\n",
       "   ['4CP', 17, 232],\n",
       "   ['/', 18, 236],\n",
       "   ['DIA', 19, 238],\n",
       "   [',', 20, 241],\n",
       "   ['atenolol', 21, 243],\n",
       "   ['25mg', 22, 252],\n",
       "   ['/', 23, 257],\n",
       "   ['dia', 24, 259],\n",
       "   [',', 25, 262],\n",
       "   ['furosemida', 26, 264],\n",
       "   ['40mg', 27, 275],\n",
       "   ['/', 28, 280],\n",
       "   ['dia', 29, 282],\n",
       "   [',', 30, 285],\n",
       "   ['omeoprazol', 31, 287],\n",
       "   ['40mg', 32, 298],\n",
       "   ['ao', 33, 303],\n",
       "   ['dia', 34, 306],\n",
       "   [',', 35, 309],\n",
       "   ['ezatimibe', 36, 311],\n",
       "   ['10', 37, 321],\n",
       "   ['mg', 38, 324],\n",
       "   [',', 39, 326],\n",
       "   ['Hidantal', 40, 328],\n",
       "   ['100mg', 41, 337],\n",
       "   ['-', 42, 343],\n",
       "   ['por', 43, 345],\n",
       "   ['cisticercose', 44, 349],\n",
       "   ['.', 45, 350]],\n",
       "  [['AAS 100 mg', [4, 5, 6, 7], 'Tratamento'],\n",
       "   ['losartana 100 mg', [9, 10, 11, 12], 'Tratamento'],\n",
       "   ['atovastatina 20 mg', [14, 15, 16, 17], 'Tratamento'],\n",
       "   ['atenolol 25mg', [21, 22, 23], 'Tratamento'],\n",
       "   ['furosemida 40mg', [26, 27, 28], 'Tratamento'],\n",
       "   ['omeoprazol 40mg', [31, 32, 33], 'Tratamento'],\n",
       "   ['ezatimibe 10 mg', [36, 37, 38, 39], 'Tratamento'],\n",
       "   ['Hidantal 100mg', [40, 41, 42], 'Tratamento'],\n",
       "   ['cisticercose', [44, 45], 'Problema']]],\n",
       " 4: [[['Penicilina', 0, 363],\n",
       "   ['G', 1, 374],\n",
       "   ['benzatina', 2, 376],\n",
       "   ['1', 3, 386],\n",
       "   [',', 4, 387],\n",
       "   ['200', 5, 388],\n",
       "   [',', 6, 391],\n",
       "   ['000U', 7, 392],\n",
       "   ['IM', 8, 397],\n",
       "   ['a', 9, 400],\n",
       "   ['cada', 10, 402],\n",
       "   ['21', 11, 407],\n",
       "   ['dia', 12, 410],\n",
       "   ['-', 13, 414],\n",
       "   ['suspenso', 14, 416],\n",
       "   ['em', 15, 425],\n",
       "   ['2013', 16, 428],\n",
       "   ['.', 17, 429]],\n",
       "  [['Penicilina G benzatina 1 , 200 , 000U',\n",
       "    [0, 1, 2, 3, 4, 5, 6, 7, 8],\n",
       "    'Tratamento']]],\n",
       " 5: [[['DOR', 0, 433],\n",
       "   ['REGIAO', 1, 437],\n",
       "   ['ESTERNAL', 2, 444],\n",
       "   [',', 3, 452],\n",
       "   ['EM', 4, 454],\n",
       "   ['FO', 5, 457],\n",
       "   [',', 6, 459],\n",
       "   ['LOCAL', 7, 461],\n",
       "   ['FIOS', 8, 467],\n",
       "   ['AÇO', 9, 472],\n",
       "   [',', 10, 475],\n",
       "   ['EM', 11, 477],\n",
       "   ['QUEIMAÇÃO', 12, 480],\n",
       "   ['.', 13, 481]],\n",
       "  [['DOR REGIAO ESTERNAL', [0, 1, 2, 3], 'Problema'],\n",
       "   ['REGIAO ESTERNAL', [1, 2, 3], 'Anatomia']]],\n",
       " 6: [[['-', 0, 492],\n",
       "   ['Labs', 1, 494],\n",
       "   [':', 2, 498],\n",
       "   ['tg', 3, 500],\n",
       "   ['221', 4, 503],\n",
       "   ['/', 5, 507],\n",
       "   ['ct', 6, 509],\n",
       "   ['166', 7, 512],\n",
       "   ['/', 8, 516],\n",
       "   ['ldl', 9, 518],\n",
       "   ['76', 10, 522],\n",
       "   [',', 11, 524],\n",
       "   ['demais', 12, 526],\n",
       "   ['ok', 13, 533],\n",
       "   ['.', 14, 534]],\n",
       "  [['tg', [3, 4], 'Teste'],\n",
       "   ['ct', [6, 7], 'Teste'],\n",
       "   ['ldl', [9, 10], 'Teste']]],\n",
       " 7: [[['-', 0, 537],\n",
       "   ['Ecocardio', 1, 539],\n",
       "   ['12', 2, 549],\n",
       "   ['/', 3, 551],\n",
       "   ['08', 4, 552],\n",
       "   ['/', 5, 554],\n",
       "   ['15', 6, 555],\n",
       "   [':', 7, 557],\n",
       "   ['A0', 8, 559],\n",
       "   ['/', 9, 562],\n",
       "   ['VD', 10, 564],\n",
       "   ['22', 11, 567],\n",
       "   ['/', 12, 570],\n",
       "   ['Septo', 13, 572],\n",
       "   ['11', 14, 578],\n",
       "   ['/', 15, 581],\n",
       "   ['FE', 16, 583],\n",
       "   ['66', 17, 586],\n",
       "   ['.', 18, 587]],\n",
       "  [['Ecocardio', [1, 2], 'Teste'],\n",
       "   ['VD', [10, 11], 'Anatomia'],\n",
       "   ['Septo', [13, 14], 'Anatomia']]],\n",
       " 8: [[['Homoenxerto', 0, 590],\n",
       "   ['aortico', 1, 602],\n",
       "   ['com', 2, 610],\n",
       "   ['mobilidade', 3, 614],\n",
       "   ['ampla', 4, 625],\n",
       "   ['folhetps', 5, 631],\n",
       "   ['.', 6, 632]],\n",
       "  []],\n",
       " 9: [[['grad', 0, 641], ['pressao', 1, 646], ['4mmHg', 2, 654], ['.', 3, 655]],\n",
       "  []],\n",
       " 10: [[['Regurgitacao', 0, 661],\n",
       "   ['grau', 1, 674],\n",
       "   ['leve', 2, 679],\n",
       "   ['a', 3, 684],\n",
       "   ['moderada', 4, 686],\n",
       "   ['com', 5, 695],\n",
       "   ['jato', 6, 699],\n",
       "   ['dirigido', 7, 704],\n",
       "   ['valva', 8, 713],\n",
       "   ['mitral', 9, 719],\n",
       "   ['(', 10, 726],\n",
       "   ['?', 11, 727],\n",
       "   [')', 12, 728],\n",
       "   ['.', 13, 729]],\n",
       "  [['Regurgitacao grau leve a moderada', [0, 1, 2, 3, 4, 5], 'Problema'],\n",
       "   ['valva mitral', [8, 9, 10], 'Anatomia']]],\n",
       " 11: [[['Exames', 0, 732], [':', 1, 738], ['.', 2, 739]],\n",
       "  [['Exames', [0, 1], 'Teste']]],\n",
       " 12: [[['Ecocardiograma', 0, 740],\n",
       "   ['-', 1, 755],\n",
       "   ['26', 2, 757],\n",
       "   ['/', 3, 759],\n",
       "   ['06', 4, 760],\n",
       "   ['/', 5, 762],\n",
       "   ['2013', 6, 763],\n",
       "   [':', 7, 767],\n",
       "   ['Homoenxerto', 8, 769],\n",
       "   ['em', 9, 781],\n",
       "   ['posição', 10, 784],\n",
       "   ['aórtica', 11, 792],\n",
       "   ['normofuncionante', 12, 800],\n",
       "   ['.', 13, 801]],\n",
       "  [['Ecocardiograma', [0, 1], 'Teste'],\n",
       "   ['Homoenxerto em posição aórtica', [8, 9, 10, 11, 12], 'Tratamento'],\n",
       "   ['aórtica', [11, 12], 'Anatomia']]],\n",
       " 13: [[['VE', 0, 818],\n",
       "   ['hipertrofiado', 1, 821],\n",
       "   ['com', 2, 835],\n",
       "   ['dimensões', 3, 839],\n",
       "   ['normais', 4, 849],\n",
       "   ['e', 5, 857],\n",
       "   ['fumção', 6, 859],\n",
       "   ['sistólica', 7, 866],\n",
       "   ['preservada', 8, 876],\n",
       "   ['.', 9, 877]],\n",
       "  [['VE hipertrofiado', [0, 1, 2], 'Problema'], ['VE', [0, 1], 'Anatomia']]],\n",
       " 14: [[['ALteração', 0, 888],\n",
       "   ['do', 1, 898],\n",
       "   ['relaxamento', 2, 901],\n",
       "   ['do', 3, 913],\n",
       "   ['VE', 4, 916],\n",
       "   ['AE', 5, 919],\n",
       "   ['dilatado', 6, 922],\n",
       "   ['.', 7, 923]],\n",
       "  [['ALteração do relaxamento do VE', [0, 1, 2, 3, 4, 5], 'Problema'],\n",
       "   ['VE', [4, 5], 'Anatomia'],\n",
       "   ['AE dilatado', [5, 6, 7], 'Problema'],\n",
       "   ['AE', [5, 6], 'Anatomia']]],\n",
       " 15: [[['Homoenxerto', 0, 932],\n",
       "   ['com', 1, 944],\n",
       "   ['mobilidade', 2, 948],\n",
       "   ['adequada', 3, 959],\n",
       "   ['.', 4, 960]],\n",
       "  [['Homoenxerto', [0, 1], 'Tratamento']]],\n",
       " 16: [[['FE', 0, 969],\n",
       "   [':', 1, 971],\n",
       "   ['66', 2, 973],\n",
       "   ['%', 3, 975],\n",
       "   ['.', 4, 976]],\n",
       "  []],\n",
       " 17: [[['-', 0, 977],\n",
       "   ['Lab', 1, 979],\n",
       "   ['(', 2, 983],\n",
       "   ['05', 3, 984],\n",
       "   ['/', 4, 986],\n",
       "   ['02', 5, 987],\n",
       "   ['/', 6, 989],\n",
       "   ['15', 7, 990],\n",
       "   [')', 8, 992],\n",
       "   [':', 9, 993],\n",
       "   ['Cr', 10, 995],\n",
       "   ['0', 11, 998],\n",
       "   [',', 12, 999],\n",
       "   ['91', 13, 1000],\n",
       "   [',', 14, 1002],\n",
       "   ['CT', 15, 1004],\n",
       "   ['174', 16, 1007],\n",
       "   [',', 17, 1010],\n",
       "   ['TGC', 18, 1012],\n",
       "   ['197', 19, 1016],\n",
       "   [',', 20, 1019],\n",
       "   ['HDL', 21, 1021],\n",
       "   ['44', 22, 1025],\n",
       "   [',', 23, 1027],\n",
       "   ['K', 24, 1029],\n",
       "   ['4', 25, 1031],\n",
       "   [',', 26, 1032],\n",
       "   ['8', 27, 1033],\n",
       "   [',', 28, 1034],\n",
       "   ['LDL', 29, 1036],\n",
       "   ['90', 30, 1040],\n",
       "   [',', 31, 1042],\n",
       "   ['Glic', 32, 1044],\n",
       "   ['107', 33, 1049],\n",
       "   ['.', 34, 1050]],\n",
       "  [['Lab', [1, 2], 'Teste'],\n",
       "   ['Cr', [10, 11], 'Teste'],\n",
       "   ['CT', [15, 16], 'Teste'],\n",
       "   ['TGC', [18, 19], 'Teste'],\n",
       "   ['HDL', [21, 22], 'Teste'],\n",
       "   ['K', [24, 25], 'Teste'],\n",
       "   ['LDL', [29, 30], 'Teste'],\n",
       "   ['Glic', [32, 33], 'Teste']]],\n",
       " 18: [[['Ao', 0, 1053], ['exame', 1, 1056], [':', 2, 1061], ['.', 3, 1062]],\n",
       "  [['exame', [1, 2], 'Teste']]],\n",
       " 19: [[['Pte', 0, 1063],\n",
       "   ['em', 1, 1067],\n",
       "   ['BEG', 2, 1070],\n",
       "   [',', 3, 1073],\n",
       "   ['eupneico', 4, 1075],\n",
       "   [',', 5, 1083],\n",
       "   ['corado', 6, 1085],\n",
       "   [',', 7, 1091],\n",
       "   ['hidratado', 8, 1093],\n",
       "   ['.', 9, 1094]],\n",
       "  []],\n",
       " 20: [[['FC79', 0, 1104],\n",
       "   ['PA', 1, 1110],\n",
       "   [':', 2, 1112],\n",
       "   ['130', 3, 1114],\n",
       "   ['/', 4, 1117],\n",
       "   ['80', 5, 1118],\n",
       "   ['.', 6, 1119]],\n",
       "  [['PA', [1, 2], 'Teste']]],\n",
       " 21: [[['CP', 0, 1121],\n",
       "   [':', 1, 1123],\n",
       "   ['sem', 2, 1125],\n",
       "   ['estase', 3, 1129],\n",
       "   ['jugular', 4, 1136],\n",
       "   ['.', 5, 1137]],\n",
       "  [['estase jugular', [3, 4, 5], 'Problema'],\n",
       "   ['jugular', [4, 5], 'Anatomia']]],\n",
       " 22: [[['AC', 0, 1144],\n",
       "   [':', 1, 1146],\n",
       "   ['BCRNF', 2, 1148],\n",
       "   ['sem', 3, 1154],\n",
       "   ['sopros', 4, 1158],\n",
       "   ['.', 5, 1159]],\n",
       "  [['sopros', [4, 5], 'Problema']]],\n",
       " 23: [[['AP', 0, 1165],\n",
       "   [':', 1, 1167],\n",
       "   ['MV', 2, 1169],\n",
       "   ['presente', 3, 1172],\n",
       "   ['bilateralmente', 4, 1181],\n",
       "   [',', 5, 1195],\n",
       "   ['sem', 6, 1197],\n",
       "   ['RA', 7, 1201],\n",
       "   ['.', 8, 1202]],\n",
       "  [['RA', [7, 8], 'Problema']]],\n",
       " 24: [[['mantenho', 0, 1205], ['medicações', 1, 1214], ['.', 2, 1215]],\n",
       "  [['medicações', [1, 2], 'Tratamento']]],\n",
       " 25: [[['encaminho', 0, 1225],\n",
       "   ['para', 1, 1235],\n",
       "   ['cir', 2, 1240],\n",
       "   ['cardiaca', 3, 1244],\n",
       "   ['para', 4, 1253],\n",
       "   ['retirada', 5, 1258],\n",
       "   ['pontos', 6, 1267],\n",
       "   ['suttura', 7, 1274],\n",
       "   ['esterno', 8, 1282],\n",
       "   ['.', 9, 1283]],\n",
       "  [['cardiaca', [3, 4], 'Anatomia'], ['esterno', [8, 9], 'Anatomia']]],\n",
       " 26: [[['retorno', 0, 1290],\n",
       "   ['6', 1, 1298],\n",
       "   ['meses', 2, 1300],\n",
       "   ['.', 3, 1301]],\n",
       "  []],\n",
       " 27: [[['e3', 0, 1306],\n",
       "   ['cardio', 1, 1309],\n",
       "   ['Doutor', 2, 1316],\n",
       "   ['Vital', 3, 1323],\n",
       "   ['Brazil', 4, 1329],\n",
       "   ['.', 5, 1330]],\n",
       "  []]}"
      ]
     },
     "execution_count": 323,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dic_sentencesTest"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 304,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[['c', 0],\n",
       "  ['#', 1],\n",
       "  ['cintilografia', 2],\n",
       "  ['miocardica', 3],\n",
       "  ['para', 4],\n",
       "  ['avaliar', 5],\n",
       "  ['angina', 6],\n",
       "  ['.', 7]],\n",
       " [['cintilografia miocardica', [2, 3], 'Teste'],\n",
       "  ['miocardica', [3], 'Anatomia'],\n",
       "  ['avaliar', [5], 'Teste'],\n",
       "  ['angina', [6], 'Problema']]]"
      ]
     },
     "execution_count": 304,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dic_sentencesTrain[2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[['Dispneia', 0],\n",
       "  ['importante', 1],\n",
       "  ['aos', 2],\n",
       "  ['esforços', 3],\n",
       "  ['+', 4],\n",
       "  ['dor', 5],\n",
       "  ['tipo', 6],\n",
       "  ['peso', 7],\n",
       "  ['no', 8],\n",
       "  ['peito', 9],\n",
       "  ['no', 10],\n",
       "  ['esforço', 11],\n",
       "  ['.', 12]],\n",
       " [['Dispneia importante aos esforços', [0, 1, 2, 3], 'Problema'],\n",
       "  ['dor tipo peso no peito no esforço', [5, 6, 7, 8, 9, 10, 11], 'Problema'],\n",
       "  ['peito', [9], 'Anatomia']]]"
      ]
     },
     "execution_count": 200,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dic_sentencesTrain[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[3, 15, 30, 30, 30, 30, 55, 55, 66, 66]"
      ]
     },
     "execution_count": 201,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "frasesComDescontinuasTrain[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "51\n"
     ]
    }
   ],
   "source": [
    "# frases com entidades descontinuas\n",
    "print(len(set(frasesComDescontinuasTrain)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 203,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[['Comorbidades', 0], [':', 1], ['DM', 2], ['há', 3], ['10', 4], ['anos', 5], ['em', 6], ['uso', 7], ['de', 8], ['metformina', 9], ['850mg', 10], ['3', 11], ['cp', 12], ['/', 13], ['dia', 14], [',', 15], ['acarbose', 16], ['1', 17], ['cp', 18], ['/', 19], ['dia', 20], ['e', 21], ['glicazida', 22], ['60mg', 23], ['2', 24], ['cp', 25], ['/', 26], ['dia', 27], ['e', 28], ['insulina', 29], ['(', 30], ['24', 31], ['-', 32], ['0', 33], ['-', 34], ['24', 35], [')', 36], ['.', 37]], [['Comorbidades', [0], 'Problema'], ['DM', [2], 'Problema'], ['metformina 850mg', [9, 10], 'Tratamento'], ['acarbose', [16], 'Tratamento'], ['glicazida 60mg', [22, 23], 'Tratamento'], ['insulina', [29], 'Tratamento']]]\n"
     ]
    }
   ],
   "source": [
    "print(dic_sentencesTest[2])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 204,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "tamanhoTrain 1334\n",
      "tamanhoDev 422\n",
      "len(dic_sentencesTrain): 1334\n",
      "len(dic_sentencesDev): 421\n",
      "[[['ICO', 0], ['.', 1]], [['ICO', [0], 'Problema']]]\n",
      "[[['SOLICITO', 0], ['EXAMES', 1], ['LABORATORIAIS', 2], ['E', 3], ['CATETERISMO', 4], ['DIAGNOSTICO', 5], ['PARA', 6], ['VER', 7], ['OSSTENTS', 8], ['IMPLANTADOS', 9], ['.', 10]], [['EXAMES LABORATORIAIS', [1, 2], 'Teste'], ['CATETERISMO DIAGNOSTICO', [4, 5], 'Teste'], ['OSSTENTS', [8], 'Tratamento']]]\n"
     ]
    }
   ],
   "source": [
    "\n",
    "save_obj('dic_sentencesTrainDev',dic_sentencesTrain)\n",
    "porc=0.76\n",
    "tamanhoTotal = len(dic_sentencesTrain)\n",
    "tamanhoTrain = int(tamanhoTotal*porc)\n",
    "print('tamanhoTrain', tamanhoTrain)\n",
    "tamanhoDev = tamanhoTotal - tamanhoTrain\n",
    "print('tamanhoDev', tamanhoDev)\n",
    "dic_sentencesDev_temp = {k: dic_sentencesTrain[k] for k in list(dic_sentencesTrain)[tamanhoTrain:-1]}\n",
    "dic_sentencesTrain = {k: dic_sentencesTrain[k] for k in list(dic_sentencesTrain)[:tamanhoTrain]}\n",
    "num=0\n",
    "dic_sentencesDev = {}\n",
    "for key, value in dic_sentencesDev_temp.items():\n",
    "    dic_sentencesDev[num] = value\n",
    "    num=num+1\n",
    "\n",
    "print('len(dic_sentencesTrain):', len(dic_sentencesTrain))\n",
    "print('len(dic_sentencesDev):', len(dic_sentencesDev))\n",
    "print(dic_sentencesTrain[tamanhoTrain-1])\n",
    "print(dic_sentencesDev[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[['SOLICITO', 0],\n",
       "  ['EXAMES', 1],\n",
       "  ['LABORATORIAIS', 2],\n",
       "  ['E', 3],\n",
       "  ['CATETERISMO', 4],\n",
       "  ['DIAGNOSTICO', 5],\n",
       "  ['PARA', 6],\n",
       "  ['VER', 7],\n",
       "  ['OSSTENTS', 8],\n",
       "  ['IMPLANTADOS', 9],\n",
       "  ['.', 10]],\n",
       " [['EXAMES LABORATORIAIS', [1, 2], 'Teste'],\n",
       "  ['CATETERISMO DIAGNOSTICO', [4, 5], 'Teste'],\n",
       "  ['OSSTENTS', [8], 'Tratamento']]]"
      ]
     },
     "execution_count": 205,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dic_sentencesDev[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[['LUCAS', 0], [',', 1], ['69', 2], ['anos', 3], ['.', 4]], []]"
      ]
     },
     "execution_count": 206,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dic_sentencesDev[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {},
   "outputs": [],
   "source": [
    "save_obj('dic_sentencesTrain',dic_sentencesTrain)\n",
    "save_obj('dic_sentencesDev',dic_sentencesDev)\n",
    "save_obj('dic_sentencesTest',dic_sentencesTest)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "num_entidade_train 13759\n",
      "num_entidade_dev 4053\n",
      "num_entidade_test: 5453\n",
      "num_entidade_total: 23265\n"
     ]
    }
   ],
   "source": [
    "# gerar arquivo treinamento\n",
    "f_train = open('nested_train.conll', 'w')\n",
    "\n",
    "num_entidade_total=0\n",
    "num_entidade_train=0\n",
    "num_entidade_dev=0\n",
    "num_entidade_test=0\n",
    "\n",
    "\n",
    "for i in range(len(dic_sentencesTrain)):\n",
    "    tokens = dic_sentencesTrain[i][0]\n",
    "    ents = dic_sentencesTrain[i][1]\n",
    "    indiceEnts=[]\n",
    "    for token in tokens:\n",
    "        #print('token:', token)\n",
    "        indiceToken = token[1]\n",
    "        tag='O'\n",
    "        for ent in ents:\n",
    "            if indiceToken in ent[1]:\n",
    "                tag = ent[2]\n",
    "                break\n",
    "        tokenGravar = token[0].replace(' ','')\n",
    "        tokenGravar = tokenGravar.strip()\n",
    "        f_train.write(tokenGravar+' '+tag+'\\n')\n",
    "        num_entidade_train=num_entidade_train+1\n",
    "    f_train.write('\\n')\n",
    "        \n",
    "f_train.close()\n",
    "\n",
    "f_dev = open('nested_dev.conll', 'w')\n",
    "for i in range(len(dic_sentencesDev)):\n",
    "    tokens = dic_sentencesDev[i][0]\n",
    "    ents = dic_sentencesDev[i][1]\n",
    "    indiceEnts=[]\n",
    "    for token in tokens:\n",
    "        #print('token:', token)\n",
    "        indiceToken = token[1]\n",
    "        tag='O'\n",
    "        for ent in ents:\n",
    "            if indiceToken in ent[1]:\n",
    "                tag = ent[2]\n",
    "                break\n",
    "        tokenGravar = token[0].replace(' ','')\n",
    "        tokenGravar = tokenGravar.strip()\n",
    "        f_dev.write(tokenGravar+' '+tag+'\\n')\n",
    "        num_entidade_dev=num_entidade_dev+1\n",
    "    f_dev.write('\\n')\n",
    "f_dev.close()\n",
    "\n",
    "\n",
    "f_test = open('nested_test.conll', 'w')\n",
    "for i in range(len(dic_sentencesTest)):\n",
    "    tokens = dic_sentencesTest[i][0]\n",
    "    ents = dic_sentencesTest[i][1]\n",
    "    indiceEnts=[]\n",
    "    for token in tokens:\n",
    "        #print('token:', token)\n",
    "        indiceToken = token[1]\n",
    "        tag='O'\n",
    "        for ent in ents:\n",
    "            if indiceToken in ent[1]:\n",
    "                tag = ent[2]\n",
    "                break\n",
    "        tokenGravar = token[0].replace(' ','')\n",
    "        tokenGravar = tokenGravar.strip()\n",
    "        f_test.write(tokenGravar+' '+tag+'\\n')\n",
    "        num_entidade_test=num_entidade_test+1\n",
    "    f_test.write('\\n')\n",
    "f_test.close()\n",
    "\n",
    "print('num_entidade_train', num_entidade_train)\n",
    "print('num_entidade_dev', num_entidade_dev)\n",
    "print('num_entidade_test:', num_entidade_test)\n",
    "num_entidade_total=num_entidade_train+num_entidade_dev+num_entidade_test\n",
    "print('num_entidade_total:', num_entidade_total)\n",
    "\n",
    "save_obj('dic_sentencesTrain',dic_sentencesTrain)\n",
    "save_obj('dic_sentencesDev',dic_sentencesDev)\n",
    "save_obj('dic_sentencesTest',dic_sentencesTest)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## parte 2- Gerar arquivo treinamento para SpanClassification\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [],
   "source": [
    "def areConsecutive(arr):\n",
    "    # Sort the array\n",
    "    arr.sort()\n",
    "    n = len(arr)\n",
    "    # checking the adjacent elements\n",
    "    for i in range (1,n):\n",
    "        if(arr[i]!=arr[i-1]+1):\n",
    "            return False;\n",
    "             \n",
    "    return True;"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{0: [[['Lucas', 0], [',', 1], ['74', 2], ['anos', 3], ['.', 4]], []],\n",
       " 1: [[['Em', 0],\n",
       "   ['acompanhamento', 1],\n",
       "   ['no', 2],\n",
       "   ['ambualtorio', 3],\n",
       "   ['há', 4],\n",
       "   ['5', 5],\n",
       "   ['anos', 6],\n",
       "   ['por', 7],\n",
       "   ['FA', 8],\n",
       "   [',', 9],\n",
       "   ['uso', 10],\n",
       "   ['de', 11],\n",
       "   ['marevan', 12],\n",
       "   ['5mg', 13],\n",
       "   ['1', 14],\n",
       "   ['x', 15],\n",
       "   ['ao', 16],\n",
       "   ['dia', 17],\n",
       "   ['.', 18]],\n",
       "  [['FA', [8], 'Problema'], ['marevan 5mg', [12, 13], 'Tratamento']]],\n",
       " 2: [[['Comorbidades', 0],\n",
       "   [':', 1],\n",
       "   ['DM', 2],\n",
       "   ['há', 3],\n",
       "   ['10', 4],\n",
       "   ['anos', 5],\n",
       "   ['em', 6],\n",
       "   ['uso', 7],\n",
       "   ['de', 8],\n",
       "   ['metformina', 9],\n",
       "   ['850mg', 10],\n",
       "   ['3', 11],\n",
       "   ['cp', 12],\n",
       "   ['/', 13],\n",
       "   ['dia', 14],\n",
       "   [',', 15],\n",
       "   ['acarbose', 16],\n",
       "   ['1', 17],\n",
       "   ['cp', 18],\n",
       "   ['/', 19],\n",
       "   ['dia', 20],\n",
       "   ['e', 21],\n",
       "   ['glicazida', 22],\n",
       "   ['60mg', 23],\n",
       "   ['2', 24],\n",
       "   ['cp', 25],\n",
       "   ['/', 26],\n",
       "   ['dia', 27],\n",
       "   ['e', 28],\n",
       "   ['insulina', 29],\n",
       "   ['(', 30],\n",
       "   ['24', 31],\n",
       "   ['-', 32],\n",
       "   ['0', 33],\n",
       "   ['-', 34],\n",
       "   ['24', 35],\n",
       "   [')', 36],\n",
       "   ['.', 37]],\n",
       "  [['Comorbidades', [0], 'Problema'],\n",
       "   ['DM', [2], 'Problema'],\n",
       "   ['metformina 850mg', [9, 10], 'Tratamento'],\n",
       "   ['acarbose', [16], 'Tratamento'],\n",
       "   ['glicazida 60mg', [22, 23], 'Tratamento'],\n",
       "   ['insulina', [29], 'Tratamento']]],\n",
       " 3: [[['HAS', 0],\n",
       "   ['há', 1],\n",
       "   ['15', 2],\n",
       "   ['anos', 3],\n",
       "   ['em', 4],\n",
       "   ['uso', 5],\n",
       "   ['de', 6],\n",
       "   ['losartana', 7],\n",
       "   ['50mg', 8],\n",
       "   ['/', 9],\n",
       "   ['dia', 10],\n",
       "   ['e', 11],\n",
       "   ['digoxina', 12],\n",
       "   ['1', 13],\n",
       "   ['/', 14],\n",
       "   ['2', 15],\n",
       "   ['cp', 16],\n",
       "   ['/', 17],\n",
       "   ['dia', 18],\n",
       "   [',', 19],\n",
       "   ['carvedilol', 20],\n",
       "   ['25', 21],\n",
       "   ['12', 22],\n",
       "   ['/', 23],\n",
       "   ['12', 24],\n",
       "   [',', 25],\n",
       "   ['HCTZ', 26],\n",
       "   ['.', 27]],\n",
       "  [['HAS', [0], 'Problema'],\n",
       "   ['losartana 50mg', [7, 8], 'Tratamento'],\n",
       "   ['digoxina', [12], 'Tratamento'],\n",
       "   ['carvedilol 25', [20, 21], 'Tratamento'],\n",
       "   ['HCTZ', [26], 'Tratamento']]],\n",
       " 4: [[['DSLP', 0],\n",
       "   ['em', 1],\n",
       "   ['uso', 2],\n",
       "   ['de', 3],\n",
       "   ['sinvastatina', 4],\n",
       "   [',', 5],\n",
       "   ['marevan', 6],\n",
       "   ['1', 7],\n",
       "   ['cp', 8],\n",
       "   ['/', 9],\n",
       "   ['dia', 10],\n",
       "   ['seg', 11],\n",
       "   ['-', 12],\n",
       "   ['sab', 13],\n",
       "   ['para', 14],\n",
       "   ['no', 15],\n",
       "   ['alvo', 16],\n",
       "   ['sic', 17],\n",
       "   ['.', 18]],\n",
       "  [['DSLP', [0], 'Problema'],\n",
       "   ['sinvastatina', [4], 'Tratamento'],\n",
       "   ['marevan', [6], 'Tratamento']]],\n",
       " 5: [[['S', 0], [':', 1], ['Assintomática', 2], ['.', 3]], []],\n",
       " 6: [[['Nega', 0],\n",
       "   ['dispnéia', 1],\n",
       "   [',', 2],\n",
       "   ['DPN', 3],\n",
       "   ['e', 4],\n",
       "   ['ortopneia', 5],\n",
       "   [',', 6],\n",
       "   ['palpitações', 7],\n",
       "   ['e', 8],\n",
       "   ['sincope', 9],\n",
       "   ['.', 10]],\n",
       "  [['dispnéia', [1], 'Problema'],\n",
       "   ['DPN', [3], 'Problema'],\n",
       "   ['ortopneia', [5], 'Problema'],\n",
       "   ['palpitações', [7], 'Problema'],\n",
       "   ['sincope', [9], 'Problema']]],\n",
       " 7: [[['O', 0],\n",
       "   [':', 1],\n",
       "   ['BEG', 2],\n",
       "   [',', 3],\n",
       "   ['corada', 4],\n",
       "   [',', 5],\n",
       "   ['hidtratada', 6],\n",
       "   [',', 7],\n",
       "   ['afebril', 8],\n",
       "   [',', 9],\n",
       "   ['fc', 10],\n",
       "   ['63', 11],\n",
       "   [',', 12],\n",
       "   ['so2', 13],\n",
       "   ['97', 14],\n",
       "   ['%', 15],\n",
       "   [',', 16],\n",
       "   ['pa', 17],\n",
       "   ['130', 18],\n",
       "   ['/', 19],\n",
       "   ['80', 20],\n",
       "   [';', 21],\n",
       "   ['MV', 22],\n",
       "   ['presente', 23],\n",
       "   [',', 24],\n",
       "   ['simétrico', 25],\n",
       "   [',', 26],\n",
       "   ['sem', 27],\n",
       "   ['RA', 28],\n",
       "   ['.', 29]],\n",
       "  [['afebril', [8], 'Problema'],\n",
       "   ['fc', [10], 'Teste'],\n",
       "   ['so2', [13], 'Teste'],\n",
       "   ['pa', [17], 'Teste'],\n",
       "   ['RA', [28], 'Problema']]],\n",
       " 8: [[['BC', 0],\n",
       "   ['arritmicas', 1],\n",
       "   [',', 2],\n",
       "   ['NF', 3],\n",
       "   ['SS', 4],\n",
       "   ['2T', 5],\n",
       "   ['.', 6]],\n",
       "  [['BC arritmicas', [0, 1], 'Problema'], ['SS', [4], 'Problema']]],\n",
       " 9: [[['Abd', 0],\n",
       "   ['globoso', 1],\n",
       "   [',', 2],\n",
       "   ['flacido', 3],\n",
       "   [',', 4],\n",
       "   ['indolor', 5],\n",
       "   ['a', 6],\n",
       "   ['palpacao', 7],\n",
       "   [',', 8],\n",
       "   ['sem', 9],\n",
       "   ['VCM', 10],\n",
       "   ['.', 11]],\n",
       "  [['Abd', [0], 'Anatomia'], ['VCM', [10], 'Problema']]],\n",
       " 10: [[['MMII', 0],\n",
       "   ['sem', 1],\n",
       "   ['edema', 2],\n",
       "   [',', 3],\n",
       "   ['panturrilhas', 4],\n",
       "   ['livres', 5],\n",
       "   ['.', 6]],\n",
       "  [['MMII', [0], 'Anatomia'],\n",
       "   ['edema', [2], 'Problema'],\n",
       "   ['panturrilhas', [4], 'Anatomia']]],\n",
       " 11: [[['Exames', 0],\n",
       "   ['-', 1],\n",
       "   ['Holter', 2],\n",
       "   [':', 3],\n",
       "   ['FC', 4],\n",
       "   ['controlada', 5],\n",
       "   [',', 6],\n",
       "   ['media', 7],\n",
       "   ['92', 8],\n",
       "   ['.', 9]],\n",
       "  [['Exames', [0], 'Teste'], ['Holter', [2], 'Teste']]],\n",
       " 12: [[['Cr', 0],\n",
       "   ['-', 1],\n",
       "   ['0', 2],\n",
       "   [',', 3],\n",
       "   ['7', 4],\n",
       "   [';', 5],\n",
       "   ['glic', 6],\n",
       "   ['-', 7],\n",
       "   ['140', 8],\n",
       "   [';', 9],\n",
       "   ['ureia', 10],\n",
       "   ['-', 11],\n",
       "   ['33', 12],\n",
       "   [';', 13],\n",
       "   ['CPK', 14],\n",
       "   ['-', 15],\n",
       "   ['68', 16],\n",
       "   [',', 17],\n",
       "   ['Triglerideos', 18],\n",
       "   ['-', 19],\n",
       "   ['335', 20],\n",
       "   [';', 21],\n",
       "   ['HDL', 22],\n",
       "   ['-', 23],\n",
       "   ['30', 24],\n",
       "   [';', 25],\n",
       "   ['CT', 26],\n",
       "   ['-', 27],\n",
       "   ['216', 28],\n",
       "   [';', 29],\n",
       "   ['LDL', 30],\n",
       "   ['-', 31],\n",
       "   ['119', 32],\n",
       "   [';', 33],\n",
       "   ['K', 34],\n",
       "   ['-', 35],\n",
       "   ['5', 36],\n",
       "   [',', 37],\n",
       "   ['1', 38],\n",
       "   [';', 39],\n",
       "   ['RNI', 40],\n",
       "   ['-', 41],\n",
       "   ['2', 42],\n",
       "   [',', 43],\n",
       "   ['07', 44],\n",
       "   [';', 45],\n",
       "   ['Microalbuminúria', 46],\n",
       "   ['308', 47],\n",
       "   ['mg', 48],\n",
       "   ['/', 49],\n",
       "   ['g', 50],\n",
       "   ['.', 51]],\n",
       "  [['Cr', [0], 'Teste'],\n",
       "   ['glic', [6], 'Teste'],\n",
       "   ['ureia', [10], 'Teste'],\n",
       "   ['CPK', [14], 'Teste'],\n",
       "   ['Triglerideos', [18], 'Teste'],\n",
       "   ['HDL', [22], 'Teste'],\n",
       "   ['CT', [26], 'Teste'],\n",
       "   ['LDL', [30], 'Teste'],\n",
       "   ['K', [34], 'Teste'],\n",
       "   ['RNI', [40], 'Teste'],\n",
       "   ['Microalbuminúria', [46], 'Teste']]],\n",
       " 13: [[['Ecocardiograma', 0],\n",
       "   ['-', 1],\n",
       "   ['ventrículo', 2],\n",
       "   ['esquerdo', 3],\n",
       "   ['com', 4],\n",
       "   ['hipertrofia', 5],\n",
       "   ['concentrica', 6],\n",
       "   ['de', 7],\n",
       "   ['grau', 8],\n",
       "   ['discreto', 9],\n",
       "   ['e', 10],\n",
       "   ['função', 11],\n",
       "   ['sistólica', 12],\n",
       "   ['preservada', 13],\n",
       "   ['.', 14]],\n",
       "  [['Ecocardiograma', [0], 'Teste'],\n",
       "   ['ventrículo esquerdo com hipertrofia concentrica de grau discreto',\n",
       "    [2, 3, 4, 5, 6, 7, 8, 9],\n",
       "    'Problema']]],\n",
       " 14: [[['aumento', 0],\n",
       "   ['moderado', 1],\n",
       "   ['de', 2],\n",
       "   ['átrio', 3],\n",
       "   ['esquerdo', 4],\n",
       "   ['.', 5]],\n",
       "  [['aumento moderado de átrio esquerdo', [0, 1, 2, 3, 4], 'Problema'],\n",
       "   ['átrio esquerdo', [3, 4], 'Anatomia']]],\n",
       " 15: [[['calcificação', 0],\n",
       "   ['mitral', 1],\n",
       "   ['e', 2],\n",
       "   ['aórtica', 3],\n",
       "   ['com', 4],\n",
       "   ['refluxo', 5],\n",
       "   ['leve', 6],\n",
       "   ['.', 7]],\n",
       "  [['calcificação mitral e aórtica com refluxo leve',\n",
       "    [0, 1, 2, 3, 4, 5, 6],\n",
       "    'Problema'],\n",
       "   ['mitral', [1], 'Anatomia'],\n",
       "   ['aórtica', [3], 'Anatomia']]],\n",
       " 16: [[['A', 0], [':', 1], ['FA', 2], ['.', 3]], [['FA', [2], 'Problema']]],\n",
       " 17: [[['PA', 0],\n",
       "   ['-', 1],\n",
       "   ['120', 2],\n",
       "   ['/', 3],\n",
       "   ['80', 4],\n",
       "   [',', 5],\n",
       "   ['P', 6],\n",
       "   ['-', 7],\n",
       "   ['72', 8],\n",
       "   [',', 9],\n",
       "   ['bulhas', 10],\n",
       "   ['arritimicas', 11],\n",
       "   [',', 12],\n",
       "   ['normofonéticas', 13],\n",
       "   [',', 14],\n",
       "   ['sem', 15],\n",
       "   ['sopros', 16],\n",
       "   ['.', 17]],\n",
       "  [['PA', [0], 'Teste'],\n",
       "   ['P', [6], 'Teste'],\n",
       "   ['bulhas arritimicas', [10, 11], 'Problema'],\n",
       "   ['sopros', [16], 'Problema']]],\n",
       " 18: [[['Abdomen', 0],\n",
       "   ['flácido', 1],\n",
       "   [',', 2],\n",
       "   ['indolor', 3],\n",
       "   [',', 4],\n",
       "   ['sem', 5],\n",
       "   ['visceromegalias', 6],\n",
       "   ['.', 7]],\n",
       "  [['Abdomen', [0], 'Anatomia'], ['visceromegalias', [6], 'Problema']]],\n",
       " 19: [[['Sem', 0], ['alterações', 1], ['nos', 2], ['MMII', 3], ['.', 4]],\n",
       "  [['alterações nos MMII', [1, 2, 3], 'Problema'], ['MMII', [3], 'Anatomia']]],\n",
       " 20: [[['Otimizo', 0],\n",
       "   ['dose', 1],\n",
       "   ['da', 2],\n",
       "   ['sinvastatina', 3],\n",
       "   ['para', 4],\n",
       "   ['40mg', 5],\n",
       "   ['/', 6],\n",
       "   ['dia', 7],\n",
       "   ['.', 8]],\n",
       "  [['sinvastatina para 40mg', [3, 4, 5], 'Tratamento']]],\n",
       " 21: [[['Atestado', 0],\n",
       "   ['para', 1],\n",
       "   ['atividades', 2],\n",
       "   ['físicas', 3],\n",
       "   ['.', 4]],\n",
       "  []],\n",
       " 22: [[['Retorno', 0], ['na', 1], ['vaga', 2], ['.', 3]], []],\n",
       " 23: [[['#', 0],\n",
       "   ['Lucas', 1],\n",
       "   [',', 2],\n",
       "   ['78', 3],\n",
       "   ['anos', 4],\n",
       "   ['*', 5],\n",
       "   ['HAS', 6],\n",
       "   [',', 7],\n",
       "   ['*', 8],\n",
       "   ['Gota', 9],\n",
       "   [',', 10],\n",
       "   ['*', 11],\n",
       "   ['DAC', 12],\n",
       "   ['-', 13],\n",
       "   ['IAM', 14],\n",
       "   ['+', 15],\n",
       "   ['ATC', 16],\n",
       "   ['em', 17],\n",
       "   ['2012', 18],\n",
       "   ['(', 19],\n",
       "   ['angina', 20],\n",
       "   ['instavel', 21],\n",
       "   [')', 22],\n",
       "   ['-', 23],\n",
       "   ['Ecocardiograma', 24],\n",
       "   ['mar', 25],\n",
       "   ['/', 26],\n",
       "   ['14', 27],\n",
       "   [':', 28],\n",
       "   ['FE', 29],\n",
       "   ['42', 30],\n",
       "   ['%', 31],\n",
       "   ['/', 32],\n",
       "   ['comprometimento', 33],\n",
       "   ['difuso', 34],\n",
       "   ['do', 35],\n",
       "   ['VE', 36],\n",
       "   ['grau', 37],\n",
       "   ['moderado', 38],\n",
       "   [',', 39],\n",
       "   ['insuf', 40],\n",
       "   ['Mi', 41],\n",
       "   ['discreta', 42],\n",
       "   [',', 43],\n",
       "   ['insuf', 44],\n",
       "   ['Tri', 45],\n",
       "   ['discreta', 46],\n",
       "   [',', 47],\n",
       "   ['hipetensao', 48],\n",
       "   ['pulmonar', 49],\n",
       "   ['discreta', 50],\n",
       "   ['.', 51]],\n",
       "  [['HAS', [6], 'Problema'],\n",
       "   ['Gota', [9], 'Problema'],\n",
       "   ['DAC', [12], 'Problema'],\n",
       "   ['IAM', [14], 'Problema'],\n",
       "   ['ATC', [16], 'Tratamento'],\n",
       "   ['angina instavel', [20, 21], 'Problema'],\n",
       "   ['Ecocardiograma', [24], 'Teste'],\n",
       "   ['comprometimento difuso do VE grau moderado',\n",
       "    [33, 34, 35, 36, 37, 38],\n",
       "    'Problema'],\n",
       "   ['VE', [36], 'Anatomia'],\n",
       "   ['insuf Mi discreta', [40, 41, 42], 'Problema'],\n",
       "   ['Mi', [41], 'Anatomia'],\n",
       "   ['insuf Tri discreta', [44, 45, 46], 'Problema'],\n",
       "   ['Tri', [45], 'Anatomia'],\n",
       "   ['hipetensao pulmonar discreta', [48, 49, 50], 'Problema'],\n",
       "   ['pulmonar', [49], 'Anatomia']]],\n",
       " 24: [[['Paciente', 0],\n",
       "   ['refere', 1],\n",
       "   ['fraqueza', 2],\n",
       "   ['intermitente', 3],\n",
       "   ['em', 4],\n",
       "   ['MMII', 5],\n",
       "   [',', 6],\n",
       "   ['associada', 7],\n",
       "   ['a', 8],\n",
       "   ['tontura', 9],\n",
       "   [',', 10],\n",
       "   ['turvação', 11],\n",
       "   ['visual', 12],\n",
       "   ['e', 13],\n",
       "   ['epigastralgia', 14],\n",
       "   ['.', 15]],\n",
       "  [['fraqueza intermitente em MMII', [2, 3, 4, 5], 'Problema'],\n",
       "   ['MMII', [5], 'Anatomia'],\n",
       "   ['tontura', [9], 'Problema'],\n",
       "   ['turvação visual', [11, 12], 'Problema'],\n",
       "   ['epigastralgia', [14], 'Problema']]],\n",
       " 25: [[['Nega', 0],\n",
       "   ['dor', 1],\n",
       "   ['toracica', 2],\n",
       "   ['associada', 3],\n",
       "   ['ou', 4],\n",
       "   ['dispneia', 5],\n",
       "   ['.', 6]],\n",
       "  [['dor toracica', [1, 2], 'Problema'],\n",
       "   ['toracica', [2], 'Anatomia'],\n",
       "   ['dispneia', [5], 'Problema']]],\n",
       " 26: [[['Nega', 0], ['DPN', 1], ['.', 2]], [['DPN', [1], 'Problema']]],\n",
       " 27: [[['Nega', 0], ['sincope', 1], ['.', 2]], [['sincope', [1], 'Problema']]],\n",
       " 28: [[['Nega', 0], ['edema', 1], ['de', 2], ['MMII', 3], ['.', 4]],\n",
       "  [['edema de MMII', [1, 2, 3], 'Problema'], ['MMII', [3], 'Anatomia']]],\n",
       " 29: [[['Alimenta', 0], ['-', 1], ['se', 2], ['bem', 3], ['.', 4]], []],\n",
       " 30: [[['Nega', 0],\n",
       "   ['queixas', 1],\n",
       "   ['urinarias', 2],\n",
       "   ['e', 3],\n",
       "   ['gastrointestinais', 4],\n",
       "   ['.', 5]],\n",
       "  [['queixas urinarias e gastrointestinais', [1, 2, 3, 4], 'Problema'],\n",
       "   ['urinarias', [2], 'Anatomia'],\n",
       "   ['gastrointestinais', [4], 'Anatomia']]],\n",
       " 31: [[['Nega', 0],\n",
       "   ['tabagismo', 1],\n",
       "   ['atual', 2],\n",
       "   ['e', 3],\n",
       "   ['pregresso', 4],\n",
       "   ['.', 5]],\n",
       "  [['tabagismo', [1], 'Problema']]],\n",
       " 32: [[['-', 0],\n",
       "   ['Laboratorio', 1],\n",
       "   ['Cr', 2],\n",
       "   ['1', 3],\n",
       "   [',', 4],\n",
       "   ['5', 5],\n",
       "   ['/', 6],\n",
       "   ['glicose', 7],\n",
       "   ['85', 8],\n",
       "   ['/', 9],\n",
       "   ['ureia', 10],\n",
       "   ['70', 11],\n",
       "   ['/', 12],\n",
       "   ['CPK', 13],\n",
       "   ['161', 14],\n",
       "   ['/', 15],\n",
       "   ['acido', 16],\n",
       "   ['urico', 17],\n",
       "   ['5', 18],\n",
       "   [',', 19],\n",
       "   ['3', 20],\n",
       "   ['/', 21],\n",
       "   ['TG', 22],\n",
       "   ['107', 23],\n",
       "   ['/', 24],\n",
       "   ['TGP', 25],\n",
       "   ['23', 26],\n",
       "   ['/', 27],\n",
       "   ['HDL', 28],\n",
       "   ['38', 29],\n",
       "   ['/', 30],\n",
       "   ['Na', 31],\n",
       "   ['143', 32],\n",
       "   ['/', 33],\n",
       "   ['TGO', 34],\n",
       "   ['26', 35],\n",
       "   ['/', 36],\n",
       "   ['CT', 37],\n",
       "   ['103', 38],\n",
       "   ['/', 39],\n",
       "   ['LDL', 40],\n",
       "   ['44', 41],\n",
       "   ['/', 42],\n",
       "   ['K', 43],\n",
       "   ['5', 44],\n",
       "   [',', 45],\n",
       "   ['1', 46],\n",
       "   ['/', 47],\n",
       "   ['HB', 48],\n",
       "   ['16', 49],\n",
       "   [',', 50],\n",
       "   ['1', 51],\n",
       "   ['/', 52],\n",
       "   ['leucocitos', 53],\n",
       "   ['6800', 54],\n",
       "   ['/', 55],\n",
       "   ['plaquetas', 56],\n",
       "   ['116000', 57],\n",
       "   ['/', 58],\n",
       "   ['-', 59],\n",
       "   ['Albumina', 60],\n",
       "   ['/', 61],\n",
       "   ['Cr', 62],\n",
       "   ['isolada', 63],\n",
       "   ['=', 64],\n",
       "   ['174', 65],\n",
       "   [',', 66],\n",
       "   ['3', 67],\n",
       "   ['***', 68],\n",
       "   ['CKD', 69],\n",
       "   ['44', 70],\n",
       "   [';', 71],\n",
       "   ['Paciente', 72],\n",
       "   ['encaminhado', 73],\n",
       "   ['via', 74],\n",
       "   ['UBS', 75],\n",
       "   ['para', 76],\n",
       "   ['nefrologia', 77],\n",
       "   ['-', 78],\n",
       "   ['aguarda', 79],\n",
       "   ['consulta', 80],\n",
       "   ['.', 81]],\n",
       "  [['Laboratorio', [1], 'Teste'],\n",
       "   ['Cr', [2], 'Teste'],\n",
       "   ['glicose', [7], 'Teste'],\n",
       "   ['ureia', [10], 'Teste'],\n",
       "   ['CPK', [13], 'Teste'],\n",
       "   ['acido urico', [16, 17], 'Teste'],\n",
       "   ['TG', [22], 'Teste'],\n",
       "   ['TGP', [25], 'Teste'],\n",
       "   ['HDL', [28], 'Teste'],\n",
       "   ['Na', [31], 'Teste'],\n",
       "   ['TGO', [34], 'Teste'],\n",
       "   ['CT', [37], 'Teste'],\n",
       "   ['LDL', [40], 'Teste'],\n",
       "   ['K', [43], 'Teste'],\n",
       "   ['HB', [48], 'Teste'],\n",
       "   ['leucocitos', [53], 'Teste'],\n",
       "   ['plaquetas', [56], 'Teste'],\n",
       "   ['Albumina / Cr isolada', [60, 61, 62, 63], 'Teste'],\n",
       "   ['CKD', [69], 'Teste']]],\n",
       " 33: [[['Em', 0],\n",
       "   ['uso', 1],\n",
       "   ['de', 2],\n",
       "   [':', 3],\n",
       "   ['alopurinol', 4],\n",
       "   ['150mg', 5],\n",
       "   ['/', 6],\n",
       "   ['dia', 7],\n",
       "   [',', 8],\n",
       "   ['anlodipino', 9],\n",
       "   ['5mg', 10],\n",
       "   ['/', 11],\n",
       "   ['noite', 12],\n",
       "   [',', 13],\n",
       "   ['enalapril', 14],\n",
       "   ['10mg', 15],\n",
       "   ['12', 16],\n",
       "   ['/', 17],\n",
       "   ['12', 18],\n",
       "   ['h', 19],\n",
       "   [',', 20],\n",
       "   ['selozok', 21],\n",
       "   ['100mg', 22],\n",
       "   ['/', 23],\n",
       "   ['dia', 24],\n",
       "   [',', 25],\n",
       "   ['AAS', 26],\n",
       "   ['100mg', 27],\n",
       "   ['/', 28],\n",
       "   ['dia', 29],\n",
       "   [',', 30],\n",
       "   ['clopidogrel', 31],\n",
       "   ['75mg', 32],\n",
       "   ['/', 33],\n",
       "   ['dia', 34],\n",
       "   [',', 35],\n",
       "   ['sinvastatina', 36],\n",
       "   ['40mg', 37],\n",
       "   ['/', 38],\n",
       "   ['dia', 39],\n",
       "   [',', 40],\n",
       "   ['colecalciferol', 41],\n",
       "   ['10', 42],\n",
       "   ['gotas', 43],\n",
       "   ['/', 44],\n",
       "   ['semana', 45],\n",
       "   ['.', 46]],\n",
       "  [['alopurinol 150mg', [4, 5], 'Tratamento'],\n",
       "   ['anlodipino 5mg', [9, 10], 'Tratamento'],\n",
       "   ['enalapril 10mg', [14, 15], 'Tratamento'],\n",
       "   ['selozok 100mg', [21, 22], 'Tratamento'],\n",
       "   ['AAS 100mg', [26, 27], 'Tratamento'],\n",
       "   ['clopidogrel 75mg', [31, 32], 'Tratamento'],\n",
       "   ['sinvastatina 40mg', [36, 37], 'Tratamento'],\n",
       "   ['colecalciferol', [41], 'Tratamento']]],\n",
       " 34: [[['Ao', 0],\n",
       "   ['exame', 1],\n",
       "   [':', 2],\n",
       "   ['PA', 3],\n",
       "   [':', 4],\n",
       "   ['90', 5],\n",
       "   ['/', 6],\n",
       "   ['60', 7],\n",
       "   ['mmHg', 8],\n",
       "   ['/', 9],\n",
       "   ['FC', 10],\n",
       "   ['55', 11],\n",
       "   ['bpm', 12],\n",
       "   [',', 13],\n",
       "   ['BCRNF', 14],\n",
       "   [',', 15],\n",
       "   ['SS', 16],\n",
       "   [',', 17],\n",
       "   ['MV', 18],\n",
       "   ['+', 19],\n",
       "   ['bilateralmente', 20],\n",
       "   [',', 21],\n",
       "   ['sem', 22],\n",
       "   ['RA', 23],\n",
       "   [',', 24],\n",
       "   ['MMII', 25],\n",
       "   ['edema', 26],\n",
       "   ['+', 27],\n",
       "   ['/', 28],\n",
       "   ['IV', 29],\n",
       "   ['bilateral', 30],\n",
       "   ['.', 31]],\n",
       "  [['exame', [1], 'Teste'],\n",
       "   ['PA', [3], 'Teste'],\n",
       "   ['FC', [10], 'Teste'],\n",
       "   ['SS', [16], 'Problema'],\n",
       "   ['RA', [23], 'Problema'],\n",
       "   ['MMII edema + / IV bilateral', [25, 26, 27, 28, 29, 30], 'Problema']]],\n",
       " 35: [[['#', 0],\n",
       "   ['#', 1],\n",
       "   ['Reduzimos', 2],\n",
       "   ['anlodipino', 3],\n",
       "   ['para', 4],\n",
       "   ['2', 5],\n",
       "   [',', 6],\n",
       "   ['5mg', 7],\n",
       "   ['/', 8],\n",
       "   ['dia', 9],\n",
       "   ['.', 10]],\n",
       "  [['anlodipino', [3], 'Tratamento']]],\n",
       " 36: [[['Suspendemos', 0], ['clopidogrel', 1], ['.', 2]],\n",
       "  [['clopidogrel', [1], 'Tratamento']]],\n",
       " 37: [[['Mantemos', 0], ['demais', 1], ['medicações', 2], ['.', 3]],\n",
       "  [['demais medicações', [1, 2], 'Tratamento']]],\n",
       " 38: [[['Retorno', 0],\n",
       "   ['em', 1],\n",
       "   ['10', 2],\n",
       "   ['meses', 3],\n",
       "   ['com', 4],\n",
       "   ['a', 5],\n",
       "   ['cardio', 6],\n",
       "   ['com', 7],\n",
       "   ['novos', 8],\n",
       "   ['exames', 9],\n",
       "   ['.', 10]],\n",
       "  [['exames', [9], 'Teste']]],\n",
       " 39: [[['Encaminhado', 0], ['à', 1], ['nefrologia', 2], ['.', 3]], []],\n",
       " 40: [[['R1', 0],\n",
       "   ['CM', 1],\n",
       "   ['Lucas', 2],\n",
       "   [',', 3],\n",
       "   ['Dr', 4],\n",
       "   ['Fortes', 5],\n",
       "   ['.', 6]],\n",
       "  []],\n",
       " 41: [[['*', 0],\n",
       "   ['HAS', 1],\n",
       "   ['*', 2],\n",
       "   ['Gota', 3],\n",
       "   ['*', 4],\n",
       "   ['DAC', 5],\n",
       "   ['-', 6],\n",
       "   ['IAM', 7],\n",
       "   ['+', 8],\n",
       "   ['ATC', 9],\n",
       "   ['em', 10],\n",
       "   ['2012', 11],\n",
       "   ['(', 12],\n",
       "   ['angina', 13],\n",
       "   ['instavel', 14],\n",
       "   [')', 15],\n",
       "   ['.', 16]],\n",
       "  [['HAS', [1], 'Problema'],\n",
       "   ['Gota', [3], 'Problema'],\n",
       "   ['DAC', [5], 'Problema'],\n",
       "   ['IAM', [7], 'Problema'],\n",
       "   ['ATC', [9], 'Tratamento'],\n",
       "   ['angina instavel', [13, 14], 'Problema']]],\n",
       " 42: [[['Paciente', 0],\n",
       "   ['em', 1],\n",
       "   ['tratamento', 2],\n",
       "   ['para', 3],\n",
       "   ['ICC', 4],\n",
       "   ['diastólica', 5],\n",
       "   ['há', 6],\n",
       "   ['cerca', 7],\n",
       "   ['de', 8],\n",
       "   ['5', 9],\n",
       "   ['a', 10],\n",
       "   ['com', 11],\n",
       "   ['melhora', 12],\n",
       "   ['importante', 13],\n",
       "   ['dos', 14],\n",
       "   ['sintomas', 15],\n",
       "   ['após', 16],\n",
       "   ['início', 17],\n",
       "   ['do', 18],\n",
       "   ['tratamento', 19],\n",
       "   ['.', 20]],\n",
       "  [['tratamento', [2], 'Tratamento'],\n",
       "   ['ICC diastólica', [4, 5], 'Problema'],\n",
       "   ['melhora importante dos sintomas após início do tratamento',\n",
       "    [12, 13, 14, 15, 16, 17, 18, 19],\n",
       "    'Problema'],\n",
       "   ['tratamento', [19], 'Tratamento']]],\n",
       " 43: [[['Hoje', 0],\n",
       "   ['relata', 1],\n",
       "   ['dispneia', 2],\n",
       "   ['aos', 3],\n",
       "   ['moderados', 4],\n",
       "   ['esforços', 5],\n",
       "   [',', 6],\n",
       "   ['nega', 7],\n",
       "   ['disnpneía', 8],\n",
       "   ['paroxística', 9],\n",
       "   ['notrna', 10],\n",
       "   [',', 11],\n",
       "   ['dorme', 12],\n",
       "   ['com', 13],\n",
       "   ['apenas', 14],\n",
       "   ['1', 15],\n",
       "   ['travesseiro', 16],\n",
       "   ['.', 17]],\n",
       "  [['dispneia aos moderados esforços', [2, 3, 4, 5], 'Problema'],\n",
       "   ['disnpneía paroxística notrna', [8, 9, 10], 'Problema']]],\n",
       " 44: [[['Nega', 0],\n",
       "   ['precordialgia', 1],\n",
       "   ['aos', 2],\n",
       "   ['moderados', 3],\n",
       "   ['esforços', 4],\n",
       "   ['.', 5]],\n",
       "  [['precordialgia aos moderados esforços', [1, 2, 3, 4], 'Problema']]],\n",
       " 45: [[['Nega', 0], ['episódios', 1], ['anginosos', 2], ['.', 3]],\n",
       "  [['episódios anginosos', [1, 2], 'Problema']]],\n",
       " 46: [[['Nega', 0], ['outras', 1], ['queixas', 2], ['.', 3]],\n",
       "  [['outras queixas', [1, 2], 'Problema']]],\n",
       " 47: [[['Rsultado', 0],\n",
       "   ['de', 1],\n",
       "   ['exames', 2],\n",
       "   [':', 3],\n",
       "   ['1', 4],\n",
       "   ['-', 5],\n",
       "   ['Microalbuminúria', 6],\n",
       "   ['12', 7],\n",
       "   ['/', 8],\n",
       "   ['04', 9],\n",
       "   [':', 10],\n",
       "   ['10', 11],\n",
       "   ['.', 12]],\n",
       "  [['exames', [2], 'Teste'], ['Microalbuminúria', [6], 'Teste']]],\n",
       " 48: [[['64', 0],\n",
       "   ['/', 1],\n",
       "   ['G', 2],\n",
       "   ['de', 3],\n",
       "   ['creatinina', 4],\n",
       "   ['2', 5],\n",
       "   ['-', 6],\n",
       "   ['US', 7],\n",
       "   ['abdome', 8],\n",
       "   ['total', 9],\n",
       "   [':', 10],\n",
       "   ['rins', 11],\n",
       "   ['com', 12],\n",
       "   ['alterações', 13],\n",
       "   ['tróficas', 14],\n",
       "   ['3', 15],\n",
       "   ['-', 16],\n",
       "   ['ECG', 17],\n",
       "   ['19', 18],\n",
       "   ['/', 19],\n",
       "   ['03', 20],\n",
       "   ['/', 21],\n",
       "   ['14', 22],\n",
       "   [':', 23],\n",
       "   ['isquemia', 24],\n",
       "   ['subepicárdica', 25],\n",
       "   ['anterior', 26],\n",
       "   [',', 27],\n",
       "   ['alteração', 28],\n",
       "   ['de', 29],\n",
       "   ['repolarização', 30],\n",
       "   ['ventricular', 31],\n",
       "   ['infero', 32],\n",
       "   ['-', 33],\n",
       "   ['lateral', 34],\n",
       "   ['4', 35],\n",
       "   ['-', 36],\n",
       "   ['Laboratoriais', 37],\n",
       "   ['12', 38],\n",
       "   ['/', 39],\n",
       "   ['04', 40],\n",
       "   ['/', 41],\n",
       "   ['14', 42],\n",
       "   [':', 43],\n",
       "   ['K', 44],\n",
       "   ['5', 45],\n",
       "   ['.', 46]],\n",
       "  [['creatinina', [4], 'Teste'],\n",
       "   ['US abdome total', [7, 8, 9], 'Teste'],\n",
       "   ['abdome', [8], 'Anatomia'],\n",
       "   ['rins com alterações tróficas', [11, 12, 13, 14], 'Problema'],\n",
       "   ['ECG', [17], 'Teste'],\n",
       "   ['isquemia subepicárdica anterior', [24, 25, 26], 'Problema'],\n",
       "   ['subepicárdica anterior', [25, 26], 'Anatomia'],\n",
       "   ['alteração de repolarização ventricular infero - lateral',\n",
       "    [28, 29, 30, 31, 32, 33, 34],\n",
       "    'Problema'],\n",
       "   ['ventricular infero - lateral', [31, 32, 33, 34], 'Anatomia'],\n",
       "   ['Laboratoriais', [37], 'Teste'],\n",
       "   ['K', [44], 'Teste']]],\n",
       " 49: [[['1', 0],\n",
       "   [',', 1],\n",
       "   ['clearence', 2],\n",
       "   ['creatinina', 3],\n",
       "   ['57', 4],\n",
       "   ['Ao', 5],\n",
       "   ['EF', 6],\n",
       "   [':', 7],\n",
       "   ['PA', 8],\n",
       "   ['170x100mmHg', 9],\n",
       "   ['(', 10],\n",
       "   ['relata', 11],\n",
       "   ['PA', 12],\n",
       "   ['normal', 13],\n",
       "   ['quando', 14],\n",
       "   ['em', 15],\n",
       "   ['ambiente', 16],\n",
       "   ['não', 17],\n",
       "   ['hospitalar', 18],\n",
       "   [')', 19],\n",
       "   [',', 20],\n",
       "   ['FC', 21],\n",
       "   ['63bpm', 22],\n",
       "   ['AR', 23],\n",
       "   [':', 24],\n",
       "   ['mv', 25],\n",
       "   ['sem', 26],\n",
       "   ['RA', 27],\n",
       "   ['ACV', 28],\n",
       "   [':', 29],\n",
       "   ['rcr', 30],\n",
       "   ['em', 31],\n",
       "   ['2T', 32],\n",
       "   ['sem', 33],\n",
       "   ['sopros', 34],\n",
       "   [',', 35],\n",
       "   ['com', 36],\n",
       "   ['hipofonese', 37],\n",
       "   ['de', 38],\n",
       "   ['bulhas', 39],\n",
       "   ['AD', 40],\n",
       "   [':', 41],\n",
       "   ['abdome', 42],\n",
       "   ['globos', 43],\n",
       "   [',', 44],\n",
       "   ['normotenso', 45],\n",
       "   [',', 46],\n",
       "   ['indolor', 47],\n",
       "   [',', 48],\n",
       "   ['ausência', 49],\n",
       "   ['de', 50],\n",
       "   ['massas', 51],\n",
       "   ['ou', 52],\n",
       "   ['visceromegalias', 53],\n",
       "   ['MMII', 54],\n",
       "   ['sem', 55],\n",
       "   ['edema', 56],\n",
       "   ['HD', 57],\n",
       "   [':', 58],\n",
       "   ['-', 59],\n",
       "   ['ICC', 60],\n",
       "   ['diastólica', 61],\n",
       "   ['classe', 62],\n",
       "   ['III', 63],\n",
       "   ['-', 64],\n",
       "   ['HAS', 65],\n",
       "   ['-', 66],\n",
       "   ['DMII', 67],\n",
       "   ['-', 68],\n",
       "   ['Dislipidemia', 69],\n",
       "   ['CD', 70],\n",
       "   [':', 71],\n",
       "   ['-', 72],\n",
       "   ['mantenho', 73],\n",
       "   ['medicação', 74],\n",
       "   ['-', 75],\n",
       "   ['retorno', 76],\n",
       "   ['em', 77],\n",
       "   ['6', 78],\n",
       "   ['m', 79],\n",
       "   ['com', 80],\n",
       "   ['exames', 81],\n",
       "   ['laboratoriais', 82],\n",
       "   ['+', 83],\n",
       "   ['ECG', 84],\n",
       "   ['.', 85]],\n",
       "  [['clearence creatinina', [2, 3], 'Teste'],\n",
       "   ['EF', [6], 'Teste'],\n",
       "   ['PA', [8], 'Teste'],\n",
       "   ['PA', [12], 'Teste'],\n",
       "   ['FC', [21], 'Teste'],\n",
       "   ['RA', [27], 'Problema'],\n",
       "   ['ACV', [28], 'Anatomia'],\n",
       "   ['sopros', [34], 'Problema'],\n",
       "   ['hipofonese de bulhas', [37, 38, 39], 'Problema'],\n",
       "   ['abdome', [42], 'Anatomia'],\n",
       "   ['massas', [51], 'Problema'],\n",
       "   ['visceromegalias', [53], 'Problema'],\n",
       "   ['MMII', [54], 'Anatomia'],\n",
       "   ['edema', [56], 'Problema'],\n",
       "   ['ICC diastólica classe III', [60, 61, 62, 63], 'Problema'],\n",
       "   ['HAS', [65], 'Problema'],\n",
       "   ['DMII', [67], 'Problema'],\n",
       "   ['Dislipidemia', [69], 'Problema'],\n",
       "   ['medicação', [74], 'Tratamento'],\n",
       "   ['exames laboratoriais', [81, 82], 'Teste'],\n",
       "   ['ECG', [84], 'Teste']]],\n",
       " 50: [[['ICC', 0],\n",
       "   ['classe', 1],\n",
       "   ['III', 2],\n",
       "   ['HAS', 3],\n",
       "   [',', 4],\n",
       "   ['em', 5],\n",
       "   ['uso', 6],\n",
       "   ['de', 7],\n",
       "   ['atenolol', 8],\n",
       "   ['100mg', 9],\n",
       "   ['/', 10],\n",
       "   ['dia', 11],\n",
       "   [',', 12],\n",
       "   ['AAS', 13],\n",
       "   [',', 14],\n",
       "   ['100mg', 15],\n",
       "   ['/', 16],\n",
       "   ['dia', 17],\n",
       "   [',', 18],\n",
       "   ['furosemida', 19],\n",
       "   ['40mg', 20],\n",
       "   ['1cp', 21],\n",
       "   ['12', 22],\n",
       "   ['/', 23],\n",
       "   ['12h', 24],\n",
       "   [',', 25],\n",
       "   ['anlodipino', 26],\n",
       "   ['5mg', 27],\n",
       "   ['1', 28],\n",
       "   ['/', 29],\n",
       "   ['2', 30],\n",
       "   ['cp', 31],\n",
       "   ['à', 32],\n",
       "   ['noite', 33],\n",
       "   [',', 34],\n",
       "   ['losartana', 35],\n",
       "   ['50mg', 36],\n",
       "   ['12', 37],\n",
       "   ['/', 38],\n",
       "   ['12h', 39],\n",
       "   ['DMII', 40],\n",
       "   [',', 41],\n",
       "   ['metformina', 42],\n",
       "   ['850mg', 43],\n",
       "   ['12', 44],\n",
       "   ['/', 45],\n",
       "   ['12h', 46],\n",
       "   ['Dislipidemia', 47],\n",
       "   [',', 48],\n",
       "   ['faz', 49],\n",
       "   ['uso', 50],\n",
       "   ['de', 51],\n",
       "   ['sinvastatina', 52],\n",
       "   ['20mg', 53],\n",
       "   ['02', 54],\n",
       "   ['cp', 55],\n",
       "   ['à', 56],\n",
       "   ['noite', 57],\n",
       "   ['.', 58]],\n",
       "  [['ICC classe III', [0, 1, 2], 'Problema'],\n",
       "   ['HAS', [3], 'Problema'],\n",
       "   ['atenolol 100mg', [8, 9], 'Tratamento'],\n",
       "   ['AAS', [13], 'Tratamento'],\n",
       "   ['furosemida 40mg', [19, 20], 'Tratamento'],\n",
       "   ['anlodipino 5mg', [26, 27], 'Tratamento'],\n",
       "   ['losartana 50mg', [35, 36], 'Tratamento'],\n",
       "   ['DMII', [40], 'Problema'],\n",
       "   ['metformina 850mg', [42, 43], 'Tratamento'],\n",
       "   ['Dislipidemia', [47], 'Problema'],\n",
       "   ['sinvastatina 20mg', [52, 53], 'Tratamento']]],\n",
       " 51: [[['POS', 0],\n",
       "   ['TX', 1],\n",
       "   ['CARDIACO', 2],\n",
       "   ['HA', 3],\n",
       "   ['20', 4],\n",
       "   ['dias', 5],\n",
       "   ['.', 6]],\n",
       "  [['TX CARDIACO', [1, 2], 'Tratamento'], ['CARDIACO', [2], 'Anatomia']]],\n",
       " 52: [[['ULTIMA', 0],\n",
       "   ['BIOPSIA', 1],\n",
       "   ['AINDA', 2],\n",
       "   ['SEM', 3],\n",
       "   ['RESULTADO', 4],\n",
       "   ['.', 5]],\n",
       "  [['ULTIMA BIOPSIA', [0, 1], 'Teste'], ['BIOPSIA', [1], 'Teste']]],\n",
       " 53: [[['EM', 0],\n",
       "   ['USO', 1],\n",
       "   ['DE', 2],\n",
       "   ['EVEROLIMUS', 3],\n",
       "   ['0', 4],\n",
       "   [',', 5],\n",
       "   ['75', 6],\n",
       "   ['DE', 7],\n",
       "   ['8', 8],\n",
       "   ['/', 9],\n",
       "   ['8', 10],\n",
       "   [',', 11],\n",
       "   ['AZATIOPRINA', 12],\n",
       "   ['50', 13],\n",
       "   ['/', 14],\n",
       "   ['50', 15],\n",
       "   ['E', 16],\n",
       "   ['PREDNISONA', 17],\n",
       "   ['20MG', 18],\n",
       "   ['/', 19],\n",
       "   ['DIA', 20],\n",
       "   ['.', 21]],\n",
       "  [['EVEROLIMUS 0 , 75', [3, 4, 5, 6], 'Tratamento'],\n",
       "   ['AZATIOPRINA', [12], 'Tratamento'],\n",
       "   ['PREDNISONA 20MG', [17, 18], 'Tratamento']]],\n",
       " 54: [[['USO', 0],\n",
       "   ['DE', 1],\n",
       "   ['FURO', 2],\n",
       "   [',', 3],\n",
       "   ['AAS', 4],\n",
       "   [',', 5],\n",
       "   ['SINVA', 6],\n",
       "   ['.', 7]],\n",
       "  [['FURO', [2], 'Tratamento'],\n",
       "   ['AAS', [4], 'Tratamento'],\n",
       "   ['SINVA', [6], 'Tratamento']]],\n",
       " 55: [[['S', 0], ['#', 1], ['SEM', 2], ['QUEIXAS', 3], ['.', 4]],\n",
       "  [['QUEIXAS', [3], 'Problema']]],\n",
       " 56: [[['REFERE', 0], ['QUEIMAÇÃO', 1], ['NAS', 2], ['PERNAS', 3], ['.', 4]],\n",
       "  [['QUEIMAÇÃO NAS PERNAS', [1, 2, 3], 'Problema'],\n",
       "   ['PERNAS', [3], 'Anatomia']]],\n",
       " 57: [[['DEMAIS', 0], ['SP', 1], ['.', 2]], []],\n",
       " 58: [[['O', 0],\n",
       "   ['#', 1],\n",
       "   ['PA', 2],\n",
       "   ['120', 3],\n",
       "   ['/', 4],\n",
       "   ['80', 5],\n",
       "   [',', 6],\n",
       "   ['FC', 7],\n",
       "   ['89', 8],\n",
       "   ['.', 9]],\n",
       "  [['PA', [2], 'Teste'], ['FC', [7], 'Teste']]],\n",
       " 59: [[['ACV', 0], ['BCRNF', 1], ['.', 2]], []],\n",
       " 60: [[['CPP', 0], ['LIVRE', 1], ['.', 2]], [['CPP', [0], 'Anatomia']]],\n",
       " 61: [[['MMII', 0], ['SEM', 1], ['EDEMA', 2], ['.', 3]],\n",
       "  [['MMII', [0], 'Anatomia'], ['EDEMA', [2], 'Problema']]],\n",
       " 62: [[['A', 0],\n",
       "   ['#', 1],\n",
       "   ['POS', 2],\n",
       "   ['TX', 3],\n",
       "   ['CARDIACO', 4],\n",
       "   [',', 5],\n",
       "   ['COM', 6],\n",
       "   ['BOA', 7],\n",
       "   ['EVOLUÇÃO', 8],\n",
       "   ['.', 9]],\n",
       "  [['TX CARDIACO', [3, 4], 'Tratamento'], ['CARDIACO', [4], 'Anatomia']]],\n",
       " 63: [[['P', 0],\n",
       "   ['#', 1],\n",
       "   ['NOVA', 2],\n",
       "   ['BIOPSIA', 3],\n",
       "   [',', 4],\n",
       "   ['ECO', 5],\n",
       "   ['E', 6],\n",
       "   ['LAB', 7],\n",
       "   ['.', 8]],\n",
       "  [['ECO', [5], 'Teste'], ['LAB', [7], 'Teste']]],\n",
       " 64: [[['PROTESE', 0], ['VALVAR', 1], ['MI', 2], ['2012', 3], ['.', 4]],\n",
       "  [['PROTESE VALVAR MI', [0, 1, 2], 'Tratamento'], ['MI', [2], 'Anatomia']]],\n",
       " 65: [[['USA', 0],\n",
       "   ['S', 1],\n",
       "   ['MAREVAN', 2],\n",
       "   ['ENL', 3],\n",
       "   ['20', 4],\n",
       "   ['12', 5],\n",
       "   ['-', 6],\n",
       "   ['12', 7],\n",
       "   ['ATL', 8],\n",
       "   ['50', 9],\n",
       "   ['12', 10],\n",
       "   ['-', 11],\n",
       "   ['12', 12],\n",
       "   ['ESPIRONOLACTONA', 13],\n",
       "   ['25', 14],\n",
       "   ['MG', 15],\n",
       "   ['-', 16],\n",
       "   ['D', 17],\n",
       "   ['SVT', 18],\n",
       "   ['20', 19],\n",
       "   ['MG', 20],\n",
       "   ['-', 21],\n",
       "   ['D', 22],\n",
       "   ['.', 23]],\n",
       "  [['MAREVAN', [2], 'Tratamento'],\n",
       "   ['ENL 20', [3, 4], 'Tratamento'],\n",
       "   ['ATL 50', [8, 9], 'Tratamento'],\n",
       "   ['ESPIRONOLACTONA 25 MG', [13, 14, 15], 'Tratamento'],\n",
       "   ['SVT 20 MG', [18, 19, 20], 'Tratamento']]],\n",
       " 66: [[['SEM', 0], ['QX', 1], ['CARDIOLOGICAS', 2], ['.', 3]],\n",
       "  [['QX CARDIOLOGICAS', [1, 2], 'Problema'],\n",
       "   ['CARDIOLOGICAS', [2], 'Anatomia']]],\n",
       " 67: [[['27', 0],\n",
       "   ['-', 1],\n",
       "   ['01', 2],\n",
       "   ['-', 3],\n",
       "   ['15', 4],\n",
       "   ['FE', 5],\n",
       "   ['50', 6],\n",
       "   ['EP', 7],\n",
       "   ['25', 8],\n",
       "   ['AE', 9],\n",
       "   ['46', 10],\n",
       "   ['VE', 11],\n",
       "   ['REMODELAMENTO', 12],\n",
       "   ['CONCENTRICO', 13],\n",
       "   ['AE', 14],\n",
       "   ['E', 15],\n",
       "   ['AD', 16],\n",
       "   ['AUMENTADOS', 17],\n",
       "   ['VD', 18],\n",
       "   ['NL', 19],\n",
       "   ['PROTESE', 20],\n",
       "   ['AO', 21],\n",
       "   ['NF', 22],\n",
       "   ['REFLUXO', 23],\n",
       "   ['DISCRETO', 24],\n",
       "   ['=', 25],\n",
       "   ['DUPLA', 26],\n",
       "   ['LESAO', 27],\n",
       "   ['MI', 28],\n",
       "   ['COM', 29],\n",
       "   ['ESTENOSE', 30],\n",
       "   ['E', 31],\n",
       "   ['INSUF', 32],\n",
       "   ['MODERADAS', 33],\n",
       "   ['-', 34],\n",
       "   ['INSUF', 35],\n",
       "   ['TRI', 36],\n",
       "   ['MODERADA', 37],\n",
       "   ['A', 38],\n",
       "   ['IMPORTANTE', 39],\n",
       "   ['=', 40],\n",
       "   ['HIPERTENSAO', 41],\n",
       "   ['PULMONAR', 42],\n",
       "   ['.', 43]],\n",
       "  [['EP', [7], 'Anatomia'],\n",
       "   ['AE', [9], 'Anatomia'],\n",
       "   ['VE REMODELAMENTO CONCENTRICO', [11, 12, 13], 'Problema'],\n",
       "   ['AE E AD AUMENTADOS', [14, 15, 16, 17], 'Problema'],\n",
       "   ['AD', [16], 'Anatomia'],\n",
       "   ['REFLUXO DISCRETO', [23, 24], 'Problema'],\n",
       "   ['DUPLA LESAO MI COM ESTENOSE E INSUF MODERADAS',\n",
       "    [26, 27, 28, 29, 30, 31, 32, 33],\n",
       "    'Problema'],\n",
       "   ['MI', [28], 'Anatomia'],\n",
       "   ['INSUF TRI MODERADA A IMPORTANTE', [35, 36, 37, 38, 39], 'Problema'],\n",
       "   ['TRI', [36], 'Anatomia'],\n",
       "   ['HIPERTENSAO PULMONAR', [41, 42], 'Problema'],\n",
       "   ['PULMONAR', [42], 'Anatomia']]],\n",
       " 68: [[['CONTROLE', 0],\n",
       "   ['NO', 1],\n",
       "   ['AMB', 2],\n",
       "   ['VALVAS', 3],\n",
       "   ['E', 4],\n",
       "   ['ACO', 5],\n",
       "   ['.', 6]],\n",
       "  [['CONTROLE', [0], 'Tratamento'], ['ACO', [5], 'Tratamento']]],\n",
       " 69: [[['Protese', 0],\n",
       "   ['Metalica', 1],\n",
       "   ['Aortica', 2],\n",
       "   ['em', 3],\n",
       "   ['2004', 4],\n",
       "   ['.', 5]],\n",
       "  [['Protese Metalica Aortica', [0, 1, 2], 'Tratamento'],\n",
       "   ['Aortica', [2], 'Anatomia']]],\n",
       " 70: [[['Uso', 0],\n",
       "   [':', 1],\n",
       "   ['Enalapril', 2],\n",
       "   ['10', 3],\n",
       "   ['-', 4],\n",
       "   ['2cp', 5],\n",
       "   ['-', 6],\n",
       "   ['12', 7],\n",
       "   ['/', 8],\n",
       "   ['12', 9],\n",
       "   ['/', 10],\n",
       "   ['Atenolol', 11],\n",
       "   ['50', 12],\n",
       "   ['-', 13],\n",
       "   ['1cp', 14],\n",
       "   ['=', 15],\n",
       "   ['12', 16],\n",
       "   ['/', 17],\n",
       "   ['12', 18],\n",
       "   [';', 19],\n",
       "   ['Espironolactona', 20],\n",
       "   ['25', 21],\n",
       "   ['-', 22],\n",
       "   ['1xd', 23],\n",
       "   [';', 24],\n",
       "   ['Sinvastatina', 25],\n",
       "   ['20', 26],\n",
       "   ['-', 27],\n",
       "   ['1x', 28],\n",
       "   ['noite', 29],\n",
       "   [';', 30],\n",
       "   ['Calcio', 31],\n",
       "   ['500', 32],\n",
       "   ['-', 33],\n",
       "   ['3x', 34],\n",
       "   ['d', 35],\n",
       "   [';', 36],\n",
       "   ['Marevan', 37],\n",
       "   ['5', 38],\n",
       "   ['-', 39],\n",
       "   ['amb', 40],\n",
       "   ['da', 41],\n",
       "   ['ACXO', 42],\n",
       "   [';', 43],\n",
       "   ['Trazodona', 44],\n",
       "   ['25', 45],\n",
       "   ['1x', 46],\n",
       "   ['d', 47],\n",
       "   ['.', 48]],\n",
       "  [['Enalapril 10', [2, 3], 'Tratamento'],\n",
       "   ['Atenolol 50', [11, 12], 'Tratamento'],\n",
       "   ['Espironolactona 25', [20, 21], 'Tratamento'],\n",
       "   ['Sinvastatina 20', [25, 26], 'Tratamento'],\n",
       "   ['Calcio 500', [31, 32], 'Tratamento'],\n",
       "   ['Marevan 5', [37, 38], 'Tratamento'],\n",
       "   ['Trazodona 25', [44, 45], 'Tratamento']]],\n",
       " 71: [[['Dispnia', 0],\n",
       "   ['esporadica', 1],\n",
       "   [',', 2],\n",
       "   ['edema', 3],\n",
       "   ['mmii', 4],\n",
       "   ['eventual', 5],\n",
       "   [',', 6],\n",
       "   ['sem', 7],\n",
       "   ['palpitação', 8],\n",
       "   [',', 9],\n",
       "   ['dor', 10],\n",
       "   ['toracica', 11],\n",
       "   ['quando', 12],\n",
       "   ['epigastralgia', 13],\n",
       "   ['que', 14],\n",
       "   ['melhoram', 15],\n",
       "   ['com', 16],\n",
       "   ['paracetamol', 17],\n",
       "   ['.', 18]],\n",
       "  [['Dispnia esporadica', [0, 1], 'Problema'],\n",
       "   ['edema mmii eventual', [3, 4, 5], 'Problema'],\n",
       "   ['mmii', [4], 'Anatomia'],\n",
       "   ['palpitação', [8], 'Problema'],\n",
       "   ['dor toracica quando epigastralgia', [10, 11, 12, 13], 'Problema'],\n",
       "   ['toracica', [11], 'Anatomia'],\n",
       "   ['melhoram com paracetamol . ', [15, 16, 17, 18], 'Problema'],\n",
       "   ['paracetamol', [17], 'Tratamento']]],\n",
       " 72: [[['Lab', 0],\n",
       "   ['(', 1],\n",
       "   ['21', 2],\n",
       "   ['/', 3],\n",
       "   ['01', 4],\n",
       "   ['/', 5],\n",
       "   ['16', 6],\n",
       "   [')', 7],\n",
       "   [':', 8],\n",
       "   ['Hbglic', 9],\n",
       "   ['6', 10],\n",
       "   [',', 11],\n",
       "   ['34', 12],\n",
       "   [';', 13],\n",
       "   ['TSH', 14],\n",
       "   ['2', 15],\n",
       "   [',', 16],\n",
       "   ['15', 17],\n",
       "   [';', 18],\n",
       "   ['Hb', 19],\n",
       "   ['13', 20],\n",
       "   [',', 21],\n",
       "   ['5', 22],\n",
       "   [';', 23],\n",
       "   ['VG', 24],\n",
       "   ['42', 25],\n",
       "   ['%', 26],\n",
       "   ['leuco', 27],\n",
       "   ['9610', 28],\n",
       "   [';', 29],\n",
       "   ['plaq', 30],\n",
       "   ['234000', 31],\n",
       "   [';', 32],\n",
       "   ['Urina', 33],\n",
       "   ['I', 34],\n",
       "   ['ok', 35],\n",
       "   [';', 36],\n",
       "   ['K', 37],\n",
       "   ['4', 38],\n",
       "   [',', 39],\n",
       "   ['4', 40],\n",
       "   [';', 41],\n",
       "   ['Cr', 42],\n",
       "   ['1', 43],\n",
       "   [',', 44],\n",
       "   ['1', 45],\n",
       "   [';', 46],\n",
       "   ['Gli', 47],\n",
       "   ['71', 48],\n",
       "   [';', 49],\n",
       "   ['U', 50],\n",
       "   ['69', 51],\n",
       "   [';', 52],\n",
       "   ['Tri', 53],\n",
       "   ['67', 54],\n",
       "   [';', 55],\n",
       "   ['HDL', 56],\n",
       "   ['-', 57],\n",
       "   ['C', 58],\n",
       "   ['77', 59],\n",
       "   [';', 60],\n",
       "   ['ColT', 61],\n",
       "   ['160', 62],\n",
       "   [';', 63],\n",
       "   ['LDL', 64],\n",
       "   ['-', 65],\n",
       "   ['C', 66],\n",
       "   ['70', 67],\n",
       "   [';', 68],\n",
       "   ['.', 69]],\n",
       "  [['Lab', [0], 'Teste'],\n",
       "   ['Hbglic', [9], 'Teste'],\n",
       "   ['TSH', [14], 'Teste'],\n",
       "   ['Hb', [19], 'Teste'],\n",
       "   ['VG', [24], 'Teste'],\n",
       "   ['leuco', [27], 'Teste'],\n",
       "   ['plaq', [30], 'Teste'],\n",
       "   ['Urina I', [33, 34], 'Teste'],\n",
       "   ['K', [37], 'Teste'],\n",
       "   ['Cr', [42], 'Teste'],\n",
       "   ['Gli', [47], 'Teste'],\n",
       "   ['U', [50], 'Teste'],\n",
       "   ['Tri', [53], 'Teste'],\n",
       "   ['HDL - C', [56, 57, 58], 'Teste'],\n",
       "   ['ColT', [61], 'Teste'],\n",
       "   ['LDL - C', [64, 65, 66], 'Teste']]],\n",
       " 73: [[['Ecocardio', 0],\n",
       "   ['(', 1],\n",
       "   ['04', 2],\n",
       "   ['/', 3],\n",
       "   ['12', 4],\n",
       "   ['/', 5],\n",
       "   ['15', 6],\n",
       "   [')', 7],\n",
       "   [':', 8],\n",
       "   ['AE', 9],\n",
       "   ['42', 10],\n",
       "   [';', 11],\n",
       "   ['DDVE', 12],\n",
       "   ['39', 13],\n",
       "   [';', 14],\n",
       "   ['DSVE', 15],\n",
       "   ['23', 16],\n",
       "   [';', 17],\n",
       "   ['FE', 18],\n",
       "   ['73', 19],\n",
       "   ['%', 20],\n",
       "   [';', 21],\n",
       "   ['VMi', 22],\n",
       "   ['area', 23],\n",
       "   ['1', 24],\n",
       "   [',', 25],\n",
       "   ['65', 26],\n",
       "   [';', 27],\n",
       "   ['calcificada', 28],\n",
       "   [',', 29],\n",
       "   ['refluxo', 30],\n",
       "   ['discreto', 31],\n",
       "   ['.', 32]],\n",
       "  [['Ecocardio', [0], 'Teste'],\n",
       "   ['AE', [9], 'Anatomia'],\n",
       "   ['VMi', [22], 'Anatomia'],\n",
       "   ['calcificada', [28], 'Problema'],\n",
       "   ['refluxo discreto', [30, 31], 'Problema']]],\n",
       " 74: [[['VAo', 0],\n",
       "   ['protese', 1],\n",
       "   ['metalica', 2],\n",
       "   ['com', 3],\n",
       "   ['espessamento', 4],\n",
       "   [',', 5],\n",
       "   ['e', 6],\n",
       "   ['relfuxo', 7],\n",
       "   ['discreto', 8],\n",
       "   ['.', 9]],\n",
       "  [['VAo protese metalica com espessamento', [0, 1, 2, 3, 4], 'Problema'],\n",
       "   ['relfuxo discreto', [7, 8], 'Problema']]],\n",
       " 75: [[['Grad', 0],\n",
       "   ['medio', 1],\n",
       "   ['de', 2],\n",
       "   ['16', 3],\n",
       "   ['e', 4],\n",
       "   ['max', 5],\n",
       "   ['de', 6],\n",
       "   ['30', 7],\n",
       "   ['.', 8]],\n",
       "  []],\n",
       " 76: [[['ARea', 0], ['de', 1], ['1', 2], [',', 3], ['12', 4], ['.', 5]], []],\n",
       " 77: [[['VTri', 0],\n",
       "   ['=', 1],\n",
       "   ['refluxo', 2],\n",
       "   ['discreto', 3],\n",
       "   ['com', 4],\n",
       "   ['PSAP', 5],\n",
       "   ['56mmHg', 6],\n",
       "   ['.', 7]],\n",
       "  [['VTri', [0], 'Anatomia'], ['refluxo discreto', [2, 3], 'Problema']]],\n",
       " 78: [[['VP', 0], ['refluxo', 1], ['minimo', 2], ['.', 3]],\n",
       "  [['VP refluxo minimo', [0, 1, 2], 'Problema']]],\n",
       " 79: [[['VE', 0], ['hipertrofiado', 1], ['.', 2]],\n",
       "  [['VE hipertrofiado', [0, 1], 'Problema']]],\n",
       " 80: [[[',', 0], ['SIv', 1], ['13', 2], ['.', 3]], []],\n",
       " 81: [[['PPVE', 0], ['14', 1], ['.', 2]], []],\n",
       " 82: [[['PA', 0],\n",
       "   ['(', 1],\n",
       "   ['sentada', 2],\n",
       "   [')', 3],\n",
       "   [':', 4],\n",
       "   ['140x60mmHg', 5],\n",
       "   [',', 6],\n",
       "   ['P', 7],\n",
       "   ['76', 8],\n",
       "   ['.', 9]],\n",
       "  [['PA ( sentada ) ', [0, 1, 2, 3], 'Teste'], ['P', [7], 'Teste']]],\n",
       " 83: [[['EM', 0], ['pé', 1], ['2', 2], ['idem', 3], ['.', 4]], []],\n",
       " 84: [[['MV', 0],\n",
       "   ['normodist', 1],\n",
       "   ['sem', 2],\n",
       "   ['ruidos', 3],\n",
       "   ['adsventicios', 4],\n",
       "   ['.', 5]],\n",
       "  [['ruidos adsventicios', [3, 4], 'Problema']]],\n",
       " 85: [[['RCR', 0],\n",
       "   [',', 1],\n",
       "   ['2T', 2],\n",
       "   [',', 3],\n",
       "   ['BNF', 4],\n",
       "   ['ss', 5],\n",
       "   ['+', 6],\n",
       "   ['+', 7],\n",
       "   ['/', 8],\n",
       "   ['6', 9],\n",
       "   ['+', 10],\n",
       "   ['FAo', 11],\n",
       "   ['e', 12],\n",
       "   ['Mi', 13],\n",
       "   ['(', 14],\n",
       "   ['+', 15],\n",
       "   ['/', 16],\n",
       "   ['6', 17],\n",
       "   ['+', 18],\n",
       "   ['.', 19]],\n",
       "  [['ss + + / 6 + FAo', [5, 6, 7, 8, 9, 10, 11], 'Problema'],\n",
       "   ['FAo', [11], 'Anatomia'],\n",
       "   ['Mi ( + / 6 + ', [13, 14, 15, 16, 17, 18], 'Problema']]],\n",
       " 86: [[['BEG', 0],\n",
       "   [',', 1],\n",
       "   ['eupneica', 2],\n",
       "   [',', 3],\n",
       "   ['acianotica', 4],\n",
       "   [',', 5],\n",
       "   ['orientada', 6],\n",
       "   [',', 7],\n",
       "   ['Sem', 8],\n",
       "   ['edema', 9],\n",
       "   ['mmii', 10],\n",
       "   ['.', 11]],\n",
       "  [['edema mmii', [9, 10], 'Problema'], ['mmii', [10], 'Anatomia']]],\n",
       " 87: [[['HD', 0],\n",
       "   [':', 1],\n",
       "   ['protese', 2],\n",
       "   ['Metalica', 3],\n",
       "   ['AOrtoca', 4],\n",
       "   ['normofuncionante', 5],\n",
       "   ['.', 6]],\n",
       "  [['AOrtoca', [4], 'Anatomia']]],\n",
       " 88: [[['Disfunbção', 0], ['diastolica', 1], ['.', 2]],\n",
       "  [['Disfunbção diastolica', [0, 1], 'Problema']]],\n",
       " 89: [[['CD', 0], [':', 1], ['Suspendo', 2], ['Espironolactona', 3], ['.', 4]],\n",
       "  [['Espironolactona', [3], 'Tratamento']]],\n",
       " 90: [[['Reduzo', 0],\n",
       "   ['Sinvastatina', 1],\n",
       "   ['para', 2],\n",
       "   ['10mg', 3],\n",
       "   ['/', 4],\n",
       "   ['noite', 5],\n",
       "   ['.', 6]],\n",
       "  [['Sinvastatina para 10mg', [1, 2, 3], 'Tratamento']]],\n",
       " 91: [[['ACrescento', 0],\n",
       "   ['Omeprazol', 1],\n",
       "   ['20', 2],\n",
       "   ['-', 3],\n",
       "   ['1x', 4],\n",
       "   ['d', 5],\n",
       "   ['.', 6]],\n",
       "  [['Omeprazol 20', [1, 2], 'Tratamento']]],\n",
       " 92: [[['Retonro', 0], ['em', 1], ['6m', 2], ['com', 3], ['lab', 4], ['.', 5]],\n",
       "  [['lab', [4], 'Teste']]],\n",
       " 93: [[['#', 0], ['Lucas', 1], [',', 2], ['55', 3], ['ANOS', 4], ['.', 5]],\n",
       "  []],\n",
       " 94: [[['#', 0],\n",
       "   ['NATURAL', 1],\n",
       "   ['DE', 2],\n",
       "   ['ALAGOAS', 3],\n",
       "   [',', 4],\n",
       "   ['RESIDENTE', 5],\n",
       "   ['EM', 6],\n",
       "   ['CURITIBA', 7],\n",
       "   ['.', 8]],\n",
       "  []],\n",
       " 95: [[['#', 0], ['DÇ', 1], ['DE', 2], ['CHAGAS', 3], ['.', 4]],\n",
       "  [['DÇ DE CHAGAS', [1, 2, 3], 'Problema']]],\n",
       " 96: [[['#', 0], ['BAVT', 1], ['.', 2]], [['BAVT', [1], 'Problema']]],\n",
       " 97: [[['#', 0],\n",
       "   ['MARCA', 1],\n",
       "   ['-', 2],\n",
       "   ['PASSO', 3],\n",
       "   ['DUPLACAMARA', 4],\n",
       "   ['EM', 5],\n",
       "   ['DDD', 6],\n",
       "   ['50', 7],\n",
       "   ['-', 8],\n",
       "   ['60', 9],\n",
       "   ['DESDE', 10],\n",
       "   ['2007', 11],\n",
       "   ['.', 12]],\n",
       "  [['MARCA - PASSO DUPLACAMARA', [1, 2, 3, 4], 'Tratamento']]],\n",
       " 98: [[['#', 0],\n",
       "   ['PACIENTE', 1],\n",
       "   ['ENCAMINHADO', 2],\n",
       "   ['DO', 3],\n",
       "   ['AMBULATÓRIO', 4],\n",
       "   ['DE', 5],\n",
       "   ['ARRITMIAS', 6],\n",
       "   ['.', 7]],\n",
       "  []],\n",
       " 99: [[['PACIENTE', 0],\n",
       "   ['NEGA', 1],\n",
       "   ['DISPNEIA', 2],\n",
       "   [',', 3],\n",
       "   ['NEGA', 4],\n",
       "   ['ORTOPNEIA', 5],\n",
       "   [',', 6],\n",
       "   ['NEGA', 7],\n",
       "   ['DPN', 8],\n",
       "   ['.', 9]],\n",
       "  [['DISPNEIA', [2], 'Problema'],\n",
       "   ['ORTOPNEIA', [5], 'Problema'],\n",
       "   ['DPN', [8], 'Problema']]],\n",
       " 100: [[['SEM', 0], ['OUTRAS', 1], ['QUEIXAS', 2], ['.', 3]],\n",
       "  [['OUTRAS QUEIXAS', [1, 2], 'Problema']]],\n",
       " 101: [[['#', 0],\n",
       "   ['NEGA', 1],\n",
       "   ['HX', 2],\n",
       "   ['DE', 3],\n",
       "   ['HAS', 4],\n",
       "   [',', 5],\n",
       "   ['DM', 6],\n",
       "   [',', 7],\n",
       "   ['DISLIPIDEMIA', 8],\n",
       "   ['E', 9],\n",
       "   ['TIREOIDOPATIAS', 10],\n",
       "   ['.', 11]],\n",
       "  [['HAS', [4], 'Problema'],\n",
       "   ['DM', [6], 'Problema'],\n",
       "   ['DISLIPIDEMIA', [8], 'Problema'],\n",
       "   ['TIREOIDOPATIAS', [10], 'Problema']]],\n",
       " 102: [[['#', 0],\n",
       "   ['MÃE', 1],\n",
       "   ['FALECIDA', 2],\n",
       "   ['POR', 3],\n",
       "   ['CARDIOPATIA', 4],\n",
       "   ['(', 5],\n",
       "   ['IAM', 6],\n",
       "   ['?', 7],\n",
       "   [')', 8],\n",
       "   [',', 9],\n",
       "   ['65', 10],\n",
       "   ['ANOS', 11],\n",
       "   ['.', 12]],\n",
       "  []],\n",
       " 103: [[['#', 0],\n",
       "   ['CX', 1],\n",
       "   ['DE', 2],\n",
       "   ['APENDICITE', 3],\n",
       "   ['E', 4],\n",
       "   ['HERNIAS', 5],\n",
       "   ['PARAMEDIANAS', 6],\n",
       "   ['HÁ', 7],\n",
       "   ['>', 8],\n",
       "   ['20', 9],\n",
       "   ['ANOS', 10],\n",
       "   ['.', 11]],\n",
       "  [['CX DE APENDICITE E HERNIAS PARAMEDIANAS',\n",
       "    [1, 2, 3, 4, 5, 6],\n",
       "    'Tratamento'],\n",
       "   ['PARAMEDIANAS', [6], 'Anatomia']]],\n",
       " 104: [[['#', 0], ['EM', 1], ['USO', 2], ['DE', 3], [':', 4], ['.', 5]], []],\n",
       " 105: [[['METOPROLOL', 0],\n",
       "   ['50MG', 1],\n",
       "   ['12', 2],\n",
       "   ['/', 3],\n",
       "   ['12HS', 4],\n",
       "   ['.', 5]],\n",
       "  [['METOPROLOL 50MG', [0, 1], 'Tratamento']]],\n",
       " 106: [[['ENALAPRIL', 0],\n",
       "   ['10', 1],\n",
       "   ['MG', 2],\n",
       "   ['12', 3],\n",
       "   ['/', 4],\n",
       "   ['12HS', 5],\n",
       "   ['.', 6]],\n",
       "  [['ENALAPRIL 10 MG', [0, 1, 2], 'Tratamento']]],\n",
       " 107: [[['ESPIRONO', 0], ['25', 1], ['.', 2]],\n",
       "  [['ESPIRONO 25', [0, 1], 'Tratamento']]],\n",
       " 108: [[['SVT', 0], ['20', 1], ['.', 2]], [['SVT 20', [0, 1], 'Tratamento']]],\n",
       " 109: [[['EXAMES', 0],\n",
       "   ['LAB', 1],\n",
       "   ['11', 2],\n",
       "   ['/', 3],\n",
       "   ['11', 4],\n",
       "   [':', 5],\n",
       "   ['CREAT', 6],\n",
       "   ['1', 7],\n",
       "   [';', 8],\n",
       "   ['UR', 9],\n",
       "   ['58', 10],\n",
       "   [';', 11],\n",
       "   ['GLI', 12],\n",
       "   ['71', 13],\n",
       "   [';', 14],\n",
       "   ['TG', 15],\n",
       "   ['88', 16],\n",
       "   [';', 17],\n",
       "   ['HDL', 18],\n",
       "   ['52', 19],\n",
       "   [';', 20],\n",
       "   ['CT', 21],\n",
       "   ['160', 22],\n",
       "   [';', 23],\n",
       "   ['LDL', 24],\n",
       "   ['90', 25],\n",
       "   [';', 26],\n",
       "   ['Na', 27],\n",
       "   ['140', 28],\n",
       "   [';', 29],\n",
       "   ['K', 30],\n",
       "   ['5', 31],\n",
       "   [',', 32],\n",
       "   ['2', 33],\n",
       "   [';', 34],\n",
       "   ['Hb', 35],\n",
       "   ['14', 36],\n",
       "   [',', 37],\n",
       "   ['8', 38],\n",
       "   ['/', 39],\n",
       "   ['Ht', 40],\n",
       "   ['46', 41],\n",
       "   [',', 42],\n",
       "   ['6', 43],\n",
       "   ['/', 44],\n",
       "   ['LEUC', 45],\n",
       "   ['5150', 46],\n",
       "   ['.', 47]],\n",
       "  [['EXAMES LAB', [0, 1], 'Teste'],\n",
       "   ['CREAT', [6], 'Teste'],\n",
       "   ['UR', [9], 'Teste'],\n",
       "   ['GLI', [12], 'Teste'],\n",
       "   ['TG', [15], 'Teste'],\n",
       "   ['HDL', [18], 'Teste'],\n",
       "   ['CT', [21], 'Teste'],\n",
       "   ['LDL', [24], 'Teste'],\n",
       "   ['Na', [27], 'Teste'],\n",
       "   ['K', [30], 'Teste'],\n",
       "   ['Hb', [35], 'Teste'],\n",
       "   ['Ht', [40], 'Teste'],\n",
       "   ['LEUC', [45], 'Teste']]],\n",
       " 110: [[['EF', 0],\n",
       "   ['#', 1],\n",
       "   ['BEG', 2],\n",
       "   [',', 3],\n",
       "   ['LOTE', 4],\n",
       "   [',', 5],\n",
       "   ['CORADO', 6],\n",
       "   ['E', 7],\n",
       "   ['HIDRATADO', 8],\n",
       "   ['.', 9]],\n",
       "  [['EF', [0], 'Teste']]],\n",
       " 111: [[['PA', 0],\n",
       "   [':', 1],\n",
       "   ['90', 2],\n",
       "   ['/', 3],\n",
       "   ['60MMHG', 4],\n",
       "   [';', 5],\n",
       "   ['P', 6],\n",
       "   [':', 7],\n",
       "   ['72', 8],\n",
       "   ['.', 9]],\n",
       "  [['PA', [0], 'Teste'], ['P', [6], 'Teste']]],\n",
       " 112: [[['AUSENCIA', 0],\n",
       "   ['DE', 1],\n",
       "   ['TURGÊNCIA', 2],\n",
       "   ['JUGULAR', 3],\n",
       "   ['.', 4]],\n",
       "  [['TURGÊNCIA JUGULAR', [2, 3], 'Problema'], ['JUGULAR', [3], 'Anatomia']]],\n",
       " 113: [[['ICTUS', 0],\n",
       "   ['PALPÁVEL', 1],\n",
       "   ['EM', 2],\n",
       "   ['7º', 3],\n",
       "   ['EIC', 4],\n",
       "   ['EM', 5],\n",
       "   ['LINHA', 6],\n",
       "   ['HEMIAXILAR', 7],\n",
       "   ['E', 8],\n",
       "   ['.', 9]],\n",
       "  [['ICTUS PALPÁVEL EM 7º EIC EM LINHA HEMIAXILAR E',\n",
       "    [0, 1, 2, 3, 4, 5, 6, 7, 8],\n",
       "    'Problema']]],\n",
       " 114: [[['BCRNF', 0], ['2T', 1], ['SS', 2], ['.', 3]],\n",
       "  [['SS', [2], 'Problema']]],\n",
       " 115: [[['AUSCULTA', 0],\n",
       "   ['PULMONAR', 1],\n",
       "   ['SEM', 2],\n",
       "   ['ALTERAÇÕES', 3],\n",
       "   ['.', 4]],\n",
       "  [['PULMONAR', [1], 'Anatomia'], ['ALTERAÇÕES', [3], 'Problema']]],\n",
       " 116: [[['ABDOME', 0], ['INOCENTE', 1], ['.', 2]],\n",
       "  [['ABDOME', [0], 'Anatomia']]],\n",
       " 117: [[['SEM', 0], ['EDEMA', 1], ['EM', 2], ['MMII', 3], ['.', 4]],\n",
       "  [['EDEMA EM MMII', [1, 2, 3], 'Problema'], ['MMII', [3], 'Anatomia']]],\n",
       " 118: [[['PULSOS', 0],\n",
       "   ['+', 1],\n",
       "   ['BL', 2],\n",
       "   [',', 3],\n",
       "   ['SIMÉTRICOS', 4],\n",
       "   ['.', 5]],\n",
       "  []],\n",
       " 119: [[['A', 0], [':', 1], ['DÇA', 2], ['CHAGAS', 3], ['.', 4]],\n",
       "  [['DÇA CHAGAS', [2, 3], 'Problema']]],\n",
       " 120: [[['P', 0], [':', 1], ['MANTENHO', 2], ['MEDICAÇÕES', 3], ['.', 4]],\n",
       "  [['MEDICAÇÕES', [3], 'Tratamento']]],\n",
       " 121: [[['SOLICITO', 0], ['ECOCARDIO', 1], ['.', 2]],\n",
       "  [['ECOCARDIO', [1], 'Teste']]],\n",
       " 122: [[['RETORNO', 0], ['EM', 1], ['3', 2], ['MESES', 3], ['.', 4]], []],\n",
       " 123: [[['sem', 0], ['queixas', 1], ['.', 2]], [['queixas', [1], 'Problema']]],\n",
       " 124: [[['sente', 0], ['-', 1], ['se', 2], ['bem', 3], ['.', 4]], []],\n",
       " 125: [[['#', 0],\n",
       "   ['hoje', 1],\n",
       "   ['avaliação', 2],\n",
       "   ['exclusiva', 3],\n",
       "   ['de', 4],\n",
       "   ['mp', 5],\n",
       "   ['.', 6]],\n",
       "  [['avaliação exclusiva de mp', [2, 3, 4, 5], 'Teste']]],\n",
       " 126: [[['#', 0], ['mp', 1], ['camara', 2], ['-', 3], ['dupla', 4], ['.', 5]],\n",
       "  [['mp camara - dupla', [1, 2, 3, 4], 'Tratamento']]],\n",
       " 127: [[['#', 0],\n",
       "   ['programado', 1],\n",
       "   ['em', 2],\n",
       "   ['modo', 3],\n",
       "   ['ddd', 4],\n",
       "   ['.', 5]],\n",
       "  []],\n",
       " 128: [[['#', 0], ['bat', 1], ['ok', 2], ['.', 3]], []],\n",
       " 129: [[['#', 0],\n",
       "   ['para', 1],\n",
       "   ['eri', 2],\n",
       "   ['=', 3],\n",
       "   ['2', 4],\n",
       "   ['meses', 5],\n",
       "   ['.', 6]],\n",
       "  []],\n",
       " 130: [[['#', 0],\n",
       "   ['a', 1],\n",
       "   ['-', 2],\n",
       "   ['imp', 3],\n",
       "   ['=', 4],\n",
       "   ['388', 5],\n",
       "   [',', 6],\n",
       "   ['sense', 7],\n",
       "   ['=', 8],\n",
       "   ['5', 9],\n",
       "   [',', 10],\n",
       "   ['0', 11],\n",
       "   [',', 12],\n",
       "   ['clv', 13],\n",
       "   ['-', 14],\n",
       "   ['menor', 15],\n",
       "   ['que', 16],\n",
       "   ['1', 17],\n",
       "   [',', 18],\n",
       "   ['0', 19],\n",
       "   ['x', 20],\n",
       "   ['0', 21],\n",
       "   [',', 22],\n",
       "   ['4', 23],\n",
       "   ['.', 24]],\n",
       "  []],\n",
       " 131: [[['#', 0],\n",
       "   ['v', 1],\n",
       "   ['-', 2],\n",
       "   ['imp', 3],\n",
       "   ['=', 4],\n",
       "   ['395', 5],\n",
       "   [',', 6],\n",
       "   ['sense', 7],\n",
       "   ['=', 8],\n",
       "   ['11', 9],\n",
       "   [',', 10],\n",
       "   ['5', 11],\n",
       "   [',', 12],\n",
       "   ['clv', 13],\n",
       "   ['=', 14],\n",
       "   ['menor', 15],\n",
       "   ['que', 16],\n",
       "   ['1', 17],\n",
       "   [',', 18],\n",
       "   ['0', 19],\n",
       "   ['x', 20],\n",
       "   ['0', 21],\n",
       "   [',', 22],\n",
       "   ['4', 23],\n",
       "   ['.', 24]],\n",
       "  []],\n",
       " 132: [[['#', 0],\n",
       "   ['cd', 1],\n",
       "   ['=', 2],\n",
       "   ['retorno', 3],\n",
       "   ['como', 4],\n",
       "   ['encaixe', 5],\n",
       "   ['para', 6],\n",
       "   ['avaliar', 7],\n",
       "   ['bat', 8],\n",
       "   ['-', 9],\n",
       "   ['pcte', 10],\n",
       "   ['ciente', 11],\n",
       "   ['que', 12],\n",
       "   ['NÃO', 13],\n",
       "   ['PODE', 14],\n",
       "   ['DIXAR', 15],\n",
       "   ['DE', 16],\n",
       "   ['VIR', 17],\n",
       "   [',', 18],\n",
       "   ['se', 19],\n",
       "   ['intercorrencias', 20],\n",
       "   ['=', 21],\n",
       "   ['procura', 22],\n",
       "   ['direta', 23],\n",
       "   ['.', 24]],\n",
       "  []],\n",
       " 133: [[['ABLAÇÃO', 0],\n",
       "   ['DE', 1],\n",
       "   ['TAQUICARDIA', 2],\n",
       "   ['ATRIAL', 3],\n",
       "   ['2009', 4],\n",
       "   ['E', 5],\n",
       "   ['A', 6],\n",
       "   ['SEGUNDA', 7],\n",
       "   ['EM', 8],\n",
       "   ['2010', 9],\n",
       "   ['.', 10]],\n",
       "  [['ABLAÇÃO DE TAQUICARDIA ATRIAL', [0, 1, 2, 3], 'Tratamento'],\n",
       "   ['ATRIAL', [3], 'Anatomia']]],\n",
       " 134: [[['HAS', 0],\n",
       "   [',', 1],\n",
       "   ['NEGA', 2],\n",
       "   ['DM', 3],\n",
       "   [',', 4],\n",
       "   ['DLPD', 5],\n",
       "   [',', 6],\n",
       "   ['TABAGISMO', 7],\n",
       "   [',', 8],\n",
       "   ['ALERGIA', 9],\n",
       "   ['MEDICAMENTOSA', 10],\n",
       "   ['.', 11]],\n",
       "  [['HAS', [0], 'Problema'],\n",
       "   ['DM', [3], 'Problema'],\n",
       "   ['DLPD', [5], 'Problema'],\n",
       "   ['TABAGISMO', [7], 'Problema'],\n",
       "   ['ALERGIA MEDICAMENTOSA', [9, 10], 'Problema']]],\n",
       " 135: [[['CATETERISMO', 0],\n",
       "   ['COM', 1],\n",
       "   ['AUSENCIA', 2],\n",
       "   ['DE', 3],\n",
       "   ['CORONARIOPATIA', 4],\n",
       "   ['.', 5]],\n",
       "  [['CATETERISMO', [0], 'Teste'], ['CORONARIOPATIA', [4], 'Problema']]],\n",
       " 136: [[['FAZ', 0],\n",
       "   ['USO', 1],\n",
       "   ['DE', 2],\n",
       "   ['ATENOLOL', 3],\n",
       "   ['100MG', 4],\n",
       "   ['12', 5],\n",
       "   ['/', 6],\n",
       "   ['12H', 7],\n",
       "   [',', 8],\n",
       "   ['DILTIAZEN', 9],\n",
       "   ['60MG', 10],\n",
       "   ['12', 11],\n",
       "   ['/', 12],\n",
       "   ['12H', 13],\n",
       "   [',', 14],\n",
       "   ['ENALAPRIL', 15],\n",
       "   ['10MG', 16],\n",
       "   ['12', 17],\n",
       "   ['/', 18],\n",
       "   ['12H', 19],\n",
       "   [',', 20],\n",
       "   ['HCTZ', 21],\n",
       "   ['25MG', 22],\n",
       "   [',', 23],\n",
       "   ['MAREVAN', 24],\n",
       "   ['5MG', 25],\n",
       "   ['.', 26]],\n",
       "  [['ATENOLOL 100MG', [3, 4], 'Tratamento'],\n",
       "   ['DILTIAZEN 60MG', [9, 10], 'Tratamento'],\n",
       "   ['ENALAPRIL 10MG', [15, 16], 'Tratamento'],\n",
       "   ['HCTZ 25MG', [21, 22], 'Tratamento'],\n",
       "   ['MAREVAN 5MG', [24, 25], 'Tratamento']]],\n",
       " 137: [[['HOLTER', 0],\n",
       "   ['COM', 1],\n",
       "   ['PRESENCA', 2],\n",
       "   ['DE', 3],\n",
       "   ['EV', 4],\n",
       "   ['E', 5],\n",
       "   ['ESV', 6],\n",
       "   ['RARAS', 7],\n",
       "   ['COM', 8],\n",
       "   ['EPISODIOS', 9],\n",
       "   ['NAO', 10],\n",
       "   ['SUSTENTADOS', 11],\n",
       "   ['DE', 12],\n",
       "   ['TAQUICARDIA', 13],\n",
       "   ['ATRIAL', 14],\n",
       "   ['SEM', 15],\n",
       "   ['PRESENCA', 16],\n",
       "   ['DE', 17],\n",
       "   ['SINTOMAS', 18],\n",
       "   ['.', 19]],\n",
       "  [['HOLTER', [0], 'Teste'],\n",
       "   ['EV E ESV RARAS COM EPISODIOS NAO SUSTENTADOS DE TAQUICARDIA ATRIAL',\n",
       "    [4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14],\n",
       "    'Problema'],\n",
       "   ['SINTOMAS', [18], 'Problema']]],\n",
       " 138: [[['#', 0],\n",
       "   ['PACIENTE', 1],\n",
       "   ['REFERE', 2],\n",
       "   ['ESTAR', 3],\n",
       "   ['SE', 4],\n",
       "   ['SENTINDO', 5],\n",
       "   ['BEM', 6],\n",
       "   ['COM', 7],\n",
       "   ['AS', 8],\n",
       "   ['MEDICAÇÕES', 9],\n",
       "   ['.', 10]],\n",
       "  [['MEDICAÇÕES', [9], 'Tratamento']]],\n",
       " 139: [[['NEGA', 0], ['SINTOMAS', 1], ['DE', 2], ['PALPITAÇÕES', 3], ['.', 4]],\n",
       "  [['PALPITAÇÕES', [3], 'Problema']]],\n",
       " 140: [[['RELATA', 0],\n",
       "   ['DISPNEIA', 1],\n",
       "   ['AOS', 2],\n",
       "   ['GRANDES', 3],\n",
       "   ['ESFORCOS', 4],\n",
       "   [',', 5],\n",
       "   ['COM', 6],\n",
       "   ['MELHORA', 7],\n",
       "   ['COM', 8],\n",
       "   ['REPOUSO', 9],\n",
       "   ['.', 10]],\n",
       "  [['DISPNEIA AOS GRANDES ESFORCOS', [1, 2, 3, 4], 'Problema'],\n",
       "   ['MELHORA COM REPOUSO', [7, 8, 9], 'Problema']]],\n",
       " 141: [[['RELATA', 0],\n",
       "   ['QUE', 1],\n",
       "   ['APRESENTOU', 2],\n",
       "   ['1', 3],\n",
       "   ['EPISODIO', 4],\n",
       "   ['DE', 5],\n",
       "   ['HIPOTENSAO', 6],\n",
       "   ['.', 7]],\n",
       "  [['HIPOTENSAO', [6], 'Problema']]],\n",
       " 142: [[['RELATA', 0], ['CEFALEIA', 1], ['.', 2]],\n",
       "  [['CEFALEIA', [1], 'Problema']]],\n",
       " 143: [[['#', 0],\n",
       "   ['BEG', 1],\n",
       "   [',', 2],\n",
       "   ['PA', 3],\n",
       "   ['120', 4],\n",
       "   ['/', 5],\n",
       "   ['80MMHG', 6],\n",
       "   [',', 7],\n",
       "   ['FC', 8],\n",
       "   ['70', 9],\n",
       "   ['.', 10]],\n",
       "  [['PA', [3], 'Teste'], ['FC', [8], 'Teste']]],\n",
       " 144: [[['C', 0],\n",
       "   ['/', 1],\n",
       "   ['P', 2],\n",
       "   [':', 3],\n",
       "   ['SEM', 4],\n",
       "   ['ALT', 5],\n",
       "   ['.', 6]],\n",
       "  []],\n",
       " 145: [[['CORAÇÃO', 0],\n",
       "   [':', 1],\n",
       "   ['BRNF', 2],\n",
       "   ['2T', 3],\n",
       "   [',', 4],\n",
       "   ['SEM', 5],\n",
       "   ['SOPRO', 6],\n",
       "   ['.', 7]],\n",
       "  [['CORAÇÃO', [0], 'Anatomia'], ['SOPRO', [6], 'Problema']]],\n",
       " 146: [[['PULMAO', 0],\n",
       "   [':', 1],\n",
       "   ['MV', 2],\n",
       "   ['+', 3],\n",
       "   [',', 4],\n",
       "   ['SEM', 5],\n",
       "   ['RA', 6],\n",
       "   ['.', 7]],\n",
       "  [['PULMAO', [0], 'Anatomia'], ['RA', [6], 'Problema']]],\n",
       " 147: [[['CD', 0], [':', 1], ['MEDICAÇÃO', 2], ['MANTIDA', 3], ['.', 4]],\n",
       "  [['MEDICAÇÃO', [2], 'Tratamento']]],\n",
       " 148: [[['SOLICITO', 0],\n",
       "   ['NOVO', 1],\n",
       "   ['HOLTER', 2],\n",
       "   ['E', 3],\n",
       "   ['ECOCARDIO', 4],\n",
       "   ['.', 5]],\n",
       "  [['HOLTER', [2], 'Teste'], ['ECOCARDIO', [4], 'Teste']]],\n",
       " 149: [[['Lucas', 0], [',', 1], ['55', 2], ['anos', 3], ['.', 4]], []],\n",
       " 150: [[['Comorbidades', 0], [':', 1], ['HAS', 2], ['.', 3]],\n",
       "  [['Comorbidades', [0], 'Problema'], ['HAS', [2], 'Problema']]],\n",
       " 151: [[['Medicamentos', 0],\n",
       "   ['atenolol', 1],\n",
       "   ['100mg', 2],\n",
       "   [',', 3],\n",
       "   ['12', 4],\n",
       "   ['-', 5],\n",
       "   ['12h', 6],\n",
       "   [',', 7],\n",
       "   ['.', 8]],\n",
       "  [['Medicamentos', [0], 'Tratamento'],\n",
       "   ['atenolol 100mg', [1, 2], 'Tratamento']]],\n",
       " 152: [[['diltiazen', 0],\n",
       "   ['60mg', 1],\n",
       "   [',', 2],\n",
       "   ['12', 3],\n",
       "   ['-', 4],\n",
       "   ['12h', 5],\n",
       "   [',', 6],\n",
       "   ['.', 7]],\n",
       "  [['diltiazen 60mg', [0, 1], 'Tratamento']]],\n",
       " 153: [[['Enalapril', 0],\n",
       "   ['10mg', 1],\n",
       "   [',', 2],\n",
       "   ['12', 3],\n",
       "   ['-', 4],\n",
       "   ['12h', 5],\n",
       "   [',', 6],\n",
       "   ['.', 7]],\n",
       "  [['Enalapril 10mg', [0, 1], 'Tratamento']]],\n",
       " 154: [[['HCTZ', 0], ['25mg', 1], [',', 2], ['.', 3]],\n",
       "  [['HCTZ 25mg', [0, 1], 'Tratamento']]],\n",
       " 155: [[['Marevan', 0], ['5mg', 1], ['.', 2]],\n",
       "  [['Marevan 5mg', [0, 1], 'Tratamento']]],\n",
       " 156: [[['HMA', 0],\n",
       "   [':', 1],\n",
       "   ['fez', 2],\n",
       "   ['02', 3],\n",
       "   ['ablações', 4],\n",
       "   ['(', 5],\n",
       "   ['2009', 6],\n",
       "   ['e', 7],\n",
       "   ['2010', 8],\n",
       "   [')', 9],\n",
       "   ['POR', 10],\n",
       "   ['PALPITAÇÕES', 11],\n",
       "   ['com', 12],\n",
       "   ['taquicardia', 13],\n",
       "   ['atrial', 14],\n",
       "   [';', 15],\n",
       "   ['.', 16]],\n",
       "  [['02 ablações', [3, 4], 'Tratamento'],\n",
       "   ['PALPITAÇÕES', [11], 'Problema'],\n",
       "   ['taquicardia atrial', [13, 14], 'Problema'],\n",
       "   ['atrial', [14], 'Anatomia']]],\n",
       " 157: [[['Atualmente', 0],\n",
       "   ['sente', 1],\n",
       "   ['palpitações', 2],\n",
       "   ['quando', 3],\n",
       "   ['faz', 4],\n",
       "   ['grandes', 5],\n",
       "   ['esforços', 6],\n",
       "   [';', 7],\n",
       "   ['dor', 8],\n",
       "   ['de', 9],\n",
       "   ['cabeça', 10],\n",
       "   ['que', 11],\n",
       "   ['melhora', 12],\n",
       "   ['com', 13],\n",
       "   ['analgésico', 14],\n",
       "   ['comum', 15],\n",
       "   ['.', 16]],\n",
       "  [['palpitações quando faz grandes esforços', [2, 3, 4, 5, 6], 'Problema'],\n",
       "   ['dor de cabeça', [8, 9, 10], 'Problema'],\n",
       "   ['cabeça', [10], 'Anatomia'],\n",
       "   ['melhora com analgésico comum', [12, 13, 14, 15], 'Problema'],\n",
       "   ['analgésico comum', [14, 15], 'Tratamento']]],\n",
       " 158: [[['Relata', 0],\n",
       "   ['que', 1],\n",
       "   ['acompanha', 2],\n",
       "   ['pressão', 3],\n",
       "   ['no', 4],\n",
       "   ['posto', 5],\n",
       "   ['e', 6],\n",
       "   ['normalmente', 7],\n",
       "   ['está', 8],\n",
       "   ['120x80', 9],\n",
       "   ['.', 10]],\n",
       "  [['pressão', [3], 'Teste']]],\n",
       " 159: [[['FC', 0], [':', 1], ['88', 2], ['.', 3]], [['FC', [0], 'Teste']]],\n",
       " 160: [[['Edema', 0], ['discreto', 1], ['mmi', 2], ['.', 3]],\n",
       "  [['Edema discreto mmi', [0, 1, 2], 'Problema'], ['mmi', [2], 'Anatomia']]],\n",
       " 161: [[['Ausculta', 0],\n",
       "   ['pulmonar', 1],\n",
       "   ['e', 2],\n",
       "   ['cardiaca', 3],\n",
       "   ['normais', 4],\n",
       "   ['.', 5]],\n",
       "  [['pulmonar', [1], 'Anatomia'], ['cardiaca', [3], 'Anatomia']]],\n",
       " 162: [[['Lucas', 0], [',', 1], ['57', 2], ['anos', 3], ['.', 4]], []],\n",
       " 163: [[['Homoenxerto', 0],\n",
       "   ['aortico', 1],\n",
       "   ['(', 2],\n",
       "   ['por', 3],\n",
       "   ['EAo', 4],\n",
       "   [')', 5],\n",
       "   ['+', 6],\n",
       "   ['MIE', 7],\n",
       "   ['-', 8],\n",
       "   ['DA', 9],\n",
       "   [';', 10],\n",
       "   ['RAD', 11],\n",
       "   ['-', 12],\n",
       "   ['CD', 13],\n",
       "   [';', 14],\n",
       "   ['SAF', 15],\n",
       "   ['-', 16],\n",
       "   ['MG', 17],\n",
       "   ['em', 18],\n",
       "   ['2006', 19],\n",
       "   ['-', 20],\n",
       "   ['Doutor', 21],\n",
       "   ['Vital', 22],\n",
       "   ['Brazil', 23],\n",
       "   ['.', 24]],\n",
       "  [['Homoenxerto aortico', [0, 1], 'Tratamento'], ['EAo', [4], 'Problema']]],\n",
       " 164: [[['Hipertenso', 0], [',', 1], ['.', 2]],\n",
       "  [['Hipertenso', [0], 'Problema']]],\n",
       " 165: [[['Em', 0],\n",
       "   ['uso', 1],\n",
       "   ['de', 2],\n",
       "   [':', 3],\n",
       "   ['AAS', 4],\n",
       "   ['100', 5],\n",
       "   ['mg', 6],\n",
       "   ['dia', 7],\n",
       "   [',', 8],\n",
       "   ['losartana', 9],\n",
       "   ['100', 10],\n",
       "   ['mg', 11],\n",
       "   ['dia', 12],\n",
       "   [',', 13],\n",
       "   ['atovastatina', 14],\n",
       "   ['20', 15],\n",
       "   ['mg', 16],\n",
       "   ['4CP', 17],\n",
       "   ['/', 18],\n",
       "   ['DIA', 19],\n",
       "   [',', 20],\n",
       "   ['atenolol', 21],\n",
       "   ['25mg', 22],\n",
       "   ['/', 23],\n",
       "   ['dia', 24],\n",
       "   [',', 25],\n",
       "   ['furosemida', 26],\n",
       "   ['40mg', 27],\n",
       "   ['/', 28],\n",
       "   ['dia', 29],\n",
       "   [',', 30],\n",
       "   ['omeoprazol', 31],\n",
       "   ['40mg', 32],\n",
       "   ['ao', 33],\n",
       "   ['dia', 34],\n",
       "   [',', 35],\n",
       "   ['ezatimibe', 36],\n",
       "   ['10', 37],\n",
       "   ['mg', 38],\n",
       "   [',', 39],\n",
       "   ['Hidantal', 40],\n",
       "   ['100mg', 41],\n",
       "   ['-', 42],\n",
       "   ['por', 43],\n",
       "   ['cisticercose', 44],\n",
       "   ['.', 45]],\n",
       "  [['AAS 100 mg', [4, 5, 6], 'Tratamento'],\n",
       "   ['losartana 100 mg', [9, 10, 11], 'Tratamento'],\n",
       "   ['atovastatina 20 mg', [14, 15, 16], 'Tratamento'],\n",
       "   ['atenolol 25mg', [21, 22], 'Tratamento'],\n",
       "   ['furosemida 40mg', [26, 27], 'Tratamento'],\n",
       "   ['omeoprazol 40mg', [31, 32], 'Tratamento'],\n",
       "   ['ezatimibe 10 mg', [36, 37, 38], 'Tratamento'],\n",
       "   ['Hidantal 100mg', [40, 41], 'Tratamento'],\n",
       "   ['cisticercose', [44], 'Problema']]],\n",
       " 166: [[['Penicilina', 0],\n",
       "   ['G', 1],\n",
       "   ['benzatina', 2],\n",
       "   ['1', 3],\n",
       "   [',', 4],\n",
       "   ['200', 5],\n",
       "   [',', 6],\n",
       "   ['000U', 7],\n",
       "   ['IM', 8],\n",
       "   ['a', 9],\n",
       "   ['cada', 10],\n",
       "   ['21', 11],\n",
       "   ['dia', 12],\n",
       "   ['-', 13],\n",
       "   ['suspenso', 14],\n",
       "   ['em', 15],\n",
       "   ['2013', 16],\n",
       "   ['.', 17]],\n",
       "  [['Penicilina G benzatina 1 , 200 , 000U',\n",
       "    [0, 1, 2, 3, 4, 5, 6, 7],\n",
       "    'Tratamento']]],\n",
       " 167: [[['DOR', 0],\n",
       "   ['REGIAO', 1],\n",
       "   ['ESTERNAL', 2],\n",
       "   [',', 3],\n",
       "   ['EM', 4],\n",
       "   ['FO', 5],\n",
       "   [',', 6],\n",
       "   ['LOCAL', 7],\n",
       "   ['FIOS', 8],\n",
       "   ['AÇO', 9],\n",
       "   [',', 10],\n",
       "   ['EM', 11],\n",
       "   ['QUEIMAÇÃO', 12],\n",
       "   ['.', 13]],\n",
       "  [['DOR REGIAO ESTERNAL', [0, 1, 2], 'Problema'],\n",
       "   ['REGIAO ESTERNAL', [1, 2], 'Anatomia']]],\n",
       " 168: [[['-', 0],\n",
       "   ['Labs', 1],\n",
       "   [':', 2],\n",
       "   ['tg', 3],\n",
       "   ['221', 4],\n",
       "   ['/', 5],\n",
       "   ['ct', 6],\n",
       "   ['166', 7],\n",
       "   ['/', 8],\n",
       "   ['ldl', 9],\n",
       "   ['76', 10],\n",
       "   [',', 11],\n",
       "   ['demais', 12],\n",
       "   ['ok', 13],\n",
       "   ['.', 14]],\n",
       "  [['tg', [3], 'Teste'], ['ct', [6], 'Teste'], ['ldl', [9], 'Teste']]],\n",
       " 169: [[['-', 0],\n",
       "   ['Ecocardio', 1],\n",
       "   ['12', 2],\n",
       "   ['/', 3],\n",
       "   ['08', 4],\n",
       "   ['/', 5],\n",
       "   ['15', 6],\n",
       "   [':', 7],\n",
       "   ['A0', 8],\n",
       "   ['/', 9],\n",
       "   ['VD', 10],\n",
       "   ['22', 11],\n",
       "   ['/', 12],\n",
       "   ['Septo', 13],\n",
       "   ['11', 14],\n",
       "   ['/', 15],\n",
       "   ['FE', 16],\n",
       "   ['66', 17],\n",
       "   ['.', 18]],\n",
       "  [['Ecocardio', [1], 'Teste'],\n",
       "   ['VD', [10], 'Anatomia'],\n",
       "   ['Septo', [13], 'Anatomia']]],\n",
       " 170: [[['Homoenxerto', 0],\n",
       "   ['aortico', 1],\n",
       "   ['com', 2],\n",
       "   ['mobilidade', 3],\n",
       "   ['ampla', 4],\n",
       "   ['folhetps', 5],\n",
       "   ['.', 6]],\n",
       "  [['Homoenxerto', [0], 'Tratamento']]],\n",
       " 171: [[['grad', 0], ['pressao', 1], ['4mmHg', 2], ['.', 3]], []],\n",
       " 172: [[['Regurgitacao', 0],\n",
       "   ['grau', 1],\n",
       "   ['leve', 2],\n",
       "   ['a', 3],\n",
       "   ['moderada', 4],\n",
       "   ['com', 5],\n",
       "   ['jato', 6],\n",
       "   ['dirigido', 7],\n",
       "   ['valva', 8],\n",
       "   ['mitral', 9],\n",
       "   ['(', 10],\n",
       "   ['?', 11],\n",
       "   [')', 12],\n",
       "   ['.', 13]],\n",
       "  [['Regurgitacao grau leve a moderada', [0, 1, 2, 3, 4], 'Problema'],\n",
       "   ['valva mitral', [8, 9], 'Anatomia']]],\n",
       " 173: [[['Exames', 0], [':', 1], ['.', 2]], [['Exames', [0], 'Teste']]],\n",
       " 174: [[['Ecocardiograma', 0],\n",
       "   ['-', 1],\n",
       "   ['26', 2],\n",
       "   ['/', 3],\n",
       "   ['06', 4],\n",
       "   ['/', 5],\n",
       "   ['2013', 6],\n",
       "   [':', 7],\n",
       "   ['Homoenxerto', 8],\n",
       "   ['em', 9],\n",
       "   ['posição', 10],\n",
       "   ['aórtica', 11],\n",
       "   ['normofuncionante', 12],\n",
       "   ['.', 13]],\n",
       "  [['Ecocardiograma', [0], 'Teste'],\n",
       "   ['Homoenxerto em posição aórtica', [8, 9, 10, 11], 'Tratamento'],\n",
       "   ['aórtica', [11], 'Anatomia']]],\n",
       " 175: [[['VE', 0],\n",
       "   ['hipertrofiado', 1],\n",
       "   ['com', 2],\n",
       "   ['dimensões', 3],\n",
       "   ['normais', 4],\n",
       "   ['e', 5],\n",
       "   ['fumção', 6],\n",
       "   ['sistólica', 7],\n",
       "   ['preservada', 8],\n",
       "   ['.', 9]],\n",
       "  [['VE hipertrofiado', [0, 1], 'Problema']]],\n",
       " 176: [[['ALteração', 0],\n",
       "   ['do', 1],\n",
       "   ['relaxamento', 2],\n",
       "   ['do', 3],\n",
       "   ['VE', 4],\n",
       "   ['AE', 5],\n",
       "   ['dilatado', 6],\n",
       "   ['.', 7]],\n",
       "  [['VE', [4], 'Anatomia'], ['AE dilatado', [5, 6], 'Problema']]],\n",
       " 177: [[['Homoenxerto', 0],\n",
       "   ['com', 1],\n",
       "   ['mobilidade', 2],\n",
       "   ['adequada', 3],\n",
       "   ['.', 4]],\n",
       "  []],\n",
       " 178: [[['FE', 0], [':', 1], ['66', 2], ['%', 3], ['.', 4]], []],\n",
       " 179: [[['-', 0],\n",
       "   ['Lab', 1],\n",
       "   ['(', 2],\n",
       "   ['05', 3],\n",
       "   ['/', 4],\n",
       "   ['02', 5],\n",
       "   ['/', 6],\n",
       "   ['15', 7],\n",
       "   [')', 8],\n",
       "   [':', 9],\n",
       "   ['Cr', 10],\n",
       "   ['0', 11],\n",
       "   [',', 12],\n",
       "   ['91', 13],\n",
       "   [',', 14],\n",
       "   ['CT', 15],\n",
       "   ['174', 16],\n",
       "   [',', 17],\n",
       "   ['TGC', 18],\n",
       "   ['197', 19],\n",
       "   [',', 20],\n",
       "   ['HDL', 21],\n",
       "   ['44', 22],\n",
       "   [',', 23],\n",
       "   ['K', 24],\n",
       "   ['4', 25],\n",
       "   [',', 26],\n",
       "   ['8', 27],\n",
       "   [',', 28],\n",
       "   ['LDL', 29],\n",
       "   ['90', 30],\n",
       "   [',', 31],\n",
       "   ['Glic', 32],\n",
       "   ['107', 33],\n",
       "   ['.', 34]],\n",
       "  [['Lab', [1], 'Teste'],\n",
       "   ['Cr', [10], 'Teste'],\n",
       "   ['CT', [15], 'Teste'],\n",
       "   ['TGC', [18], 'Teste'],\n",
       "   ['HDL', [21], 'Teste'],\n",
       "   ['K', [24], 'Teste'],\n",
       "   ['LDL', [29], 'Teste'],\n",
       "   ['Glic', [32], 'Teste']]],\n",
       " 180: [[['Ao', 0], ['exame', 1], [':', 2], ['.', 3]],\n",
       "  [['exame', [1], 'Teste']]],\n",
       " 181: [[['Pte', 0],\n",
       "   ['em', 1],\n",
       "   ['BEG', 2],\n",
       "   [',', 3],\n",
       "   ['eupneico', 4],\n",
       "   [',', 5],\n",
       "   ['corado', 6],\n",
       "   [',', 7],\n",
       "   ['hidratado', 8],\n",
       "   ['.', 9]],\n",
       "  []],\n",
       " 182: [[['FC79', 0],\n",
       "   ['PA', 1],\n",
       "   [':', 2],\n",
       "   ['130', 3],\n",
       "   ['/', 4],\n",
       "   ['80', 5],\n",
       "   ['.', 6]],\n",
       "  [['PA', [1], 'Teste']]],\n",
       " 183: [[['CP', 0],\n",
       "   [':', 1],\n",
       "   ['sem', 2],\n",
       "   ['estase', 3],\n",
       "   ['jugular', 4],\n",
       "   ['.', 5]],\n",
       "  [['estase jugular', [3, 4], 'Problema'], ['jugular', [4], 'Anatomia']]],\n",
       " 184: [[['AC', 0],\n",
       "   [':', 1],\n",
       "   ['BCRNF', 2],\n",
       "   ['sem', 3],\n",
       "   ['sopros', 4],\n",
       "   ['.', 5]],\n",
       "  [['sopros', [4], 'Problema']]],\n",
       " 185: [[['AP', 0],\n",
       "   [':', 1],\n",
       "   ['MV', 2],\n",
       "   ['presente', 3],\n",
       "   ['bilateralmente', 4],\n",
       "   [',', 5],\n",
       "   ['sem', 6],\n",
       "   ['RA', 7],\n",
       "   ['.', 8]],\n",
       "  [['RA', [7], 'Problema']]],\n",
       " 186: [[['mantenho', 0], ['medicações', 1], ['.', 2]],\n",
       "  [['medicações', [1], 'Tratamento']]],\n",
       " 187: [[['encaminho', 0],\n",
       "   ['para', 1],\n",
       "   ['cir', 2],\n",
       "   ['cardiaca', 3],\n",
       "   ['para', 4],\n",
       "   ['retirada', 5],\n",
       "   ['pontos', 6],\n",
       "   ['suttura', 7],\n",
       "   ['esterno', 8],\n",
       "   ['.', 9]],\n",
       "  [['cardiaca', [3], 'Anatomia'], ['esterno', [8], 'Anatomia']]],\n",
       " 188: [[['retorno', 0], ['6', 1], ['meses', 2], ['.', 3]], []],\n",
       " 189: [[['e3', 0],\n",
       "   ['cardio', 1],\n",
       "   ['Doutor', 2],\n",
       "   ['Vital', 3],\n",
       "   ['Brazil', 4],\n",
       "   ['.', 5]],\n",
       "  []],\n",
       " 190: [[['USUÁRIA', 0],\n",
       "   ['DE', 1],\n",
       "   ['MAREVAN', 2],\n",
       "   ['DEVIDO', 3],\n",
       "   [\"'\", 4],\n",
       "   ['ARRITMIA', 5],\n",
       "   [\"'\", 6],\n",
       "   [',', 7],\n",
       "   ['NÃO', 8],\n",
       "   ['SABE', 9],\n",
       "   ['INFORMAR', 10],\n",
       "   ['MAIORES', 11],\n",
       "   ['DETALHES', 12],\n",
       "   ['.', 13]],\n",
       "  [['MAREVAN', [2], 'Tratamento'], ['ARRITMIA', [5], 'Problema']]],\n",
       " 191: [[['sEM', 0], ['QUEIXAS', 1], ['.', 2]], [['QUEIXAS', [1], 'Problema']]],\n",
       " 192: [[['SEM', 0], ['RELATO', 1], ['DE', 2], ['SANGRAMENTOS', 3], ['.', 4]],\n",
       "  [['SANGRAMENTOS', [3], 'Problema']]],\n",
       " 193: [[['EM', 0],\n",
       "   ['USO', 1],\n",
       "   ['DE', 2],\n",
       "   ['2CP', 3],\n",
       "   ['DE', 4],\n",
       "   ['SEG', 5],\n",
       "   ['A', 6],\n",
       "   ['SEXT', 7],\n",
       "   ['E', 8],\n",
       "   ['1CP', 9],\n",
       "   ['SAB', 10],\n",
       "   ['E', 11],\n",
       "   ['DOM', 12],\n",
       "   ['.', 13]],\n",
       "  []],\n",
       " 194: [[['RNI', 0], ['3', 1], [',', 2], ['40', 3], ['.', 4]],\n",
       "  [['RNI', [0], 'Teste']]],\n",
       " 195: [[['CD', 0], [':', 1], ['ORIENTO', 2], ['PACIENTE', 3], ['.', 4]], []],\n",
       " 196: [[['ALERTO', 0], ['SINAIS', 1], ['DE', 2], ['GRAVIDADE', 3], ['.', 4]],\n",
       "  []],\n",
       " 197: [[['MANTER', 0],\n",
       "   ['AS', 1],\n",
       "   ['MEDICAÇÃO', 2],\n",
       "   ['DE', 3],\n",
       "   ['USO', 4],\n",
       "   [',', 5],\n",
       "   ['COM', 6],\n",
       "   ['AJUSTE', 7],\n",
       "   ['DE', 8],\n",
       "   ['COSE', 9],\n",
       "   ['.', 10]],\n",
       "  [['MEDICAÇÃO', [2], 'Tratamento']]],\n",
       " 198: [[['RETORNO', 0], ['AGENDADA', 1], ['.', 2]], []],\n",
       " 199: [[['Lucas', 0], [',', 1], ['62', 2], ['anos', 3], ['.', 4]], []],\n",
       " 200: [[['#', 0], ['EAo', 1], ['.', 2]], [['EAo', [1], 'Problema']]],\n",
       " 201: [[['#', 0], ['HAS', 1], ['.', 2]], [['HAS', [1], 'Problema']]],\n",
       " 202: [[['#', 0],\n",
       "   ['Uso', 1],\n",
       "   [':', 2],\n",
       "   ['Atenol', 3],\n",
       "   ['50', 4],\n",
       "   ['-', 5],\n",
       "   ['1xd', 6],\n",
       "   ['.', 7]],\n",
       "  [['Atenol 50', [3, 4], 'Tratamento']]],\n",
       " 203: [[['Enalapril', 0],\n",
       "   ['5', 1],\n",
       "   ['-', 2],\n",
       "   ['12', 3],\n",
       "   ['/', 4],\n",
       "   ['12h', 5],\n",
       "   ['.', 6]],\n",
       "  [['Enalapril 5', [0, 1], 'Tratamento']]],\n",
       " 204: [[['#', 0],\n",
       "   ['Sem', 1],\n",
       "   ['queixas', 2],\n",
       "   ['cardiovasculares', 3],\n",
       "   ['.', 4]],\n",
       "  [['queixas cardiovasculares', [2, 3], 'Problema'],\n",
       "   ['cardiovasculares', [3], 'Anatomia']]],\n",
       " 205: [[['Ecocardio', 0],\n",
       "   ['(', 1],\n",
       "   ['13', 2],\n",
       "   ['/', 3],\n",
       "   ['02', 4],\n",
       "   ['/', 5],\n",
       "   ['15', 6],\n",
       "   [')', 7],\n",
       "   [':', 8],\n",
       "   ['AE', 9],\n",
       "   ['=', 10],\n",
       "   ['45', 11],\n",
       "   [';', 12],\n",
       "   ['SIV', 13],\n",
       "   ['=', 14],\n",
       "   ['PPVE', 15],\n",
       "   ['=', 16],\n",
       "   ['14mm', 17],\n",
       "   [';', 18],\n",
       "   ['DDVE', 19],\n",
       "   ['=', 20],\n",
       "   ['48', 21],\n",
       "   [';', 22],\n",
       "   ['DSVE', 23],\n",
       "   ['=', 24],\n",
       "   ['29', 25],\n",
       "   [';', 26],\n",
       "   ['FE', 27],\n",
       "   ['=', 28],\n",
       "   ['69', 29],\n",
       "   ['%', 30],\n",
       "   ['.', 31]],\n",
       "  [['Ecocardio', [0], 'Teste'],\n",
       "   ['AE', [9], 'Anatomia'],\n",
       "   ['SIV', [13], 'Anatomia'],\n",
       "   ['PPVE', [15], 'Anatomia']]],\n",
       " 206: [[['VMi', 0],\n",
       "   ['=', 1],\n",
       "   ['folhetos', 2],\n",
       "   ['espessados', 3],\n",
       "   [',', 4],\n",
       "   ['abertura', 5],\n",
       "   ['preservada', 6],\n",
       "   [',', 7],\n",
       "   ['refluxo', 8],\n",
       "   ['discreto', 9],\n",
       "   [',', 10],\n",
       "   ['fluxo', 11],\n",
       "   ['diastólico', 12],\n",
       "   ['com', 13],\n",
       "   ['padrão', 14],\n",
       "   ['E', 15],\n",
       "   ['<', 16],\n",
       "   ['A', 17],\n",
       "   ['VAo', 18],\n",
       "   ['=', 19],\n",
       "   ['cúspides', 20],\n",
       "   ['calcificadas', 21],\n",
       "   [',', 22],\n",
       "   ['com', 23],\n",
       "   ['dupla', 24],\n",
       "   ['lesão', 25],\n",
       "   [';', 26],\n",
       "   ['Grad', 27],\n",
       "   ['max', 28],\n",
       "   ['de', 29],\n",
       "   ['54', 30],\n",
       "   ['e', 31],\n",
       "   ['medio', 32],\n",
       "   ['de', 33],\n",
       "   ['28mmHg', 34],\n",
       "   ['.', 35]],\n",
       "  [['VMi', [0], 'Anatomia'],\n",
       "   ['folhetos espessados', [2, 3], 'Problema'],\n",
       "   ['refluxo discreto', [8, 9], 'Problema'],\n",
       "   ['VAo', [18], 'Anatomia'],\n",
       "   ['cúspides calcificadas', [20, 21], 'Problema'],\n",
       "   ['dupla lesão', [24, 25], 'Problema']]],\n",
       " 207: [[['VP', 0],\n",
       "   ['=', 1],\n",
       "   ['ok', 2],\n",
       "   [';', 3],\n",
       "   ['Vtri', 4],\n",
       "   ['=', 5],\n",
       "   ['folhetos', 6],\n",
       "   ['espessados', 7],\n",
       "   [',', 8],\n",
       "   ['abertura', 9],\n",
       "   ['preservada', 10],\n",
       "   [',', 11],\n",
       "   ['insuficiente', 12],\n",
       "   [',', 13],\n",
       "   ['PSAP', 14],\n",
       "   ['=', 15],\n",
       "   ['39mmHg', 16],\n",
       "   ['.', 17]],\n",
       "  [['VP', [0], 'Anatomia'],\n",
       "   ['Vtri', [4], 'Anatomia'],\n",
       "   ['folhetos espessados', [6, 7], 'Problema'],\n",
       "   ['insuficiente', [12], 'Problema']]],\n",
       " 208: [[['VE', 0],\n",
       "   ['=', 1],\n",
       "   ['hipertrofiado', 2],\n",
       "   [',', 3],\n",
       "   ['cavidade', 4],\n",
       "   [',', 5],\n",
       "   ['função', 6],\n",
       "   ['sistólica', 7],\n",
       "   ['e', 8],\n",
       "   ['contração', 9],\n",
       "   ['preservadas', 10],\n",
       "   [',', 11],\n",
       "   ['alt', 12],\n",
       "   ['de', 13],\n",
       "   ['relaxamento', 14],\n",
       "   ['.', 15]],\n",
       "  [['VE', [0], 'Anatomia'],\n",
       "   ['hipertrofiado', [2], 'Problema'],\n",
       "   ['alt de relaxamento', [12, 13, 14], 'Problema']]],\n",
       " 209: [[['AD', 0], ['=', 1], ['aumentado', 2], ['.', 3]],\n",
       "  [['AD', [0], 'Anatomia'], ['aumentado', [2], 'Problema']]],\n",
       " 210: [[['Eco', 0],\n",
       "   ['(', 1],\n",
       "   ['14', 2],\n",
       "   ['/', 3],\n",
       "   ['04', 4],\n",
       "   ['/', 5],\n",
       "   ['16', 6],\n",
       "   [')', 7],\n",
       "   ['RA', 8],\n",
       "   [':', 9],\n",
       "   ['38', 10],\n",
       "   ['AE', 11],\n",
       "   [':', 12],\n",
       "   ['45', 13],\n",
       "   ['VD', 14],\n",
       "   [':', 15],\n",
       "   ['20', 16],\n",
       "   ['S', 17],\n",
       "   [':', 18],\n",
       "   ['14', 19],\n",
       "   ['PP', 20],\n",
       "   [':', 21],\n",
       "   ['14', 22],\n",
       "   ['VE', 23],\n",
       "   [':', 24],\n",
       "   ['50', 25],\n",
       "   ['/', 26],\n",
       "   ['30', 27],\n",
       "   ['FE', 28],\n",
       "   [':', 29],\n",
       "   ['70', 30],\n",
       "   ['%', 31],\n",
       "   ['.', 32]],\n",
       "  [['Eco', [0], 'Teste'],\n",
       "   ['RA', [8], 'Anatomia'],\n",
       "   ['AE', [11], 'Anatomia'],\n",
       "   ['VD', [14], 'Anatomia'],\n",
       "   ['S', [17], 'Anatomia'],\n",
       "   ['PP', [20], 'Anatomia'],\n",
       "   ['VE', [23], 'Anatomia']]],\n",
       " 211: [[['AO', 0],\n",
       "   [':', 1],\n",
       "   ['CUSPIDES', 2],\n",
       "   ['CALCIFICADAS', 3],\n",
       "   [',', 4],\n",
       "   ['COM', 5],\n",
       "   ['DUPLA', 6],\n",
       "   ['LESAO', 7],\n",
       "   ['.', 8]],\n",
       "  [['AO', [0], 'Anatomia'],\n",
       "   ['CUSPIDES CALCIFICADAS', [2, 3], 'Problema'],\n",
       "   ['DUPLA LESAO', [6, 7], 'Problema']]],\n",
       " 212: [[['GRAD', 0],\n",
       "   ['MAX', 1],\n",
       "   ['=', 2],\n",
       "   ['43', 3],\n",
       "   ['MEDIO', 4],\n",
       "   ['25', 5],\n",
       "   ['.', 6]],\n",
       "  []],\n",
       " 213: [[['INSUF', 0],\n",
       "   ['TRICU', 1],\n",
       "   ['DISCRETA', 2],\n",
       "   ['(', 3],\n",
       "   ['PSAP', 4],\n",
       "   ['39', 5],\n",
       "   [')', 6],\n",
       "   ['.', 7]],\n",
       "  [['INSUF TRICU DISCRETA', [0, 1, 2], 'Problema'],\n",
       "   ['TRICU', [1], 'Anatomia']]],\n",
       " 214: [[['REFLUXO', 0], ['MITRAL', 1], ['DISCRETO', 2], ['.', 3]],\n",
       "  [['REFLUXO MITRAL DISCRETO', [0, 1, 2], 'Problema'],\n",
       "   ['MITRAL', [1], 'Anatomia']]],\n",
       " 215: [[['AO', 0], ['ASC', 1], [':', 2], ['41MM', 3], ['.', 4]],\n",
       "  [['AO ASC', [0, 1], 'Anatomia']]],\n",
       " 216: [[['S', 0], ['.', 1]], []],\n",
       " 217: [[['Nega', 0],\n",
       "   ['dispneia', 1],\n",
       "   [',', 2],\n",
       "   ['dor', 3],\n",
       "   ['toracica', 4],\n",
       "   [',', 5],\n",
       "   ['sincope', 6],\n",
       "   ['.', 7]],\n",
       "  [['dispneia', [1], 'Problema'],\n",
       "   ['dor toracica', [3, 4], 'Problema'],\n",
       "   ['toracica', [4], 'Anatomia'],\n",
       "   ['sincope', [6], 'Problema']]],\n",
       " 218: [[['Nega', 0], ['edema', 1], ['de', 2], ['MMII', 3], ['.', 4]],\n",
       "  [['edema de MMII', [1, 2, 3], 'Problema'], ['MMII', [3], 'Anatomia']]],\n",
       " 219: [[['Refere', 0],\n",
       "   ['controle', 1],\n",
       "   ['pressorico', 2],\n",
       "   ['bom', 3],\n",
       "   ['em', 4],\n",
       "   ['casa', 5],\n",
       "   [',', 6],\n",
       "   ['porem', 7],\n",
       "   ['nas', 8],\n",
       "   ['consultas', 9],\n",
       "   ['sempre', 10],\n",
       "   ['apresenta', 11],\n",
       "   ['elevação', 12],\n",
       "   ['de', 13],\n",
       "   ['PA', 14],\n",
       "   ['.', 15]],\n",
       "  [['elevação de PA', [12, 13, 14], 'Problema'], ['PA', [14], 'Teste']]],\n",
       " 220: [[['Sd', 0], ['Jaleco', 1], ['Branco', 2], ['?', 3], [')', 4], ['.', 5]],\n",
       "  [['Sd Jaleco Branco', [0, 1, 2], 'Problema']]],\n",
       " 221: [[['BEG', 0],\n",
       "   [',', 1],\n",
       "   ['eupneico', 2],\n",
       "   [',', 3],\n",
       "   ['acianotico', 4],\n",
       "   [',', 5],\n",
       "   ['anicterico', 6],\n",
       "   ['.', 7]],\n",
       "  []],\n",
       " 222: [[['PA', 0],\n",
       "   [':', 1],\n",
       "   ['170', 2],\n",
       "   ['/', 3],\n",
       "   ['100', 4],\n",
       "   ['mmHg', 5],\n",
       "   [',', 6],\n",
       "   ['P', 7],\n",
       "   ['=', 8],\n",
       "   ['99', 9],\n",
       "   ['spm', 10],\n",
       "   ['.', 11]],\n",
       "  [['P', [7], 'Teste']]],\n",
       " 223: [[['MV', 0], ['normodist', 1], ['sem', 2], ['ruidos', 3], ['.', 4]],\n",
       "  [['ruidos', [3], 'Problema']]],\n",
       " 224: [[['RCR', 0],\n",
       "   [',', 1],\n",
       "   ['2T', 2],\n",
       "   [',', 3],\n",
       "   ['taquicardico', 4],\n",
       "   [',', 5],\n",
       "   ['ss', 6],\n",
       "   ['+', 7],\n",
       "   ['+', 8],\n",
       "   ['/', 9],\n",
       "   ['6', 10],\n",
       "   ['+', 11],\n",
       "   [',', 12],\n",
       "   ['crsec', 13],\n",
       "   ['/', 14],\n",
       "   ['decres', 15],\n",
       "   [',', 16],\n",
       "   ['FAO', 17],\n",
       "   [',', 18],\n",
       "   ['irradiação', 19],\n",
       "   ['para', 20],\n",
       "   ['todo', 21],\n",
       "   ['o', 22],\n",
       "   ['precórdio', 23],\n",
       "   ['.', 24]],\n",
       "  [['taquicardico', [4], 'Problema'],\n",
       "   ['ss + + / 6 + , crsec / decres , FAO , irradiação para todo o precórdio',\n",
       "    [6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23],\n",
       "    'Problema'],\n",
       "   ['FAO', [17], 'Anatomia'],\n",
       "   ['precórdio', [23], 'Anatomia']]],\n",
       " 225: [[['A', 0],\n",
       "   ['#', 1],\n",
       "   ['DUPLA', 2],\n",
       "   ['LESAO', 3],\n",
       "   ['AO', 4],\n",
       "   ['ESTENOSE', 5],\n",
       "   ['MOD', 6],\n",
       "   ['+', 7],\n",
       "   ['INSUF', 8],\n",
       "   ['DISCRETA', 9],\n",
       "   ['.', 10]],\n",
       "  [['DUPLA LESAO AO ESTENOSE MOD + INSUF DISCRETA',\n",
       "    [2, 3, 4, 5, 6, 7, 8, 9],\n",
       "    'Problema'],\n",
       "   ['AO', [4], 'Anatomia']]],\n",
       " 226: [[['P', 0],\n",
       "   ['#', 1],\n",
       "   ['aumento', 2],\n",
       "   ['Enalapril', 3],\n",
       "   ['20mg', 4],\n",
       "   ['2x', 5],\n",
       "   ['d', 6],\n",
       "   ['e', 7],\n",
       "   ['inicio', 8],\n",
       "   ['HCTZ', 9],\n",
       "   ['25mg', 10],\n",
       "   ['/', 11],\n",
       "   ['dia', 12],\n",
       "   ['.', 13]],\n",
       "  [['Enalapril 20mg', [3, 4], 'Tratamento'],\n",
       "   ['HCTZ 25mg', [9, 10], 'Tratamento']]],\n",
       " 227: [[['retorno', 0],\n",
       "   ['em', 1],\n",
       "   ['1', 2],\n",
       "   ['ano', 3],\n",
       "   ['com', 4],\n",
       "   ['novo', 5],\n",
       "   ['eco', 6],\n",
       "   ['.', 7]],\n",
       "  [['eco', [6], 'Teste']]],\n",
       " 228: [[['R4', 0],\n",
       "   ['cardio', 1],\n",
       "   ['Doutor', 2],\n",
       "   ['Vital', 3],\n",
       "   ['Brazil', 4],\n",
       "   ['.', 5]],\n",
       "  []],\n",
       " 229: [[['#', 0],\n",
       "   ['MCP', 1],\n",
       "   ['dilatada', 2],\n",
       "   ['-', 3],\n",
       "   ['IDIOPÁTICA', 4],\n",
       "   ['.', 5]],\n",
       "  [['MCP dilatada - IDIOPÁTICA', [1, 2, 3, 4], 'Problema']]],\n",
       " 230: [[['CF', 0],\n",
       "   ['II', 1],\n",
       "   ['-', 2],\n",
       "   ['III', 3],\n",
       "   ['OSCILANTE', 4],\n",
       "   ['.', 5]],\n",
       "  [['CF II - III OSCILANTE', [0, 1, 2, 3, 4], 'Problema']]],\n",
       " 231: [[['**', 0],\n",
       "   ['PROTOCOLO', 1],\n",
       "   ['DE', 2],\n",
       "   ['TX', 3],\n",
       "   ['CARDIACO', 4],\n",
       "   ['-', 5],\n",
       "   ['INDICE', 6],\n",
       "   ['CARDIACO', 7],\n",
       "   ['.', 8]],\n",
       "  [['CARDIACO', [4], 'Anatomia'],\n",
       "   ['INDICE CARDIACO', [6, 7], 'Teste'],\n",
       "   ['CARDIACO', [7], 'Anatomia']]],\n",
       " 232: [[['***', 0],\n",
       "   ['Em', 1],\n",
       "   ['uso', 2],\n",
       "   ['de', 3],\n",
       "   [':', 4],\n",
       "   ['Omeprazol', 5],\n",
       "   ['20mg', 6],\n",
       "   ['/', 7],\n",
       "   ['d', 8],\n",
       "   [',', 9],\n",
       "   ['AAS', 10],\n",
       "   ['100mg', 11],\n",
       "   ['/', 12],\n",
       "   ['d', 13],\n",
       "   [',', 14],\n",
       "   ['Losartan', 15],\n",
       "   ['50mg', 16],\n",
       "   ['12', 17],\n",
       "   ['/', 18],\n",
       "   ['12hs', 19],\n",
       "   [',', 20],\n",
       "   ['Espironolactona', 21],\n",
       "   ['25mg', 22],\n",
       "   ['/', 23],\n",
       "   ['d', 24],\n",
       "   [',', 25],\n",
       "   ['Hidralazina', 26],\n",
       "   ['50mg', 27],\n",
       "   ['12', 28],\n",
       "   ['/', 29],\n",
       "   ['12hs', 30],\n",
       "   [',', 31],\n",
       "   ['Carvedilol', 32],\n",
       "   ['12', 33],\n",
       "   [',', 34],\n",
       "   ['5mg', 35],\n",
       "   ['12', 36],\n",
       "   ['/', 37],\n",
       "   ['12hs', 38],\n",
       "   ['.', 39]],\n",
       "  [['Omeprazol 20mg', [5, 6], 'Tratamento'],\n",
       "   ['AAS 100mg', [10, 11], 'Tratamento'],\n",
       "   ['Losartan 50mg', [15, 16], 'Tratamento'],\n",
       "   ['Espironolactona 25mg', [21, 22], 'Tratamento'],\n",
       "   ['Hidralazina 50mg', [26, 27], 'Tratamento'],\n",
       "   ['Carvedilol 12 , 5mg', [32, 33, 34, 35], 'Tratamento']]],\n",
       " 233: [[['S', 0],\n",
       "   ['#', 1],\n",
       "   ['QUEIXA', 2],\n",
       "   ['DE', 3],\n",
       "   ['CANSAÇO', 4],\n",
       "   ['AOS', 5],\n",
       "   ['ESFORÇOS', 6],\n",
       "   ['.', 7]],\n",
       "  [['CANSAÇO AOS ESFORÇOS', [4, 5, 6], 'Problema']]],\n",
       " 234: [[['DEMAIS', 0], ['SP', 1], ['.', 2]], []],\n",
       " 235: [[['TG', 0],\n",
       "   ['111', 1],\n",
       "   [',', 2],\n",
       "   ['HDL', 3],\n",
       "   ['37', 4],\n",
       "   [',', 5],\n",
       "   ['K', 6],\n",
       "   ['4', 7],\n",
       "   [',', 8],\n",
       "   ['7', 9],\n",
       "   [',', 10],\n",
       "   ['CT', 11],\n",
       "   ['162', 12],\n",
       "   [',', 13],\n",
       "   ['LDL', 14],\n",
       "   ['103', 15],\n",
       "   [',', 16],\n",
       "   ['TSH', 17],\n",
       "   ['1', 18],\n",
       "   [',', 19],\n",
       "   ['59', 20],\n",
       "   [',', 21],\n",
       "   ['HB', 22],\n",
       "   ['13', 23],\n",
       "   [',', 24],\n",
       "   ['HB', 25],\n",
       "   ['GLICASADA', 26],\n",
       "   ['5', 27],\n",
       "   [',', 28],\n",
       "   ['87', 29],\n",
       "   [',', 30],\n",
       "   ['NA', 31],\n",
       "   ['145', 32],\n",
       "   [',', 33],\n",
       "   ['TGP', 34],\n",
       "   ['28', 35],\n",
       "   [',', 36],\n",
       "   ['TGO', 37],\n",
       "   ['26', 38],\n",
       "   [',', 39],\n",
       "   ['CREAT', 40],\n",
       "   ['0', 41],\n",
       "   [',', 42],\n",
       "   ['98', 43],\n",
       "   ['.', 44]],\n",
       "  [['TG', [0], 'Teste'],\n",
       "   ['HDL', [3], 'Teste'],\n",
       "   ['K', [6], 'Teste'],\n",
       "   ['CT', [11], 'Teste'],\n",
       "   ['LDL', [14], 'Teste'],\n",
       "   ['TSH', [17], 'Teste'],\n",
       "   ['HB', [22], 'Teste'],\n",
       "   ['HB GLICASADA', [25, 26], 'Teste'],\n",
       "   ['NA', [31], 'Teste'],\n",
       "   ['TGP', [34], 'Teste'],\n",
       "   ['TGO', [37], 'Teste'],\n",
       "   ['CREAT', [40], 'Teste']]],\n",
       " 236: [[['AO', 0],\n",
       "   ['EAXME', 1],\n",
       "   ['PA', 2],\n",
       "   ['110', 3],\n",
       "   ['/', 4],\n",
       "   ['70', 5],\n",
       "   ['FC', 6],\n",
       "   ['64', 7],\n",
       "   ['.', 8]],\n",
       "  [['EAXME', [1], 'Teste'], ['PA', [2], 'Teste'], ['FC', [6], 'Teste']]],\n",
       " 237: [[['SEM', 0], ['TURGÊNCIA', 1], ['JUGULAR', 2], ['.', 3]],\n",
       "  [['TURGÊNCIA JUGULAR', [1, 2], 'Problema'], ['JUGULAR', [2], 'Anatomia']]],\n",
       " 238: [[['CPP', 0],\n",
       "   ['-', 1],\n",
       "   ['MV', 2],\n",
       "   ['+', 3],\n",
       "   ['BILATERAL', 4],\n",
       "   [',', 5],\n",
       "   ['.', 6]],\n",
       "  [['CPP', [0], 'Anatomia']]],\n",
       " 239: [[['PC', 0],\n",
       "   ['-', 1],\n",
       "   ['BCR', 2],\n",
       "   ['SEM', 3],\n",
       "   ['SOPROS', 4],\n",
       "   [',', 5],\n",
       "   ['.', 6]],\n",
       "  [['SOPROS', [4], 'Problema']]],\n",
       " 240: [[['MMII', 0], ['-', 1], ['SEM', 2], ['EDEM', 3], ['.', 4]],\n",
       "  [['MMII', [0], 'Anatomia'], ['EDEM', [3], 'Problema']]],\n",
       " 241: [[['A', 0],\n",
       "   ['#', 1],\n",
       "   ['MCP', 2],\n",
       "   ['dilatada', 3],\n",
       "   ['CF', 4],\n",
       "   ['II', 5],\n",
       "   ['/', 6],\n",
       "   ['III', 7],\n",
       "   ['.', 8]],\n",
       "  [['MCP dilatada CF II / III', [2, 3, 4, 5, 6, 7], 'Problema']]],\n",
       " 242: [[['P', 0], ['#', 1], ['MEDICAÇÕES', 2], ['MANTIDAS', 3], ['.', 4]],\n",
       "  [['MEDICAÇÕES', [2], 'Tratamento']]],\n",
       " 243: [[['Lucas', 0], [',', 1], ['65', 2], ['ANOS', 3], ['.', 4]], []],\n",
       " 244: [[['#', 0],\n",
       "   ['PO', 1],\n",
       "   ['DE', 2],\n",
       "   ['TX', 3],\n",
       "   ['CARDÍACO', 4],\n",
       "   ['(', 5],\n",
       "   ['23', 6],\n",
       "   ['/', 7],\n",
       "   ['10', 8],\n",
       "   ['/', 9],\n",
       "   ['15', 10],\n",
       "   [')', 11],\n",
       "   ['-', 12],\n",
       "   ['(', 13],\n",
       "   ['IC', 14],\n",
       "   ['DILATADA', 15],\n",
       "   ['IDIOPÁTICA', 16],\n",
       "   [')', 17],\n",
       "   ['.', 18]],\n",
       "  [['TX CARDÍACO', [3, 4], 'Tratamento'],\n",
       "   ['CARDÍACO', [4], 'Anatomia'],\n",
       "   ['IC DILATADA IDIOPÁTICA', [14, 15, 16], 'Problema']]],\n",
       " 245: [[['#', 0],\n",
       "   ['EM', 1],\n",
       "   ['USO', 2],\n",
       "   ['DE', 3],\n",
       "   [':', 4],\n",
       "   ['CICLOSPORINA', 5],\n",
       "   ['150MG', 6],\n",
       "   ['12', 7],\n",
       "   ['/', 8],\n",
       "   ['12H', 9],\n",
       "   [',', 10],\n",
       "   ['MICOFENOLATO', 11],\n",
       "   ['720MG', 12],\n",
       "   ['8', 13],\n",
       "   ['/', 14],\n",
       "   ['8H', 15],\n",
       "   [',', 16],\n",
       "   ['PREDNISONA', 17],\n",
       "   ['20MG', 18],\n",
       "   ['/', 19],\n",
       "   ['DIA', 20],\n",
       "   ['.', 21]],\n",
       "  [['CICLOSPORINA 150MG', [5, 6], 'Tratamento'],\n",
       "   ['MICOFENOLATO 720MG', [11, 12], 'Tratamento'],\n",
       "   ['PREDNISONA 20MG', [17, 18], 'Tratamento']]],\n",
       " 246: [[['AAS', 0],\n",
       "   ['100MG', 1],\n",
       "   ['/', 2],\n",
       "   ['DIA', 3],\n",
       "   [',', 4],\n",
       "   ['OMZ', 5],\n",
       "   ['20MG', 6],\n",
       "   ['/', 7],\n",
       "   ['DIA', 8],\n",
       "   ['.', 9]],\n",
       "  [['AAS 100MG', [0, 1], 'Tratamento'], ['OMZ 20MG', [5, 6], 'Tratamento']]],\n",
       " 247: [[['-', 0],\n",
       "   ['EXAMES', 1],\n",
       "   ['-', 2],\n",
       "   ['AVALIAÇÃO', 3],\n",
       "   ['PRÉ', 4],\n",
       "   ['-', 5],\n",
       "   ['TRANSPLANTE', 6],\n",
       "   [':', 7],\n",
       "   ['.', 8]],\n",
       "  [['EXAMES', [1], 'Teste'],\n",
       "   ['AVALIAÇÃO PRÉ - TRANSPLANTE', [3, 4, 5, 6], 'Teste']]],\n",
       " 248: [[['#', 0],\n",
       "   ['DOPPLER', 1],\n",
       "   ['CAROTIDAS', 2],\n",
       "   ['+', 3],\n",
       "   ['VERTEBRAIS', 4],\n",
       "   [':', 5],\n",
       "   ['CERVICAL', 6],\n",
       "   ['D', 7],\n",
       "   [':', 8],\n",
       "   ['CAROTIDA', 9],\n",
       "   [':', 10],\n",
       "   ['ESTENOSE', 11],\n",
       "   ['20', 12],\n",
       "   ['-', 13],\n",
       "   ['30', 14],\n",
       "   ['%', 15],\n",
       "   ['NA', 16],\n",
       "   ['BIFURCAÇÃO', 17],\n",
       "   ['CAROTÍDEA', 18],\n",
       "   [';', 19],\n",
       "   ['.', 20]],\n",
       "  [['DOPPLER CAROTIDAS + VERTEBRAIS', [1, 2, 3, 4], 'Teste'],\n",
       "   ['CAROTIDAS', [2], 'Anatomia'],\n",
       "   ['VERTEBRAIS', [4], 'Anatomia'],\n",
       "   ['CERVICAL D', [6, 7], 'Anatomia'],\n",
       "   ['CAROTIDA', [9], 'Anatomia'],\n",
       "   ['ESTENOSE 20 - 30 % NA BIFURCAÇÃO CAROTÍDEA',\n",
       "    [11, 12, 13, 14, 15, 16, 17, 18],\n",
       "    'Problema'],\n",
       "   ['BIFURCAÇÃO CAROTÍDEA', [17, 18], 'Anatomia']]],\n",
       " 249: [[['#', 0],\n",
       "   ['CATE', 1],\n",
       "   ['16', 2],\n",
       "   ['/', 3],\n",
       "   ['06', 4],\n",
       "   ['/', 5],\n",
       "   ['15', 6],\n",
       "   [':', 7],\n",
       "   ['PRÉ', 8],\n",
       "   [':', 9],\n",
       "   ['RVP', 10],\n",
       "   ['2', 11],\n",
       "   [',', 12],\n",
       "   ['19', 13],\n",
       "   ['/', 14],\n",
       "   ['/', 15],\n",
       "   ['RVS', 16],\n",
       "   ['24', 17],\n",
       "   [',', 18],\n",
       "   ['PÓS', 19],\n",
       "   [':', 20],\n",
       "   ['RVP', 21],\n",
       "   ['2', 22],\n",
       "   [',', 23],\n",
       "   ['08', 24],\n",
       "   ['/', 25],\n",
       "   ['/', 26],\n",
       "   ['RVS', 27],\n",
       "   ['12', 28],\n",
       "   [',', 29],\n",
       "   ['59', 30],\n",
       "   ['.', 31]],\n",
       "  [['CATE', [1], 'Teste']]],\n",
       " 250: [[['#', 0],\n",
       "   ['ECOTT', 1],\n",
       "   ['28', 2],\n",
       "   ['/', 3],\n",
       "   ['8', 4],\n",
       "   ['/', 5],\n",
       "   ['15', 6],\n",
       "   ['AE', 7],\n",
       "   ['46', 8],\n",
       "   ['/', 9],\n",
       "   ['VD', 10],\n",
       "   ['26', 11],\n",
       "   ['/', 12],\n",
       "   ['S', 13],\n",
       "   ['/', 14],\n",
       "   ['PP', 15],\n",
       "   ['10', 16],\n",
       "   ['/', 17],\n",
       "   ['10', 18],\n",
       "   ['VE', 19],\n",
       "   ['81', 20],\n",
       "   ['/', 21],\n",
       "   ['66', 22],\n",
       "   ['FE', 23],\n",
       "   ['37', 24],\n",
       "   ['%', 25],\n",
       "   ['.', 26]],\n",
       "  [['ECOTT', [1], 'Teste'],\n",
       "   ['AE', [7], 'Anatomia'],\n",
       "   ['VD', [10], 'Anatomia'],\n",
       "   ['PP', [15], 'Anatomia'],\n",
       "   ['VE', [19], 'Anatomia']]],\n",
       " 251: [[['VE', 0],\n",
       "   ['DILATADO', 1],\n",
       "   ['COM', 2],\n",
       "   ['HIPOCONTRATIBILIDADE', 3],\n",
       "   ['DIFUSA', 4],\n",
       "   ['MODERADA', 5],\n",
       "   ['/', 6],\n",
       "   ['I', 7],\n",
       "   ['AO', 8],\n",
       "   ['LEVE', 9],\n",
       "   ['/', 10],\n",
       "   ['IMI', 11],\n",
       "   ['LEVE', 12],\n",
       "   ['AMODERADA', 13],\n",
       "   ['/', 14],\n",
       "   ['PSAP', 15],\n",
       "   ['39', 16],\n",
       "   ['MMHG', 17],\n",
       "   ['.', 18]],\n",
       "  [['VE DILATADO COM HIPOCONTRATIBILIDADE DIFUSA MODERADA',\n",
       "    [0, 1, 2, 3, 4, 5],\n",
       "    'Problema'],\n",
       "   ['I AO LEVE', [7, 8, 9], 'Problema'],\n",
       "   ['AO', [8], 'Anatomia'],\n",
       "   ['IMI LEVE AMODERADA', [11, 12, 13], 'Problema']]],\n",
       " 252: [[['S', 0],\n",
       "   ['#', 1],\n",
       "   ['SENTE', 2],\n",
       "   ['BEM', 3],\n",
       "   [',', 4],\n",
       "   ['DOR', 5],\n",
       "   ['NA', 6],\n",
       "   ['FO', 7],\n",
       "   ['SOMENTE', 8],\n",
       "   ['AOS', 9],\n",
       "   ['MOVIMENTOS', 10],\n",
       "   ['E', 11],\n",
       "   ['AO', 12],\n",
       "   ['TOSSIR', 13],\n",
       "   ['.', 14]],\n",
       "  [['DOR NA FO', [5, 6, 7], 'Problema'],\n",
       "   ['AOS MOVIMENTOS E AO TOSSIR', [9, 10, 11, 12, 13], 'Problema']]],\n",
       " 253: [[['NEGA', 0], ['DISPNÉIA', 1], ['.', 2]],\n",
       "  [['DISPNÉIA', [1], 'Problema']]],\n",
       " 254: [[['RELATA', 0],\n",
       "   ['QUE', 1],\n",
       "   ['FO', 2],\n",
       "   ['ESTÁ', 3],\n",
       "   ['SECA', 4],\n",
       "   [',', 5],\n",
       "   ['SEM', 6],\n",
       "   ['SINAIS', 7],\n",
       "   ['FLOGÍSTICOS', 8],\n",
       "   ['.', 9]],\n",
       "  [['FO', [2], 'Problema'], ['SINAIS FLOGÍSTICOS', [7, 8], 'Problema']]],\n",
       " 255: [[['NEGA', 0], ['FEBRE', 1], ['.', 2]], [['FEBRE', [1], 'Problema']]],\n",
       " 256: [[['O', 0],\n",
       "   ['#', 1],\n",
       "   ['BEG', 2],\n",
       "   [',', 3],\n",
       "   ['HIPOCORADO', 4],\n",
       "   ['+', 5],\n",
       "   ['/', 6],\n",
       "   ['4', 7],\n",
       "   ['+', 8],\n",
       "   [',', 9],\n",
       "   ['EUPNEICO', 10],\n",
       "   [',', 11],\n",
       "   ['AFEBRIL', 12],\n",
       "   ['.', 13]],\n",
       "  [['HIPOCORADO + / 4 + ', [4, 5, 6, 7, 8], 'Problema'],\n",
       "   ['AFEBRIL', [12], 'Problema']]],\n",
       " 257: [[['PA', 0],\n",
       "   [':', 1],\n",
       "   ['110X70', 2],\n",
       "   ['/', 3],\n",
       "   ['/', 4],\n",
       "   ['FC', 5],\n",
       "   [':', 6],\n",
       "   ['80', 7],\n",
       "   ['.', 8]],\n",
       "  [['PA', [0], 'Teste'], ['FC', [5], 'Teste']]],\n",
       " 258: [[['CV', 0], [':', 1], ['BRNF', 2], ['SS', 3], ['.', 4]],\n",
       "  [['SS', [3], 'Problema']]],\n",
       " 259: [[['CPP', 0],\n",
       "   [':', 1],\n",
       "   ['MV', 2],\n",
       "   ['+', 3],\n",
       "   ['BILATERAL', 4],\n",
       "   [',', 5],\n",
       "   ['SEM', 6],\n",
       "   ['RA', 7],\n",
       "   ['.', 8]],\n",
       "  [['CPP', [0], 'Anatomia'], ['RA', [7], 'Problema']]],\n",
       " 260: [[['FO', 0],\n",
       "   ['LIMPA', 1],\n",
       "   ['E', 2],\n",
       "   ['SECA', 3],\n",
       "   [',', 4],\n",
       "   ['SEM', 5],\n",
       "   ['PRESENÇA', 6],\n",
       "   ['DE', 7],\n",
       "   ['SECREÇÃO', 8],\n",
       "   ['.', 9]],\n",
       "  [['FO', [0], 'Problema'], ['SECREÇÃO', [8], 'Problema']]],\n",
       " 261: [[['ABD', 0],\n",
       "   [':', 1],\n",
       "   ['RHA', 2],\n",
       "   ['+', 3],\n",
       "   [',', 4],\n",
       "   ['INDOLOR', 5],\n",
       "   ['.', 6]],\n",
       "  [['ABD', [0], 'Anatomia']]],\n",
       " 262: [[['MMII', 0],\n",
       "   [':', 1],\n",
       "   ['SEM', 2],\n",
       "   ['EDEMA', 3],\n",
       "   ['OU', 4],\n",
       "   ['EMPASTAMENTO', 5],\n",
       "   ['.', 6]],\n",
       "  [['MMII', [0], 'Anatomia'],\n",
       "   ['EDEMA', [3], 'Problema'],\n",
       "   ['EMPASTAMENTO', [5], 'Problema']]],\n",
       " 263: [[['A', 0],\n",
       "   ['#', 1],\n",
       "   ['PO', 2],\n",
       "   ['DE', 3],\n",
       "   ['TX', 4],\n",
       "   ['CARDIACO', 5],\n",
       "   ['(', 6],\n",
       "   ['23', 7],\n",
       "   ['/', 8],\n",
       "   ['10', 9],\n",
       "   ['/', 10],\n",
       "   ['15', 11],\n",
       "   [')', 12],\n",
       "   ['.', 13]],\n",
       "  [['TX CARDIACO', [4, 5], 'Tratamento'], ['CARDIACO', [5], 'Anatomia']]],\n",
       " 264: [[['P', 0],\n",
       "   ['#', 1],\n",
       "   ['MANTIDO', 2],\n",
       "   ['RETORNO', 3],\n",
       "   ['EM', 4],\n",
       "   ['30', 5],\n",
       "   ['DIAS', 6],\n",
       "   ['.', 7]],\n",
       "  []],\n",
       " 265: [[['POS', 0],\n",
       "   ['TX', 1],\n",
       "   ['CARDIACO', 2],\n",
       "   ['HA', 3],\n",
       "   ['3', 4],\n",
       "   ['ANOS', 5],\n",
       "   ['e', 6],\n",
       "   ['7', 7],\n",
       "   ['MESES', 8],\n",
       "   ['.', 9]],\n",
       "  [['TX CARDIACO', [1, 2], 'Tratamento'], ['CARDIACO', [2], 'Anatomia']]],\n",
       " 266: [[['ULTIMA', 0], ['BIOPSIA', 1], ['.', 2]],\n",
       "  [['ULTIMA BIOPSIA', [0, 1], 'Teste'], ['BIOPSIA', [1], 'Teste']]],\n",
       " 267: [[['EM', 0],\n",
       "   ['USO', 1],\n",
       "   ['DE', 2],\n",
       "   ['CICLOSPORINA', 3],\n",
       "   ['100', 4],\n",
       "   ['/', 5],\n",
       "   ['100', 6],\n",
       "   [',', 7],\n",
       "   ['MYFORTIC', 8],\n",
       "   ['360', 9],\n",
       "   ['/', 10],\n",
       "   ['360', 11],\n",
       "   ['(', 12],\n",
       "   ['REDUZIU', 13],\n",
       "   ['PARA', 14],\n",
       "   ['ESSA', 15],\n",
       "   ['DOSE', 16],\n",
       "   ['POR', 17],\n",
       "   ['ENGANO', 18],\n",
       "   [')', 19],\n",
       "   [',', 20],\n",
       "   ['E', 21],\n",
       "   ['PREDNISONA', 22],\n",
       "   ['5', 23],\n",
       "   ['MG', 24],\n",
       "   ['/', 25],\n",
       "   ['DIA', 26],\n",
       "   ['.', 27]],\n",
       "  [['CICLOSPORINA', [3], 'Tratamento'],\n",
       "   ['MYFORTIC', [8], 'Tratamento'],\n",
       "   ['PREDNISONA 5 MG', [22, 23, 24], 'Tratamento']]],\n",
       " 268: [[['USO', 0],\n",
       "   ['DE', 1],\n",
       "   ['AAS', 2],\n",
       "   ['100', 3],\n",
       "   ['/', 4],\n",
       "   ['DIA', 5],\n",
       "   ['.', 6]],\n",
       "  [['AAS 100', [2, 3], 'Tratamento']]],\n",
       " 269: [[['S', 0], ['#', 1], ['SEM', 2], ['QUEIXAS', 3], ['.', 4]],\n",
       "  [['QUEIXAS', [3], 'Problema']]],\n",
       " 270: [[['DEMAIS', 0], ['SP', 1], ['.', 2]], []],\n",
       " 271: [[['AGURADA', 0], ['BIOPSIA', 1], ['.', 2]], []],\n",
       " 272: [[['O', 0],\n",
       "   ['#', 1],\n",
       "   ['PA', 2],\n",
       "   ['120', 3],\n",
       "   ['/', 4],\n",
       "   ['80', 5],\n",
       "   [',', 6],\n",
       "   ['FC', 7],\n",
       "   ['70', 8],\n",
       "   ['.', 9]],\n",
       "  [['PA', [2], 'Teste'], ['FC', [7], 'Teste']]],\n",
       " 273: [[['ACV', 0], ['BCRNF', 1], ['.', 2]], []],\n",
       " 274: [[['CPP', 0], ['LIVRE', 1], ['.', 2]], [['CPP', [0], 'Anatomia']]],\n",
       " 275: [[['MMII', 0], ['SEM', 1], ['EDEMA', 2], ['.', 3]],\n",
       "  [['MMII', [0], 'Anatomia'], ['EDEMA', [2], 'Problema']]],\n",
       " 276: [[['A', 0],\n",
       "   ['#', 1],\n",
       "   ['POS', 2],\n",
       "   ['TX', 3],\n",
       "   ['CARDIACO', 4],\n",
       "   [',', 5],\n",
       "   ['COM', 6],\n",
       "   ['BOA', 7],\n",
       "   ['EVOLUÇÃO', 8],\n",
       "   ['.', 9]],\n",
       "  [['TX CARDIACO', [3, 4], 'Tratamento'],\n",
       "   ['CARDIACO', [4], 'Anatomia'],\n",
       "   ['BOA EVOLUÇÃO', [7, 8], 'Teste']]],\n",
       " 277: [[['AGUARDA', 0], ['LAUDO', 1], ['DA', 2], ['BIOPSIA', 3], ['.', 4]],\n",
       "  []],\n",
       " 278: [[['P', 0],\n",
       "   ['#', 1],\n",
       "   ['SOLICITO', 2],\n",
       "   ['LAB', 3],\n",
       "   ['E', 4],\n",
       "   ['ECOCC', 5],\n",
       "   ['.', 6]],\n",
       "  [['LAB', [3], 'Teste'], ['ECOCC', [5], 'Teste']]],\n",
       " 279: [[['has', 0],\n",
       "   ['/', 1],\n",
       "   ['10', 2],\n",
       "   ['anos', 3],\n",
       "   ['/', 4],\n",
       "   ['maço', 5],\n",
       "   ['.', 6]],\n",
       "  [['has', [0], 'Problema']]],\n",
       " 280: [[['aas', 0],\n",
       "   ['100', 1],\n",
       "   ['/', 2],\n",
       "   ['carvedilol', 3],\n",
       "   ['25', 4],\n",
       "   ['1x', 5],\n",
       "   ['/', 6],\n",
       "   ['d', 7],\n",
       "   ['/', 8],\n",
       "   ['sinva', 9],\n",
       "   ['40', 10],\n",
       "   ['mg', 11],\n",
       "   ['1', 12],\n",
       "   ['x', 13],\n",
       "   ['/', 14],\n",
       "   ['d', 15],\n",
       "   ['/', 16],\n",
       "   ['enalapril', 17],\n",
       "   ['1', 18],\n",
       "   ['cp', 19],\n",
       "   ['12', 20],\n",
       "   ['/', 21],\n",
       "   ['12', 22],\n",
       "   ['hs', 23],\n",
       "   ['/', 24],\n",
       "   ['furo', 25],\n",
       "   ['40', 26],\n",
       "   ['mg', 27],\n",
       "   ['1', 28],\n",
       "   ['x', 29],\n",
       "   ['/', 30],\n",
       "   ['d', 31],\n",
       "   ['.', 32]],\n",
       "  [['aas 100', [0, 1], 'Tratamento'],\n",
       "   ['carvedilol 25', [3, 4], 'Tratamento'],\n",
       "   ['sinva 40 mg', [9, 10, 11], 'Tratamento'],\n",
       "   ['enalapril', [17], 'Tratamento'],\n",
       "   ['furo 40 mg', [25, 26, 27], 'Tratamento']]],\n",
       " 281: [[['fez', 0],\n",
       "   ['cat', 1],\n",
       "   ['+', 2],\n",
       "   ['angioplastia', 3],\n",
       "   ['2013', 4],\n",
       "   ['.', 5]],\n",
       "  [['cat', [1], 'Teste'], ['angioplastia', [3], 'Tratamento']]],\n",
       " 282: [[['ECOCARDIO', 0],\n",
       "   ['25', 1],\n",
       "   ['/', 2],\n",
       "   ['11', 3],\n",
       "   ['/', 4],\n",
       "   ['13', 5],\n",
       "   ['-', 6],\n",
       "   ['VE', 7],\n",
       "   ['HIPERTROFIADO', 8],\n",
       "   ['COM', 9],\n",
       "   ['DIMENSÃO', 10],\n",
       "   ['INTERNA', 11],\n",
       "   ['AUMENTADA', 12],\n",
       "   ['E', 13],\n",
       "   ['FUNÇÃO', 14],\n",
       "   ['GLOBAL', 15],\n",
       "   ['MODERADAMENTE', 16],\n",
       "   ['DIMINUIDA', 17],\n",
       "   ['POR', 18],\n",
       "   ['ACINESIA', 19],\n",
       "   ['MEDIO', 20],\n",
       "   ['APICAL', 21],\n",
       "   ['PAREDE', 22],\n",
       "   ['ANTERO', 23],\n",
       "   ['SEPTAL', 24],\n",
       "   ['.', 25]],\n",
       "  [['ECOCARDIO', [0], 'Teste'],\n",
       "   ['VE HIPERTROFIADO COM DIMENSÃO INTERNA AUMENTADA',\n",
       "    [7, 8, 9, 10, 11, 12],\n",
       "    'Problema'],\n",
       "   ['FUNÇÃO GLOBAL MODERADAMENTE DIMINUIDA', [14, 15, 16, 17], 'Problema'],\n",
       "   ['ACINESIA MEDIO APICAL PAREDE ANTERO SEPTAL',\n",
       "    [19, 20, 21, 22, 23, 24],\n",
       "    'Problema'],\n",
       "   ['MEDIO APICAL PAREDE ANTERO SEPTAL', [20, 21, 22, 23, 24], 'Anatomia']]],\n",
       " 283: [[['AUMENTO', 0], ['DE', 1], ['AE', 2], ['.', 3]],\n",
       "  [['AUMENTO DE AE', [0, 1, 2], 'Problema'], ['AE', [2], 'Anatomia']]],\n",
       " 284: [[['REFLUXO', 0], ['MITRAL', 1], ['DISCRETO', 2], ['.', 3]],\n",
       "  [['REFLUXO MITRAL DISCRETO', [0, 1, 2], 'Problema'],\n",
       "   ['MITRAL', [1], 'Anatomia']]],\n",
       " 285: [[['ECTASIA', 0], ['DA', 1], ['RAIZ', 2], ['AÓRTICA', 3], ['.', 4]],\n",
       "  [['ECTASIA DA RAIZ AÓRTICA', [0, 1, 2, 3], 'Problema'],\n",
       "   ['RAIZ AÓRTICA', [2, 3], 'Anatomia']]],\n",
       " 286: [[['REFLUXO', 0], ['AÓRTICO', 1], ['DISCRETO', 2], ['.', 3]],\n",
       "  [['REFLUXO AÓRTICO DISCRETO', [0, 1, 2], 'Problema'],\n",
       "   ['AÓRTICO', [1], 'Anatomia']]],\n",
       " 287: [[['FEVE', 0], [':', 1], ['47', 2], ['%', 3], ['.', 4]], []],\n",
       " 288: [[['01', 0],\n",
       "   ['/', 1],\n",
       "   ['03', 2],\n",
       "   ['/', 3],\n",
       "   ['11', 4],\n",
       "   [':', 5],\n",
       "   ['ECODOPPLER', 6],\n",
       "   ['ARTERIAR', 7],\n",
       "   ['MMII', 8],\n",
       "   ['ESQ', 9],\n",
       "   [':', 10],\n",
       "   ['ESTENOSE', 11],\n",
       "   ['APROX', 12],\n",
       "   ['DE', 13],\n",
       "   ['75', 14],\n",
       "   ['%', 15],\n",
       "   ['DA', 16],\n",
       "   ['BIFURCAÇÃO', 17],\n",
       "   ['DOS', 18],\n",
       "   ['VASOS', 19],\n",
       "   ['FEMORAIS', 20],\n",
       "   ['.', 21]],\n",
       "  [['ECODOPPLER ARTERIAR', [6, 7], 'Teste'],\n",
       "   ['MMII ESQ', [8, 9], 'Anatomia'],\n",
       "   ['ESTENOSE APROX DE 75 % DA BIFURCAÇÃO DOS VASOS FEMORAIS',\n",
       "    [11, 12, 13, 14, 15, 16, 17, 18, 19, 20],\n",
       "    'Problema'],\n",
       "   ['BIFURCAÇÃO DOS VASOS FEMORAIS', [17, 18, 19, 20], 'Anatomia']]],\n",
       " 289: [[['LESOES', 0],\n",
       "   ['SUBSEGMENTARES', 1],\n",
       "   ['DAS', 2],\n",
       "   ['ART', 3],\n",
       "   [',', 4],\n",
       "   ['TIBIAL', 5],\n",
       "   ['ANTERIOR', 6],\n",
       "   ['E', 7],\n",
       "   ['FIBULAR', 8],\n",
       "   ['.', 9]],\n",
       "  [['LESOES SUBSEGMENTARES DAS ART , TIBIAL ANTERIOR E FIBULAR',\n",
       "    [0, 1, 2, 3, 4, 5, 6, 7, 8],\n",
       "    'Problema'],\n",
       "   ['ART , TIBIAL ANTERIOR E FIBULAR', [3, 4, 5, 6, 7, 8], 'Anatomia']]],\n",
       " 290: [[['OCLUSÃO', 0],\n",
       "   ['DA', 1],\n",
       "   ['ARTERIA', 2],\n",
       "   ['POPLITEA', 3],\n",
       "   ['MEDIO', 4],\n",
       "   ['DISTAL', 5],\n",
       "   ['.', 6]],\n",
       "  [['OCLUSÃO DA ARTERIA POPLITEA MEDIO DISTAL',\n",
       "    [0, 1, 2, 3, 4, 5],\n",
       "    'Problema'],\n",
       "   ['ARTERIA POPLITEA MEDIO DISTAL', [2, 3, 4, 5], 'Anatomia']]],\n",
       " 291: [[['ESTENOSE', 0],\n",
       "   ['>', 1],\n",
       "   ['A', 2],\n",
       "   ['OCLUSAO', 3],\n",
       "   ['DA', 4],\n",
       "   ['ARTERIA', 5],\n",
       "   ['TIBIAL', 6],\n",
       "   ['POSTERIOR', 7],\n",
       "   ['.', 8]],\n",
       "  [['ESTENOSE', [0], 'Problema'],\n",
       "   ['OCLUSAO DA ARTERIA TIBIAL POSTERIOR', [3, 4, 5, 6, 7], 'Problema'],\n",
       "   ['ARTERIA TIBIAL POSTERIOR', [5, 6, 7], 'Anatomia']]],\n",
       " 292: [[['quadro', 0],\n",
       "   ['de', 1],\n",
       "   ['disfunção', 2],\n",
       "   ['sistólica', 3],\n",
       "   ['após', 4],\n",
       "   ['iam', 5],\n",
       "   [',', 6],\n",
       "   ['perdeu', 7],\n",
       "   ['seguimento', 8],\n",
       "   ['.', 9]],\n",
       "  [['disfunção sistólica', [2, 3], 'Problema'], ['iam', [5], 'Problema']]],\n",
       " 293: [[['LUCAS', 0], [',', 1], ['18', 2], ['anos', 3], ['.', 4]], []],\n",
       " 294: [[['#', 0], ['HF', 1], ['-', 2], ['cardiopatia', 3], ['.', 4]], []],\n",
       " 295: [[['Em', 0],\n",
       "   ['uso', 1],\n",
       "   ['de', 2],\n",
       "   [':', 3],\n",
       "   ['Diclin', 4],\n",
       "   ['(', 5],\n",
       "   ['ACO', 6],\n",
       "   [')', 7],\n",
       "   ['.', 8]],\n",
       "  [['Diclin ( ACO ) ', [4, 5, 6, 7], 'Tratamento']]],\n",
       " 296: [[['HMA', 0],\n",
       "   [':', 1],\n",
       "   ['Pcte', 2],\n",
       "   ['relata', 3],\n",
       "   ['síncope', 4],\n",
       "   ['inicio', 5],\n",
       "   ['em', 6],\n",
       "   ['janeiro', 7],\n",
       "   ['mais', 8],\n",
       "   ['com', 9],\n",
       "   ['o', 10],\n",
       "   ['calor', 11],\n",
       "   ['.', 12]],\n",
       "  [['síncope', [4], 'Problema'],\n",
       "   ['mais com o calor', [8, 9, 10, 11], 'Problema']]],\n",
       " 297: [[['pródromos', 0],\n",
       "   [':', 1],\n",
       "   ['tontura', 2],\n",
       "   ['e', 3],\n",
       "   ['sudorese', 4],\n",
       "   ['e', 5],\n",
       "   ['escurecimento', 6],\n",
       "   ['visual', 7],\n",
       "   ['.', 8]],\n",
       "  [['tontura', [2], 'Problema'],\n",
       "   ['sudorese', [4], 'Problema'],\n",
       "   ['escurecimento visual', [6, 7], 'Problema']]],\n",
       " 298: [[['Refere', 0],\n",
       "   ['exacerbação', 1],\n",
       "   ['dos', 2],\n",
       "   ['sintomas', 3],\n",
       "   ['com', 4],\n",
       "   ['nervosismo', 5],\n",
       "   ['.', 6]],\n",
       "  [['exacerbação dos sintomas com nervosismo', [1, 2, 3, 4, 5], 'Problema'],\n",
       "   ['sintomas', [3], 'Problema']]],\n",
       " 299: [[['Duração', 0],\n",
       "   ['de', 1],\n",
       "   ['segundos', 2],\n",
       "   ['há', 3],\n",
       "   ['3', 4],\n",
       "   ['minutos', 5],\n",
       "   ['.', 6]],\n",
       "  []],\n",
       " 300: [[['Nega', 0],\n",
       "   ['confusão', 1],\n",
       "   ['mental', 2],\n",
       "   ['ou', 3],\n",
       "   ['salivação', 4],\n",
       "   ['após', 5],\n",
       "   ['.', 6]],\n",
       "  [['confusão mental', [1, 2], 'Problema'], ['salivação', [4], 'Problema']]],\n",
       " 301: [[['Nega', 0], ['uso', 1], ['de', 2], ['medicação', 3], ['.', 4]],\n",
       "  [['medicação', [3], 'Tratamento']]],\n",
       " 302: [[['Nega', 0], ['outros', 1], ['sintomas', 2], ['.', 3]],\n",
       "  [['outros sintomas', [1, 2], 'Problema']]],\n",
       " 303: [[['Refere', 0],\n",
       "   ['que', 1],\n",
       "   ['continua', 2],\n",
       "   ['apresentando', 3],\n",
       "   ['sintomas', 4],\n",
       "   [',', 5],\n",
       "   ['com', 6],\n",
       "   ['ultimo', 7],\n",
       "   ['ha', 8],\n",
       "   ['1', 9],\n",
       "   ['semana', 10],\n",
       "   [',', 11],\n",
       "   ['durante', 12],\n",
       "   ['atividades', 13],\n",
       "   ['domesticas', 14],\n",
       "   ['.', 15]],\n",
       "  [['durante atividades domesticas', [12, 13, 14], 'Problema']]],\n",
       " 304: [[['Ecocardio', 0],\n",
       "   ['18', 1],\n",
       "   ['/', 2],\n",
       "   ['02', 3],\n",
       "   ['/', 4],\n",
       "   ['15', 5],\n",
       "   [':', 6],\n",
       "   ['RA', 7],\n",
       "   ['20', 8],\n",
       "   [',', 9],\n",
       "   ['AE', 10],\n",
       "   ['26', 11],\n",
       "   [',', 12],\n",
       "   ['VD', 13],\n",
       "   ['17', 14],\n",
       "   [',', 15],\n",
       "   ['Septo', 16],\n",
       "   ['08', 17],\n",
       "   [',', 18],\n",
       "   ['PP', 19],\n",
       "   ['08', 20],\n",
       "   [',', 21],\n",
       "   ['VE', 22],\n",
       "   ['39', 23],\n",
       "   [',', 24],\n",
       "   ['FE', 25],\n",
       "   ['63', 26],\n",
       "   ['%', 27],\n",
       "   ['.', 28]],\n",
       "  [['Ecocardio', [0], 'Teste'],\n",
       "   ['RA', [7], 'Anatomia'],\n",
       "   ['AE', [10], 'Anatomia'],\n",
       "   ['VD', [13], 'Anatomia'],\n",
       "   ['Septo', [16], 'Anatomia'],\n",
       "   ['PP', [19], 'Anatomia'],\n",
       "   ['VE', [22], 'Anatomia']]],\n",
       " 305: [[['Exame', 0], ['dentro', 1], ['da', 2], ['normalidade', 3], ['.', 4]],\n",
       "  [['Exame', [0], 'Teste']]],\n",
       " 306: [[['Teste', 0],\n",
       "   ['ergometrico', 1],\n",
       "   [':', 2],\n",
       "   ['Resposta', 3],\n",
       "   ['cardiovascular', 4],\n",
       "   ['normal', 5],\n",
       "   ['ao', 6],\n",
       "   ['esforço', 7],\n",
       "   ['fisico', 8],\n",
       "   [',', 9],\n",
       "   ['atingiu', 10],\n",
       "   ['11', 11],\n",
       "   ['METS', 12],\n",
       "   ['.', 13]],\n",
       "  [['Teste ergometrico', [0, 1], 'Teste'],\n",
       "   ['cardiovascular', [4], 'Anatomia']]],\n",
       " 307: [[['ECG', 0], [':', 1], ['DCRD', 2], ['.', 3]],\n",
       "  [['ECG', [0], 'Teste'], ['DCRD', [2], 'Problema']]],\n",
       " 308: [[['HF', 0], ['-', 1], ['p', 2], ['cardiopatia', 3], ['.', 4]], []],\n",
       " 309: [[['BEG', 0],\n",
       "   [',', 1],\n",
       "   ['hidratada', 2],\n",
       "   [',', 3],\n",
       "   ['normocorada', 4],\n",
       "   [',', 5],\n",
       "   ['acianotica', 6],\n",
       "   ['.', 7]],\n",
       "  []],\n",
       " 310: [[['FC', 0],\n",
       "   ['80', 1],\n",
       "   [',', 2],\n",
       "   ['FR', 3],\n",
       "   ['18', 4],\n",
       "   [',', 5],\n",
       "   ['PA', 6],\n",
       "   [':', 7],\n",
       "   ['120', 8],\n",
       "   ['/', 9],\n",
       "   ['70', 10],\n",
       "   [',', 11],\n",
       "   ['SAT', 12],\n",
       "   ['98', 13],\n",
       "   ['%', 14],\n",
       "   ['AA', 15],\n",
       "   ['.', 16]],\n",
       "  [['FC', [0], 'Teste'],\n",
       "   ['FR', [3], 'Teste'],\n",
       "   ['PA', [6], 'Teste'],\n",
       "   ['SAT', [12], 'Teste']]],\n",
       " 311: [[['AC', 0],\n",
       "   ['bnf', 1],\n",
       "   ['em', 2],\n",
       "   ['2t', 3],\n",
       "   ['e', 4],\n",
       "   ['ss', 5],\n",
       "   [',', 6],\n",
       "   ['.', 7]],\n",
       "  [['ss', [5], 'Problema']]],\n",
       " 312: [[['AP', 0],\n",
       "   ['mv', 1],\n",
       "   ['+', 2],\n",
       "   ['simetrico', 3],\n",
       "   ['s', 4],\n",
       "   ['ra', 5],\n",
       "   [',', 6],\n",
       "   ['.', 7]],\n",
       "  [['ra', [5], 'Problema']]],\n",
       " 313: [[['Abd', 0], ['sp', 1], ['.', 2]], [['Abd', [0], 'Anatomia']]],\n",
       " 314: [[['mmii', 0], ['s', 1], ['edema', 2], [',', 3], ['.', 4]],\n",
       "  [['mmii', [0], 'Anatomia'], ['edema', [2], 'Problema']]],\n",
       " 315: [[['Vaso', 0], ['vagal', 1], ['?', 2], ['.', 3]],\n",
       "  [['Vaso vagal', [0, 1], 'Problema']]],\n",
       " 316: [[['A', 0], [':', 1], ['Aguarda', 2], ['holter', 3], ['.', 4]],\n",
       "  [['holter', [3], 'Teste']]],\n",
       " 317: [[['Retorno', 0], ['com', 1], ['exames', 2], ['.', 3]],\n",
       "  [['exames', [2], 'Teste']]],\n",
       " 318: [[['Paciente', 0],\n",
       "   ['em', 1],\n",
       "   ['acompanhamento', 2],\n",
       "   ['com', 3],\n",
       "   ['a', 4],\n",
       "   ['cardiologia', 5],\n",
       "   ['devido', 6],\n",
       "   ['a', 7],\n",
       "   ['2', 8],\n",
       "   ['IAM', 9],\n",
       "   ['previos', 10],\n",
       "   ['(', 11],\n",
       "   ['em', 12],\n",
       "   ['2004', 13],\n",
       "   ['e', 14],\n",
       "   ['2012', 15],\n",
       "   [')', 16],\n",
       "   ['.', 17]],\n",
       "  [['IAM', [9], 'Problema']]],\n",
       " 319: [[['Em', 0],\n",
       "   ['uso', 1],\n",
       "   ['de', 2],\n",
       "   [':', 3],\n",
       "   ['carvediol', 4],\n",
       "   ['6', 5],\n",
       "   [',', 6],\n",
       "   ['25', 7],\n",
       "   ['mg', 8],\n",
       "   [',', 9],\n",
       "   ['AAS', 10],\n",
       "   ['100', 11],\n",
       "   ['mg', 12],\n",
       "   [',', 13],\n",
       "   ['furosemida', 14],\n",
       "   ['40', 15],\n",
       "   ['mg', 16],\n",
       "   [',', 17],\n",
       "   ['losartana', 18],\n",
       "   ['100', 19],\n",
       "   ['mg', 20],\n",
       "   [',', 21],\n",
       "   ['atorvstatina', 22],\n",
       "   ['20', 23],\n",
       "   ['mg', 24],\n",
       "   [',', 25],\n",
       "   ['omeprazol', 26],\n",
       "   ['20', 27],\n",
       "   ['mg', 28],\n",
       "   ['.', 29]],\n",
       "  [['carvediol 6 , 25 mg', [4, 5, 6, 7, 8], 'Tratamento'],\n",
       "   ['AAS 100 mg', [10, 11, 12], 'Tratamento'],\n",
       "   ['furosemida 40 mg', [14, 15, 16], 'Tratamento'],\n",
       "   ['losartana 100 mg', [18, 19, 20], 'Tratamento'],\n",
       "   ['atorvstatina 20 mg', [22, 23, 24], 'Tratamento'],\n",
       "   ['omeprazol 20 mg', [26, 27, 28], 'Tratamento']]],\n",
       " 320: [[['Paciente', 0],\n",
       "   ['relata', 1],\n",
       "   ['dispnéia', 2],\n",
       "   ['e', 3],\n",
       "   ['cansaço', 4],\n",
       "   ['aos', 5],\n",
       "   ['minimos', 6],\n",
       "   ['esforços', 7],\n",
       "   ['.', 8]],\n",
       "  [['dispnéia', [2], 'Problema'],\n",
       "   ['cansaço aos minimos esforços', [4, 5, 6, 7], 'Problema']]],\n",
       " 321: [[['Paciente', 0],\n",
       "   ['relata', 1],\n",
       "   ['parestesias', 2],\n",
       "   ['em', 3],\n",
       "   ['pé', 4],\n",
       "   ['direito', 5],\n",
       "   [',', 6],\n",
       "   ['dores', 7],\n",
       "   ['musculares', 8],\n",
       "   ['na', 9],\n",
       "   ['panturrilha', 10],\n",
       "   ['.', 11]],\n",
       "  [['parestesias em pé direito', [2, 3, 4, 5], 'Problema'],\n",
       "   ['pé direito', [4, 5], 'Anatomia'],\n",
       "   ['dores musculares na panturrilha', [7, 8, 9, 10], 'Problema'],\n",
       "   ['panturrilha', [10], 'Anatomia']]],\n",
       " 322: [[['Comorbidades', 0],\n",
       "   [':', 1],\n",
       "   ['HAS', 2],\n",
       "   [',', 3],\n",
       "   ['ICC', 4],\n",
       "   [',', 5],\n",
       "   ['dislipidemia', 6],\n",
       "   ['.', 7]],\n",
       "  [['Comorbidades', [0], 'Problema'],\n",
       "   ['HAS', [2], 'Problema'],\n",
       "   ['ICC', [4], 'Problema'],\n",
       "   ['dislipidemia', [6], 'Problema']]],\n",
       " 323: [[['Eventos', 0],\n",
       "   [':', 1],\n",
       "   ['2', 2],\n",
       "   ['CATs', 3],\n",
       "   ['e', 4],\n",
       "   ['3', 5],\n",
       "   ['angioplastias', 6],\n",
       "   ['.', 7]],\n",
       "  [['2 CATs', [2, 3], 'Teste'], ['3 angioplastias', [5, 6], 'Tratamento']]],\n",
       " 324: [[['Na', 0],\n",
       "   ['ultima', 1],\n",
       "   ['consulta', 2],\n",
       "   ['foi', 3],\n",
       "   ['trocada', 4],\n",
       "   ['sinvastatina', 5],\n",
       "   ['por', 6],\n",
       "   ['atorvastatina', 7],\n",
       "   ['mas', 8],\n",
       "   ['o', 9],\n",
       "   ['paciente', 10],\n",
       "   ['continua', 11],\n",
       "   ['em', 12],\n",
       "   ['uso', 13],\n",
       "   ['de', 14],\n",
       "   ['sinvastatina', 15],\n",
       "   ['.', 16]],\n",
       "  [['sinvastatina', [5], 'Tratamento'],\n",
       "   ['atorvastatina', [7], 'Tratamento'],\n",
       "   ['sinvastatina', [15], 'Tratamento']]],\n",
       " 325: [[['Paciente', 0], ['nao', 1], ['traz', 2], ['exames', 3], ['.', 4]],\n",
       "  [['exames', [3], 'Teste']]],\n",
       " 326: [[['EF', 0], [':', 1], ['.', 2]], [['EF', [0], 'Teste']]],\n",
       " 327: [[['PA', 0],\n",
       "   [':', 1],\n",
       "   ['120', 2],\n",
       "   ['/', 3],\n",
       "   ['80', 4],\n",
       "   ['mmhg', 5],\n",
       "   ['.', 6]],\n",
       "  [['PA', [0], 'Teste']]],\n",
       " 328: [[['BCRNF', 0], ['.', 1]], []],\n",
       " 329: [[['Pulso', 0],\n",
       "   ['MMII', 1],\n",
       "   ['pedioso', 2],\n",
       "   ['poplíteo', 3],\n",
       "   ['e', 4],\n",
       "   ['femoral', 5],\n",
       "   ['presentes', 6],\n",
       "   ['.', 7]],\n",
       "  [['MMII pedioso poplíteo', [1, 2, 3], 'Anatomia'],\n",
       "   ['femoral', [5], 'Anatomia']]],\n",
       " 330: [[['Conduta', 0],\n",
       "   [':', 1],\n",
       "   ['troco', 2],\n",
       "   ['sinvastatina', 3],\n",
       "   ['por', 4],\n",
       "   ['atorvastatina', 5],\n",
       "   ['e', 6],\n",
       "   ['solicito', 7],\n",
       "   ['exames', 8],\n",
       "   ['de', 9],\n",
       "   ['sangue', 10],\n",
       "   ['(', 11],\n",
       "   ['CT', 12],\n",
       "   [',', 13],\n",
       "   ['LDL', 14],\n",
       "   [',', 15],\n",
       "   ['HDL', 16],\n",
       "   [',', 17],\n",
       "   ['TG', 18],\n",
       "   [')', 19],\n",
       "   [';', 20],\n",
       "   ['mantenho', 21],\n",
       "   ['as', 22],\n",
       "   ['demais', 23],\n",
       "   ['medicações', 24],\n",
       "   ['.', 25]],\n",
       "  [['sinvastatina', [3], 'Tratamento'],\n",
       "   ['atorvastatina', [5], 'Tratamento'],\n",
       "   ['exames de sangue', [8, 9, 10], 'Teste'],\n",
       "   ['sangue', [10], 'Anatomia'],\n",
       "   ['CT', [12], 'Teste'],\n",
       "   ['LDL', [14], 'Teste'],\n",
       "   ['HDL', [16], 'Teste'],\n",
       "   ['TG', [18], 'Teste'],\n",
       "   ['demais medicações', [23, 24], 'Tratamento']]],\n",
       " 331: [[['LUCAS', 0], [',', 1], ['61', 2], ['ANOS', 3], ['.', 4]], []],\n",
       " 332: [[['ACOMPANHAMENTO', 0], ['HÁ', 1], ['1', 2], ['ANO', 3], ['.', 4]], []],\n",
       " 333: [[['#', 0], ['HAS', 1], ['.', 2]], [['HAS', [1], 'Problema']]],\n",
       " 334: [[['#', 0], ['IC', 1], ['.', 2]], [['IC', [1], 'Problema']]],\n",
       " 335: [[['#', 0], ['HIPOTIREOIDISMO', 1], ['.', 2]],\n",
       "  [['HIPOTIREOIDISMO', [1], 'Problema']]],\n",
       " 336: [[['#', 0], ['MIOCARDIOPATIA', 1], ['DILATADA', 2], ['.', 3]],\n",
       "  [['MIOCARDIOPATIA DILATADA', [1, 2], 'Problema']]],\n",
       " 337: [[['#', 0],\n",
       "   ['IMPLANTE', 1],\n",
       "   ['DE', 2],\n",
       "   ['RESSINCRONIZADOR', 3],\n",
       "   ['/', 4],\n",
       "   ['DESFIBRILADOR', 5],\n",
       "   ['EM', 6],\n",
       "   ['MAIO', 7],\n",
       "   ['/', 8],\n",
       "   ['2012', 9],\n",
       "   ['.', 10]],\n",
       "  [['IMPLANTE DE RESSINCRONIZADOR', [1, 2, 3], 'Tratamento'],\n",
       "   ['DESFIBRILADOR', [5], 'Tratamento']]],\n",
       " 338: [[['#', 0], ['DEPRESSÃO', 1], ['.', 2]],\n",
       "  [['DEPRESSÃO', [1], 'Problema']]],\n",
       " 339: [[['EM', 0],\n",
       "   ['USO', 1],\n",
       "   [':', 2],\n",
       "   ['CARVEDILOL', 3],\n",
       "   ['50', 4],\n",
       "   ['MG', 5],\n",
       "   ['2X', 6],\n",
       "   ['/', 7],\n",
       "   ['D', 8],\n",
       "   ['+', 9],\n",
       "   ['MONONITRATO', 10],\n",
       "   ['DE', 11],\n",
       "   ['ISOSSORBITA', 12],\n",
       "   ['20', 13],\n",
       "   ['MG', 14],\n",
       "   ['2X', 15],\n",
       "   ['/', 16],\n",
       "   ['D', 17],\n",
       "   ['+', 18],\n",
       "   ['ANCORON', 19],\n",
       "   ['(', 20],\n",
       "   ['AMIODARONA', 21],\n",
       "   [')', 22],\n",
       "   ['200', 23],\n",
       "   ['MG', 24],\n",
       "   ['2X', 25],\n",
       "   ['/', 26],\n",
       "   ['D', 27],\n",
       "   ['+', 28],\n",
       "   ['ESPIRONOLACTONA', 29],\n",
       "   ['25', 30],\n",
       "   ['MG', 31],\n",
       "   ['/', 32],\n",
       "   ['D', 33],\n",
       "   ['+', 34],\n",
       "   ['FUROSEMIDA', 35],\n",
       "   ['40', 36],\n",
       "   ['MG', 37],\n",
       "   ['2X', 38],\n",
       "   ['/', 39],\n",
       "   ['D', 40],\n",
       "   ['+', 41],\n",
       "   ['LOSARTAN', 42],\n",
       "   ['50', 43],\n",
       "   ['MG', 44],\n",
       "   ['2X', 45],\n",
       "   ['/', 46],\n",
       "   ['D', 47],\n",
       "   ['+', 48],\n",
       "   ['DIGOXINA', 49],\n",
       "   ['0', 50],\n",
       "   [',', 51],\n",
       "   ['25', 52],\n",
       "   ['MG', 53],\n",
       "   ['/', 54],\n",
       "   ['D', 55],\n",
       "   ['+', 56],\n",
       "   ['SINVASTATINA', 57],\n",
       "   ['40', 58],\n",
       "   ['MG', 59],\n",
       "   ['/', 60],\n",
       "   ['D', 61],\n",
       "   ['+', 62],\n",
       "   ['LEVOTIROXINA', 63],\n",
       "   ['50', 64],\n",
       "   ['MCG', 65],\n",
       "   ['/', 66],\n",
       "   ['D', 67],\n",
       "   ['+', 68],\n",
       "   ['RIVOTRIL', 69],\n",
       "   ['2', 70],\n",
       "   ['MG', 71],\n",
       "   ['/', 72],\n",
       "   ['D', 73],\n",
       "   ['+', 74],\n",
       "   ['CITALOPRAM', 75],\n",
       "   ['20MG', 76],\n",
       "   ['3X', 77],\n",
       "   ['/', 78],\n",
       "   ['DIA', 79],\n",
       "   ['.', 80]],\n",
       "  [['CARVEDILOL 50 MG', [3, 4, 5], 'Tratamento'],\n",
       "   ['MONONITRATO DE ISOSSORBITA 20 MG', [10, 11, 12, 13, 14], 'Tratamento'],\n",
       "   ['ANCORON ( AMIODARONA ) 200 MG', [19, 20, 21, 22, 23, 24], 'Tratamento'],\n",
       "   ['ESPIRONOLACTONA 25 MG', [29, 30, 31], 'Tratamento'],\n",
       "   ['FUROSEMIDA 40 MG', [35, 36, 37], 'Tratamento'],\n",
       "   ['LOSARTAN 50 MG', [42, 43, 44], 'Tratamento'],\n",
       "   ['DIGOXINA 0 , 25 MG', [49, 50, 51, 52, 53], 'Tratamento'],\n",
       "   ['SINVASTATINA 40 MG', [57, 58, 59], 'Tratamento'],\n",
       "   ['LEVOTIROXINA 50 MCG', [63, 64, 65], 'Tratamento'],\n",
       "   ['RIVOTRIL 2 MG', [69, 70, 71], 'Tratamento'],\n",
       "   ['CITALOPRAM 20MG', [75, 76], 'Tratamento']]],\n",
       " 340: [[['REFERE', 0],\n",
       "   ['DOR', 1],\n",
       "   ['PRECORDIAL', 2],\n",
       "   ['LEVE', 3],\n",
       "   ['A', 4],\n",
       "   ['MODERADA', 5],\n",
       "   ['DO', 6],\n",
       "   ['TIPO', 7],\n",
       "   ['QUEIMAÇÃO', 8],\n",
       "   [',', 9],\n",
       "   ['SEM', 10],\n",
       "   ['IRRADIAÇÃO', 11],\n",
       "   [',', 12],\n",
       "   ['SEM', 13],\n",
       "   ['PIORA', 14],\n",
       "   ['AOS', 15],\n",
       "   ['ESFORÇOS', 16],\n",
       "   ['.', 17]],\n",
       "  [['DOR PRECORDIAL LEVE A MODERADA DO TIPO QUEIMAÇÃO',\n",
       "    [1, 2, 3, 4, 5, 6, 7, 8],\n",
       "    'Problema'],\n",
       "   ['PRECORDIAL', [2], 'Anatomia'],\n",
       "   ['SEM IRRADIAÇÃO', [10, 11], 'Problema']]],\n",
       " 341: [[['DISPNEIA', 0],\n",
       "   [',', 1],\n",
       "   ['MAL', 2],\n",
       "   ['ESTAR', 3],\n",
       "   ['E', 4],\n",
       "   ['FADIGA', 5],\n",
       "   ['AO', 6],\n",
       "   ['SUBIR', 7],\n",
       "   ['ESCADAS', 8],\n",
       "   [',', 9],\n",
       "   ['ANDAR', 10],\n",
       "   ['RÁPIDO', 11],\n",
       "   ['.', 12]],\n",
       "  [['DISPNEIA , MAL ESTAR E FADIGA AO SUBIR ESCADAS , ANDAR RÁPIDO',\n",
       "    [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11],\n",
       "    'Problema']]],\n",
       " 342: [[['NEGA', 0], ['ORTOPNEIA', 1], ['E', 2], ['DPN', 3], ['.', 4]],\n",
       "  [['ORTOPNEIA', [1], 'Problema'], ['DPN', [3], 'Problema']]],\n",
       " 343: [[['NEGA', 0], ['TOSSE', 1], ['E', 2], ['EDEMA', 3], ['.', 4]],\n",
       "  [['TOSSE', [1], 'Problema'], ['EDEMA', [3], 'Problema']]],\n",
       " 344: [[['HÁ', 0],\n",
       "   ['2', 1],\n",
       "   ['ANOS', 2],\n",
       "   ['NÃO', 3],\n",
       "   ['FAZ', 4],\n",
       "   ['REVISÃO', 5],\n",
       "   ['DO', 6],\n",
       "   ['MP', 7],\n",
       "   ['.', 8]],\n",
       "  []],\n",
       " 345: [[['#', 0], ['HMP', 1], [':', 2], ['.', 3]], []],\n",
       " 346: [[['1', 0],\n",
       "   ['CESAREA', 1],\n",
       "   ['PRÉVIA', 2],\n",
       "   [',', 3],\n",
       "   ['LIPOASPIRAÇÃO', 4],\n",
       "   ['.', 5]],\n",
       "  [['CESAREA', [1], 'Tratamento'], ['LIPOASPIRAÇÃO', [4], 'Tratamento']]],\n",
       " 347: [[['NEGA', 0], ['ALERGIAS', 1], ['.', 2]],\n",
       "  [['ALERGIAS', [1], 'Problema']]],\n",
       " 348: [[['JÁ', 0],\n",
       "   ['REALIZOU', 1],\n",
       "   ['3', 2],\n",
       "   ['CATETERISMOS', 3],\n",
       "   [',', 4],\n",
       "   ['CINTILOGRAFIA', 5],\n",
       "   ['E', 6],\n",
       "   ['EXAMES', 7],\n",
       "   ['PARA', 8],\n",
       "   ['DOENÇA', 9],\n",
       "   ['DE', 10],\n",
       "   ['CHAGAS', 11],\n",
       "   ['.', 12]],\n",
       "  [['3 CATETERISMOS', [2, 3], 'Teste'],\n",
       "   ['CINTILOGRAFIA', [5], 'Teste'],\n",
       "   ['EXAMES PARA DOENÇA DE CHAGAS', [7, 8, 9, 10, 11], 'Teste']]],\n",
       " 349: [[['#', 0], ['HMA', 1], [':', 2], ['.', 3]], []],\n",
       " 350: [[['AVÓ', 0], ['TEVE', 1], ['IAM', 2], ['.', 3]], []],\n",
       " 351: [[['#', 0], ['CHDV', 1], ['.', 2]], []],\n",
       " 352: [[['NEGA', 0],\n",
       "   ['TABAGISMO', 1],\n",
       "   [',', 2],\n",
       "   ['NEGA', 3],\n",
       "   ['ETILISMO', 4],\n",
       "   [',', 5],\n",
       "   ['NEGA', 6],\n",
       "   ['EXERCÍCIO', 7],\n",
       "   ['FÍSICO', 8],\n",
       "   ['.', 9]],\n",
       "  [['TABAGISMO', [1], 'Problema'], ['ETILISMO', [4], 'Problema']]],\n",
       " 353: [[['EXAMES', 0], [':', 1], ['.', 2]], [['EXAMES', [0], 'Teste']]],\n",
       " 354: [[['HB', 0], ['14', 1], [',', 2], ['7', 3], ['.', 4]],\n",
       "  [['HB', [0], 'Teste']]],\n",
       " 355: [[['HT', 0], ['43', 1], [',', 2], ['9', 3], ['%', 4], ['.', 5]],\n",
       "  [['HT', [0], 'Teste']]],\n",
       " 356: [[['CREATININA', 0], ['0', 1], [',', 2], ['8', 3], ['.', 4]],\n",
       "  [['CREATININA', [0], 'Teste']]],\n",
       " 357: [[['GLICOSE', 0], ['87', 1], ['.', 2]], [['GLICOSE', [0], 'Teste']]],\n",
       " 358: [[['UREIA', 0], ['48', 1], ['48', 2], ['.', 3]],\n",
       "  [['UREIA', [0], 'Teste']]],\n",
       " 359: [[['HDL', 0], ['28', 1], ['.', 2]], [['HDL', [0], 'Teste']]],\n",
       " 360: [[['SODIO', 0], ['139', 1], ['.', 2]], [['SODIO', [0], 'Teste']]],\n",
       " 361: [[['POTASSIO', 0], ['4', 1], [',', 2], ['8', 3], ['.', 4]],\n",
       "  [['POTASSIO', [0], 'Teste']]],\n",
       " 362: [[['COLESTEROL', 0], ['TOTAL', 1], ['268', 2], ['.', 3]],\n",
       "  [['COLESTEROL TOTAL', [0, 1], 'Teste']]],\n",
       " 363: [[['TSH', 0], ['5', 1], [',', 2], ['87', 3], ['.', 4]],\n",
       "  [['TSH', [0], 'Teste']]],\n",
       " 364: [[['T4', 0], ['LIVRE', 1], ['0', 2], [',', 3], ['86', 4], ['.', 5]],\n",
       "  [['T4 LIVRE', [0, 1], 'Teste']]],\n",
       " 365: [[['TRIGLICERIDEOS', 0], ['626', 1], ['.', 2]],\n",
       "  [['TRIGLICERIDEOS', [0], 'Teste']]],\n",
       " 366: [[['STRESS', 0],\n",
       "   ['FARMACOLOGICO', 1],\n",
       "   ['COM', 2],\n",
       "   ['DIPIRIDAMOL', 3],\n",
       "   [':', 4],\n",
       "   ['ECOCARDIOGRAMA', 5],\n",
       "   ['DE', 6],\n",
       "   ['STRESS', 7],\n",
       "   ['FARMACOLOGICO', 8],\n",
       "   ['SEM', 9],\n",
       "   ['EVIDENCIA', 10],\n",
       "   ['DE', 11],\n",
       "   ['ISQUEMIA', 12],\n",
       "   ['MIOCÁRDICA', 13],\n",
       "   ['.', 14]],\n",
       "  [['STRESS FARMACOLOGICO COM DIPIRIDAMOL', [0, 1, 2, 3], 'Teste'],\n",
       "   ['ECOCARDIOGRAMA DE STRESS FARMACOLOGICO', [5, 6, 7, 8], 'Teste'],\n",
       "   ['ISQUEMIA MIOCÁRDICA', [12, 13], 'Problema'],\n",
       "   ['MIOCÁRDICA', [13], 'Anatomia']]],\n",
       " 367: [[['CONDUTA', 0],\n",
       "   [':', 1],\n",
       "   ['AUMENTO', 2],\n",
       "   ['DA', 3],\n",
       "   ['DOSE', 4],\n",
       "   ['DE', 5],\n",
       "   ['LEVOTIROXINA', 6],\n",
       "   ['PARA', 7],\n",
       "   ['100', 8],\n",
       "   ['MCG', 9],\n",
       "   [',', 10],\n",
       "   ['CIPROFIBRATO', 11],\n",
       "   ['100MG', 12],\n",
       "   ['1X', 13],\n",
       "   ['/', 14],\n",
       "   ['DIA', 15],\n",
       "   [',', 16],\n",
       "   ['FUROSEMIDA', 17],\n",
       "   ['SE', 18],\n",
       "   ['FALTA', 19],\n",
       "   ['DE', 20],\n",
       "   ['AR', 21],\n",
       "   [',', 22],\n",
       "   ['SINVASTATINA', 23],\n",
       "   ['PARA', 24],\n",
       "   ['40MG', 25],\n",
       "   [',', 26],\n",
       "   ['HIDRALAZINA', 27],\n",
       "   ['25MG', 28],\n",
       "   ['DE', 29],\n",
       "   ['8', 30],\n",
       "   ['/', 31],\n",
       "   ['8H', 32],\n",
       "   ['.', 33]],\n",
       "  [['LEVOTIROXINA PARA 100 MCG', [6, 7, 8, 9], 'Tratamento'],\n",
       "   ['CIPROFIBRATO 100MG', [11, 12], 'Tratamento'],\n",
       "   ['FUROSEMIDA', [17], 'Tratamento'],\n",
       "   ['FALTA DE AR', [19, 20, 21], 'Problema'],\n",
       "   ['SINVASTATINA PARA 40MG', [23, 24, 25], 'Tratamento'],\n",
       "   ['HIDRALAZINA 25MG', [27, 28], 'Tratamento']]],\n",
       " 368: [[['EXAMES', 0],\n",
       "   ['DE', 1],\n",
       "   ['TSH', 2],\n",
       "   [',', 3],\n",
       "   ['T4', 4],\n",
       "   ['LIVRE', 5],\n",
       "   [',', 6],\n",
       "   ['CK', 7],\n",
       "   [',', 8],\n",
       "   ['TGO', 9],\n",
       "   ['E', 10],\n",
       "   ['TGP', 11],\n",
       "   ['E', 12],\n",
       "   ['PERFIL', 13],\n",
       "   ['LIPIDICO', 14],\n",
       "   ['.', 15]],\n",
       "  [['EXAMES', [0], 'Teste'],\n",
       "   ['TSH', [2], 'Teste'],\n",
       "   ['T4 LIVRE', [4, 5], 'Teste'],\n",
       "   ['CK', [7], 'Teste'],\n",
       "   ['TGO', [9], 'Teste'],\n",
       "   ['TGP', [11], 'Teste'],\n",
       "   ['PERFIL LIPIDICO', [13, 14], 'Teste']]],\n",
       " 369: [[['LUCAS', 0], [',', 1], ['50', 2], ['ANOS', 3], ['.', 4]], []],\n",
       " 370: [[['#', 0], ['HAS', 1], ['HÁ', 2], ['10', 3], ['ANOS', 4], ['.', 5]],\n",
       "  [['HAS', [1], 'Problema']]],\n",
       " 371: [[['#', 0], ['DSLP', 1], ['.', 2]], [['DSLP', [1], 'Problema']]],\n",
       " 372: [[['#', 0], ['HIPOTIREOIDISMO', 1], ['.', 2]],\n",
       "  [['HIPOTIREOIDISMO', [1], 'Problema']]],\n",
       " 373: [[['#', 0], ['DEPRESSÃO', 1], ['.', 2]],\n",
       "  [['DEPRESSÃO', [1], 'Problema']]],\n",
       " 374: [[['#', 0], ['OSTEOARTRITE', 1], ['DE', 2], ['MÃOS', 3], ['.', 4]],\n",
       "  [['OSTEOARTRITE DE MÃOS', [1, 2, 3], 'Problema'],\n",
       "   ['MÃOS', [3], 'Anatomia']]],\n",
       " 375: [[['#', 0],\n",
       "   ['CAT', 1],\n",
       "   ['EM', 2],\n",
       "   ['JUNHO', 3],\n",
       "   ['/', 4],\n",
       "   ['2011', 5],\n",
       "   [':', 6],\n",
       "   ['SEM', 7],\n",
       "   ['INDICAÇÃO', 8],\n",
       "   ['CIRÚRGICA', 9],\n",
       "   ['.', 10]],\n",
       "  [['CAT', [1], 'Teste']]],\n",
       " 376: [[['#', 0],\n",
       "   ['HÉRNIA', 1],\n",
       "   ['DE', 2],\n",
       "   ['DISCO', 3],\n",
       "   ['HÁ', 4],\n",
       "   ['3', 5],\n",
       "   ['ANOS', 6],\n",
       "   ['.', 7]],\n",
       "  [['HÉRNIA DE DISCO', [1, 2, 3], 'Problema']]],\n",
       " 377: [[['#', 0],\n",
       "   ['CINTILOGRAFIA', 1],\n",
       "   ['MIOCÁRDICA', 2],\n",
       "   ['COM', 3],\n",
       "   ['DIPIRIDAMOL', 4],\n",
       "   ['EM', 5],\n",
       "   ['2011', 6],\n",
       "   [':', 7],\n",
       "   ['HIPOCAPTAÇÃO', 8],\n",
       "   ['APICAL', 9],\n",
       "   [',', 10],\n",
       "   ['DO', 11],\n",
       "   ['SEGMENTO', 12],\n",
       "   ['ANTERIOR', 13],\n",
       "   ['(', 14],\n",
       "   ['TRANSITÓRIO', 15],\n",
       "   [')', 16],\n",
       "   ['E', 17],\n",
       "   ['BASAL', 18],\n",
       "   ['DO', 19],\n",
       "   ['SEGMENTO', 20],\n",
       "   ['ANTERO', 21],\n",
       "   ['-', 22],\n",
       "   ['SEPTAL', 23],\n",
       "   ['.', 24]],\n",
       "  [['CINTILOGRAFIA MIOCÁRDICA COM DIPIRIDAMOL', [1, 2, 3, 4], 'Teste'],\n",
       "   ['MIOCÁRDICA', [2], 'Anatomia'],\n",
       "   ['HIPOCAPTAÇÃO APICAL , DO SEGMENTO ANTERIOR ( TRANSITÓRIO ) E BASAL DO SEGMENTO ANTERO - SEPTAL',\n",
       "    [8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23],\n",
       "    'Problema'],\n",
       "   ['APICAL , DO SEGMENTO ANTERIOR ( TRANSITÓRIO ) E BASAL DO SEGMENTO ANTERO - SEPTAL',\n",
       "    [9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23],\n",
       "    'Anatomia']]],\n",
       " 378: [[['#', 0],\n",
       "   ['EM', 1],\n",
       "   ['USO', 2],\n",
       "   [':', 3],\n",
       "   ['FLUOXETINA', 4],\n",
       "   ['20MG', 5],\n",
       "   ['/', 6],\n",
       "   ['D', 7],\n",
       "   [';', 8],\n",
       "   ['ENALAPRIL', 9],\n",
       "   ['10MG', 10],\n",
       "   ['12', 11],\n",
       "   ['/', 12],\n",
       "   ['12H', 13],\n",
       "   [';', 14],\n",
       "   ['AAS', 15],\n",
       "   ['100MG', 16],\n",
       "   ['/', 17],\n",
       "   ['D', 18],\n",
       "   [';', 19],\n",
       "   ['CARVEDILOL', 20],\n",
       "   ['12', 21],\n",
       "   [',', 22],\n",
       "   ['5MG', 23],\n",
       "   ['12', 24],\n",
       "   ['/', 25],\n",
       "   ['12H', 26],\n",
       "   [';', 27],\n",
       "   ['HCTZ', 28],\n",
       "   ['25MG', 29],\n",
       "   ['/', 30],\n",
       "   ['D', 31],\n",
       "   [';', 32],\n",
       "   ['SUSTRATE', 33],\n",
       "   ['10MG', 34],\n",
       "   ['SN', 35],\n",
       "   [';', 36],\n",
       "   ['SINVASTATINA', 37],\n",
       "   ['40MG', 38],\n",
       "   ['/', 39],\n",
       "   ['D', 40],\n",
       "   [';', 41],\n",
       "   ['LEVOTIROXINA', 42],\n",
       "   ['25UCG', 43],\n",
       "   ['/', 44],\n",
       "   ['D', 45],\n",
       "   [';', 46],\n",
       "   ['HIDROXICLOROQUINA', 47],\n",
       "   ['400MG', 48],\n",
       "   ['/', 49],\n",
       "   ['D', 50],\n",
       "   [';', 51],\n",
       "   ['.', 52]],\n",
       "  [['FLUOXETINA 20MG', [4, 5], 'Tratamento'],\n",
       "   ['ENALAPRIL 10MG', [9, 10], 'Tratamento'],\n",
       "   ['AAS 100MG', [15, 16], 'Tratamento'],\n",
       "   ['CARVEDILOL 12 , 5MG', [20, 21, 22, 23], 'Tratamento'],\n",
       "   ['HCTZ 25MG', [28, 29], 'Tratamento'],\n",
       "   ['SUSTRATE 10MG', [33, 34], 'Tratamento'],\n",
       "   ['SINVASTATINA 40MG', [37, 38], 'Tratamento'],\n",
       "   ['LEVOTIROXINA 25UCG', [42, 43], 'Tratamento'],\n",
       "   ['HIDROXICLOROQUINA 400MG', [47, 48], 'Tratamento']]],\n",
       " 379: [[['LAB', 0],\n",
       "   ['08', 1],\n",
       "   ['/', 2],\n",
       "   ['04', 3],\n",
       "   ['/', 4],\n",
       "   ['15', 5],\n",
       "   [':', 6],\n",
       "   ['TG', 7],\n",
       "   ['144', 8],\n",
       "   ['/', 9],\n",
       "   ['HDL', 10],\n",
       "   ['50', 11],\n",
       "   ['/', 12],\n",
       "   ['CT', 13],\n",
       "   ['178', 14],\n",
       "   ['/', 15],\n",
       "   ['LDL', 16],\n",
       "   ['99', 17],\n",
       "   ['/', 18],\n",
       "   ['.', 19]],\n",
       "  [['LAB', [0], 'Teste'],\n",
       "   ['TG', [7], 'Teste'],\n",
       "   ['HDL', [10], 'Teste'],\n",
       "   ['CT', [13], 'Teste'],\n",
       "   ['LDL', [16], 'Teste']]],\n",
       " 380: [[['S', 0],\n",
       "   ['#', 1],\n",
       "   ['REFERE', 2],\n",
       "   ['DOR', 3],\n",
       "   ['TORÁCICA', 4],\n",
       "   ['PRECORDIAL', 5],\n",
       "   ['TIPO', 6],\n",
       "   ['QUEIMAÇÃO', 7],\n",
       "   [',', 8],\n",
       "   ['COM', 9],\n",
       "   ['MELHORA', 10],\n",
       "   ['AO', 11],\n",
       "   ['USO', 12],\n",
       "   ['DE', 13],\n",
       "   ['NITRATO', 14],\n",
       "   [',', 15],\n",
       "   ['SEM', 16],\n",
       "   ['IRRADIAÇÃO', 17],\n",
       "   [',', 18],\n",
       "   ['SEM', 19],\n",
       "   ['RELAÇÃO', 20],\n",
       "   ['COM', 21],\n",
       "   ['O', 22],\n",
       "   ['ESFORÇO', 23],\n",
       "   [',', 24],\n",
       "   ['2', 25],\n",
       "   ['EPISÓDIOS', 26],\n",
       "   ['AO', 27],\n",
       "   ['MÊS', 28],\n",
       "   ['.', 29]],\n",
       "  [['DOR TORÁCICA PRECORDIAL TIPO QUEIMAÇÃO', [3, 4, 5, 6, 7], 'Problema'],\n",
       "   ['TORÁCICA PRECORDIAL', [4, 5], 'Anatomia'],\n",
       "   ['IRRADIAÇÃO', [17], 'Problema']]],\n",
       " 381: [[['NEGA', 0],\n",
       "   ['DISPNÉIA', 1],\n",
       "   [',', 2],\n",
       "   ['ORTOPNEIA', 3],\n",
       "   ['OU', 4],\n",
       "   ['DPN', 5],\n",
       "   ['.', 6]],\n",
       "  [['DISPNÉIA', [1], 'Problema'],\n",
       "   ['ORTOPNEIA', [3], 'Problema'],\n",
       "   ['DPN', [5], 'Problema']]],\n",
       " 382: [[['NEGA', 0], ['EDEMA', 1], ['DE', 2], ['MMII', 3], ['.', 4]],\n",
       "  [['EDEMA DE MMII', [1, 2, 3], 'Problema'], ['MMII', [3], 'Anatomia']]],\n",
       " 383: [[['CONTA', 0],\n",
       "   ['APRESENTAR', 1],\n",
       "   ['ARTRALGIAS', 2],\n",
       "   ['DIFUSAS', 3],\n",
       "   ['PIORES', 4],\n",
       "   ['AO', 5],\n",
       "   ['FRIO', 6],\n",
       "   ['.', 7]],\n",
       "  [['ARTRALGIAS DIFUSAS', [2, 3], 'Problema']]],\n",
       " 384: [[['O', 0], ['#', 1], ['.', 2]], []],\n",
       " 385: [[['BEG', 0],\n",
       "   [',', 1],\n",
       "   ['CORADA', 2],\n",
       "   [',', 3],\n",
       "   ['HIDRATADA', 4],\n",
       "   [',', 5],\n",
       "   ['EUPNEICA', 6],\n",
       "   [',', 7],\n",
       "   ['LOTE', 8],\n",
       "   ['.', 9]],\n",
       "  []],\n",
       " 386: [[['PA', 0],\n",
       "   ['120', 1],\n",
       "   ['/', 2],\n",
       "   ['70', 3],\n",
       "   [';', 4],\n",
       "   ['FC', 5],\n",
       "   ['62', 6],\n",
       "   [';', 7],\n",
       "   ['SATO2', 8],\n",
       "   ['98', 9],\n",
       "   ['%', 10],\n",
       "   ['EM', 11],\n",
       "   ['AA', 12],\n",
       "   ['.', 13]],\n",
       "  [['PA', [0], 'Teste'], ['FC', [5], 'Teste'], ['SATO2', [8], 'Teste']]],\n",
       " 387: [[['MV', 0],\n",
       "   ['PRESENTE', 1],\n",
       "   ['E', 2],\n",
       "   ['SIMÉTRICO', 3],\n",
       "   ['SEM', 4],\n",
       "   ['RA', 5],\n",
       "   ['.', 6]],\n",
       "  [['RA', [5], 'Problema']]],\n",
       " 388: [[['BCRNF', 0],\n",
       "   ['SEM', 1],\n",
       "   ['SOPROS', 2],\n",
       "   ['EM', 3],\n",
       "   ['2T', 4],\n",
       "   ['.', 5]],\n",
       "  [['SOPROS', [2], 'Problema']]],\n",
       " 389: [[['ABDOME', 0],\n",
       "   ['GLOBOSO', 1],\n",
       "   [',', 2],\n",
       "   ['FLÁCIDO', 3],\n",
       "   [',', 4],\n",
       "   ['INDOLOR', 5],\n",
       "   [',', 6],\n",
       "   ['SEM', 7],\n",
       "   ['VCM', 8],\n",
       "   ['OU', 9],\n",
       "   ['MASSAS', 10],\n",
       "   ['PALPAVEIS', 11],\n",
       "   ['.', 12]],\n",
       "  [['ABDOME', [0], 'Anatomia'],\n",
       "   ['VCM', [8], 'Problema'],\n",
       "   ['MASSAS PALPAVEIS', [10, 11], 'Problema']]],\n",
       " 390: [[['MMII', 0],\n",
       "   ['SEM', 1],\n",
       "   ['EDEMA', 2],\n",
       "   ['OU', 3],\n",
       "   ['EMPASTAMENTO', 4],\n",
       "   ['.', 5]],\n",
       "  [['MMII', [0], 'Anatomia'],\n",
       "   ['EDEMA', [2], 'Problema'],\n",
       "   ['EMPASTAMENTO', [4], 'Problema']]],\n",
       " 391: [[['A', 0], ['#', 1], ['.', 2]], []],\n",
       " 392: [[['-', 0], ['ANGINA', 1], ['INSTÁVEL', 2], ['?', 3], ['.', 4]],\n",
       "  [['ANGINA INSTÁVEL', [1, 2], 'Problema']]],\n",
       " 393: [[['P', 0], ['#', 1], ['.', 2]], []],\n",
       " 394: [[['-', 0],\n",
       "   ['ACRESCENTO', 1],\n",
       "   ['MONOCORDIL', 2],\n",
       "   ['20MG', 3],\n",
       "   ['12', 4],\n",
       "   ['/', 5],\n",
       "   ['12H', 6],\n",
       "   ['.', 7]],\n",
       "  [['MONOCORDIL 20MG', [2, 3], 'Tratamento']]],\n",
       " 395: [[['-', 0],\n",
       "   ['SOLICITO', 1],\n",
       "   ['NOVA', 2],\n",
       "   ['CINTILOGRAFIA', 3],\n",
       "   ['DE', 4],\n",
       "   ['PERFUSÃO', 5],\n",
       "   ['MIOCÁRDICA', 6],\n",
       "   ['.', 7]],\n",
       "  [['CINTILOGRAFIA DE PERFUSÃO MIOCÁRDICA', [3, 4, 5, 6], 'Teste'],\n",
       "   ['MIOCÁRDICA', [6], 'Anatomia']]],\n",
       " 396: [[['-', 0], ['SOLICITO', 1], ['NOVO', 2], ['LAB', 3], ['.', 4]],\n",
       "  [['LAB', [3], 'Teste']]],\n",
       " 397: [[['-', 0], ['RETORNO', 1], ['COM', 2], ['EXAMES', 3], ['.', 4]],\n",
       "  [['EXAMES', [3], 'Teste']]],\n",
       " 398: [[['R2', 0], ['XX', 1], ['LUCAS', 2], ['.', 3]], []],\n",
       " 399: [[['Doutor', 0], ['Vital', 1], ['Brazil', 2], ['.', 3]], []],\n",
       " 400: [[['LUCAS', 0], [',', 1], ['.', 2]], []],\n",
       " 401: [[['ACOMPANHANDO', 0],\n",
       "   ['COM', 1],\n",
       "   ['Doutor', 2],\n",
       "   ['Vital', 3],\n",
       "   ['Brazil', 4],\n",
       "   ['SEM', 5],\n",
       "   ['SINCOPE', 6],\n",
       "   [',', 7],\n",
       "   ['SEM', 8],\n",
       "   ['PALPITAÇÃO', 9],\n",
       "   [',', 10],\n",
       "   ['SEM', 11],\n",
       "   ['TERAPIAS', 12],\n",
       "   ['.', 13]],\n",
       "  [['SINCOPE', [6], 'Problema'],\n",
       "   ['PALPITAÇÃO', [9], 'Problema'],\n",
       "   ['TERAPIAS', [12], 'Tratamento']]],\n",
       " 402: [[['SENTE', 0], ['-', 1], ['SE', 2], ['BEM', 3], ['.', 4]], []],\n",
       " 403: [[['#', 0],\n",
       "   ['hoje', 1],\n",
       "   ['realizou', 2],\n",
       "   ['avaliação', 3],\n",
       "   ['de', 4],\n",
       "   ['mp', 5],\n",
       "   ['-', 6],\n",
       "   ['cdi', 7],\n",
       "   [',', 8],\n",
       "   ['.', 9]],\n",
       "  [['avaliação de mp - cdi', [3, 4, 5, 6, 7], 'Teste']]],\n",
       " 404: [[['#', 0],\n",
       "   ['sem', 1],\n",
       "   ['registro', 2],\n",
       "   ['de', 3],\n",
       "   ['arritmias', 4],\n",
       "   ['-', 5],\n",
       "   ['terapias', 6],\n",
       "   [',', 7],\n",
       "   ['.', 8]],\n",
       "  [['arritmias - terapias', [4, 5, 6], 'Tratamento']]],\n",
       " 405: [[['#', 0], ['bat', 1], ['ok', 2], ['.', 3]], []],\n",
       " 406: [[['#', 0],\n",
       "   ['eletrodos', 1],\n",
       "   ['atrial', 2],\n",
       "   ['e', 3],\n",
       "   ['ventricular', 4],\n",
       "   ['=', 5],\n",
       "   ['ok', 6],\n",
       "   ['.', 7]],\n",
       "  [['atrial', [2], 'Anatomia'], ['ventricular', [4], 'Anatomia']]],\n",
       " 407: [[['#', 0],\n",
       "   ['retorno', 1],\n",
       "   ['em', 2],\n",
       "   ['6', 3],\n",
       "   ['meses', 4],\n",
       "   ['+', 5],\n",
       "   ['se', 6],\n",
       "   ['sintomas', 7],\n",
       "   ['=', 8],\n",
       "   ['procura', 9],\n",
       "   ['direta', 10],\n",
       "   ['+', 11],\n",
       "   ['nunca', 12],\n",
       "   ['faltar', 13],\n",
       "   ['Doutor', 14],\n",
       "   ['Vital', 15],\n",
       "   ['Brazil', 16],\n",
       "   ['.', 17]],\n",
       "  []],\n",
       " 408: [[['Lucas', 0], [',', 1], ['68A', 2], ['.', 3]], []],\n",
       " 409: [[['#', 0], ['CAT', 1], ['EM', 2], ['2008', 3], ['.', 4]],\n",
       "  [['CAT', [1], 'Teste']]],\n",
       " 410: [[['VALVOPLASTIA', 0], ['AÓRTICA', 1], ['EM', 2], ['2008', 3], ['.', 4]],\n",
       "  [['VALVOPLASTIA AÓRTICA', [0, 1], 'Tratamento'],\n",
       "   ['AÓRTICA', [1], 'Anatomia']]],\n",
       " 411: [[['VALVA', 0],\n",
       "   ['BIOLÓGICA', 1],\n",
       "   ['-', 2],\n",
       "   ['40', 3],\n",
       "   ['DIAS', 4],\n",
       "   ['DEPOIS', 5],\n",
       "   ['DA', 6],\n",
       "   ['CX', 7],\n",
       "   ['CARDIOVERSÃO', 8],\n",
       "   ['.', 9]],\n",
       "  [['VALVA BIOLÓGICA', [0, 1], 'Tratamento'],\n",
       "   ['CX', [7], 'Tratamento'],\n",
       "   ['CARDIOVERSÃO', [8], 'Tratamento']]],\n",
       " 412: [[['#', 0],\n",
       "   ['PACIENTE', 1],\n",
       "   ['ASSINTOMÁTICO', 2],\n",
       "   ['HÁ', 3],\n",
       "   ['BASTANTE', 4],\n",
       "   ['TEMPO', 5],\n",
       "   ['(', 6],\n",
       "   ['APROX', 7],\n",
       "   ['1', 8],\n",
       "   ['ANO', 9],\n",
       "   [')', 10],\n",
       "   [',', 11],\n",
       "   ['A', 12],\n",
       "   ['CAMINHO', 13],\n",
       "   ['DO', 14],\n",
       "   ['TRABALHO', 15],\n",
       "   ['NO', 16],\n",
       "   ['DIA', 17],\n",
       "   ['16', 18],\n",
       "   ['/', 19],\n",
       "   ['08', 20],\n",
       "   ['COMEÇOU', 21],\n",
       "   ['A', 22],\n",
       "   ['SENTIR', 23],\n",
       "   ['TONTURA', 24],\n",
       "   ['.', 25]],\n",
       "  [['TONTURA', [24], 'Problema']]],\n",
       " 413: [[['PROCUROU', 0],\n",
       "   ['ATENDIMENTO', 1],\n",
       "   ['NO', 2],\n",
       "   ['XXXX', 3],\n",
       "   ['E', 4],\n",
       "   ['PERMANECEU', 5],\n",
       "   ['INTERNADO', 6],\n",
       "   ['ATÉ', 7],\n",
       "   ['DIA', 8],\n",
       "   ['21', 9],\n",
       "   ['/', 10],\n",
       "   ['08', 11],\n",
       "   ['.', 12]],\n",
       "  []],\n",
       " 414: [[['HOJE', 0],\n",
       "   ['PACIENTE', 1],\n",
       "   ['ESTÁ', 2],\n",
       "   ['ASSINTOMÁTICO', 3],\n",
       "   [',', 4],\n",
       "   ['NEGA', 5],\n",
       "   ['QUEIXAS', 6],\n",
       "   [',', 7],\n",
       "   ['VOLTOU', 8],\n",
       "   ['AS', 9],\n",
       "   ['ATIVIDADES', 10],\n",
       "   ['NORMAIS', 11],\n",
       "   ['(', 12],\n",
       "   ['FRENTISTA', 13],\n",
       "   [')', 14],\n",
       "   ['.', 15]],\n",
       "  [['QUEIXAS', [6], 'Problema']]],\n",
       " 415: [[['RELATA', 0],\n",
       "   ['EMAGRECIMENTO', 1],\n",
       "   ['DE', 2],\n",
       "   ['7', 3],\n",
       "   ['KG', 4],\n",
       "   ['EM', 5],\n",
       "   ['10', 6],\n",
       "   ['DIAS', 7],\n",
       "   ['.', 8]],\n",
       "  [['EMAGRECIMENTO DE 7 KG', [1, 2, 3, 4], 'Problema']]],\n",
       " 416: [[['#', 0],\n",
       "   ['EX', 1],\n",
       "   ['-', 2],\n",
       "   ['TABAGISTA', 3],\n",
       "   ['40', 4],\n",
       "   ['MAÇOS', 5],\n",
       "   ['-', 6],\n",
       "   ['ANO', 7],\n",
       "   ['-', 8],\n",
       "   ['EM', 9],\n",
       "   ['ABSTINÊNCIA', 10],\n",
       "   ['HÁ', 11],\n",
       "   ['7', 12],\n",
       "   ['ANOS', 13],\n",
       "   ['.', 14]],\n",
       "  [['EX - TABAGISTA', [1, 2, 3], 'Problema']]],\n",
       " 417: [[['#', 0],\n",
       "   ['EM', 1],\n",
       "   ['USO', 2],\n",
       "   ['DE', 3],\n",
       "   [':', 4],\n",
       "   ['ENALAPRIL', 5],\n",
       "   ['10MG', 6],\n",
       "   ['2X', 7],\n",
       "   ['/', 8],\n",
       "   ['DIA', 9],\n",
       "   [',', 10],\n",
       "   ['CARVEDILOL', 11],\n",
       "   ['12', 12],\n",
       "   [',', 13],\n",
       "   ['5', 14],\n",
       "   ['MG', 15],\n",
       "   ['2X', 16],\n",
       "   ['/', 17],\n",
       "   ['DIA', 18],\n",
       "   [',', 19],\n",
       "   ['ATORVASTATINA', 20],\n",
       "   ['40', 21],\n",
       "   ['MG', 22],\n",
       "   [',', 23],\n",
       "   ['CLOPIDOGREL', 24],\n",
       "   ['E', 25],\n",
       "   ['MAREVAN', 26],\n",
       "   ['.', 27]],\n",
       "  [['ENALAPRIL 10MG', [5, 6], 'Tratamento'],\n",
       "   ['CARVEDILOL 12 , 5 MG', [11, 12, 13, 14, 15], 'Tratamento'],\n",
       "   ['ATORVASTATINA 40 MG', [20, 21, 22], 'Tratamento'],\n",
       "   ['CLOPIDOGREL', [24], 'Tratamento'],\n",
       "   ['MAREVAN', [26], 'Tratamento']]],\n",
       " 418: [[['#', 0], ['EXAME', 1], ['FÍSICO', 2], [':', 3], ['.', 4]],\n",
       "  [['EXAME FÍSICO', [1, 2], 'Teste']]],\n",
       " 419: [[['LOTE', 0],\n",
       "   [',', 1],\n",
       "   ['BEG', 2],\n",
       "   [',', 3],\n",
       "   ['NORMOCORADO', 4],\n",
       "   [',', 5],\n",
       "   ['HIDRATADO', 6],\n",
       "   [',', 7],\n",
       "   ['ANICTÉRICO', 8],\n",
       "   [',', 9],\n",
       "   ['ACIANÓTICO', 10],\n",
       "   ['.', 11]],\n",
       "  []],\n",
       " 420: [[['FC', 0],\n",
       "   ['80', 1],\n",
       "   ['BPM', 2],\n",
       "   [',', 3],\n",
       "   ['PA', 4],\n",
       "   ['120', 5],\n",
       "   ['/', 6],\n",
       "   ['80', 7],\n",
       "   ['MMHG', 8],\n",
       "   ['.', 9]],\n",
       "  [['FC', [0], 'Teste'], ['PA', [4], 'Teste']]],\n",
       " 421: [[['BCRNF', 0],\n",
       "   ['-', 1],\n",
       "   ['HIPOFONESE', 2],\n",
       "   ['DE', 3],\n",
       "   ['BULHAS', 4],\n",
       "   ['EM', 5],\n",
       "   ['FOCO', 6],\n",
       "   ['AÓRTICO', 7],\n",
       "   ['.', 8]],\n",
       "  [['HIPOFONESE DE BULHAS EM FOCO AÓRTICO', [2, 3, 4, 5, 6, 7], 'Problema'],\n",
       "   ['FOCO AÓRTICO', [6, 7], 'Anatomia']]],\n",
       " 422: [[['MV', 0],\n",
       "   ['+', 1],\n",
       "   ['SIM', 2],\n",
       "   ['BILATERAL', 3],\n",
       "   [',', 4],\n",
       "   ['S', 5],\n",
       "   ['/', 6],\n",
       "   ['RA', 7],\n",
       "   ['.', 8]],\n",
       "  [['RA', [7], 'Problema']]],\n",
       " 423: [[['RHA', 0],\n",
       "   ['+', 1],\n",
       "   [',', 2],\n",
       "   ['ABDOME', 3],\n",
       "   ['NÃO', 4],\n",
       "   ['DOLOROSO', 5],\n",
       "   ['A', 6],\n",
       "   ['PALPAÇÃO', 7],\n",
       "   [',', 8],\n",
       "   ['SEM', 9],\n",
       "   ['VISCEROMEGALIAS', 10],\n",
       "   ['.', 11]],\n",
       "  [['ABDOME', [3], 'Anatomia'],\n",
       "   ['DOLOROSO A PALPAÇÃO', [5, 6, 7], 'Problema'],\n",
       "   ['VISCEROMEGALIAS', [10], 'Problema']]],\n",
       " 424: [[['HERNIA', 0], ['UMBILICAL', 1], ['REDUTÍVEL', 2], ['.', 3]],\n",
       "  [['HERNIA UMBILICAL REDUTÍVEL', [0, 1, 2], 'Problema'],\n",
       "   ['UMBILICAL', [1], 'Anatomia']]],\n",
       " 425: [[['MMII', 0],\n",
       "   ['EDEMA', 1],\n",
       "   ['+', 2],\n",
       "   ['/', 3],\n",
       "   ['4', 4],\n",
       "   ['+', 5],\n",
       "   [',', 6],\n",
       "   ['PANTURRILHAS', 7],\n",
       "   ['LIVRES', 8],\n",
       "   ['.', 9]],\n",
       "  [['MMII EDEMA + / 4 + ', [0, 1, 2, 3, 4, 5], 'Problema'],\n",
       "   ['PANTURRILHAS', [7], 'Anatomia']]],\n",
       " 426: [[['#', 0],\n",
       "   ['ECG', 1],\n",
       "   ['19', 2],\n",
       "   ['/', 3],\n",
       "   ['08', 4],\n",
       "   [':', 5],\n",
       "   ['FLUTTER', 6],\n",
       "   ['ATRIAL', 7],\n",
       "   ['PROVAVELMENTE', 8],\n",
       "   ['ANTI', 9],\n",
       "   ['-', 10],\n",
       "   ['HORÁRIO', 11],\n",
       "   ['2', 12],\n",
       "   ['/', 13],\n",
       "   ['3', 14],\n",
       "   [':', 15],\n",
       "   ['1', 16],\n",
       "   [',', 17],\n",
       "   ['EXTRASSÍSTOLES', 18],\n",
       "   ['VENTRICULARES', 19],\n",
       "   ['.', 20]],\n",
       "  [['ECG', [1], 'Teste'],\n",
       "   ['FLUTTER ATRIAL', [6, 7], 'Problema'],\n",
       "   ['ATRIAL', [7], 'Anatomia'],\n",
       "   ['EXTRASSÍSTOLES VENTRICULARES', [18, 19], 'Problema'],\n",
       "   ['VENTRICULARES', [19], 'Anatomia']]],\n",
       " 427: [[['#', 0],\n",
       "   ['CD', 1],\n",
       "   [':', 2],\n",
       "   ['CONTROLE', 3],\n",
       "   ['DO', 4],\n",
       "   ['TAP', 5],\n",
       "   ['.', 6]],\n",
       "  [['CONTROLE', [3], 'Tratamento'], ['TAP', [5], 'Teste']]],\n",
       " 428: [[['PRECISA', 0],\n",
       "   ['DE', 1],\n",
       "   ['4', 2],\n",
       "   ['TAP', 3],\n",
       "   ['ENTRE', 4],\n",
       "   ['2', 5],\n",
       "   ['E', 6],\n",
       "   ['3', 7],\n",
       "   ['.', 8]],\n",
       "  [['TAP', [3], 'Teste']]],\n",
       " 429: [[['RETORNO', 0], ['EM', 1], ['30', 2], ['DIAS', 3], ['.', 4]], []],\n",
       " 430: [[['#', 0],\n",
       "   ['CONTINUA', 1],\n",
       "   ['com', 2],\n",
       "   ['Doutor', 3],\n",
       "   ['Vital', 4],\n",
       "   ['Brazil', 5],\n",
       "   [',', 6],\n",
       "   ['.', 7]],\n",
       "  []],\n",
       " 431: [[['#', 0],\n",
       "   ['ficou', 1],\n",
       "   ['bem', 2],\n",
       "   ['neste', 3],\n",
       "   ['ano', 4],\n",
       "   ['que', 5],\n",
       "   ['passou', 6],\n",
       "   [',', 7],\n",
       "   ['.', 8]],\n",
       "  []],\n",
       " 432: [[['#', 0], ['sem', 1], ['intercorrencias', 2], [',', 3], ['.', 4]],\n",
       "  [['intercorrencias', [2], 'Problema']]],\n",
       " 433: [[['#', 0],\n",
       "   ['hoje', 1],\n",
       "   ['avaliação', 2],\n",
       "   ['exclusiva', 3],\n",
       "   ['de', 4],\n",
       "   ['mp', 5],\n",
       "   ['.', 6]],\n",
       "  [['avaliação exclusiva de mp', [2, 3, 4, 5], 'Teste']]],\n",
       " 434: [[['#', 0],\n",
       "   ['mp', 1],\n",
       "   ['camara', 2],\n",
       "   ['-', 3],\n",
       "   ['dupla', 4],\n",
       "   [',', 5],\n",
       "   ['.', 6]],\n",
       "  [['mp camara - dupla', [1, 2, 3, 4], 'Tratamento']]],\n",
       " 435: [[['#', 0],\n",
       "   ['programado', 1],\n",
       "   ['em', 2],\n",
       "   ['modo', 3],\n",
       "   ['DDD', 4],\n",
       "   ['60', 5],\n",
       "   ['-', 6],\n",
       "   ['50', 7],\n",
       "   [',', 8],\n",
       "   ['.', 9]],\n",
       "  []],\n",
       " 436: [[['#', 0],\n",
       "   ['ritmo', 1],\n",
       "   ['espontaneo', 2],\n",
       "   ['=', 3],\n",
       "   ['bav', 4],\n",
       "   ['-', 5],\n",
       "   ['t', 6],\n",
       "   ['com', 7],\n",
       "   ['escape', 8],\n",
       "   ['minimo', 9],\n",
       "   [',', 10],\n",
       "   ['.', 11]],\n",
       "  []],\n",
       " 437: [[['#', 0], ['bat', 1], ['=', 2], ['ok', 3], ['.', 4]], []],\n",
       " 438: [[['#', 0],\n",
       "   ['para', 1],\n",
       "   ['eri', 2],\n",
       "   ['=', 3],\n",
       "   ['7', 4],\n",
       "   ['anos', 5],\n",
       "   [',', 6],\n",
       "   ['.', 7]],\n",
       "  []],\n",
       " 439: [[['#', 0],\n",
       "   ['operando', 1],\n",
       "   ['=', 2],\n",
       "   ['96', 3],\n",
       "   ['%', 4],\n",
       "   ['em', 5],\n",
       "   ['vat', 6],\n",
       "   ['.', 7]],\n",
       "  []],\n",
       " 440: [[['#', 0],\n",
       "   ['A', 1],\n",
       "   ['-', 2],\n",
       "   ['IMP', 3],\n",
       "   ['=', 4],\n",
       "   ['390', 5],\n",
       "   [',', 6],\n",
       "   ['SENSE', 7],\n",
       "   ['=', 8],\n",
       "   ['2', 9],\n",
       "   [',', 10],\n",
       "   ['3', 11],\n",
       "   [',', 12],\n",
       "   ['CLA', 13],\n",
       "   ['=', 14],\n",
       "   ['MENOR', 15],\n",
       "   ['QUE', 16],\n",
       "   ['1', 17],\n",
       "   [',', 18],\n",
       "   ['0', 19],\n",
       "   ['X', 20],\n",
       "   ['0', 21],\n",
       "   [',', 22],\n",
       "   ['4', 23],\n",
       "   [';', 24],\n",
       "   ['.', 25]],\n",
       "  []],\n",
       " 441: [[['#', 0],\n",
       "   ['V', 1],\n",
       "   ['-', 2],\n",
       "   ['IMP', 3],\n",
       "   ['=', 4],\n",
       "   ['487', 5],\n",
       "   [',', 6],\n",
       "   ['SENSE', 7],\n",
       "   ['=', 8],\n",
       "   ['3', 9],\n",
       "   [',', 10],\n",
       "   ['8', 11],\n",
       "   [',', 12],\n",
       "   ['CLV', 13],\n",
       "   ['=', 14],\n",
       "   ['MENOR', 15],\n",
       "   ['QUE', 16],\n",
       "   ['1', 17],\n",
       "   [',', 18],\n",
       "   ['0', 19],\n",
       "   ['X', 20],\n",
       "   ['0', 21],\n",
       "   [',', 22],\n",
       "   ['4', 23],\n",
       "   ['.', 24]],\n",
       "  []],\n",
       " 442: [[['#', 0],\n",
       "   ['cd', 1],\n",
       "   ['=', 2],\n",
       "   ['programada', 3],\n",
       "   ['auto', 4],\n",
       "   ['captura', 5],\n",
       "   [',', 6],\n",
       "   ['retorno', 7],\n",
       "   ['em', 8],\n",
       "   ['12', 9],\n",
       "   ['meses', 10],\n",
       "   [',', 11],\n",
       "   ['se', 12],\n",
       "   ['intercorrencias', 13],\n",
       "   ['=', 14],\n",
       "   ['procura', 15],\n",
       "   ['direta', 16],\n",
       "   [',', 17],\n",
       "   ['sempre', 18],\n",
       "   ['manter', 19],\n",
       "   ['com', 20],\n",
       "   ['cardio', 21],\n",
       "   ['geral', 22],\n",
       "   ['.', 23]],\n",
       "  []],\n",
       " 443: [[['Humberto', 0], [',', 1], ['65anos', 2], ['.', 3]], []],\n",
       " 444: [[['#', 0],\n",
       "   ['DAC', 1],\n",
       "   ['-', 2],\n",
       "   ['IAM', 3],\n",
       "   ['2007', 4],\n",
       "   ['+', 5],\n",
       "   ['RVM', 6],\n",
       "   ['/', 7],\n",
       "   ['/', 8],\n",
       "   ['ATC', 9],\n",
       "   ['2010', 10],\n",
       "   [',', 11],\n",
       "   ['.', 12]],\n",
       "  [['DAC', [1], 'Problema'],\n",
       "   ['IAM', [3], 'Problema'],\n",
       "   ['RVM', [6], 'Tratamento'],\n",
       "   ['ATC', [9], 'Tratamento']]],\n",
       " 445: [[['#', 0],\n",
       "   ['Hipertireoidismo', 1],\n",
       "   ['?', 2],\n",
       "   ['?', 3],\n",
       "   ['-', 4],\n",
       "   ['em', 5],\n",
       "   ['acompanhamento', 6],\n",
       "   ['com', 7],\n",
       "   ['endocrinologia', 8],\n",
       "   [',', 9],\n",
       "   ['.', 10]],\n",
       "  [['Hipertireoidismo', [1], 'Problema']]],\n",
       " 446: [[['#', 0], ['FA', 1], ['paroxistica', 2], [',', 3], ['.', 4]],\n",
       "  [['FA paroxistica', [1, 2], 'Problema']]],\n",
       " 447: [[['#', 0],\n",
       "   ['Internamento', 1],\n",
       "   ['em', 2],\n",
       "   ['jan', 3],\n",
       "   ['/', 4],\n",
       "   ['15', 5],\n",
       "   ['por', 6],\n",
       "   ['taquicardia', 7],\n",
       "   ['atrial', 8],\n",
       "   ['com', 9],\n",
       "   ['aberrância', 10],\n",
       "   ['-', 11],\n",
       "   ['-', 12],\n",
       "   ['DESCARTADO', 13],\n",
       "   ['ISQUEMIA', 14],\n",
       "   ['MIOCÁRDICA', 15],\n",
       "   ['COMO', 16],\n",
       "   ['CAUSA', 17],\n",
       "   [',', 18],\n",
       "   ['PROVÁVEL', 19],\n",
       "   ['FATOR', 20],\n",
       "   ['DESCOMPENSADOR', 21],\n",
       "   ['SERIA', 22],\n",
       "   ['O', 23],\n",
       "   ['HIPERTIREOIDISMO', 24],\n",
       "   ['QUE', 25],\n",
       "   ['AINDA', 26],\n",
       "   ['APRESENTA', 27],\n",
       "   ['-', 28],\n",
       "   ['SE', 29],\n",
       "   ['DESCOMPENSADO', 30],\n",
       "   ['(', 31],\n",
       "   ['TSH', 32],\n",
       "   ['<', 33],\n",
       "   ['0', 34],\n",
       "   [',', 35],\n",
       "   ['01', 36],\n",
       "   [')', 37],\n",
       "   ['.', 38]],\n",
       "  [['taquicardia atrial com aberrância', [7, 8, 9, 10], 'Problema'],\n",
       "   ['atrial', [8], 'Anatomia'],\n",
       "   ['ISQUEMIA MIOCÁRDICA', [14, 15], 'Problema'],\n",
       "   ['MIOCÁRDICA', [15], 'Anatomia'],\n",
       "   ['HIPERTIREOIDISMO', [24], 'Problema'],\n",
       "   ['TSH', [32], 'Teste']]],\n",
       " 448: [[['Medicamentos', 0],\n",
       "   [':', 1],\n",
       "   ['tapazol', 2],\n",
       "   ['10', 3],\n",
       "   ['mg', 4],\n",
       "   [',', 5],\n",
       "   ['enalapril', 6],\n",
       "   ['10', 7],\n",
       "   ['mg', 8],\n",
       "   ['12', 9],\n",
       "   ['/', 10],\n",
       "   ['12h', 11],\n",
       "   [',', 12],\n",
       "   ['selozok', 13],\n",
       "   ['50', 14],\n",
       "   ['mg', 15],\n",
       "   [',', 16],\n",
       "   ['sinvastatina', 17],\n",
       "   ['40', 18],\n",
       "   ['mg', 19],\n",
       "   [',', 20],\n",
       "   ['xarelto', 21],\n",
       "   ['20', 22],\n",
       "   ['mg', 23],\n",
       "   [',', 24],\n",
       "   ['AAS100mg', 25],\n",
       "   ['.', 26]],\n",
       "  [['Medicamentos', [0], 'Tratamento'],\n",
       "   ['tapazol 10 mg', [2, 3, 4], 'Tratamento'],\n",
       "   ['enalapril 10 mg', [6, 7, 8], 'Tratamento'],\n",
       "   ['selozok 50 mg', [13, 14, 15], 'Tratamento'],\n",
       "   ['sinvastatina 40 mg', [17, 18, 19], 'Tratamento'],\n",
       "   ['xarelto 20 mg', [21, 22, 23], 'Tratamento'],\n",
       "   ['AAS100mg', [25], 'Tratamento']]],\n",
       " 449: [[['S', 0],\n",
       "   ['#', 1],\n",
       "   ['Nega', 2],\n",
       "   ['palpitações', 3],\n",
       "   [',', 4],\n",
       "   ['dispneia', 5],\n",
       "   [',', 6],\n",
       "   ['dor', 7],\n",
       "   ['torácica', 8],\n",
       "   ['.', 9]],\n",
       "  [['palpitações', [3], 'Problema'],\n",
       "   ['dispneia', [5], 'Problema'],\n",
       "   ['dor torácica', [7, 8], 'Problema'],\n",
       "   ['torácica', [8], 'Anatomia']]],\n",
       " 450: [[['Epigastralgia', 0], ['.', 1]], [['Epigastralgia', [0], 'Problema']]],\n",
       " 451: [[['Consuçta', 0],\n",
       "   ['com', 1],\n",
       "   ['endocrino', 2],\n",
       "   ['amanhã', 3],\n",
       "   ['.', 4]],\n",
       "  []],\n",
       " 452: [[['Traz', 0], ['exames', 1], [':', 2], ['.', 3]],\n",
       "  [['exames', [1], 'Teste']]],\n",
       " 453: [[['-', 0],\n",
       "   ['Holter', 1],\n",
       "   ['28', 2],\n",
       "   ['/', 3],\n",
       "   ['04', 4],\n",
       "   ['/', 5],\n",
       "   ['15', 6],\n",
       "   [':', 7],\n",
       "   ['ritmo', 8],\n",
       "   ['sinusal', 9],\n",
       "   [',', 10],\n",
       "   ['FC', 11],\n",
       "   ['media', 12],\n",
       "   ['53', 13],\n",
       "   [',', 14],\n",
       "   ['Intervalo', 15],\n",
       "   ['PR', 16],\n",
       "   ['normal', 17],\n",
       "   [',', 18],\n",
       "   ['QRS', 19],\n",
       "   ['normais', 20],\n",
       "   [',', 21],\n",
       "   ['extra', 22],\n",
       "   ['-', 23],\n",
       "   ['sistole', 24],\n",
       "   ['ventricular', 25],\n",
       "   ['unica', 26],\n",
       "   [',', 27],\n",
       "   ['extra', 28],\n",
       "   ['-', 29],\n",
       "   ['sistole', 30],\n",
       "   ['supra', 31],\n",
       "   ['-', 32],\n",
       "   ['ventricular', 33],\n",
       "   ['rara', 34],\n",
       "   [',', 35],\n",
       "   ['ausencia', 36],\n",
       "   ['de', 37],\n",
       "   ['pausas', 38],\n",
       "   [',', 39],\n",
       "   ['ST', 40],\n",
       "   ['sem', 41],\n",
       "   ['alterações', 42],\n",
       "   ['.', 43]],\n",
       "  [['Holter', [1], 'Teste'],\n",
       "   ['extra - sistole ventricular unica', [22, 23, 24, 25, 26], 'Problema'],\n",
       "   ['ventricular', [25], 'Anatomia'],\n",
       "   ['extra - sistole supra - ventricular rara',\n",
       "    [28, 29, 30, 31, 32, 33, 34],\n",
       "    'Problema'],\n",
       "   ['pausas', [38], 'Problema'],\n",
       "   ['alterações', [42], 'Problema']]],\n",
       " 454: [[['-', 0],\n",
       "   ['TSH', 1],\n",
       "   ['47', 2],\n",
       "   ['/', 3],\n",
       "   ['t4l', 4],\n",
       "   ['0', 5],\n",
       "   [',', 6],\n",
       "   ['55', 7],\n",
       "   ['.', 8]],\n",
       "  [['TSH', [1], 'Teste'], ['t4l', [4], 'Teste']]],\n",
       " 455: [[['-', 0],\n",
       "   ['Ecocardiograma', 1],\n",
       "   ['out', 2],\n",
       "   ['/', 3],\n",
       "   ['14', 4],\n",
       "   ['(', 5],\n",
       "   ['externo', 6],\n",
       "   [')', 7],\n",
       "   [':', 8],\n",
       "   ['AE', 9],\n",
       "   ['43', 10],\n",
       "   ['/', 11],\n",
       "   ['VD', 12],\n",
       "   ['23', 13],\n",
       "   ['/', 14],\n",
       "   ['septo', 15],\n",
       "   ['10', 16],\n",
       "   ['/', 17],\n",
       "   ['VE', 18],\n",
       "   ['52', 19],\n",
       "   ['-', 20],\n",
       "   ['26', 21],\n",
       "   ['/', 22],\n",
       "   ['FE', 23],\n",
       "   ['81', 24],\n",
       "   ['%', 25],\n",
       "   ['.', 26]],\n",
       "  [['Ecocardiograma', [1], 'Teste'],\n",
       "   ['AE', [9], 'Anatomia'],\n",
       "   ['VD', [12], 'Anatomia'],\n",
       "   ['septo', [15], 'Anatomia'],\n",
       "   ['VE', [18], 'Anatomia']]],\n",
       " 456: [[['VE', 0],\n",
       "   ['com', 1],\n",
       "   ['contração', 2],\n",
       "   ['segmentar', 3],\n",
       "   ['e', 4],\n",
       "   ['função', 5],\n",
       "   ['sistólica', 6],\n",
       "   ['adequadas', 7],\n",
       "   ['.', 8]],\n",
       "  [['VE', [0], 'Anatomia']]],\n",
       " 457: [[['Ao', 0], ['exame', 1], [':', 2], ['.', 3]],\n",
       "  [['exame', [1], 'Teste']]],\n",
       " 458: [[['PA130', 0],\n",
       "   ['/', 1],\n",
       "   ['70', 2],\n",
       "   [',', 3],\n",
       "   ['FC', 4],\n",
       "   ['55', 5],\n",
       "   ['.', 6]],\n",
       "  [['FC', [4], 'Teste']]],\n",
       " 459: [[['CV', 0],\n",
       "   [':', 1],\n",
       "   ['BRNF', 2],\n",
       "   [',', 3],\n",
       "   ['sem', 4],\n",
       "   ['sopros', 5],\n",
       "   ['.', 6]],\n",
       "  [['sopros', [5], 'Problema']]],\n",
       " 460: [[['CPP', 0], ['livres', 1], ['.', 2]], [['CPP', [0], 'Anatomia']]],\n",
       " 461: [[['A', 0], ['#', 1], ['FA', 2], ['paroxística', 3], ['.', 4]],\n",
       "  [['FA paroxística', [2, 3], 'Problema']]],\n",
       " 462: [[['CHADS', 0],\n",
       "   ['=', 1],\n",
       "   ['1', 2],\n",
       "   ['/', 3],\n",
       "   ['CHADS', 4],\n",
       "   ['Vasc', 5],\n",
       "   ['=', 6],\n",
       "   ['3', 7],\n",
       "   ['.', 8]],\n",
       "  []],\n",
       " 463: [[['P', 0],\n",
       "   ['#', 1],\n",
       "   ['Retorno', 2],\n",
       "   ['precoce', 3],\n",
       "   ['na', 4],\n",
       "   ['endocrino', 5],\n",
       "   ['para', 6],\n",
       "   ['ajuste', 7],\n",
       "   ['medicação', 8],\n",
       "   ['.', 9]],\n",
       "  [['medicação', [8], 'Tratamento']]],\n",
       " 464: [[['Suspendemos', 0], ['selozok', 1], ['.', 2]],\n",
       "  [['selozok', [1], 'Tratamento']]],\n",
       " 465: [[['prescrevemos', 0],\n",
       "   ['propafenona', 1],\n",
       "   ['300mg', 2],\n",
       "   ['12', 3],\n",
       "   ['/', 4],\n",
       "   ['12h', 5],\n",
       "   ['.', 6]],\n",
       "  [['propafenona 300mg', [1, 2], 'Tratamento']]],\n",
       " 466: [[['Retorno', 0],\n",
       "   ['com', 1],\n",
       "   ['ecocardio', 2],\n",
       "   ['em', 3],\n",
       "   ['3', 4],\n",
       "   ['meses', 5],\n",
       "   ['.', 6]],\n",
       "  [['ecocardio', [2], 'Teste']]],\n",
       " 467: [[['Demais', 0], ['medicações', 1], ['mantidas', 2], ['.', 3]],\n",
       "  [['Demais medicações', [0, 1], 'Tratamento']]],\n",
       " 468: [[['E3', 0], ['cardio', 1], ['Lucas', 2], ['.', 3]], []],\n",
       " 469: [[['Doutor', 0], ['Vital', 1], ['Brasil', 2], ['.', 3]], []],\n",
       " 470: [[['joão', 0], [',', 1], ['68', 2], ['anos', 3], ['.', 4]], []],\n",
       " 471: [[['#', 0], ['IAM', 1], ['2009', 2], ['-', 3], ['ATC', 4], ['.', 5]],\n",
       "  [['IAM', [1], 'Problema'], ['ATC', [4], 'Tratamento']]],\n",
       " 472: [[['#', 0],\n",
       "   ['CA', 1],\n",
       "   ['de', 2],\n",
       "   ['Laringe', 3],\n",
       "   [',', 4],\n",
       "   ['fez', 5],\n",
       "   ['QT', 6],\n",
       "   ['e', 7],\n",
       "   ['RT', 8],\n",
       "   ['-', 9],\n",
       "   ['TERMINO', 10],\n",
       "   ['TTO', 11],\n",
       "   ['2002', 12],\n",
       "   [',', 13],\n",
       "   ['.', 14]],\n",
       "  [['CA de Laringe', [1, 2, 3], 'Problema'],\n",
       "   ['Laringe', [3], 'Anatomia'],\n",
       "   ['QT', [6], 'Tratamento'],\n",
       "   ['RT', [8], 'Tratamento'],\n",
       "   ['TTO', [11], 'Tratamento']]],\n",
       " 473: [[['#', 0], ['HAS', 1], [',', 2], ['.', 3]], [['HAS', [1], 'Problema']]],\n",
       " 474: [[['#', 0], ['HPB', 1], [',', 2], ['.', 3]], [['HPB', [1], 'Problema']]],\n",
       " 475: [[['#', 0], ['DLP', 1], [',', 2], ['.', 3]], [['DLP', [1], 'Problema']]],\n",
       " 476: [[['#', 0], ['ARTRITE', 1], ['REUMATÓIDE', 2], ['.', 3]],\n",
       "  [['ARTRITE REUMATÓIDE', [1, 2], 'Problema']]],\n",
       " 477: [[['#', 0],\n",
       "   ['Em', 1],\n",
       "   ['um', 2],\n",
       "   ['uso', 3],\n",
       "   ['de', 4],\n",
       "   [':', 5],\n",
       "   ['.', 6]],\n",
       "  []],\n",
       " 478: [[['-', 0],\n",
       "   ['Enalapril', 1],\n",
       "   ['20', 2],\n",
       "   ['mg', 3],\n",
       "   [',', 4],\n",
       "   ['2', 5],\n",
       "   ['cp', 6],\n",
       "   ['de', 7],\n",
       "   ['12', 8],\n",
       "   ['/', 9],\n",
       "   ['12', 10],\n",
       "   ['hr', 11],\n",
       "   ['.', 12]],\n",
       "  [['Enalapril 20 mg', [1, 2, 3], 'Tratamento']]],\n",
       " 479: [[['-', 0],\n",
       "   ['CARVEDILOL', 1],\n",
       "   ['12', 2],\n",
       "   [',', 3],\n",
       "   ['5MG', 4],\n",
       "   [',', 5],\n",
       "   ['1', 6],\n",
       "   ['cp', 7],\n",
       "   ['de', 8],\n",
       "   ['12', 9],\n",
       "   ['/', 10],\n",
       "   ['12', 11],\n",
       "   ['hr', 12],\n",
       "   [',', 13],\n",
       "   ['.', 14]],\n",
       "  [['CARVEDILOL 12 , 5MG', [1, 2, 3, 4], 'Tratamento']]],\n",
       " 480: [[['-', 0],\n",
       "   ['AAS', 1],\n",
       "   ['100', 2],\n",
       "   ['mg', 3],\n",
       "   [',', 4],\n",
       "   ['1', 5],\n",
       "   ['/', 6],\n",
       "   ['dia', 7],\n",
       "   [';', 8],\n",
       "   ['.', 9]],\n",
       "  [['AAS 100 mg', [1, 2, 3], 'Tratamento']]],\n",
       " 481: [[['-', 0],\n",
       "   ['Sinvastatina', 1],\n",
       "   ['20mg', 2],\n",
       "   [',', 3],\n",
       "   ['4', 4],\n",
       "   ['cp', 5],\n",
       "   ['de', 6],\n",
       "   ['noite', 7],\n",
       "   [';', 8],\n",
       "   ['.', 9]],\n",
       "  [['Sinvastatina 20mg', [1, 2], 'Tratamento']]],\n",
       " 482: [[['-', 0],\n",
       "   ['Omeprazol', 1],\n",
       "   ['20m', 2],\n",
       "   ['mg', 3],\n",
       "   [',', 4],\n",
       "   ['2', 5],\n",
       "   ['cp', 6],\n",
       "   ['de', 7],\n",
       "   ['12', 8],\n",
       "   ['/', 9],\n",
       "   ['12hr', 10],\n",
       "   [';', 11],\n",
       "   ['.', 12]],\n",
       "  [['Omeprazol 20m', [1, 2], 'Tratamento']]],\n",
       " 483: [[['-', 0], ['PREDNISONA', 1], ['5', 2], ['MG', 3], [';', 4], ['.', 5]],\n",
       "  [['PREDNISONA 5 MG', [1, 2, 3], 'Tratamento']]],\n",
       " 484: [[['-', 0], ['COMBODART', 1], ['.', 2]],\n",
       "  [['COMBODART', [1], 'Tratamento']]],\n",
       " 485: [[['***Teste', 0],\n",
       "   ['de', 1],\n",
       "   ['esforço', 2],\n",
       "   ['10', 3],\n",
       "   ['/', 4],\n",
       "   ['2013', 5],\n",
       "   [':', 6],\n",
       "   ['eficaz', 7],\n",
       "   [',', 8],\n",
       "   ['submáximo', 9],\n",
       "   [',', 10],\n",
       "   ['sem', 11],\n",
       "   ['alterações', 12],\n",
       "   ['.', 13]],\n",
       "  [['alterações', [12], 'Problema']]],\n",
       " 486: [[['Clínicas', 0],\n",
       "   ['ou', 1],\n",
       "   ['eletrocardiográficas', 2],\n",
       "   ['de', 3],\n",
       "   ['iesquemia', 4],\n",
       "   ['miocardica', 5],\n",
       "   ['.', 6]],\n",
       "  [['iesquemia miocardica', [4, 5], 'Problema'],\n",
       "   ['miocardica', [5], 'Anatomia']]],\n",
       " 487: [[['Resposta', 0], ['cronotrópica', 1], ['adequada', 2], ['.', 3]], []],\n",
       " 488: [[['Resposta', 0], ['tensional', 1], ['fisiológica', 2], ['.', 3]], []],\n",
       " 489: [[['Decréscimo', 0],\n",
       "   ['normal', 1],\n",
       "   ['da', 2],\n",
       "   ['FC', 3],\n",
       "   ['no', 4],\n",
       "   ['1°', 5],\n",
       "   ['minuto', 6],\n",
       "   ['de', 7],\n",
       "   ['recuperação', 8],\n",
       "   ['.', 9]],\n",
       "  [['Decréscimo', [0], 'Problema'], ['FC', [3], 'Problema']]],\n",
       " 490: [[['***', 0],\n",
       "   ['ECOCARDIO', 1],\n",
       "   ['11', 2],\n",
       "   ['/', 3],\n",
       "   ['06', 4],\n",
       "   ['/', 5],\n",
       "   ['2014', 6],\n",
       "   [':', 7],\n",
       "   ['VE', 8],\n",
       "   ['COM', 9],\n",
       "   ['DIMENSÃO', 10],\n",
       "   ['INTERNA', 11],\n",
       "   [',', 12],\n",
       "   ['CONTRATILIDADE', 13],\n",
       "   ['MIOCARDICA', 14],\n",
       "   ['E', 15],\n",
       "   ['FUNÇÃO', 16],\n",
       "   ['SISTOLICA', 17],\n",
       "   ['GLOBAL', 18],\n",
       "   ['NORMAIS', 19],\n",
       "   [',', 20],\n",
       "   ['DISFUNÇÃO', 21],\n",
       "   ['DIASTOLICA', 22],\n",
       "   ['TIPO', 23],\n",
       "   ['ALTERAÇÃO', 24],\n",
       "   ['DE', 25],\n",
       "   ['RELAXAMENTO', 26],\n",
       "   [',', 27],\n",
       "   ['AE', 28],\n",
       "   ['E', 29],\n",
       "   ['CAVIDADES', 30],\n",
       "   ['DIREITAS', 31],\n",
       "   ['NORMAIS', 32],\n",
       "   [',', 33],\n",
       "   ['.', 34]],\n",
       "  [['ECOCARDIO', [1], 'Teste'],\n",
       "   ['VE', [8], 'Anatomia'],\n",
       "   ['MIOCARDICA', [14], 'Anatomia'],\n",
       "   ['DISFUNÇÃO DIASTOLICA TIPO ALTERAÇÃO DE RELAXAMENTO',\n",
       "    [21, 22, 23, 24, 25, 26],\n",
       "    'Problema'],\n",
       "   ['AE', [28], 'Anatomia'],\n",
       "   ['CAVIDADES DIREITAS', [30, 31], 'Anatomia']]],\n",
       " 491: [[['FLUXOS', 0],\n",
       "   ['VALVARES', 1],\n",
       "   ['SEM', 2],\n",
       "   ['ANORMALIDADES', 3],\n",
       "   ['MORFOFUNCIONAIS', 4],\n",
       "   ['SIGNIFICATIVAS', 5],\n",
       "   ['.', 6]],\n",
       "  [['ANORMALIDADES MORFOFUNCIONAIS SIGNIFICATIVAS', [3, 4, 5], 'Problema']]],\n",
       " 492: [[['AE', 0],\n",
       "   ['31', 1],\n",
       "   [',', 2],\n",
       "   ['VDC', 3],\n",
       "   ['21', 4],\n",
       "   [',', 5],\n",
       "   ['DDDV', 6],\n",
       "   ['44', 7],\n",
       "   [',', 8],\n",
       "   ['DSVE', 9],\n",
       "   ['30', 10],\n",
       "   [',', 11],\n",
       "   ['FE', 12],\n",
       "   ['60', 13],\n",
       "   ['.', 14]],\n",
       "  [['AE', [0], 'Anatomia']]],\n",
       " 493: [[['S', 0], ['#', 1], ['SEM', 2], ['QUEIXAS', 3], ['.', 4]],\n",
       "  [['QUEIXAS', [3], 'Problema']]],\n",
       " 494: [[['NEGA', 0],\n",
       "   ['DISPNEIA', 1],\n",
       "   ['E', 2],\n",
       "   ['DOR', 3],\n",
       "   ['NO', 4],\n",
       "   ['PEITO', 5],\n",
       "   ['.', 6]],\n",
       "  [['DISPNEIA', [1], 'Problema'],\n",
       "   ['DOR NO PEITO', [3, 4, 5], 'Problema'],\n",
       "   ['PEITO', [5], 'Anatomia']]],\n",
       " 495: [[['ESTÁVEL', 0], ['.', 1]], []],\n",
       " 496: [[['O', 0], ['#', 1], ['.', 2]], []],\n",
       " 497: [[['BEG', 0],\n",
       "   [',', 1],\n",
       "   ['LOTE', 2],\n",
       "   [',', 3],\n",
       "   ['CORADO', 4],\n",
       "   [',', 5],\n",
       "   ['HIDRATADO', 6],\n",
       "   [',', 7],\n",
       "   ['EUPNEICO', 8],\n",
       "   ['.', 9]],\n",
       "  []],\n",
       " 498: [[['PA', 0],\n",
       "   ['120', 1],\n",
       "   ['/', 2],\n",
       "   ['80', 3],\n",
       "   [',', 4],\n",
       "   ['FC', 5],\n",
       "   ['80', 6],\n",
       "   [';', 7],\n",
       "   ['.', 8]],\n",
       "  [['PA', [0], 'Teste'], ['FC', [5], 'Teste']]],\n",
       " 499: [[['ACV', 0], [':', 1], ['BCRNF2TSS', 2], [';', 3], ['.', 4]], []],\n",
       " 500: [[['AP', 0],\n",
       "   [':', 1],\n",
       "   ['MV', 2],\n",
       "   ['+', 3],\n",
       "   ['BILAT', 4],\n",
       "   [',', 5],\n",
       "   ['SEM', 6],\n",
       "   ['RA', 7],\n",
       "   [';', 8],\n",
       "   ['.', 9]],\n",
       "  [['RA', [7], 'Problema']]],\n",
       " 501: [[['ABD', 0], [':', 1], ['SP', 2], [';', 3], ['.', 4]],\n",
       "  [['ABD', [0], 'Anatomia']]],\n",
       " 502: [[['MMII', 0], [':', 1], ['SP', 2], [';', 3], ['.', 4]],\n",
       "  [['MMII', [0], 'Anatomia']]],\n",
       " 503: [[['A', 0], ['#', 1], ['DAC', 2], ['.', 3]], [['DAC', [2], 'Problema']]],\n",
       " 504: [[['P', 0],\n",
       "   ['#', 1],\n",
       "   ['CONDUTA', 2],\n",
       "   ['MANTIDA', 3],\n",
       "   [',', 4],\n",
       "   ['.', 5]],\n",
       "  []],\n",
       " 505: [[['EX', 0], ['DE', 1], ['CONTROLE', 2], ['.', 3]],\n",
       "  [['EX', [0], 'Teste']]],\n",
       " 506: [[['R3', 0], ['CARDIO', 1], ['Lucas', 2], [',', 3], ['.', 4]], []],\n",
       " 507: [[['Doutor', 0], ['Lucas', 1], ['.', 2]], []]}"
      ]
     },
     "execution_count": 85,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dic_sentencesTest"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [],
   "source": [
    "def getCombinacaoEntidades(dic_predictions):\n",
    "    num=0\n",
    "    numDescontinuas=0\n",
    "    erro_corpus=0\n",
    "    num_frases_sem_entidade=0\n",
    "    lista_erro_corpus=list()\n",
    "    combinacaoEntidadesAll = list()\n",
    "    combinacaoEntidades = list()\n",
    "    pulando_termos_postagger = list()\n",
    "    for key, value in dic_predictions.items():\n",
    "        num=num+1\n",
    "        combinacaoEntidades = list()\n",
    "        #print('key:', key)\n",
    "        #print(value)\n",
    "        tokens=value[0].copy()\n",
    "        so_tokens = [t[0] for t in tokens]\n",
    "        entidades=value[1].copy()\n",
    "        #print(so_tokens)\n",
    "        #print(entidades)\n",
    "        for entidade in entidades:\n",
    "            erros_entidade = list()\n",
    "            #print(entidade[1])\n",
    "            texto_entidade=entidade[0].strip()\n",
    "            indices = entidade[1]\n",
    "            tipo_entidade = entidade[2]\n",
    "            #print('texto_entidade:', texto_entidade)\n",
    "            frase = so_tokens.copy()\n",
    "            #print(frase)\n",
    "            inicio=indices[0]\n",
    "            fim=indices[-1]\n",
    "            frase.insert(inicio, '<e1>')\n",
    "            frase.insert(fim+2, '</e1>')\n",
    "            if texto_entidade=='-' or texto_entidade=='=' or texto_entidade=='+' or texto_entidade==':' or texto_entidade==',' or texto_entidade==\"'\" or texto_entidade=='\"' or texto_entidade=='.' or texto_entidade==';' or texto_entidade=='/' or texto_entidade=='(' or texto_entidade==')' or texto_entidade=='[' or texto_entidade==']':\n",
    "                pass\n",
    "            #print('--texto_entidade--:', texto_entidade)\n",
    "            #print('frase:', frase)\n",
    "            #print('frase[inicio+1:fim+2]:', frase[inicio+1:fim+2])\n",
    "            #print(tokens[indice])\n",
    "            # o corpus tem alguns problemas, ex tem entidade descontinua sem o CC\n",
    "            # entao aqui, se não bater, não adicionar no arquivo de treinamento\n",
    "            # ex frase \"The inhibition of c - fos and c - jun expression by IL - 4 in LPS - treated cells was shown to be due to a lower transcription rate of the c - fos and c - jun genes .\"\n",
    "            #print(\"texto_entidade:\", texto_entidade)\n",
    "            #print(\"frase[inicio:fim]:\", frase[inicio+1:fim+2])\n",
    "            texto_entidade_comparar=texto_entidade.replace('/','').replace(')','').replace('(','').replace(']','').replace('[','').replace(',','').replace('.','').replace(';','').replace('-','').replace('+','').replace(\"'\",'')\n",
    "            texto_entidade_comparar = replaceWhiteSpaces(texto_entidade_comparar)\n",
    "            texto_frase_comparar = ' '.join(frase[inicio+1:fim+2]).strip().replace('/','').replace(')','').replace('(','').replace(']','').replace('[','').replace(',','').replace('.','').replace(';','').replace('-','').replace('+','').replace(\"'\",'')\n",
    "            texto_frase_comparar = replaceWhiteSpaces(texto_frase_comparar)\n",
    "            #tem = [palavra.lower() in texto_entidade_comparar for palavra in palavrasDescontinua]\n",
    "            #tem = [' '+palavra.lower()+' ' in ' '+texto_entidade_comparar+' ' for palavra in palavrasDescontinua]\n",
    "            #texto_frase_comparar2 = texto_frase_comparar.replace('and not', 'and_not').replace('as well as', 'as_well_as').replace('neither nor', 'neither_nor').replace('but not', 'but_not')\n",
    "            #print(frase)\n",
    "            #print('texto_frase_comparar2.split():', texto_frase_comparar2.split())\n",
    "            #print('texto_frase_comparar2.split():', texto_frase_comparar2.split()[0])\n",
    "            #tem = [texto_frase_comparar2.split()[-1]==palavra.lower() for palavra in palavrasDescontinua_underline]\n",
    "            #tem = [' '+palavra.lower()+' ' in ' '+texto_frase_comparar2+' ' for palavra in palavrasDescontinua_underline]\n",
    "            #tem_boolean = True in tem\n",
    "            #print([' '.join(frase).strip(), tipo_entidade])\n",
    "            #if not tem_boolean:\n",
    "            #if 1==2:\n",
    "            #    combinacaoEntidades.append([' '.join(frase).strip(), tipo_entidade]) # apendando entidades reais\n",
    "            texto_entidade_comparar = texto_entidade_comparar.lower()\n",
    "            texto_frase_comparar = texto_frase_comparar.lower()\n",
    "            if ((texto_entidade_comparar == texto_frase_comparar) or (texto_entidade_comparar+'s' == texto_frase_comparar) or (texto_entidade_comparar+'es' == texto_frase_comparar) or (texto_entidade_comparar+'ies' == texto_frase_comparar) or (texto_entidade_comparar[:-1]+'ies' == texto_frase_comparar) or (texto_entidade_comparar[:-1] == texto_frase_comparar.replace(' /','/'))):\n",
    "                combinacaoEntidades.append([' '.join(frase).strip(), tipo_entidade]) # apendando entidades reais\n",
    "            else:\n",
    "                #print('erooooooor')\n",
    "                #print('texto_entidade_comparar:', texto_entidade_comparar)\n",
    "                #print('texto_frase_comparar:', texto_frase_comparar)\n",
    "                #print('texto_frase_comparar2.split()[-1]:', texto_frase_comparar2.split()[-1])\n",
    "                #print(\"' '.join(texto_entidade).strip()):\", ' '.join(texto_entidade).strip())\n",
    "                #print(\"' '.join(frase[inicio:fim].strip():\", ' '.join(frase[inicio+1:fim+2]).strip())\n",
    "                erro_corpus=erro_corpus+1\n",
    "                erros_entidade.append(indices)\n",
    "                #print('-----indices erro:----', indices)\n",
    "                lista_erro_corpus.append([' '.join(frase).strip(), tipo_entidade, ' '.join(so_tokens), entidade])\n",
    "\n",
    "        for entidade in entidades:\n",
    "                indices = entidade[1]\n",
    "                #print('indices:', indices)\n",
    "                if indices in erros_entidade:\n",
    "                    continue\n",
    "                inicio=indices[0]\n",
    "                fim=indices[-1]\n",
    "                # agora, fazer a combinacao entre eles.. todas a seguir serão do tipo 'O'           \n",
    "                for indice in indices:\n",
    "                    for i in range(indice, fim+1):\n",
    "                        # ver se nao tem antes\n",
    "                        frase = so_tokens.copy()\n",
    "                        termo = frase[indice:i+2]\n",
    "                        frase.insert(indice, '<e1>')\n",
    "                        #print(i)\n",
    "\n",
    "                        frase.insert(i+2, '</e1>')\n",
    "                        frase_string=' '.join(frase).strip()\n",
    "                        #print(frase_string)\n",
    "                        #if frase_string not in combinacaoEntidades:\n",
    "                        # ver se frase não termina com pontuacao ('.',',',';','-',')','(',']','[','/','\"',)\n",
    "                        devePular = 0\n",
    "                        if '. </e1>' in frase_string or ', </e1>' in frase_string  or '; </e1>' in frase_string or '- </e1>' in frase_string  or ': </e1>' in frase_string  or '= </e1>' in frase_string  or '/ </e1>' in frase_string  or '( </e1>' in frase_string  or ') </e1>' in frase_string  or '[ </e1>' in frase_string  or '] </e1>' in frase_string  or ': </e1>' in frase_string or 'and </e1>' in frase_string or 'or </e1>' in frase_string:\n",
    "                            devePular=1\n",
    "                        if '<e1> .' in frase_string or '<e1> ,' in frase_string  or '<e1> ;' in frase_string or '<e1> -' in frase_string  or '<e1> :' in frase_string  or '<e1> =' in frase_string  or '<e1> /' in frase_string  or '<e1> (' in frase_string  or '<e1> )' in frase_string  or '<e1> [' in frase_string  or '<e1> ]' in frase_string  or '<e1> :' in frase_string  or '<e1> and' in frase_string  or '<e1> or' in frase_string:\n",
    "                            devePular=1\n",
    "                        if re.search(\"<e1> [0-9]* </e1>\", frase_string):\n",
    "                            devePular=1\n",
    "\n",
    "                        tem_frase = 0\n",
    "                        for frase in combinacaoEntidades:\n",
    "                            if frase[0] == frase_string:\n",
    "                                tem_frase=''\n",
    "                                break\n",
    "                        if tem_frase==0 and devePular==0:\n",
    "                            #print('inserindo:', frase_string)\n",
    "                            #print('indice:', indice)\n",
    "                            combinacaoEntidades.append([frase_string, 'O'])\n",
    "        # shuffle no combinacaoEntidades\n",
    "        if len(combinacaoEntidades)>0:\n",
    "            random.shuffle(combinacaoEntidades)\n",
    "            combinacaoEntidadesAll.append([' '.join(so_tokens).strip(), combinacaoEntidades])\n",
    "        else:\n",
    "            num_frases_sem_entidade = num_frases_sem_entidade+1\n",
    "            combinacaoEntidadesAll.append([])\n",
    "            #print(\"key sem entidade\", key)\n",
    "            #print('len(combinacaoEntidades):', len(combinacaoEntidades))\n",
    "            #print(so_tokens)\n",
    "        combinacaoEntidades = list()\n",
    "        if (num % 1000) ==0:\n",
    "        #if (num %3) == 0:\n",
    "            print('key:', key)\n",
    "            #break\n",
    "\n",
    "    print('numDescontinuas:', numDescontinuas)\n",
    "    print('erro_corpus:', erro_corpus)\n",
    "    print('num_frases_sem_entidade:', num_frases_sem_entidade)\n",
    "    print('len(combinacaoEntidadesAll:)',len(combinacaoEntidadesAll))\n",
    "    \n",
    "    return combinacaoEntidadesAll"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--Train--\n",
      "key: 999\n",
      "numDescontinuas: 0\n",
      "erro_corpus: 0\n",
      "num_frases_sem_entidade: 239\n",
      "len(combinacaoEntidadesAll:) 1334\n",
      "\n",
      "--Dev--\n",
      "numDescontinuas: 0\n",
      "erro_corpus: 0\n",
      "num_frases_sem_entidade: 90\n",
      "len(combinacaoEntidadesAll:) 421\n",
      "\n",
      "--Test--\n",
      "numDescontinuas: 0\n",
      "erro_corpus: 0\n",
      "num_frases_sem_entidade: 108\n",
      "len(combinacaoEntidadesAll:) 508\n"
     ]
    }
   ],
   "source": [
    "print('--Train--')\n",
    "combinacaoEntidadesTrain = getCombinacaoEntidades(dic_sentencesTrain)\n",
    "print('\\n--Dev--')\n",
    "combinacaoEntidadesDev = getCombinacaoEntidades(dic_sentencesDev)\n",
    "print('\\n--Test--')\n",
    "combinacaoEntidadesTest = getCombinacaoEntidades(dic_sentencesTest)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['DSLP em uso de sinvastatina , marevan 1 cp / dia seg - sab para no alvo sic .',\n",
       " [['DSLP em uso de sinvastatina , <e1> marevan </e1> 1 cp / dia seg - sab para no alvo sic .',\n",
       "   'Tratamento'],\n",
       "  ['<e1> DSLP </e1> em uso de sinvastatina , marevan 1 cp / dia seg - sab para no alvo sic .',\n",
       "   'Problema'],\n",
       "  ['DSLP em uso de <e1> sinvastatina </e1> , marevan 1 cp / dia seg - sab para no alvo sic .',\n",
       "   'Tratamento']]]"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combinacaoEntidadesTest[4]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Obeso , has , icc .',\n",
       " [['Obeso , has , <e1> icc </e1> .', 'Problema'],\n",
       "  ['<e1> Obeso </e1> , has , icc .', 'Problema'],\n",
       "  ['Obeso , <e1> has </e1> , icc .', 'Problema']]]"
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combinacaoEntidadesTrain[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['SOLICITO EXAMES LABORATORIAIS E CATETERISMO DIAGNOSTICO PARA VER OSSTENTS IMPLANTADOS .',\n",
       " [['SOLICITO EXAMES LABORATORIAIS E CATETERISMO <e1> DIAGNOSTICO </e1> PARA VER OSSTENTS IMPLANTADOS .',\n",
       "   'O'],\n",
       "  ['SOLICITO <e1> EXAMES LABORATORIAIS </e1> E CATETERISMO DIAGNOSTICO PARA VER OSSTENTS IMPLANTADOS .',\n",
       "   'Teste'],\n",
       "  ['SOLICITO EXAMES LABORATORIAIS E CATETERISMO DIAGNOSTICO PARA VER <e1> OSSTENTS </e1> IMPLANTADOS .',\n",
       "   'Tratamento'],\n",
       "  ['SOLICITO <e1> EXAMES </e1> LABORATORIAIS E CATETERISMO DIAGNOSTICO PARA VER OSSTENTS IMPLANTADOS .',\n",
       "   'O'],\n",
       "  ['SOLICITO EXAMES <e1> LABORATORIAIS </e1> E CATETERISMO DIAGNOSTICO PARA VER OSSTENTS IMPLANTADOS .',\n",
       "   'O'],\n",
       "  ['SOLICITO EXAMES LABORATORIAIS E <e1> CATETERISMO DIAGNOSTICO </e1> PARA VER OSSTENTS IMPLANTADOS .',\n",
       "   'Teste'],\n",
       "  ['SOLICITO EXAMES LABORATORIAIS E <e1> CATETERISMO </e1> DIAGNOSTICO PARA VER OSSTENTS IMPLANTADOS .',\n",
       "   'O']]]"
      ]
     },
     "execution_count": 90,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combinacaoEntidadesDev[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [],
   "source": [
    "def gravarArquivosTreinamento(combinacaoEntidadesTrain, combinacaoEntidadesDev, combinacaoEntidadesTest):\n",
    "\n",
    "    # já ir gravando arquivos treinamento, test e dev...\n",
    "    # pra fazer um teste sem descontinuas\n",
    "    numTotalEntidades=0\n",
    "    numTotalEntidadesTrain=0\n",
    "    numTotalEntidadesDev=0\n",
    "    numTotalEntidadesTest=0\n",
    "\n",
    "    f_train = open('span.train', 'w')\n",
    "\n",
    "    for i, combinacaoEntidades in enumerate(combinacaoEntidadesTrain):\n",
    "        if len(combinacaoEntidades)>0:\n",
    "            frase = combinacaoEntidades[0]\n",
    "            frases_entidade = combinacaoEntidades[1]\n",
    "            f_train.write(frase+'\\n')\n",
    "            for frase_entidade in frases_entidade:\n",
    "                f_train.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "                numTotalEntidades=numTotalEntidades+1\n",
    "                numTotalEntidadesTrain=numTotalEntidadesTrain+1\n",
    "\n",
    "    f_train.close()\n",
    "\n",
    "    f_dev = open('span.dev', 'w')\n",
    "\n",
    "    for i, combinacaoEntidades in enumerate(combinacaoEntidadesDev):\n",
    "        if len(combinacaoEntidades)>0:\n",
    "            frase = combinacaoEntidades[0]\n",
    "            frases_entidade = combinacaoEntidades[1]\n",
    "            f_dev.write(frase+'\\n')\n",
    "            for frase_entidade in frases_entidade:\n",
    "                f_dev.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "                numTotalEntidades=numTotalEntidades+1\n",
    "                numTotalEntidadesDev=numTotalEntidadesDev+1\n",
    "\n",
    "    f_dev.close()\n",
    "\n",
    "    f_test = open('span.test', 'w')\n",
    "    for i, combinacaoEntidades in enumerate(combinacaoEntidadesTest):\n",
    "        #print(dicSentences[i])\n",
    "        if len(combinacaoEntidades)>0:\n",
    "            frase = combinacaoEntidades[0]\n",
    "            frases_entidade = combinacaoEntidades[1]\n",
    "            f_test.write(frase+'\\n')\n",
    "            for frase_entidade in frases_entidade:\n",
    "                f_test.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "                numTotalEntidades=numTotalEntidades+1\n",
    "                numTotalEntidadesTest=numTotalEntidadesTest+1\n",
    "\n",
    "    f_test.close()\n",
    "\n",
    "    print('numTotalEntidades:', numTotalEntidades)\n",
    "    print('numTotalEntidadesTrain:', numTotalEntidadesTrain)\n",
    "    print('numTotalEntidadesDev:', numTotalEntidadesDev)\n",
    "    print('numTotalEntidadesTest:', numTotalEntidadesTest)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "numTotalEntidades: 13879\n",
      "numTotalEntidadesTrain: 8822\n",
      "numTotalEntidadesDev: 2022\n",
      "numTotalEntidadesTest: 3035\n"
     ]
    }
   ],
   "source": [
    "gravarArquivosTreinamento(combinacaoEntidadesTrain, combinacaoEntidadesDev, combinacaoEntidadesTest)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### parte 2.2 - tentar tirar verbos, CC, etc do O para nao ficar com mto FP\n",
    "### depois, com nosso pos tagger"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 700,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 1o. encontrar os postaggers das entidades...\n",
    "import xml.dom.minidom\n",
    "\n",
    "def getDicPosTagger(XMLfileName):\n",
    "    dicPostagger = {}\n",
    "    doc = xml.dom.minidom.parse(XMLfileName);\n",
    "    sentences = doc.getElementsByTagName(\"sentence\")\n",
    "    for sentence in sentences:\n",
    "        # ver se tem entidade do interesse\n",
    "        cons = sentence.getElementsByTagName(\"cons\")\n",
    "        list_w = sentence.getElementsByTagName(\"w\")\n",
    "        for i, w in enumerate(list_w):\n",
    "            texto = w.firstChild.data.lower()\n",
    "            postagger = w.getAttribute(\"c\")\n",
    "            if (postagger == '*'):\n",
    "                postagger = 'NN'\n",
    "            dicPostagger[texto] = postagger\n",
    "            for palavra in texto.split('-'):\n",
    "                if not palavra in dicPostagger.keys():\n",
    "                    dicPostagger[palavra] = postagger\n",
    "            for palavra in texto.split('/'):\n",
    "                if not palavra in dicPostagger.keys():\n",
    "                    dicPostagger[palavra] = postagger\n",
    "            if not texto.replace('(','').replace(')','') in dicPostagger.keys():\n",
    "                dicPostagger[texto] = postagger\n",
    "    #save_obj('dicPostagger', dicPostagger)\n",
    "    return dicPostagger\n",
    "\n",
    "pathCorpus=r'GENIAcorpus3.02.merged_corrigido.xml'\n",
    "dicPosTagger = getDicPosTagger(pathCorpus)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 706,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'-NN-NN-CD-NN-NN'"
      ]
     },
     "execution_count": 706,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def tipoPostaggerTokens(entidade_token, dicPostagger):\n",
    "    postagger = ''\n",
    "    for p in entidade_token:\n",
    "        if p.lower() in dicPostagger.keys():\n",
    "            postagger = postagger + '-' + dicPostagger.get(p.lower())\n",
    "        elif p.lower().replace(',','') in dicPostagger.keys():\n",
    "            postagger = postagger + '-' + dicPostagger.get(p.lower().replace(',',''))\n",
    "        elif p.lower().replace('(','').replace(')','') in dicPostagger.keys():\n",
    "            postagger = postagger + '-' + dicPostagger.get(p.lower().replace('(','').replace(')',''))\n",
    "        else:\n",
    "            for palavra in p.lower().split('/'):\n",
    "                if palavra in dicPostagger.keys():\n",
    "                    postagger = postagger + '-' + dicPostagger.get(palavra)\n",
    "            for palavra in p.lower().split('-'):\n",
    "                if palavra in dicPostagger.keys():\n",
    "                    postagger = postagger + '-' + dicPostagger.get(palavra)\n",
    "    return postagger\n",
    "\n",
    "tipoPostaggerTokens(['IL','-','2','gene','CD28'], dicPosTagger)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 732,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['bHLH'], [10], 'PROTEIN']\n",
      "[['It', 0], ['contains', 1], ['a', 2], ['basic', 3], ['helix', 4], ['-', 5], ['loop', 6], ['-', 7], ['helix', 8], ['(', 9], ['bHLH', 10], [')', 11], ['motif', 12], ['contiguous', 13], ['with', 14], ['a', 15], ['Per', 16], ['-', 17], ['Arnt', 18], ['-', 19], ['Sim', 20], ['(', 21], ['PAS', 22], [')', 23], ['homology', 24], ['region', 25], ['.', 26]]\n",
      "[['GSSG'], [16], 'PROTEIN']\n",
      "[['Because', 0], ['inhibition', 1], ['of', 2], ['DNA', 3], ['-', 4], ['binding', 5], ['activity', 6], ['is', 7], ['associated', 8], ['with', 9], ['increasing', 10], ['intracellular', 11], ['glutathione', 12], ['disulfide', 13], ['levels', 14], ['and', 15], ['GSSG', 16], ['can', 17], ['be', 18], ['shown', 19], ['to', 20], ['inhibit', 21], ['the', 22], ['DNA', 23], ['-', 24], ['binding', 25], ['activity', 26], ['directly', 27], ['in', 28], ['cell', 29], ['-', 30], ['free', 31], ['systems', 32], [',', 33], ['our', 34], ['studies', 35], ['suggest', 36], ['that', 37], ['GSSG', 38], ['is', 39], ['a', 40], ['physiologically', 41], ['relevant', 42], ['inhibitor', 43], ['in', 44], ['intact', 45], ['cells', 46], ['also', 47], ['.', 48]]\n",
      "[['GSSG'], [38], 'PROTEIN']\n",
      "[['Because', 0], ['inhibition', 1], ['of', 2], ['DNA', 3], ['-', 4], ['binding', 5], ['activity', 6], ['is', 7], ['associated', 8], ['with', 9], ['increasing', 10], ['intracellular', 11], ['glutathione', 12], ['disulfide', 13], ['levels', 14], ['and', 15], ['GSSG', 16], ['can', 17], ['be', 18], ['shown', 19], ['to', 20], ['inhibit', 21], ['the', 22], ['DNA', 23], ['-', 24], ['binding', 25], ['activity', 26], ['directly', 27], ['in', 28], ['cell', 29], ['-', 30], ['free', 31], ['systems', 32], [',', 33], ['our', 34], ['studies', 35], ['suggest', 36], ['that', 37], ['GSSG', 38], ['is', 39], ['a', 40], ['physiologically', 41], ['relevant', 42], ['inhibitor', 43], ['in', 44], ['intact', 45], ['cells', 46], ['also', 47], ['.', 48]]\n",
      "[['CD4+'], [33], 'PROTEIN']\n",
      "[['OBJECTIVE', 0], [':', 1], ['To', 2], ['determine', 3], ['whether', 4], ['activation', 5], ['of', 6], ['Epstein', 7], ['-', 8], ['Barr', 9], ['virus', 10], ['(', 11], ['EBV', 12], [')', 13], ['replication', 14], ['in', 15], ['tumour', 16], ['cells', 17], ['of', 18], ['AIDS', 19], ['-', 20], ['related', 21], ['non', 22], ['-', 23], ['Hodgkin', 24], ['s', 25], ['lymphoma', 26], ['(', 27], ['ARNHL', 28], [')', 29], ['is', 30], ['correlated', 31], ['with', 32], ['CD4+', 33], ['cell', 34], ['counts', 35], ['and', 36], ['influences', 37], ['antibody', 38], ['response', 39], ['to', 40], ['EBV', 41], ['[', 42], ['anti', 43], ['-', 44], ['Z', 45], ['Epstein', 46], ['-', 47], ['Barr', 48], ['replicative', 49], ['activator', 50], ['(', 51], ['ZEBRA', 52], [')', 53], [',', 54], ['anti', 55], ['-', 56], ['early', 57], ['antigen', 58], ['(', 59], ['EA', 60], [')', 61], [',', 62], ['anti', 63], ['-', 64], ['viral', 65], ['capsid', 66], ['antigen', 67], ['(', 68], ['VCA', 69], [')', 70], [']', 71], ['.', 72]]\n",
      "[['CD4+'], [28], 'PROTEIN']\n",
      "[['DESIGN', 0], [':', 1], ['Retrospective', 2], ['study', 3], ['based', 4], ['on', 5], ['immunohistochemistry', 6], ['and', 7], ['in', 8], ['situ', 9], ['hybridization', 10], ['to', 11], ['detect', 12], ['EBV', 13], ['replicative', 14], ['gene', 15], ['products', 16], ['in', 17], ['tissue', 18], ['samples', 19], ['from', 20], ['patients', 21], ['affected', 22], ['by', 23], ['ARNHL', 24], ['and', 25], ['correlation', 26], ['with', 27], ['CD4+', 28], ['cell', 29], ['counts', 30], ['and', 31], ['results', 32], ['of', 33], ['EBV', 34], ['serology', 35], ['(', 36], ['including', 37], ['anti', 38], ['-', 39], ['ZEBRA', 40], ['activity', 41], [')', 42], ['in', 43], ['sera', 44], ['from', 45], ['the', 46], ['same', 47], ['patients', 48], ['.', 49]]\n",
      "[['CD4+'], [7], 'PROTEIN']\n",
      "[['Results', 0], ['were', 1], ['statistically', 2], ['correlated', 3], ['with', 4], ['those', 5], ['of', 6], ['CD4+', 7], ['cell', 8], ['counts', 9], ['(', 10], ['17', 11], ['out', 12], ['of', 13], ['17', 14], [')', 15], ['and', 16], ['with', 17], ['anti', 18], ['-', 19], ['EBV', 20], ['antibody', 21], ['titres', 22], ['(', 23], ['13', 24], ['out', 25], ['of', 26], ['17', 27], [')', 28], ['assessed', 29], ['using', 30], ['standard', 31], ['immunofluorescence', 32], ['method', 33], ['and', 34], ['enzyme', 35], ['-', 36], ['linked', 37], ['immunosorbent', 38], ['assay', 39], ['procedure', 40], ['using', 41], ['recombinant', 42], ['ZEBRA', 43], ['protein', 44], ['and', 45], ['synthetic', 46], ['peptides', 47], ['as', 48], ['antigens', 49], ['.', 50]]\n",
      "[['CD4+'], [11], 'PROTEIN']\n",
      "[['Demonstration', 0], ['of', 1], ['replicative', 2], ['gene', 3], ['expression', 4], ['did', 5], ['not', 6], ['correlate', 7], ['with', 8], ['either', 9], ['low', 10], ['CD4+', 11], ['cell', 12], ['counts', 13], ['(', 14], ['P', 15], ['>', 16], ['0.05', 17], [')', 18], ['or', 19], ['anti', 20], ['-', 21], ['EBV', 22], ['antibody', 23], ['titres', 24], ['(', 25], ['P', 26], ['>', 27], ['0.05', 28], [')', 29], ['.', 30]]\n",
      "[['HRBC'], [41], 'CELL_TYPE']\n",
      "[['To', 0], ['study', 1], ['putative', 2], ['teratogenic', 3], ['effects', 4], ['of', 5], ['a', 6], ['T', 7], ['cell', 8], ['-', 9], ['mediated', 10], ['immune', 11], ['response', 12], ['versus', 13], ['an', 14], ['endotoxic', 15], ['challenge', 16], [',', 17], ['10', 18], ['-', 19], ['d', 20], ['-', 21], ['pregnant', 22], ['rats', 23], ['received', 24], ['a', 25], ['single', 26], ['intraperitoneal', 27], ['injection', 28], ['of', 29], ['5', 30], ['x', 31], ['10', 32], ['(', 33], ['8', 34], [')', 35], ['human', 36], ['red', 37], ['blood', 38], ['cells', 39], ['(', 40], ['HRBC', 41], [')', 42], ['or', 43], ['gram', 44], ['-', 45], ['negative', 46], ['bacterial', 47], ['endotoxin', 48], ['(', 49], ['Escherichia', 50], ['coli', 51], ['LPS', 52], [':', 53], ['30', 54], ['micrograms/kg', 55], [')', 56], ['.', 57]]\n",
      "[['HRBC'], [7], 'CELL_TYPE']\n",
      "[['In', 0], ['addition', 1], [',', 2], ['after', 3], ['novelty', 4], ['stress', 5], ['the', 6], ['HRBC', 7], ['group', 8], [',', 9], ['but', 10], ['not', 11], ['the', 12], ['LPS', 13], ['group', 14], [',', 15], ['showed', 16], ['increased', 17], ['ACTH', 18], ['and', 19], ['corticosterone', 20], ['levels', 21], ['.', 22]]\n",
      "[['HRBC'], [0], 'CELL_TYPE']\n",
      "[['HRBC', 0], ['and', 1], ['LPS', 2], ['indeed', 3], ['stimulated', 4], ['the', 5], ['maternal', 6], ['immune', 7], ['system', 8], ['as', 9], ['revealed', 10], ['by', 11], ['specific', 12], ['anti', 13], ['-', 14], ['HRBC', 15], ['antibody', 16], ['production', 17], ['and', 18], ['enhanced', 19], ['IL', 20], ['-', 21], ['1', 22], ['beta', 23], ['mRNA', 24], ['expression', 25], ['in', 26], ['splenocytes', 27], [',', 28], ['respectively', 29], ['.', 30]]\n",
      "[['HRBC'], [15], 'CELL_TYPE']\n",
      "[['HRBC', 0], ['and', 1], ['LPS', 2], ['indeed', 3], ['stimulated', 4], ['the', 5], ['maternal', 6], ['immune', 7], ['system', 8], ['as', 9], ['revealed', 10], ['by', 11], ['specific', 12], ['anti', 13], ['-', 14], ['HRBC', 15], ['antibody', 16], ['production', 17], ['and', 18], ['enhanced', 19], ['IL', 20], ['-', 21], ['1', 22], ['beta', 23], ['mRNA', 24], ['expression', 25], ['in', 26], ['splenocytes', 27], [',', 28], ['respectively', 29], ['.', 30]]\n",
      "[['myeloperoxidase'], [24], 'PROTEIN']\n",
      "[['In', 0], ['mice', 1], ['exposed', 2], ['to', 3], ['hypoxia', 4], ['(', 5], ['PO2', 6], ['approximately', 7], ['30', 8], ['-', 9], ['40', 10], ['Torr', 11], [')', 12], [',', 13], ['there', 14], ['was', 15], ['increased', 16], ['pulmonary', 17], ['leukostasis', 18], [',', 19], ['as', 20], ['evidenced', 21], ['by', 22], ['increased', 23], ['myeloperoxidase', 24], ['activity', 25], ['in', 26], ['tissue', 27], ['homogenates', 28], ['.', 29]]\n",
      "[['myeloperoxidase'], [31], 'PROTEIN']\n",
      "[['Leukemic', 0], ['blasts', 1], ['in', 2], ['the', 3], ['majority', 4], ['of', 5], ['cases', 6], ['shared', 7], ['unique', 8], ['morphologic', 9], ['features', 10], ['(', 11], ['deeply', 12], ['invaginated', 13], ['nuclear', 14], ['membranes', 15], [',', 16], ['scant', 17], ['cytoplasm', 18], ['with', 19], ['fine', 20], ['azurophilic', 21], ['granularity', 22], [',', 23], ['and', 24], ['finely', 25], ['granular', 26], ['Sudan', 27], ['black', 28], ['B', 29], ['and', 30], ['myeloperoxidase', 31], ['cytochemical', 32], ['reactivity', 33], [')', 34], ['that', 35], ['were', 36], ['remarkably', 37], ['similar', 38], ['to', 39], ['those', 40], ['of', 41], ['acute', 42], ['promyelocytic', 43], ['leukemia', 44], ['(', 45], ['APL', 46], [')', 47], [';', 48], ['particularly', 49], ['the', 50], ['microgranular', 51], ['variant', 52], ['(', 53], ['FAB', 54], ['AML', 55], ['-', 56], ['M3v', 57], [')', 58], ['.', 59]]\n",
      "[['RARE'], [23], 'DNA']\n",
      "[['VitD3', 0], ['induction', 1], ['resulted', 2], ['in', 3], ['the', 4], ['formation', 5], ['of', 6], ['VDR', 7], ['/RXR', 8], ['DNA', 9], ['-', 10], ['binding', 11], ['complexes', 12], ['on', 13], ['both', 14], ['VitD3', 15], ['response', 16], ['elements', 17], ['and', 18], ['RA', 19], ['response', 20], ['elements', 21], ['(', 22], ['RAREs', 23], [')', 24], ['.', 25]]\n",
      "[['RARE'], [8], 'DNA']\n",
      "[['Several', 0], ['DNA', 1], ['-', 2], ['binding', 3], ['complexes', 4], ['were', 5], ['detected', 6], ['on', 7], ['RAREs', 8], ['in', 9], ['undifferentiated', 10], ['cells', 11], ['.', 12]]\n",
      "[['RARE'], [13], 'DNA']\n",
      "[['Stimulation', 0], ['by', 1], ['RA', 2], ['resulted', 3], ['in', 4], ['increased', 5], ['RAR', 6], ['beta', 7], ['/RXR', 8], ['DNA', 9], ['binding', 10], [',', 11], ['activated', 12], ['RARE', 13], ['-', 14], ['dependent', 15], ['transcription', 16], [',', 17], ['and', 18], ['increased', 19], ['expression', 20], ['of', 21], ['RAR', 22], ['-', 23], ['beta', 24], ['.', 25]]\n",
      "[['RARE'], [22], 'DNA']\n",
      "[['Concomitant', 0], ['stimulation', 1], ['by', 2], ['VitD3', 3], ['inhibited', 4], ['the', 5], ['RA', 6], ['-', 7], ['stimulated', 8], ['formation', 9], ['of', 10], ['RAR', 11], ['beta', 12], ['/RXR', 13], ['heterodimers', 14], [',', 15], ['favoring', 16], ['VDR', 17], ['/RXR', 18], ['binding', 19], ['to', 20], ['the', 21], ['RARE', 22], ['.', 23]]\n",
      "[['RARE'], [9], 'DNA']\n",
      "[['In', 0], ['contrast', 1], [',', 2], ['neither', 3], ['the', 4], ['RA', 5], ['-', 6], ['stimulated', 7], [',', 8], ['RARE', 9], ['-', 10], ['mediated', 11], ['transcription', 12], ['nor', 13], ['the', 14], ['induced', 15], ['RAR', 16], ['-', 17], ['beta', 18], ['expression', 19], ['was', 20], ['suppressed', 21], ['by', 22], ['VitD3', 23], [',', 24], ['suggesting', 25], ['that', 26], ['VitD3', 27], ['selectively', 28], ['inhibited', 29], ['the', 30], ['retinoid', 31], ['-', 32], ['induced', 33], ['differentiation', 34], ['program', 35], ['but', 36], ['not', 37], ['the', 38], ['RARE', 39], ['-', 40], ['mediated', 41], ['signal', 42], ['.', 43]]\n",
      "[['RARE'], [39], 'DNA']\n",
      "[['In', 0], ['contrast', 1], [',', 2], ['neither', 3], ['the', 4], ['RA', 5], ['-', 6], ['stimulated', 7], [',', 8], ['RARE', 9], ['-', 10], ['mediated', 11], ['transcription', 12], ['nor', 13], ['the', 14], ['induced', 15], ['RAR', 16], ['-', 17], ['beta', 18], ['expression', 19], ['was', 20], ['suppressed', 21], ['by', 22], ['VitD3', 23], [',', 24], ['suggesting', 25], ['that', 26], ['VitD3', 27], ['selectively', 28], ['inhibited', 29], ['the', 30], ['retinoid', 31], ['-', 32], ['induced', 33], ['differentiation', 34], ['program', 35], ['but', 36], ['not', 37], ['the', 38], ['RARE', 39], ['-', 40], ['mediated', 41], ['signal', 42], ['.', 43]]\n",
      "[['myeloperoxidase'], [30], 'PROTEIN']\n",
      "[['The', 0], ['involvement', 1], ['of', 2], ['members', 3], ['of', 4], ['the', 5], ['PEBP2/CBF', 6], ['family', 7], ['in', 8], ['chromosome', 9], ['translocations', 10], ['associated', 11], ['with', 12], ['acute', 13], ['myeloid', 14], ['leukemia', 15], [',', 16], ['and', 17], ['in', 18], ['the', 19], ['transcriptional', 20], ['regulation', 21], ['of', 22], ['the', 23], ['myeloid', 24], ['-', 25], ['specific', 26], ['genes', 27], ['encoding', 28], ['for', 29], ['myeloperoxidase', 30], [',', 31], ['elastase', 32], [',', 33], ['and', 34], ['the', 35], ['colony', 36], ['-', 37], ['stimulating', 38], ['factor', 39], ['receptor', 40], [',', 41], ['highlights', 42], ['the', 43], ['relevance', 44], ['of', 45], ['the', 46], ['regulation', 47], ['of', 48], ['the', 49], ['CD36', 50], ['gene', 51], ['promoter', 52], ['in', 53], ['monocytic', 54], ['cells', 55], ['by', 56], ['members', 57], ['of', 58], ['the', 59], ['PEBP2/CBF', 60], ['family', 61], ['.', 62]]\n",
      "[['rhu', 'TNFR:Fc'], [28, 29], 'PROTEIN']\n",
      "[['Recombinant', 0], ['human', 1], ['tumor', 2], ['necrosis', 3], ['factor', 4], ['(', 5], ['TNF', 6], [')', 7], ['binding', 8], ['protein', 9], ['-', 10], ['1', 11], ['(', 12], ['r', 13], ['-', 14], ['h', 15], ['TBP', 16], ['-', 17], ['1', 18], [')', 19], ['and', 20], ['recombinant', 21], ['human', 22], ['soluble', 23], ['dimeric', 24], ['TNF', 25], ['receptor', 26], ['(', 27], ['rhu', 28], ['TNFR', 29], [':', 30], ['Fc', 31], [')', 32], ['were', 33], ['used', 34], ['to', 35], ['determine', 36], ['the', 37], ['relative', 38], ['contributions', 39], ['of', 40], ['TNF', 41], ['to', 42], ['phorbol', 43], ['myristate', 44], ['acetate', 45], ['(', 46], ['PMA', 47], [')', 48], ['and', 49], ['cytokine', 50], ['-', 51], ['induced', 52], ['human', 53], ['immunodeficiency', 54], ['virus', 55], ['type', 56], ['1', 57], ['(', 58], ['HIV', 59], ['-', 60], ['1', 61], [')', 62], ['replication', 63], ['in', 64], ['chronically', 65], ['infected', 66], ['cell', 67], ['lines', 68], ['.', 69]]\n",
      "[['rhu', 'TNFR:Fc'], [19, 20], 'PROTEIN']\n",
      "[['Treatment', 0], ['of', 1], ['HIV', 2], ['-', 3], ['1', 4], ['-', 5], ['infected', 6], ['promonocytic', 7], ['U1', 8], ['cells', 9], ['with', 10], ['r', 11], ['-', 12], ['h', 13], ['-', 14], ['TBP', 15], ['-', 16], ['1', 17], ['or', 18], ['rhu', 19], ['TNFR', 20], [':', 21], ['Fc', 22], ['reduced', 23], ['PMA', 24], ['-', 25], ['induced', 26], ['HIV', 27], ['-', 28], ['1', 29], ['p24', 30], ['antigen', 31], ['production', 32], ['in', 33], ['a', 34], ['concentration', 35], ['-', 36], ['dependent', 37], ['manner', 38], [',', 39], ['with', 40], ['a', 41], ['maximal', 42], ['inhibition', 43], ['of', 44], ['approximately', 45], ['90', 46], ['%', 47], ['.', 48]]\n",
      "[['rhu', 'TNFR:Fc'], [27, 28], 'PROTEIN']\n",
      "[['Maximal', 0], ['inhibition', 1], ['of', 2], ['p24', 3], ['antigen', 4], ['production', 5], ['in', 6], ['T', 7], ['-', 8], ['lymphocytic', 9], ['ACH', 10], ['-', 11], ['2', 12], ['cells', 13], ['was', 14], ['47', 15], ['%', 16], ['with', 17], ['r', 18], ['-', 19], ['hTBP', 20], ['-', 21], ['1', 22], ['and', 23], ['42', 24], ['%', 25], ['with', 26], ['rhu', 27], ['TNFR', 28], [':', 29], ['Fc', 30], ['.', 31]]\n",
      "[['rhu', 'TNFR:Fc'], [6, 7], 'PROTEIN']\n",
      "[['r', 0], ['-', 1], ['hTBP', 2], ['-', 3], ['1', 4], ['and', 5], ['rhu', 6], ['TNFR', 7], [':', 8], ['Fc', 9], ['also', 10], ['decreased', 11], ['p24', 12], ['antigen', 13], ['synthesized', 14], ['by', 15], ['U1', 16], ['cells', 17], ['in', 18], ['response', 19], ['to', 20], ['other', 21], ['stimuli', 22], [',', 23], ['including', 24], ['phytohemagglutinin', 25], ['(', 26], ['PHA', 27], [')', 28], ['-', 29], ['induced', 30], ['supernatant', 31], [',', 32], ['granulocyte', 33], ['-', 34], ['macrophage', 35], ['colony', 36], ['-', 37], ['stimulating', 38], ['factor', 39], [',', 40], ['interleukin', 41], ['-', 42], ['6', 43], [',', 44], ['and', 45], ['TNF', 46], ['.', 47]]\n",
      "[['rhu', 'TNFR:Fc'], [7, 8], 'PROTEIN']\n",
      "[['Both', 0], ['r', 1], ['-', 2], ['hTBP', 3], ['-', 4], ['1', 5], ['and', 6], ['rhu', 7], ['TNFR', 8], [':', 9], ['Fc', 10], ['blocked', 11], ['PMA', 12], ['induction', 13], ['of', 14], ['nuclear', 15], ['factor', 16], ['(', 17], ['NK', 18], [')', 19], ['-', 20], [' kappa', 21], ['B', 22], ['DNA', 23], ['-', 24], ['binding', 25], ['activity', 26], ['in', 27], ['U1', 28], ['cells', 29], ['in', 30], ['association', 31], ['with', 32], ['decreases', 33], ['in', 34], ['HIV', 35], ['-', 36], ['1', 37], ['replication', 38], ['.', 39]]\n",
      "[['CD4+'], [64], 'PROTEIN']\n",
      "[['We', 0], ['show', 1], ['here', 2], ['that', 3], ['the', 4], ['response', 5], ['of', 6], ['the', 7], ['HIV', 8], ['-', 9], ['1', 10], ['long', 11], ['terminal', 12], ['repeat', 13], ['(', 14], ['LTR', 15], [')', 16], ['to', 17], ['mitogenic', 18], ['or', 19], ['Tat', 20], ['-', 21], ['mediated', 22], ['activation', 23], ['is', 24], ['sensitive', 25], ['to', 26], ['the', 27], ['suppressive', 28], ['action', 29], ['of', 30], ['a', 31], ['Herpesvirus', 32], ['saimiri', 33], ['(', 34], ['HVS', 35], [')', 36], ['-', 37], ['transformed', 38], ['CD8+', 39], ['T', 40], ['cell', 41], ['clone', 42], ['from', 43], ['an', 44], ['HIV', 45], ['-', 46], ['infected', 47], ['individual', 48], ['and', 49], ['supernatants', 50], ['from', 51], ['CD8+', 52], ['T', 53], ['cells', 54], ['of', 55], ['HIV', 56], ['-', 57], ['1', 58], ['-', 59], ['infected', 60], ['asymptomatic', 61], ['subjects', 62], ['(', 63], ['CD4+', 64], ['>', 65], ['350/microliters', 66], [')', 67], ['.', 68]]\n",
      "[['CIS'], [18], 'PROTEIN']\n",
      "[['SOCS', 0], ['-', 1], ['3', 2], ['protein', 3], ['was', 4], ['detectable', 5], ['as', 6], ['early', 7], ['as', 8], ['30', 9], ['min', 10], ['following', 11], ['IL', 12], ['-', 13], ['2', 14], ['stimulation', 15], [',', 16], ['while', 17], ['CIS', 18], ['was', 19], ['seen', 20], ['only', 21], ['at', 22], ['low', 23], ['levels', 24], ['after', 25], ['2', 26], ['h', 27], ['.', 28]]\n",
      "[['CIS'], [1], 'PROTEIN']\n",
      "[['Unlike', 0], ['CIS', 1], [',', 2], ['SOCS', 3], ['-', 4], ['3', 5], ['was', 6], ['rapidly', 7], ['tyrosine', 8], ['phosphorylated', 9], ['in', 10], ['response', 11], ['to', 12], ['IL', 13], ['-', 14], ['2', 15], ['.', 16]]\n",
      "[['CIS'], [10], 'PROTEIN']\n",
      "[['Additionally', 0], [',', 1], ['in', 2], ['lymphocytes', 3], ['expressing', 4], ['SOCS', 5], ['-', 6], ['3', 7], ['but', 8], ['not', 9], ['CIS', 10], [',', 11], ['IL', 12], ['-', 13], ['2', 14], ['-', 15], ['induced', 16], ['tyrosine', 17], ['phosphorylation', 18], ['of', 19], ['STAT5b', 20], ['was', 21], ['markedly', 22], ['reduced', 23], [',', 24], ['while', 25], ['there', 26], ['was', 27], ['only', 28], ['a', 29], ['weak', 30], ['effect', 31], ['on', 32], ['IL', 33], ['-', 34], ['3', 35], ['-', 36], ['mediated', 37], ['STAT5b', 38], ['tyrosine', 39], ['phosphorylation', 40], ['.', 41]]\n",
      "[['transfectant'], [0], 'CELL_LINE']\n",
      "[['Transfectants', 0], ['grown', 1], ['in', 2], ['the', 3], ['presence', 4], ['of', 5], ['tetracycline', 6], ['contained', 7], ['about', 8], ['15,000', 9], ['receptors/cell', 10], [',', 11], ['a', 12], ['value', 13], ['approximately', 14], ['equal', 15], ['to', 16], ['basal', 17], ['level', 18], ['GR', 19], ['expression', 20], ['in', 21], ['glucocorticoid', 22], ['-', 23], ['sensitive', 24], ['6TG1.1', 25], ['cells', 26], ['before', 27], ['steroid', 28], ['treatment', 29], ['.', 30]]\n",
      "[['EVI1'], [13], 'PROTEIN']\n",
      "[['By', 0], ['using', 1], ['the', 2], ['yeast', 3], ['two', 4], ['-', 5], ['hybrid', 6], ['system', 7], [',', 8], ['we', 9], ['also', 10], ['show', 11], ['that', 12], ['EVI1', 13], ['(', 14], ['contained', 15], ['in', 16], ['its', 17], ['entirety', 18], ['in', 19], ['MDS1/EVI1', 20], ['and', 21], ['AML1/MDS1/EVI1', 22], [')', 23], ['physically', 24], ['interacts', 25], ['with', 26], ['SMAD3', 27], [',', 28], ['which', 29], ['is', 30], ['an', 31], ['intracellular', 32], ['mediator', 33], ['of', 34], ['TGF', 35], ['-', 36], ['beta1', 37], ['signaling', 38], ['.', 39]]\n",
      "[['EVI1'], [15], 'PROTEIN']\n",
      "[['We', 0], ['propose', 1], ['that', 2], ['mutations', 3], ['of', 4], ['MDS1/EVI1', 5], ['either', 6], ['by', 7], ['gene', 8], ['truncation', 9], ['resulting', 10], ['in', 11], ['the', 12], ['transcription', 13], ['repressor', 14], ['EVI1', 15], ['or', 16], ['by', 17], ['gene', 18], ['fusion', 19], ['to', 20], ['AML1', 21], ['lead', 22], ['to', 23], ['an', 24], ['altered', 25], ['cellular', 26], ['response', 27], ['to', 28], ['growth', 29], ['and', 30], ['differentiation', 31], ['factors', 32], ['that', 33], ['could', 34], ['result', 35], ['in', 36], ['leukemic', 37], ['transformation', 38], ['.', 39]]\n",
      "[['CD3+'], [20], 'CELL_TYPE']\n",
      "[['The', 0], ['percentage', 1], ['of', 2], ['CD3+', 3], ['cells', 4], ['among', 5], ['all', 6], ['immunocompetent', 7], ['cells', 8], ['in', 9], ['the', 10], ['tubal', 11], ['epithelium', 12], ['was', 13], ['46.6', 14], ['%', 15], ['(', 16], ['39.9', 17], ['%', 18], ['of', 19], ['CD3+', 20], ['were', 21], ['also', 22], ['CD8+', 23], [')', 24], [';', 25], ['the', 26], ['other', 27], ['53.4', 28], ['%', 29], ['were', 30], ['CD68+', 31], ['cells', 32], ['.', 33]]\n",
      "[['CD8+'], [23], 'CELL_TYPE']\n",
      "[['The', 0], ['percentage', 1], ['of', 2], ['CD3+', 3], ['cells', 4], ['among', 5], ['all', 6], ['immunocompetent', 7], ['cells', 8], ['in', 9], ['the', 10], ['tubal', 11], ['epithelium', 12], ['was', 13], ['46.6', 14], ['%', 15], ['(', 16], ['39.9', 17], ['%', 18], ['of', 19], ['CD3+', 20], ['were', 21], ['also', 22], ['CD8+', 23], [')', 24], [';', 25], ['the', 26], ['other', 27], ['53.4', 28], ['%', 29], ['were', 30], ['CD68+', 31], ['cells', 32], ['.', 33]]\n",
      "[['BCL2+'], [3], 'CELL_TYPE']\n",
      "[['The', 0], ['ratio', 1], ['of', 2], ['BCL2+', 3], ['to', 4], ['CD3+', 5], ['cells', 6], ['in', 7], ['ectopic', 8], ['pregnancy', 9], ['was', 10], ['0.41', 11], ['.', 12]]\n",
      "[['ringed', 'sideroblast'], [26, 27], 'CELL_TYPE']\n",
      "[['In', 0], ['the', 1], ['present', 2], ['study', 3], [',', 4], ['the', 5], ['clonal', 6], ['relationship', 7], ['was', 8], ['investigated', 9], ['between', 10], ['peripheral', 11], ['blood', 12], ['polymorphonuclear', 13], ['cells', 14], ['(', 15], ['PMNCs', 16], [')', 17], ['and', 18], ['marrow', 19], ['progenitor', 20], ['cells', 21], ['and', 22], ['the', 23], ['origin', 24], ['of', 25], ['ringed', 26], ['sideroblasts', 27], ['in', 28], ['patients', 29], ['with', 30], ['refractory', 31], ['anemia', 32], ['with', 33], ['ring', 34], ['sideroblasts', 35], ['(', 36], ['RARS', 37], [')', 38], ['by', 39], ['polymerase', 40], ['chain', 41], ['reaction', 42], ['(', 43], ['PCR', 44], [')', 45], ['of', 46], ['HUMARA', 47], ['gene', 48], ['.', 49]]\n",
      "[['ringed', 'sideroblast'], [3, 4], 'CELL_TYPE']\n",
      "[['The', 0], ['development', 1], ['of', 2], ['ringed', 3], ['sideroblasts', 4], ['in', 5], ['erythroid', 6], ['colonies', 7], ['by', 8], ['iron', 9], ['staining', 10], ['and', 11], ['their', 12], ['X', 13], ['-', 14], ['inactivation', 15], ['pattern', 16], ['were', 17], ['also', 18], ['examined', 19], ['.', 20]]\n",
      "[['ringed', 'sideroblast'], [5, 6], 'CELL_TYPE']\n",
      "[['All', 0], ['erythroid', 1], ['colonies', 2], ['consisted', 3], ['of', 4], ['ringed', 5], ['sideroblasts', 6], ['exclusively', 7], ['showed', 8], ['one', 9], ['pattern', 10], ['dominant', 11], ['in', 12], ['those', 13], ['of', 14], ['PMNCs', 15], ['.', 16]]\n",
      "[['ringed', 'sideroblast'], [23, 24], 'CELL_TYPE']\n",
      "[['Our', 0], ['findings', 1], ['suggest', 2], ['that', 3], ['non', 4], ['-', 5], ['clonal', 6], ['progenitor', 7], ['cells', 8], ['persist', 9], ['in', 10], ['some', 11], ['RARS', 12], ['cases', 13], [',', 14], ['that', 15], ['erythroid', 16], ['progenitors', 17], ['show', 18], ['mosaicism', 19], [',', 20], ['and', 21], ['that', 22], ['ringed', 23], ['sideroblasts', 24], ['may', 25], ['be', 26], ['derived', 27], ['from', 28], ['an', 29], ['abnormal', 30], ['clone', 31], ['involved', 32], ['in', 33], ['the', 34], ['pathogenesis', 35], ['of', 36], ['this', 37], ['disease', 38], ['.', 39]]\n",
      "[['CD4+'], [3], 'PROTEIN']\n",
      "[['The', 0], ['induction', 1], ['of', 2], ['CD4+', 3], ['T', 4], ['cell', 5], ['proliferative', 6], ['responses', 7], ['to', 8], ['eight', 9], ['synthetic', 10], ['peptides', 11], ['representing', 12], ['amino', 13], ['acid', 14], ['sequences', 15], ['of', 16], ['the', 17], ['VZV', 18], ['IE62', 19], ['protein', 20], ['was', 21], ['assessed', 22], ['using', 23], ['T', 24], ['cells', 25], ['and', 26], ['DC', 27], ['from', 28], ['VZV', 29], ['-', 30], ['susceptible', 31], ['donors', 32], ['.', 33]]\n",
      "[['EVI1'], [28], 'DNA']\n",
      "[['In', 0], ['UCSD', 1], ['-', 2], ['AML1', 3], [',', 4], ['we', 5], ['have', 6], ['also', 7], ['identified', 8], ['fusion', 9], ['transcripts', 10], ['between', 11], ['the', 12], ['ecotropic', 13], ['viral', 14], ['insertion', 15], ['site', 16], ['I', 17], ['(', 18], ['EVI1', 19], [')', 20], ['gene', 21], ['in', 22], ['3q26', 23], ['and', 24], ['GR6', 25], ['and', 26], ['between', 27], ['EVI1', 28], ['and', 29], ['Ribophorin', 30], ['I', 31], ['that', 32], ['maps', 33], ['30', 34], ['kb', 35], ['telomeric', 36], ['to', 37], ['GR6', 38], ['in', 39], ['3q21', 40], ['.', 41]]\n",
      "[['EVI1'], [15], 'DNA']\n",
      "[['All', 0], ['fusions', 1], ['splice', 2], ['the', 3], ['5', 4], ['ends', 5], ['of', 6], ['the', 7], ['3q21', 8], ['genes', 9], ['into', 10], ['exon', 11], ['2', 12], ['of', 13], ['the', 14], ['EVI1', 15], ['gene', 16], [',', 17], ['an', 18], ['event', 19], ['that', 20], ['is', 21], ['similar', 22], ['to', 23], ['the', 24], ['normal', 25], ['intergenic', 26], ['splicing', 27], ['of', 28], ['MDS1', 29], ['-', 30], ['EVI1', 31], ['and', 32], ['to', 33], ['those', 34], ['previously', 35], ['documented', 36], ['in', 37], ['leukemias', 38], ['with', 39], ['t(3;21)', 40], ['and', 41], ['t(3;12)', 42], [',', 43], ['in', 44], ['which', 45], ['acute', 46], ['myelogenous', 47], ['leukemia', 48], ['1', 49], ['-', 50], ['EVI1', 51], ['fusions', 52], ['and', 53], ['ETV6', 54], ['-', 55], ['EVI1', 56], ['fusions', 57], [',', 58], ['respectively', 59], [',', 60], ['occur', 61], ['.', 62]]\n",
      "[['EVI1'], [31], 'DNA']\n",
      "[['All', 0], ['fusions', 1], ['splice', 2], ['the', 3], ['5', 4], ['ends', 5], ['of', 6], ['the', 7], ['3q21', 8], ['genes', 9], ['into', 10], ['exon', 11], ['2', 12], ['of', 13], ['the', 14], ['EVI1', 15], ['gene', 16], [',', 17], ['an', 18], ['event', 19], ['that', 20], ['is', 21], ['similar', 22], ['to', 23], ['the', 24], ['normal', 25], ['intergenic', 26], ['splicing', 27], ['of', 28], ['MDS1', 29], ['-', 30], ['EVI1', 31], ['and', 32], ['to', 33], ['those', 34], ['previously', 35], ['documented', 36], ['in', 37], ['leukemias', 38], ['with', 39], ['t(3;21)', 40], ['and', 41], ['t(3;12)', 42], [',', 43], ['in', 44], ['which', 45], ['acute', 46], ['myelogenous', 47], ['leukemia', 48], ['1', 49], ['-', 50], ['EVI1', 51], ['fusions', 52], ['and', 53], ['ETV6', 54], ['-', 55], ['EVI1', 56], ['fusions', 57], [',', 58], ['respectively', 59], [',', 60], ['occur', 61], ['.', 62]]\n",
      "[['EVI1'], [51], 'DNA']\n",
      "[['All', 0], ['fusions', 1], ['splice', 2], ['the', 3], ['5', 4], ['ends', 5], ['of', 6], ['the', 7], ['3q21', 8], ['genes', 9], ['into', 10], ['exon', 11], ['2', 12], ['of', 13], ['the', 14], ['EVI1', 15], ['gene', 16], [',', 17], ['an', 18], ['event', 19], ['that', 20], ['is', 21], ['similar', 22], ['to', 23], ['the', 24], ['normal', 25], ['intergenic', 26], ['splicing', 27], ['of', 28], ['MDS1', 29], ['-', 30], ['EVI1', 31], ['and', 32], ['to', 33], ['those', 34], ['previously', 35], ['documented', 36], ['in', 37], ['leukemias', 38], ['with', 39], ['t(3;21)', 40], ['and', 41], ['t(3;12)', 42], [',', 43], ['in', 44], ['which', 45], ['acute', 46], ['myelogenous', 47], ['leukemia', 48], ['1', 49], ['-', 50], ['EVI1', 51], ['fusions', 52], ['and', 53], ['ETV6', 54], ['-', 55], ['EVI1', 56], ['fusions', 57], [',', 58], ['respectively', 59], [',', 60], ['occur', 61], ['.', 62]]\n",
      "[['EVI1'], [56], 'DNA']\n",
      "[['All', 0], ['fusions', 1], ['splice', 2], ['the', 3], ['5', 4], ['ends', 5], ['of', 6], ['the', 7], ['3q21', 8], ['genes', 9], ['into', 10], ['exon', 11], ['2', 12], ['of', 13], ['the', 14], ['EVI1', 15], ['gene', 16], [',', 17], ['an', 18], ['event', 19], ['that', 20], ['is', 21], ['similar', 22], ['to', 23], ['the', 24], ['normal', 25], ['intergenic', 26], ['splicing', 27], ['of', 28], ['MDS1', 29], ['-', 30], ['EVI1', 31], ['and', 32], ['to', 33], ['those', 34], ['previously', 35], ['documented', 36], ['in', 37], ['leukemias', 38], ['with', 39], ['t(3;21)', 40], ['and', 41], ['t(3;12)', 42], [',', 43], ['in', 44], ['which', 45], ['acute', 46], ['myelogenous', 47], ['leukemia', 48], ['1', 49], ['-', 50], ['EVI1', 51], ['fusions', 52], ['and', 53], ['ETV6', 54], ['-', 55], ['EVI1', 56], ['fusions', 57], [',', 58], ['respectively', 59], [',', 60], ['occur', 61], ['.', 62]]\n",
      "[['EVI1'], [3], 'DNA']\n",
      "[['The', 0], ['Ribophorin', 1], ['I', 2], ['-', 3], ['EVI1', 4], ['fusion', 5], ['in', 6], ['particular', 7], ['may', 8], ['be', 9], ['a', 10], ['common', 11], ['occurrence', 12], ['in', 13], ['t(3;3)', 14], ['.', 15]]\n",
      "[['bHLH'], [32], 'PROTEIN']\n",
      "[['Cdk2', 0], ['-', 1], ['dependent', 2], ['phosphorylation', 3], ['therefore', 4], ['provides', 5], ['a', 6], ['switch', 7], ['during', 8], ['late', 9], ['G1', 10], ['-', 11], ['to', 12], ['-', 13], ['S', 14], ['phase', 15], ['that', 16], ['both', 17], ['nullifies', 18], ['an', 19], ['early', 20], ['G1', 21], ['cell', 22], ['cycle', 23], ['regulatory', 24], ['function', 25], ['of', 26], ['Id3', 27], ['and', 28], ['modulates', 29], ['its', 30], ['target', 31], ['bHLH', 32], ['specificity', 33], ['.', 34]]\n",
      "[['bHLH'], [26], 'PROTEIN']\n",
      "[['These', 0], ['data', 1], ['also', 2], ['demonstrate', 3], ['that', 4], ['the', 5], ['ability', 6], ['of', 7], ['Id3', 8], ['to', 9], ['promote', 10], ['cell', 11], ['cycle', 12], ['S', 13], ['phase', 14], ['entry', 15], ['is', 16], ['not', 17], ['simply', 18], ['a', 19], ['function', 20], ['of', 21], ['its', 22], ['ability', 23], ['to', 24], ['modulate', 25], ['bHLH', 26], ['heterodimer', 27], ['-', 28], ['dependent', 29], ['gene', 30], ['expression', 31], ['and', 32], ['establish', 33], ['a', 34], ['biologically', 35], ['important', 36], ['mechanism', 37], ['through', 38], ['which', 39], ['Cdk2', 40], ['and', 41], ['Id', 42], ['-', 43], ['bHLH', 44], ['functions', 45], ['are', 46], ['integrated', 47], ['in', 48], ['the', 49], ['coordination', 50], ['of', 51], ['cell', 52], ['proliferation', 53], ['and', 54], ['differentiation', 55], ['.', 56]]\n",
      "[['myeloperoxidase'], [8], 'PROTEIN']\n",
      "[['The', 0], ['highly', 1], ['specific', 2], ['granulomonocyte', 3], ['-', 4], ['associated', 5], ['lysosomal', 6], ['proteins', 7], ['myeloperoxidase', 8], ['(', 9], ['MPO', 10], [')', 11], ['and', 12], ['lysozyme', 13], ['(', 14], ['LZ', 15], [')', 16], [',', 17], ['as', 18], ['well', 19], ['as', 20], ['the', 21], ['transcription', 22], ['factor', 23], ['PU.1', 24], [',', 25], ['were', 26], ['already', 27], ['detectable', 28], ['in', 29], ['the', 30], ['most', 31], ['immature', 32], ['CD34+Thy', 33], ['-', 34], ['1+', 35], ['subset', 36], ['.', 37]]\n",
      "[['CD4+'], [16], 'PROTEIN']\n",
      "[['Addition', 0], ['of', 1], ['neutralizing', 2], ['anti', 3], ['-', 4], ['TNF', 5], ['-', 6], ['alpha', 7], ['antibodies', 8], ['drastically', 9], ['reduced', 10], ['p24', 11], ['antigen', 12], ['release', 13], ['and', 14], ['prevented', 15], ['CD4+', 16], ['cell', 17], ['depletion', 18], ['associated', 19], ['with', 20], ['infection', 21], ['.', 22]]\n",
      "[['hematopoietic', 'precurser'], [12, 13], 'CELL_TYPE']\n",
      "[['These', 0], ['studies', 1], ['demonstrate', 2], ['that', 3], ['E2a', 4], ['-', 5], ['Pbx1', 6], ['initiates', 7], ['programmed', 8], ['cell', 9], ['death', 10], ['of', 11], ['hematopoietic', 12], ['precursers', 13], ['by', 14], ['a', 15], ['mechanism', 16], ['that', 17], ['requires', 18], ['its', 19], ['chimeric', 20], ['transcriptional', 21], ['properties', 22], [',', 23], ['but', 24], [',', 25], ['unlike', 26], ['other', 27], ['nuclear', 28], ['oncoproteins', 29], [',', 30], ['is', 31], ['independent', 32], ['of', 33], ['p53', 34], ['.', 35]]\n",
      "[['topoisomerase'], [10], 'PROTEIN']\n",
      "[['One', 0], ['hour', 1], ['pulse', 2], ['-', 3], ['treatment', 4], ['with', 5], ['3', 6], ['microM', 7], ['etoposide', 8], ['caused', 9], ['topoisomerase', 10], ['associated', 11], [',', 12], ['primary', 13], ['DNA', 14], ['breakage', 15], [',', 16], ['which', 17], ['was', 18], ['rapidly', 19], ['followed', 20], ['by', 21], ['apoptosis', 22], ['.', 23]]\n",
      "[['topoisomerase'], [9], 'PROTEIN']\n",
      "[['When', 0], ['used', 1], ['at', 2], ['differentiation', 3], ['-', 4], ['inducing', 5], ['concentrations', 6], [',', 7], ['both', 8], ['topoisomerase', 9], ['inhibitors', 10], ['caused', 11], ['a', 12], ['great', 13], ['stimulation', 14], ['of', 15], ['AP', 16], ['-', 17], ['1', 18], ['binding', 19], ['activity', 20], [',', 21], ['with', 22], ['maximum', 23], ['value', 24], ['at', 25], ['hour', 26], ['12', 27], ['in', 28], ['etoposide', 29], ['-', 30], ['treated', 31], ['cells', 32], ['and', 33], ['at', 34], ['hour', 35], ['48', 36], ['in', 37], ['ICRF', 38], ['-', 39], ['193', 40], ['-', 41], ['treated', 42], ['cells', 43], ['.', 44]]\n",
      "[['topoisomerase'], [32], 'PROTEIN']\n",
      "[['However', 0], [',', 1], ['there', 2], ['are', 3], ['other', 4], ['effects', 5], [',', 6], ['such', 7], ['as', 8], ['the', 9], ['early', 10], ['activation', 11], ['of', 12], ['protein', 13], ['kinase', 14], ['C', 15], [',', 16], ['which', 17], ['are', 18], ['probably', 19], ['derived', 20], ['from', 21], ['the', 22], ['production', 23], ['of', 24], ['primary', 25], ['DNA', 26], ['breakage', 27], ['by', 28], ['some', 29], ['anti', 30], ['-', 31], ['topoisomerase', 32], ['drugs', 33], ['.', 34]]\n",
      "[['chemoattractant'], [5], 'PROTEIN']\n",
      "[['Stimulation', 0], ['of', 1], ['human', 2], ['neutrophils', 3], ['with', 4], ['chemoattractants', 5], ['FMLP', 6], ['or', 7], ['platelet', 8], ['activating', 9], ['factor', 10], ['(', 11], ['PAF', 12], [')', 13], ['results', 14], ['in', 15], ['different', 16], ['but', 17], ['overlapping', 18], ['functional', 19], ['responses', 20], ['.', 21]]\n",
      "[['chemoattractant'], [10], 'PROTEIN']\n",
      "[['These', 0], ['results', 1], ['demonstrate', 2], ['distinct', 3], ['patterns', 4], ['of', 5], ['intracellular', 6], ['signaling', 7], ['for', 8], ['two', 9], ['chemoattractants', 10], ['and', 11], ['suggest', 12], ['that', 13], ['selective', 14], ['activation', 15], ['of', 16], ['intracellular', 17], ['signaling', 18], ['cascades', 19], ['may', 20], ['underlie', 21], ['different', 22], ['patterns', 23], ['of', 24], ['functional', 25], ['responses', 26], ['.', 27]]\n",
      "[['chemoattractant'], [38], 'PROTEIN']\n",
      "[['In', 0], ['order', 1], ['to', 2], ['elucidate', 3], ['possible', 4], ['mechanisms', 5], ['underlying', 6], ['endothelial', 7], ['cell', 8], ['activation', 9], ['by', 10], ['xenogeneic', 11], ['serum', 12], [',', 13], ['we', 14], ['focussed', 15], ['on', 16], ['transcription', 17], ['factor', 18], ['NF', 19], ['-', 20], ['kappa', 21], ['B', 22], [',', 23], ['a', 24], ['central', 25], ['regulator', 26], ['for', 27], ['the', 28], ['induction', 29], ['of', 30], ['different', 31], ['genes', 32], [',', 33], ['including', 34], ['adhesive', 35], ['molecules', 36], ['and', 37], ['chemoattractants', 38], ['.', 39]]\n",
      "[['CIS'], [36], 'PROTEIN']\n",
      "[['While', 0], ['IL', 1], ['-', 2], ['4', 3], ['does', 4], ['not', 5], ['activate', 6], ['STAT5', 7], [',', 8], ['it', 9], ['induces', 10], ['expression', 11], ['of', 12], ['STAT5', 13], ['-', 14], ['regulated', 15], ['genes', 16], ['in', 17], ['CTLL', 18], ['-', 19], ['2', 20], [',', 21], ['i.e.', 22], ['beta', 23], ['-', 24], ['casein', 25], [',', 26], ['a', 27], ['cytokine', 28], ['-', 29], ['inducible', 30], ['SH2', 31], ['-', 32], ['containing', 33], ['protein', 34], ['(', 35], ['CIS', 36], [')', 37], [',', 38], ['and', 39], ['oncostatin', 40], ['M', 41], ['(', 42], ['OSM', 43], [')', 44], [',', 45], ['suggesting', 46], ['that', 47], ['STAT6', 48], ['activated', 49], ['by', 50], ['IL', 51], ['-', 52], ['4', 53], ['substitutes', 54], ['for', 55], ['the', 56], ['function', 57], ['of', 58], ['STAT5', 59], ['in', 60], ['T', 61], ['cells', 62], ['.', 63]]\n",
      "[['CIS'], [40], 'PROTEIN']\n",
      "[['In', 0], ['apparent', 1], ['contradiction', 2], [',', 3], ['expression', 4], ['of', 5], ['a', 6], ['dominant', 7], ['negative', 8], ['form', 9], ['of', 10], ['Ras', 11], [',', 12], ['RasN17', 13], [',', 14], ['also', 15], ['inhibited', 16], ['IL', 17], ['-', 18], ['2', 19], ['-', 20], ['induced', 21], ['activation', 22], ['of', 23], ['the', 24], ['promoter', 25], ['containing', 26], ['the', 27], ['minimal', 28], ['beta', 29], ['-', 30], ['casein', 31], ['STAT5', 32], ['element', 33], ['as', 34], ['well', 35], ['as', 36], ['the', 37], ['promoters', 38], ['of', 39], ['CIS', 40], ['and', 41], ['OSM', 42], ['.', 43]]\n",
      "[['alphaCD3'], [20], 'PROTEIN']\n",
      "[['MATERIALS', 0], ['AND', 1], ['METHODS', 2], [':', 3], ['Lymphocyte', 4], ['samples', 5], ['were', 6], ['stimulated', 7], ['by', 8], ['either', 9], ['the', 10], ['mitogen', 11], ['phytohemagglutinin', 12], ['(', 13], ['PHA', 14], [')', 15], ['or', 16], ['the', 17], ['monoclonal', 18], ['antibody', 19], ['alphaCD3', 20], ['.', 21]]\n",
      "[['alphaCD3'], [12], 'PROTEIN']\n",
      "[['RESULTS', 0], [':', 1], ['Cord', 2], ['lymphocytes', 3], ['revealed', 4], ['a', 5], ['greater', 6], ['proliferation', 7], ['rate', 8], ['with', 9], ['PHA', 10], ['and', 11], ['alphaCD3', 12], ['than', 13], ['adult', 14], ['lymphocytes', 15], ['(', 16], ['p', 17], ['0.0081', 18], ['and', 19], ['0.0023', 20], [',', 21], ['respectively', 22], [')', 23], ['.', 24]]\n",
      "[['alphaCD3'], [61], 'PROTEIN']\n",
      "[['In', 0], ['addition', 1], [',', 2], ['Northern', 3], ['blot', 4], ['analysis', 5], ['of', 6], ['cord', 7], ['and', 8], ['adult', 9], ['samples', 10], ['revealed', 11], ['similar', 12], ['maximal', 13], ['increases', 14], ['in', 15], ['c', 16], ['-', 17], ['fos', 18], ['(', 19], ['99+/', 20], ['-', 21], ['15', 22], ['and', 23], ['126+/', 24], ['-', 25], ['11', 26], ['%', 27], [',', 28], ['p', 29], ['0.0126', 30], [')', 31], ['and', 32], ['c', 33], ['-', 34], ['jun', 35], ['(', 36], ['123+/', 37], ['-', 38], ['9', 39], ['and', 40], ['185+/', 41], ['-', 42], ['38', 43], ['%', 44], [',', 45], ['p', 46], ['0.0291', 47], [')', 48], ['mRNA', 49], ['expression', 50], [',', 51], ['respectively', 52], [',', 53], ['as', 54], ['early', 55], ['as', 56], ['15', 57], ['min', 58], ['post', 59], ['-', 60], ['alphaCD3', 61], ['stimulation', 62], ['.', 63]]\n",
      "[['autosome'], [13], 'DNA']\n",
      "[['We', 0], ['have', 1], ['analyzed', 2], ['de', 3], ['novo', 4], ['radiation', 5], ['-', 6], ['induced', 7], ['translocations', 8], ['between', 9], ['the', 10], ['Xi', 11], ['and', 12], ['autosomes', 13], ['to', 14], ['study', 15], ['the', 16], ['maintenance', 17], ['and', 18], ['spreading', 19], ['of', 20], ['X', 21], ['-', 22], ['chromosome', 23], ['inactivation', 24], ['(', 25], ['X', 26], ['inactivation', 27], [')', 28], ['in', 29], ['relation', 30], ['to', 31], ['the', 32], ['position', 33], ['of', 34], ['the', 35], ['X', 36], ['-', 37], ['inactivation', 38], ['center', 39], ['(', 40], ['XIC', 41], [')', 42], ['/XIST', 43], ['in', 44], ['differentiated', 45], ['cells', 46], ['.', 47]]\n",
      "[['branchial', 'myoblast'], [29, 30], 'CELL_TYPE']\n",
      "[['Musculin', 0], ['protein', 1], ['is', 2], ['closely', 3], ['related', 4], ['to', 5], ['human', 6], ['ABF', 7], ['-', 8], ['1', 9], [',', 10], ['which', 11], ['is', 12], ['expressed', 13], ['in', 14], ['activated', 15], ['B', 16], ['cells', 17], [',', 18], ['and', 19], ['to', 20], ['epicardin/capsulin/Pod', 21], ['-', 22], ['1', 23], [',', 24], ['which', 25], ['is', 26], ['expressed', 27], ['in', 28], ['branchial', 29], ['myoblasts', 30], [',', 31], ['visceral', 32], ['and', 33], ['urogenital', 34], ['mesoderm', 35], ['and', 36], ['epicardium', 37], ['.', 38]]\n",
      "[['CD4+'], [11], 'PROTEIN']\n",
      "[['In', 0], ['all', 1], ['3', 2], ['cases', 3], [',', 4], ['a', 5], ['characteristic', 6], ['immunophenotype', 7], ['was', 8], ['identified', 9], ['(', 10], ['CD4+', 11], [',', 12], ['CD13', 13], ['-', 14], [',', 15], ['CD33+', 16], [',', 17], ['CD65s+', 18], [')', 19], ['.', 20]]\n",
      "[['CD8+CD56+/CD57+'], [26], 'CELL_TYPE']\n",
      "[['These', 0], ['are', 1], [':', 2], ['helper', 3], ['-', 4], ['inducer', 5], ['(', 6], ['memory', 7], [')', 8], ['T', 9], ['cells', 10], ['(', 11], ['CD4+CD45R0+CD45RA', 12], ['-', 13], ['CD29highCD11a+', 14], [')', 15], [',', 16], ['CD20+', 17], ['B', 18], ['cells', 19], [',', 20], ['and', 21], ['cytotoxic', 22], ['T', 23], ['cells', 24], ['(', 25], ['CD8+CD56+/CD57+', 26], [')', 27], ['.', 28]]\n",
      "[['heterodimeric', 'NFKB2(p49)/RelA(p65)'], [6, 7], 'PROTEIN']\n",
      "[['In', 0], ['contrast', 1], [',', 2], ['the', 3], ['affinity', 4], ['of', 5], ['heterodimeric', 6], ['NFKB2', 7], ['(', 8], ['p49', 9], [')', 10], ['/', 11], ['RelA', 12], ['(', 13], ['p65', 14], [')', 15], ['for', 16], ['this', 17], ['site', 18], ['is', 19], ['approximately', 20], ['6', 21], ['-', 22], ['fold', 23], ['higher', 24], ['(', 25], ['KD', 26], ['11.8', 27], ['pM', 28], [')', 29], ['than', 30], ['that', 31], ['of', 32], ['p49', 33], ['alone', 34], ['.', 35]]\n",
      "[['autoantibody'], [4], 'PROTEIN']\n",
      "[['Target', 0], ['antigens', 1], ['defined', 2], ['by', 3], ['autoantibodies', 4], ['in', 5], ['IDDM', 6], ['include', 7], ['insulin', 8], [',', 9], ['a', 10], ['putative', 11], ['glycolipid', 12], ['that', 13], ['reacts', 14], ['with', 15], ['islet', 16], ['cell', 17], ['antibodies', 18], [',', 19], ['and', 20], ['a', 21], ['64,000', 22], ['-', 23], ['M(r)', 24], ['protein', 25], ['recently', 26], ['identified', 27], ['as', 28], ['glutamic', 29], ['acid', 30], ['decarboxylase', 31], ['.', 32]]\n",
      "[['CD34+/CD38(2+)'], [20], 'CELL_LINE']\n",
      "[['In', 0], ['immature', 1], ['hematopoietic', 2], ['cells', 3], [',', 4], ['tal', 5], ['-', 6], ['1', 7], ['and', 8], ['GATA', 9], ['-', 10], ['1', 11], ['genes', 12], ['are', 13], ['coexpressed', 14], ['in', 15], ['committed', 16], ['progenitors', 17], ['cells', 18], ['(', 19], ['CD34+/CD38(2+)', 20], [')', 21], [',', 22], ['whereas', 23], ['they', 24], ['are', 25], ['not', 26], ['detectable', 27], ['in', 28], ['the', 29], ['most', 30], ['primitive', 31], ['cells', 32], ['(', 33], ['CD34(2+)/CD38', 34], ['-', 35], [')', 36], ['.', 37]]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['myeloblast'], [20], 'CELL_TYPE']\n",
      "[['Egr', 0], ['-', 1], ['1', 2], ['antisense', 3], ['oligomers', 4], ['in', 5], ['the', 6], ['culture', 7], ['media', 8], ['blocked', 9], ['macrophage', 10], ['differentiation', 11], ['in', 12], ['both', 13], ['myeloid', 14], ['leukemia', 15], ['cell', 16], ['lines', 17], ['and', 18], ['normal', 19], ['myeloblasts', 20], ['.', 21]]\n",
      "[['myeloblast'], [16], 'CELL_TYPE']\n",
      "[['These', 0], ['observations', 1], ['indicate', 2], ['that', 3], ['expression', 4], ['of', 5], ['Egr', 6], ['-', 7], ['1', 8], ['is', 9], ['essential', 10], ['for', 11], ['and', 12], ['restricts', 13], ['differentiation', 14], ['of', 15], ['myeloblasts', 16], ['along', 17], ['the', 18], ['macrophage', 19], ['lineage', 20], ['.', 21]]\n",
      "[['PRE'], [5], 'DNA']\n",
      "[['The', 0], ['enhancer', 1], ['activity', 2], ['of', 3], ['the', 4], ['PRE', 5], ['was', 6], ['completely', 7], ['suppressed', 8], ['when', 9], ['the', 10], ['NRE', 11], ['was', 12], ['present', 13], ['.', 14]]\n",
      "[['hepatic', 'microsome'], [0, 1], 'CELL_TYPE']\n",
      "[['Hepatic', 0], ['microsomes', 1], ['and', 2], ['isolated', 3], ['hepatocytes', 4], ['in', 5], ['short', 6], ['term', 7], ['culture', 8], ['desulfate', 9], ['T3', 10], ['sulfate', 11], ['(', 12], ['T3SO4', 13], [')', 14], ['.', 15]]\n",
      "[['hepatic', 'microsome'], [19, 20], 'CELL_TYPE']\n",
      "[['Human', 0], ['fibroblasts', 1], ['contained', 2], ['roughly', 3], ['the', 4], ['same', 5], ['level', 6], ['of', 7], ['microsomal', 8], ['p', 9], ['-', 10], ['nitrophenyl', 11], ['sulfatase', 12], ['activity', 13], ['as', 14], ['that', 15], ['previously', 16], ['observed', 17], ['in', 18], ['hepatic', 19], ['microsomes', 20], ['.', 21]]\n",
      "[['transfectant'], [21], 'CELL_LINE']\n",
      "[['A', 0], ['number', 1], ['of', 2], ['genes', 3], ['known', 4], ['to', 5], ['be', 6], ['induced', 7], ['following', 8], ['T', 9], ['cell', 10], ['activation', 11], ['and', 12], ['associated', 13], ['with', 14], ['cell', 15], ['growth', 16], ['were', 17], ['increased', 18], ['in', 19], ['the', 20], ['transfectants', 21], [',', 22], ['including', 23], ['c', 24], ['-', 25], ['fos', 26], [',', 27], ['c', 28], ['-', 29], ['myc', 30], [',', 31], ['c', 32], ['-', 33], ['myb', 34], [',', 35], ['HLA', 36], ['-', 37], ['DR', 38], [',', 39], ['lck', 40], [',', 41], ['NF', 42], ['-', 43], ['kappa', 44], ['B', 45], [',', 46], ['interleukin', 47], ['-', 48], ['2', 49], ['and', 50], ['interleukin', 51], ['-', 52], ['2', 53], ['receptor', 54], ['alpha', 55], ['(', 56], ['IL', 57], ['-', 58], ['2R', 59], ['alpha', 60], [')', 61], ['.', 62]]\n",
      "[['myeloblast'], [32], 'CELL_LINE']\n",
      "[['Constitutive', 0], ['expression', 1], ['of', 2], ['1,25(OH)2D3', 3], ['receptor', 4], ['RNA', 5], ['was', 6], ['detected', 7], ['in', 8], ['various', 9], ['kinds', 10], ['of', 11], ['hematopoietic', 12], ['cells', 13], [',', 14], ['including', 15], ['macrophages', 16], ['and', 17], ['activated', 18], ['T', 19], ['lymphocytes', 20], [',', 21], ['as', 22], ['well', 23], ['as', 24], ['in', 25], ['cell', 26], ['lines', 27], ['KG', 28], ['-', 29], ['1', 30], ['(', 31], ['myeloblasts', 32], [')', 33], [',', 34], ['HL', 35], ['-', 36], ['60', 37], ['(', 38], ['promyelocytes', 39], [')', 40], [',', 41], ['ML', 42], ['-', 43], ['3', 44], ['(', 45], ['myelomonoblasts', 46], [')', 47], [',', 48], ['U937', 49], [',', 50], ['THP', 51], ['-', 52], ['1', 53], ['(', 54], ['monoblasts', 55], [')', 56], [',', 57], ['K562', 58], ['(', 59], ['erythroblasts', 60], [')', 61], [',', 62], ['and', 63], ['S', 64], ['-', 65], ['LB1', 66], ['(', 67], ['HTLV', 68], ['-', 69], ['1', 70], ['-', 71], ['transfected', 72], ['T', 73], ['lymphocytes', 74], [')', 75], ['.', 76]]\n",
      "[['CD4+'], [27], 'CELL_TYPE']\n",
      "[['1,25(OH)2D3', 0], ['receptors', 1], ['were', 2], ['expressed', 3], ['to', 4], ['a', 5], ['greater', 6], ['extent', 7], ['on', 8], ['CD8+', 9], ['T', 10], ['-', 11], ['lymphocytes', 12], ['than', 13], ['on', 14], ['CD4+', 15], ['T', 16], ['-', 17], ['lymphocytes', 18], ['in', 19], ['sarcoidosis', 20], [',', 21], ['whereas', 22], ['a', 23], ['greater', 24], ['proportion', 25], ['of', 26], ['CD4+', 27], ['than', 28], ['of', 29], ['CD8+', 30], ['T', 31], ['-', 32], ['lymphocytes', 33], ['from', 34], ['patients', 35], ['with', 36], ['tuberculosis', 37], ['were', 38], ['receptor', 39], ['-', 40], ['positive', 41], ['.', 42]]\n",
      "[['hematologic', 'neoplasm'], [34, 35], 'CELL_TYPE']\n",
      "[['The', 0], ['TCF', 1], ['-', 2], ['1', 3], ['protein', 4], ['was', 5], ['detectable', 6], ['in', 7], ['all', 8], ['samples', 9], ['of', 10], ['a', 11], ['set', 12], ['of', 13], ['22', 14], ['T', 15], ['-', 16], ['cell', 17], ['malignancies', 18], ['of', 19], ['various', 20], ['stages', 21], ['of', 22], ['maturation', 23], [',', 24], ['but', 25], ['was', 26], ['absent', 27], ['from', 28], ['a', 29], ['large', 30], ['number', 31], ['of', 32], ['other', 33], ['hematologic', 34], ['neoplasms', 35], ['.', 36]]\n",
      "[['nonpilus', 'adhesin'], [10, 11], 'PROTEIN']\n",
      "[['The', 0], ['RpoN', 1], ['mutant', 2], [',', 3], ['lacking', 4], ['pili', 5], [',', 6], ['flagella', 7], [',', 8], ['and', 9], ['nonpilus', 10], ['adhesins', 11], [',', 12], ['bound', 13], ['poorly', 14], ['and', 15], ['was', 16], ['resistant', 17], ['to', 18], ['ingestion', 19], ['by', 20], ['both', 21], ['macrophages', 22], ['and', 23], ['neutrophils', 24], ['.', 25]]\n",
      "[['autoantibody'], [15], 'PROTEIN']\n",
      "[['Several', 0], ['of', 1], ['the', 2], ['heterogeneous', 3], ['clinical', 4], ['manifestations', 5], ['of', 6], ['systemic', 7], ['lupus', 8], ['erythematosus', 9], ['have', 10], ['been', 11], ['associated', 12], ['with', 13], ['specific', 14], ['autoantibodies', 15], ['.', 16]]\n",
      "[['autoantibody'], [7], 'PROTEIN']\n",
      "[['Associations', 0], ['between', 1], ['HLA', 2], ['class', 3], ['II', 4], ['antigens', 5], ['and', 6], ['autoantibodies', 7], ['to', 8], ['the', 9], ['ribonucleoproteins', 10], ['Ro(SSA)', 11], ['and', 12], ['La(SSB)', 13], ['have', 14], ['been', 15], ['reported', 16], ['in', 17], ['these', 18], ['patients', 19], ['.', 20]]\n",
      "[['transcriptional', \"'silencer'\"], [6, 7], 'DNA']\n",
      "[['The', 0], ['element', 1], ['has', 2], ['characteristics', 3], ['of', 4], ['a', 5], ['transcriptional', 6], ['silencer', 7], ['since', 8], ['it', 9], ['functions', 10], ['in', 11], ['both', 12], ['orientations', 13], ['.', 14]]\n",
      "[['HLH'], [6], 'PROTEIN']\n",
      "[['Helix', 0], ['-', 1], ['loop', 2], ['-', 3], ['helix', 4], ['(', 5], ['HLH', 6], [')', 7], ['transcription', 8], ['factors', 9], ['are', 10], ['involved', 11], ['in', 12], ['cellular', 13], ['growth', 14], ['and', 15], ['differentiation', 16], ['.', 17]]\n"
     ]
    }
   ],
   "source": [
    "lista_postaggers_entidades = []\n",
    "for key, value in dicSentences.items():\n",
    "    entidades = value[1]\n",
    "    #print(entidades)\n",
    "    for entidade in entidades:\n",
    "        pos_tagger=tipoPostaggerTokens(entidade[0], dicPosTagger)\n",
    "        if pos_tagger=='-JJ':\n",
    "            print(entidade)\n",
    "            print(value[0])\n",
    "        if pos_tagger not in lista_postaggers_entidades:\n",
    "            lista_postaggers_entidades.append(pos_tagger)\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 719,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['-NN-NN-CD-NN',\n",
       " '-NN-NN-NN-NN',\n",
       " '-NN',\n",
       " '-CD-NN-NN',\n",
       " '-NN-NN-NN',\n",
       " '-NN-NN-CD',\n",
       " '-JJ-NN-NNS',\n",
       " '-NN-NN-JJ-NN',\n",
       " '-NN-NN',\n",
       " '-NN-NN-NN-NN-NN']"
      ]
     },
     "execution_count": 719,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lista_postaggers_entidades[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 725,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "True\n",
      "True\n",
      "True\n",
      "False\n",
      "False\n",
      "False\n",
      "False\n",
      "True\n"
     ]
    }
   ],
   "source": [
    "print('-CD' in lista_postaggers_entidades)\n",
    "print('-CC' in lista_postaggers_entidades) # -> só tem 2, um é em maiusculo, excluir\n",
    "print('-JJ' in lista_postaggers_entidades)\n",
    "print('-VBN' in lista_postaggers_entidades)\n",
    "print('-IN' in lista_postaggers_entidades)\n",
    "print('-TO' in lista_postaggers_entidades)\n",
    "print('-VBZ' in lista_postaggers_entidades)\n",
    "print('-NNS' in lista_postaggers_entidades)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 833,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "174085\n"
     ]
    }
   ],
   "source": [
    "print(len(pulando_termos_postagger))\n",
    "#print(pulando_termos_postagger)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 834,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[['CD28', 'requires'], '-NN-VBZ'],\n",
       " [['5', '-', 'lipoxygenase', '.'], '-CD-NN-NN-.'],\n",
       " [['-', 'lipoxygenase', '.'], '-NN-NN-.'],\n",
       " [['lipoxygenase', '.'], '-NN-.'],\n",
       " [['CD28', 'surface', 'receptor', 'provides'], '-NN-NN-NN-VBZ'],\n",
       " [['surface', 'receptor', 'provides'], '-NN-NN-VBZ'],\n",
       " [['receptor', 'provides'], '-NN-VBZ'],\n",
       " [['interleukin', '-', '2', '('], '-NN-NN-CD-('],\n",
       " [['-', '2', '('], '-NN-CD-('],\n",
       " [['2', '('], '-CD-(']]"
      ]
     },
     "execution_count": 834,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pulando_termos_postagger[:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 835,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['IL - 2 gene expression and NF - kappa B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "  [['IL - <e1> 2 gene </e1> expression and NF - kappa B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['IL - 2 gene expression and NF - kappa <e1> B </e1> activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['IL - 2 <e1> gene </e1> expression and NF - kappa B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['IL - 2 gene expression and NF - kappa B activation through <e1> CD28 </e1> requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'PROTEIN'],\n",
       "   ['IL - 2 gene expression and <e1> NF </e1> - kappa B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['IL - 2 gene expression and NF - <e1> kappa B </e1> activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['IL - 2 gene expression and NF - kappa B activation through CD28 requires reactive oxygen production by <e1> 5 - lipoxygenase </e1> .',\n",
       "    'PROTEIN'],\n",
       "   ['<e1> IL </e1> - 2 gene expression and NF - kappa B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['IL - 2 gene expression and <e1> NF - kappa B </e1> activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'PROTEIN'],\n",
       "   ['<e1> IL - 2 gene </e1> expression and NF - kappa B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'DNA'],\n",
       "   ['IL - 2 gene expression and <e1> NF - kappa </e1> B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['<e1> IL - 2 </e1> gene expression and NF - kappa B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O'],\n",
       "   ['IL - 2 gene expression and NF - <e1> kappa </e1> B activation through CD28 requires reactive oxygen production by 5 - lipoxygenase .',\n",
       "    'O']]]]"
      ]
     },
     "execution_count": 835,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combinacaoEntidadesAll[:1]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Agora, tratar descontinuas"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 836,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['This delay is associated with down - regulation of many erythroid cell - specific genes , including alpha - globin, band 3 , band 4.1 , and the erythroid cell - specific histone H5 .',\n",
       "  [['This delay is associated with down - regulation of many erythroid cell - specific genes , including <e1> alpha - globin </e1> , band 3 , band 4.1 , and the erythroid cell - specific histone H5 .',\n",
       "    'DNA']]],\n",
       " ['Transient transfection experiments show that several elements in the promoter - proximal region of the IL - 2R alpha gene contribute to IL - 1 responsiveness , most importantly an NF - kappa B site conserved in the human gene .',\n",
       "  [['Transient transfection experiments show that several elements in the promoter - proximal region of the IL - 2R alpha gene contribute to IL - 1 responsiveness , most importantly an NF - kappa B site conserved in the <e1> human gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Finally , the status of our current knowledge concerning the roles of transcription factors in the commitment to erythroid cell typeis summarized .',\n",
       "  [['Finally , the status of our current knowledge concerning the roles of transcription factors in the commitment to <e1> erythroid cell type </e1> is summarized .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Finally , the status of our current knowledge concerning the roles of transcription factors in the commitment to erythroid , myeloid cell typeis summarized .',\n",
       "  [['Finally , the status of our current knowledge concerning the roles of transcription factors in the commitment to erythroid , <e1> myeloid cell type </e1> is summarized .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The results suggest that activation of protein kinase C , but not new protein synthesis , is required for IL - 2 induction of ifn - gamma cytoplasmic mrna .',\n",
       "  [['The results suggest that activation of protein kinase C , but not new protein synthesis , is required for IL - 2 induction of <e1> ifn - gamma cytoplasmic mrna </e1> .',\n",
       "    'RNA']]],\n",
       " ['One substrate is p95vav , which is expressed exclusively in hematopoietic cells .',\n",
       "  [['One substrate is p95vav , which is expressed exclusively in <e1> hematopoietic cells </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The granulocyte - macrophage colony - stimulating factor ( GM - CSF ) gene promoter contains a consensus sequence for the polyomavirus enhancer binding - protein 2 ( PEBP2 ) transcription factor , which consists of alpha subunits .',\n",
       "  [['The granulocyte - macrophage colony - stimulating factor ( GM - CSF ) gene promoter contains a consensus sequence for the polyomavirus enhancer binding - protein 2 ( PEBP2 ) transcription factor , which consists of <e1> alpha subunits </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['PEBP2 alpha A1 and alpha B1 enhanced the expression of the GM - CSF promoter - driven reporter plasmid in unstimulated human jurkat t cell .',\n",
       "  [['PEBP2 alpha A1 and alpha B1 enhanced the expression of the GM - CSF promoter - driven reporter plasmid in <e1> unstimulated human jurkat t cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In both transformed lymphocytes, OKA stimulation induced intense phosphorylation of the constitutively expressed Sp1 protein in the nucleus , a property of OKA not shared by TNF , phorbol ester , or PHA and interleukin 2 .',\n",
       "  [['In both <e1> transformed lymphocytes </e1> , OKA stimulation induced intense phosphorylation of the constitutively expressed Sp1 protein in the nucleus , a property of OKA not shared by TNF , phorbol ester , or PHA and interleukin 2 .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The gata - 1 binding sitesin this promoter lacking a TATA sequence were necessary for a high level of transcription activation .',\n",
       "  [['The <e1> gata - 1 binding sites </e1> in this promoter lacking a TATA sequence were necessary for a high level of transcription activation .',\n",
       "    'DNA']]],\n",
       " ['Protein - DNA binding studies suggested that Sp1 and two other CCGCCC binding proteins from erythroid cellscould bind to the Sp1 binding motif .',\n",
       "  [['Protein - DNA binding studies suggested that Sp1 and two other CCGCCC binding proteins from <e1> erythroid cells </e1> could bind to the Sp1 binding motif .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The interleukin - 5 /receptor interaction activates lyn tyrosine kinaseand the Jak - STAT pathways in eosinophils .',\n",
       "  [['The interleukin - 5 /receptor interaction activates <e1> lyn tyrosine kinase </e1> and the Jak - STAT pathways in eosinophils .',\n",
       "    'PROTEIN']]],\n",
       " ['The interleukin - 5 /receptor interaction activates Lyn and jak2 tyrosine kinaseand the Jak - STAT pathways in eosinophils .',\n",
       "  [['The interleukin - 5 /receptor interaction activates Lyn and <e1> jak2 tyrosine kinase </e1> and the Jak - STAT pathways in eosinophils .',\n",
       "    'PROTEIN']]],\n",
       " ['It is a potent transcriptional activator of many viral promoter .',\n",
       "  [['It is a potent transcriptional activator of many <e1> viral promoter </e1> .',\n",
       "    'DNA']]],\n",
       " ['We have recently demonstrated that stimulation of human t cellwith IL - 12 induces tyrosine phosphorylation of the Janus family tyrosine kinase JAK2 and Tyk2 , implicating these kinases in the immediate biochemical response to IL - 12 .',\n",
       "  [['We have recently demonstrated that stimulation of <e1> human t cell </e1> with IL - 12 induces tyrosine phosphorylation of the Janus family tyrosine kinase JAK2 and Tyk2 , implicating these kinases in the immediate biochemical response to IL - 12 .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We have recently demonstrated that stimulation of human natural killer cellwith IL - 12 induces tyrosine phosphorylation of the Janus family tyrosine kinase JAK2 and Tyk2 , implicating these kinases in the immediate biochemical response to IL - 12 .',\n",
       "  [['We have recently demonstrated that stimulation of <e1> human natural killer cell </e1> with IL - 12 induces tyrosine phosphorylation of the Janus family tyrosine kinase JAK2 and Tyk2 , implicating these kinases in the immediate biochemical response to IL - 12 .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In the human monocytic cell line THP - 1 , either integrin - dependent cell adhesion to fibronectin or ligation of beta 1 integrins with antibodies causes a rapid and intense tyrosine phosphorylation of two sets of proteins of about 65 - 75 kda .',\n",
       "  [['In the human monocytic cell line THP - 1 , either integrin - dependent cell adhesion to fibronectin or ligation of beta 1 integrins with antibodies causes a rapid and intense tyrosine phosphorylation of two sets of proteins of about <e1> 65 - 75 kda </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['With the recent characterization of the fya allele, these findings provide the molecular basis of the Duffy blood group system and an explanation for the erythroid - specific repression of the DARC gene in Duffy - negative individuals .',\n",
       "  [['With the recent characterization of the <e1> fya allele </e1> , these findings provide the molecular basis of the Duffy blood group system and an explanation for the erythroid - specific repression of the DARC gene in Duffy - negative individuals .',\n",
       "    'DNA']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 4, - 5 and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ <e1> human interleukin - 4 </e1> , - 5 and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 5and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ <e1> human interleukin - 5 </e1> and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 13and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ <e1> human interleukin - 13 </e1> and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 2 , - 4 , - 5 and - 13 and human granulocyte/macrophage - colony stimulating factor( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 2 , - 4 , - 5 and - 13 and <e1> human granulocyte/macrophage - colony stimulating factor </e1> ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 4, - 5 and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ <e1> human interleukin - 4 </e1> , - 5 and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 5and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ <e1> human interleukin - 5 </e1> and - 13 and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 13and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ <e1> human interleukin - 13 </e1> and human and mouse granulocyte/macrophage - colony stimulating factor ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 2 , - 4 , - 5 and - 13 and human granulocyte/macrophage - colony stimulating factor( GM - CSF ) ] .',\n",
       "  [['We have recently found a novel conserved motif in the promoters of several T - cell - expressed cytokines [ human interleukin - 2 , - 4 , - 5 and - 13 and <e1> human granulocyte/macrophage - colony stimulating factor </e1> ( GM - CSF ) ] .',\n",
       "    'PROTEIN']]],\n",
       " ['Our studies demonstrate that incubation of human T cells with low concentrations ( approximately 1 - 5 nM ) of calyculin A or okadaic acid , potent inhibitors of protein phosphatase type 1 pp - 1) and type 2A ( PP - 2A ) , induces the phosphorylation of I kappa B alpha even in the absence of any cellular stimulus .',\n",
       "  [['Our studies demonstrate that incubation of human T cells with low concentrations ( approximately 1 - 5 nM ) of calyculin A or okadaic acid , potent inhibitors of <e1> protein phosphatase type 1 pp - 1 </e1> ) and type 2A ( PP - 2A ) , induces the phosphorylation of I kappa B alpha even in the absence of any cellular stimulus .',\n",
       "    'PROTEIN']]],\n",
       " ['Our studies demonstrate that incubation of human T cells with low concentrations ( approximately 1 - 5 nM ) of calyculin A or okadaic acid , potent inhibitors of protein phosphatase type 2a( PP - 2A ) , induces the phosphorylation of I kappa B alpha even in the absence of any cellular stimulus .',\n",
       "  [['Our studies demonstrate that incubation of human T cells with low concentrations ( approximately 1 - 5 nM ) of calyculin A or okadaic acid , potent inhibitors of <e1> protein phosphatase type 2a </e1> ( PP - 2A ) , induces the phosphorylation of I kappa B alpha even in the absence of any cellular stimulus .',\n",
       "    'PROTEIN']]],\n",
       " ['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including hl - 60 cell .',\n",
       "  [['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including <e1> hl - 60 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , u937 cell .',\n",
       "  [['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , <e1> u937 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , U937 , hel/s cell .',\n",
       "  [['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , U937 , <e1> hel/s cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , U937 , HEL/S , ku812f cell .',\n",
       "  [['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , U937 , HEL/S , <e1> ku812f cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , U937 , HEL/S , KU812F , k562 cell .',\n",
       "  [['In the present experiment , we examined the effect of nm23 proteins on various differentiation induction systems of human leukemic cells including HL - 60 , U937 , HEL/S , KU812F , <e1> k562 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of hel cell, but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of <e1> hel cell </e1> , but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , ku812 cell, but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , <e1> ku812 cell </e1> , but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of hl - 60 cell .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of <e1> hl - 60 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of HL - 60 , u937 cell .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of HL - 60 , <e1> u937 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of hel cell, but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of <e1> hel cell </e1> , but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , ku812 cell, but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , <e1> ku812 cell </e1> , but not the induction of monocytic or granulocytic differentiation of HL - 60 , U937 and HEL/S cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of hl - 60 cell .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of <e1> hl - 60 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of HL - 60 , u937 cell .',\n",
       "  [['Native human erythrocyte NDP kinase protein inhibited the induction of erythroid differentiation of HEL , KU812 and K562 cells , but not the induction of monocytic or granulocytic differentiation of HL - 60 , <e1> u937 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - h1 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant human nm23 - h1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human - h2 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant human - h2 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant mouse nm23 - m1 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant mouse nm23 - m1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant mouse nm23 - m2 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant mouse nm23 - m2 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - h2, mouse nm23 - M1 , and - M2 proteins .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by recombinant <e1> human nm23 - h2 </e1> , mouse nm23 - M1 , and - M2 proteins .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - H1 , - H2 , mouse nm23 - m2proteins .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - H1 , - H2 , <e1> mouse nm23 - m2 </e1> proteins .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - h1 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant human nm23 - h1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human - h2 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant human - h2 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant mouse nm23 - m1 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant mouse nm23 - m1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant mouse nm23 - m2 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant mouse nm23 - m2 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - h2, mouse nm23 - M1 , and - M2 proteins .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by recombinant <e1> human nm23 - h2 </e1> , mouse nm23 - M1 , and - M2 proteins .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - H1 , - H2 , mouse nm23 - m2proteins .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - H1 , - H2 , <e1> mouse nm23 - m2 </e1> proteins .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - h1 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant human nm23 - h1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human - h2 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant human - h2 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant mouse nm23 - m1 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant mouse nm23 - m1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant mouse nm23 - m2 protein .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by <e1> recombinant mouse nm23 - m2 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - h2, mouse nm23 - M1 , and - M2 proteins .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by recombinant <e1> human nm23 - h2 </e1> , mouse nm23 - M1 , and - M2 proteins .',\n",
       "    'PROTEIN']]],\n",
       " ['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - H1 , - H2 , mouse nm23 - m2proteins .',\n",
       "  [['The erythroid differentiation of HEL cells was inhibited by recombinant human nm23 - H1 , - H2 , <e1> mouse nm23 - m2 </e1> proteins .',\n",
       "    'PROTEIN']]],\n",
       " ['Antigen complexed with major histocompatibility complex class i moleculeon the surface of antigen presenting cells interacts with the T cell receptor ( TCR ) on the surface of T cells and initiates an activation cascade .',\n",
       "  [['Antigen complexed with <e1> major histocompatibility complex class i molecule </e1> on the surface of antigen presenting cells interacts with the T cell receptor ( TCR ) on the surface of T cells and initiates an activation cascade .',\n",
       "    'PROTEIN']]],\n",
       " ['Antigen complexed with major histocompatibility complex class ii moleculeon the surface of antigen presenting cells interacts with the T cell receptor ( TCR ) on the surface of T cells and initiates an activation cascade .',\n",
       "  [['Antigen complexed with <e1> major histocompatibility complex class ii molecule </e1> on the surface of antigen presenting cells interacts with the T cell receptor ( TCR ) on the surface of T cells and initiates an activation cascade .',\n",
       "    'PROTEIN']]],\n",
       " ['DNase I footprinting and sequence analysis revealed the existence of multiple ap2 binding sitein the 5 - untranslated and proximal coding regions .',\n",
       "  [['DNase I footprinting and sequence analysis revealed the existence of multiple <e1> ap2 binding site </e1> in the 5 - untranslated and proximal coding regions .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting and sequence analysis revealed the existence of multiple AP2 and Sp1 binding sites in the 5 - untranslated region .',\n",
       "  [['DNase I footprinting and sequence analysis revealed the existence of multiple AP2 and Sp1 binding sites in the <e1> 5 - untranslated region </e1> .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting and sequence analysis revealed the existence of multiple ap2 binding sitein the 5 - untranslated and proximal coding regions .',\n",
       "  [['DNase I footprinting and sequence analysis revealed the existence of multiple <e1> ap2 binding site </e1> in the 5 - untranslated and proximal coding regions .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting and sequence analysis revealed the existence of multiple AP2 and Sp1 binding sites in the 5 - untranslated region .',\n",
       "  [['DNase I footprinting and sequence analysis revealed the existence of multiple AP2 and Sp1 binding sites in the <e1> 5 - untranslated region </e1> .',\n",
       "    'DNA']]],\n",
       " ['Synergy between signal transduction pathways is obligatory for expression of c - fos in b cell line: implication for c - fos control via surface immunoglobulin and T cell antigen receptors .',\n",
       "  [['Synergy between signal transduction pathways is obligatory for expression of c - fos in <e1> b cell line </e1> : implication for c - fos control via surface immunoglobulin and T cell antigen receptors .',\n",
       "    'CELL_LINE']]],\n",
       " ['These results suggest that cell - specific patterns of synergy are an essential feature for c - fos induction and may be relevant for c - fos control through b cell antigen receptor .',\n",
       "  [['These results suggest that cell - specific patterns of synergy are an essential feature for c - fos induction and may be relevant for c - fos control through <e1> b cell antigen receptor </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['In this report , we show that in monocytes and T cells IL - 10 stimulates tyrosine phosphorylation of the signal transducers and activators of transcription , STAT1 alpha and STAT3 , in a differential manner such that the relative formation of homo - dimervaries between the two cell types .',\n",
       "  [['In this report , we show that in monocytes and T cells IL - 10 stimulates tyrosine phosphorylation of the signal transducers and activators of transcription , STAT1 alpha and STAT3 , in a differential manner such that the relative formation of <e1> homo - dimer </e1> varies between the two cell types .',\n",
       "    'PROTEIN']]],\n",
       " ['Chemical cross - linking with 125I - labeled TGF - beta 1 showed an essentially normal TGF - beta receptor profile in ebv - positive burkitt s lymphoma cell line, and these receptors were shown to be functional in transducing signals , as evidenced by the TGF - beta 1 - mediated modulation of junB gene expression .',\n",
       "  [['Chemical cross - linking with 125I - labeled TGF - beta 1 showed an essentially normal TGF - beta receptor profile in <e1> ebv - positive burkitt s lymphoma cell line </e1> , and these receptors were shown to be functional in transducing signals , as evidenced by the TGF - beta 1 - mediated modulation of junB gene expression .',\n",
       "    'CELL_LINE']]],\n",
       " ['Chemical cross - linking with 125I - labeled TGF - beta 1 showed an essentially normal TGF - beta receptor profile in ebv - negative burkitt s lymphoma cell line, and these receptors were shown to be functional in transducing signals , as evidenced by the TGF - beta 1 - mediated modulation of junB gene expression .',\n",
       "  [['Chemical cross - linking with 125I - labeled TGF - beta 1 showed an essentially normal TGF - beta receptor profile in <e1> ebv - negative burkitt s lymphoma cell line </e1> , and these receptors were shown to be functional in transducing signals , as evidenced by the TGF - beta 1 - mediated modulation of junB gene expression .',\n",
       "    'CELL_LINE']]],\n",
       " ['By immunization with a recombinant fusion protein , we show that TCF - 1 knock - out mice readily yield antiserum titers against human tcf - 1 protein .',\n",
       "  [['By immunization with a recombinant fusion protein , we show that TCF - 1 knock - out mice readily yield antiserum titers against <e1> human tcf - 1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Electromobility shift assays utilizing this sequence demonstrated binding of a fetal - specific protein .',\n",
       "  [['Electromobility shift assays utilizing this sequence demonstrated binding of a <e1> fetal - specific protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Alanine substitutions introduced at two serine residues positioned within this N - terminal regulatory region of I kappa B alpha also yielded constitutive repressors that escaped from Tax - induced turnover and that potently inhibited immune activation pathways for NF - kappa B induction , including those initiated from antigen receptor .',\n",
       "  [['Alanine substitutions introduced at two serine residues positioned within this N - terminal regulatory region of I kappa B alpha also yielded constitutive repressors that escaped from Tax - induced turnover and that potently inhibited immune activation pathways for NF - kappa B induction , including those initiated from <e1> antigen receptor </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The expression of several cytoskeleton proteinincreased as determined by immunostaining on slides and by ELISA procedures .',\n",
       "  [['The expression of several <e1> cytoskeleton protein </e1> increased as determined by immunostaining on slides and by ELISA procedures .',\n",
       "    'PROTEIN']]],\n",
       " ['The interaction between lymphoid cellin growth and development is described .',\n",
       "  [['The interaction between <e1> lymphoid cell </e1> in growth and development is described .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We have previously described a mutant B lymphoblastoid cell line , Clone - 13 , that expresses HLA - DQ in the absence of hla - dp .',\n",
       "  [['We have previously described a mutant B lymphoblastoid cell line , Clone - 13 , that expresses HLA - DQ in the absence of <e1> hla - dp </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Indeed , transient transfection of hla - dra reporter constructindicated that the affected factor operates via cis - elements located between - 141 base pairs and the transcription initiation site .',\n",
       "  [['Indeed , transient transfection of <e1> hla - dra reporter construct </e1> indicated that the affected factor operates via cis - elements located between - 141 base pairs and the transcription initiation site .',\n",
       "    'DNA']]],\n",
       " ['Substitution promoters were then generated where the entire X - box , or only the x1 - boxesof HLA - DRA were replaced with the analogous regions of HLA - DQB .',\n",
       "  [['Substitution promoters were then generated where the entire X - box , or only the <e1> x1 - boxes </e1> of HLA - DRA were replaced with the analogous regions of HLA - DQB .',\n",
       "    'DNA']]],\n",
       " ['PU.1 expression is maintained at stable levels during differentiation of myeloid cell lines U937 and HL - 60 to monocytic cell .',\n",
       "  [['PU.1 expression is maintained at stable levels during differentiation of myeloid cell lines U937 and HL - 60 to <e1> monocytic cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Analysis of the role of protein kinase c - epsilon, and - zeta in T cell activation .',\n",
       "  [['Analysis of the role of <e1> protein kinase c - epsilon </e1> , and - zeta in T cell activation .',\n",
       "    'PROTEIN']]],\n",
       " ['Analysis of the role of protein kinase c - zetain T cell activation .',\n",
       "  [['Analysis of the role of <e1> protein kinase c - zeta </e1> in T cell activation .',\n",
       "    'PROTEIN']]],\n",
       " ['To address this issue , mutationally active pkc - epsilon, or - zeta have been transfected into T cells and the consequences for T cell activation determined .',\n",
       "  [['To address this issue , mutationally active <e1> pkc - epsilon </e1> , or - zeta have been transfected into T cells and the consequences for T cell activation determined .',\n",
       "    'PROTEIN']]],\n",
       " ['To address this issue , mutationally active pkc - zetahave been transfected into T cells and the consequences for T cell activation determined .',\n",
       "  [['To address this issue , mutationally active <e1> pkc - zeta </e1> have been transfected into T cells and the consequences for T cell activation determined .',\n",
       "    'PROTEIN']]],\n",
       " ['Infection and replication of Tat - human immunodeficiency viruses : genetic analyses of LTR and tat mutations in primary human lymphoid cell .',\n",
       "  [['Infection and replication of Tat - human immunodeficiency viruses : genetic analyses of LTR and tat mutations in <e1> primary human lymphoid cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying wild - type ltrwith modifications in the NF - kappa B and/or Sp1 binding sites .',\n",
       "  [['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying <e1> wild - type ltr </e1> with modifications in the NF - kappa B and/or Sp1 binding sites .',\n",
       "    'DNA']]],\n",
       " ['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying wild - type or mutant LTRs with modifications in the nf - kappa b binding site .',\n",
       "  [['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying wild - type or mutant LTRs with modifications in the <e1> nf - kappa b binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying wild - type ltrwith modifications in the NF - kappa B and/or Sp1 binding sites .',\n",
       "  [['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying <e1> wild - type ltr </e1> with modifications in the NF - kappa B and/or Sp1 binding sites .',\n",
       "    'DNA']]],\n",
       " ['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying wild - type or mutant LTRs with modifications in the nf - kappa b binding site .',\n",
       "  [['To test these hypotheses , we studied several tat mutants , including two stop codon mutants and one deletion mutant using replication - competent HIV - 1 constructs carrying wild - type or mutant LTRs with modifications in the <e1> nf - kappa b binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( ebna 2, 3A , 3B , 3C , - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "  [['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( <e1> ebna 2 </e1> , 3A , 3B , 3C , - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "    'PROTEIN']]],\n",
       " ['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( ebna 3a, 3B , 3C , - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "  [['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( <e1> ebna 3a </e1> , 3B , 3C , - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "    'PROTEIN']]],\n",
       " ['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( ebna 3b, 3C , - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "  [['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( <e1> ebna 3b </e1> , 3C , - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "    'PROTEIN']]],\n",
       " ['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( ebna 3c, - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "  [['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( <e1> ebna 3c </e1> , - LP ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "    'PROTEIN']]],\n",
       " ['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( ebna - lp) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "  [['In lymphoblastoid cell lines , six EBV - encoded nuclear antigens ( <e1> ebna - lp </e1> ) , three latent membrane proteins ( LMP1 , 2A , 2B ) , and two nuclear RNAs ( EBERs ) are expressed .',\n",
       "    'PROTEIN']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the dr beta gene, always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the <e1> dr beta gene </e1> , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the DR beta , dp alpha gene, always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the DR beta , <e1> dp alpha gene </e1> , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the DR beta , dp - beta gene, always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the DR beta , <e1> dp - beta gene </e1> , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the DR beta , dp - beta, and DQ alpha genes , always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the DR beta , <e1> dp - beta </e1> , and DQ alpha genes , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the dr beta gene, always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the <e1> dr beta gene </e1> , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the DR beta , dp alpha gene, always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the DR beta , <e1> dp alpha gene </e1> , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the DR beta , dp - beta gene, always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the DR beta , <e1> dp - beta gene </e1> , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['Significant sequence homology of this element was found in the DNA of the DR beta , dp - beta, and DQ alpha genes , always located downstream of the transcriptional start site .',\n",
       "  [['Significant sequence homology of this element was found in the DNA of the DR beta , <e1> dp - beta </e1> , and DQ alpha genes , always located downstream of the transcriptional start site .',\n",
       "    'DNA']]],\n",
       " ['The nuclear factor binds to the dr alpha elementbut not to the element in the DQ alpha gene .',\n",
       "  [['The nuclear factor binds to the <e1> dr alpha element </e1> but not to the element in the DQ alpha gene .',\n",
       "    'DNA']]],\n",
       " ['Transcriptional activity of core binding factor - alpha aml1) and beta subunits on murine leukemia virus enhancer cores .',\n",
       "  [['Transcriptional activity of <e1> core binding factor - alpha aml1 </e1> ) and beta subunits on murine leukemia virus enhancer cores .',\n",
       "    'PROTEIN']]],\n",
       " ['Transcriptional activity of core binding factor - betasubunits on murine leukemia virus enhancer cores .',\n",
       "  [['Transcriptional activity of <e1> core binding factor - beta </e1> subunits on murine leukemia virus enhancer cores .',\n",
       "    'PROTEIN']]],\n",
       " ['DNA - swap experiments using chloramphenicol acetyl - transferase assay demonstrated that androgen can suppress the transcriptional activities of SV40 early promoter via the interaction between this TR2RE - SV40 and the chimeric receptor AR/TR2/AR with the DNA - binding domain of the TR2 orphan receptor flanked by the n - terminal domainof the androgen receptor .',\n",
       "  [['DNA - swap experiments using chloramphenicol acetyl - transferase assay demonstrated that androgen can suppress the transcriptional activities of SV40 early promoter via the interaction between this TR2RE - SV40 and the chimeric receptor AR/TR2/AR with the DNA - binding domain of the TR2 orphan receptor flanked by the <e1> n - terminal domain </e1> of the androgen receptor .',\n",
       "    'PROTEIN']]],\n",
       " ['In addition , this TR2RE - SV40 can function as a repressor to suppress the transcriptional activities of both sv40 early promoter .',\n",
       "  [['In addition , this TR2RE - SV40 can function as a repressor to suppress the transcriptional activities of both <e1> sv40 early promoter </e1> .',\n",
       "    'DNA']]],\n",
       " ['In addition , this TR2RE - SV40 can function as a repressor to suppress the transcriptional activities of both sv40 late promoter .',\n",
       "  [['In addition , this TR2RE - SV40 can function as a repressor to suppress the transcriptional activities of both <e1> sv40 late promoter </e1> .',\n",
       "    'DNA']]],\n",
       " ['Mutations of the aml1 - binding sitecompletely eliminated EBNA - 2 responses .',\n",
       "  [['Mutations of the <e1> aml1 - binding site </e1> completely eliminated EBNA - 2 responses .',\n",
       "    'DNA']]],\n",
       " ['Mutations of the AML1 - or lbf4 - binding sitecompletely eliminated EBNA - 2 responses .',\n",
       "  [['Mutations of the AML1 - or <e1> lbf4 - binding site </e1> completely eliminated EBNA - 2 responses .',\n",
       "    'DNA']]],\n",
       " ['EBNA - 2 trans - activates cellular genelike CD23 , c - fgr , latent membrane protein 1 ( LMP1 ) and terminal protein 1 ( TP1 ) .',\n",
       "  [['EBNA - 2 trans - activates <e1> cellular gene </e1> like CD23 , c - fgr , latent membrane protein 1 ( LMP1 ) and terminal protein 1 ( TP1 ) .',\n",
       "    'DNA']]],\n",
       " ['Both positive regulatory elementwere identified in the proximal 5 flanking region of the IL - 3 gene .',\n",
       "  [['Both <e1> positive regulatory element </e1> were identified in the proximal 5 flanking region of the IL - 3 gene .',\n",
       "    'DNA']]],\n",
       " ['The pebp2 alpha a geneencode the DNA - binding subunit of a murine transcription factor , PEBP2 , which is implicated as a T - cell - specific transcriptional regulator .',\n",
       "  [['The <e1> pebp2 alpha a gene </e1> encode the DNA - binding subunit of a murine transcription factor , PEBP2 , which is implicated as a T - cell - specific transcriptional regulator .',\n",
       "    'DNA']]],\n",
       " ['The pebp2 alpha b geneencode the DNA - binding subunit of a murine transcription factor , PEBP2 , which is implicated as a T - cell - specific transcriptional regulator .',\n",
       "  [['The <e1> pebp2 alpha b gene </e1> encode the DNA - binding subunit of a murine transcription factor , PEBP2 , which is implicated as a T - cell - specific transcriptional regulator .',\n",
       "    'DNA']]],\n",
       " ['PEBP2 alpha B is the murine counterpart of human AML1 , which is located at the breakpoints of the 8;21 chromosome translocationassociated with acute myeloid leukemia .',\n",
       "  [['PEBP2 alpha B is the murine counterpart of human AML1 , which is located at the breakpoints of the <e1> 8;21 chromosome translocation </e1> associated with acute myeloid leukemia .',\n",
       "    'DNA']]],\n",
       " ['The transcripts were detected in all the cd4 - cd8 - cell population .',\n",
       "  [['The transcripts were detected in all the <e1> cd4 - cd8 - cell population </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The transcripts were detected in all the CD4 - CD8 - , cd4+ cd8+ cell population .',\n",
       "  [['The transcripts were detected in all the CD4 - CD8 - , <e1> cd4+ cd8+ cell population </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The transcripts were detected in all the CD4 - CD8 - , cd4+ cd8 - cell population .',\n",
       "  [['The transcripts were detected in all the CD4 - CD8 - , <e1> cd4+ cd8 - cell population </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The transcripts were detected in all the cd4 - cd8+ cell population .',\n",
       "  [['The transcripts were detected in all the <e1> cd4 - cd8+ cell population </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The amount of TcR at the cell surface is downmodulated , whereas the cd2 receptorare increased .',\n",
       "  [['The amount of TcR at the cell surface is downmodulated , whereas the <e1> cd2 receptor </e1> are increased .',\n",
       "    'PROTEIN']]],\n",
       " ['The measured binding to a TCF - 1 site is the same in the nuclei of resting cell .',\n",
       "  [['The measured binding to a TCF - 1 site is the same in the nuclei of <e1> resting cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The measured binding to a TCF - 1 site is the same in the nuclei of resting , activated cell .',\n",
       "  [['The measured binding to a TCF - 1 site is the same in the nuclei of resting , <e1> activated cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Using a highly sensitive mutation screening assay of the dna - binding region, none of the treated patients revealed any mutation , suggesting that the glucocorticoid receptor in the CLL patients tested is not altered in these domains .',\n",
       "  [['Using a highly sensitive mutation screening assay of the <e1> dna - binding region </e1> , none of the treated patients revealed any mutation , suggesting that the glucocorticoid receptor in the CLL patients tested is not altered in these domains .',\n",
       "    'PROTEIN']]],\n",
       " ['This conclusion is discussed in relation to the mechanism of corticoid resistance in mouse lymphoma cellin culture .',\n",
       "  [['This conclusion is discussed in relation to the mechanism of corticoid resistance in <e1> mouse lymphoma cell </e1> in culture .',\n",
       "    'CELL_LINE']]],\n",
       " ['The protein components of this complex are specifically competed by an AP1 consensus motif and were shown by supershift to include c - Jun and c - Fos , suggesting that this binding site is an AP1 motif and that the jun familyof transcription factors play a role in the regulation of the kappa light chain gene .',\n",
       "  [['The protein components of this complex are specifically competed by an AP1 consensus motif and were shown by supershift to include c - Jun and c - Fos , suggesting that this binding site is an AP1 motif and that the <e1> jun family </e1> of transcription factors play a role in the regulation of the kappa light chain gene .',\n",
       "    'PROTEIN']]],\n",
       " ['Distinct DNase - I hypersensitive sites are associated with TAL - 1 transcription in erythroid cell line .',\n",
       "  [['Distinct DNase - I hypersensitive sites are associated with TAL - 1 transcription in <e1> erythroid cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['The tal - 1 gene , frequently activated in human T - cell acute lymphoblastic leukemia ( T - ALL ) , is expressed in the erythroid cell lineageduring normal hematopoiesis .',\n",
       "  [['The tal - 1 gene , frequently activated in human T - cell acute lymphoblastic leukemia ( T - ALL ) , is expressed in the <e1> erythroid cell lineage </e1> during normal hematopoiesis .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The tal - 1 gene , frequently activated in human T - cell acute lymphoblastic leukemia ( T - ALL ) , is expressed in the erythroid , megakaryocytic cell lineageduring normal hematopoiesis .',\n",
       "  [['The tal - 1 gene , frequently activated in human T - cell acute lymphoblastic leukemia ( T - ALL ) , is expressed in the erythroid , <e1> megakaryocytic cell lineage </e1> during normal hematopoiesis .',\n",
       "    'CELL_TYPE']]],\n",
       " [' hs iv, and V were exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "  [[' <e1> hs iv </e1> , and V were exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "    'DNA']]],\n",
       " [' hs vwere exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "  [[' <e1> hs v </e1> were exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "    'DNA']]],\n",
       " [' hs iv, and V were exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "  [[' <e1> hs iv </e1> , and V were exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "    'DNA']]],\n",
       " [' hs vwere exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "  [[' <e1> hs v </e1> were exclusively observed in the erythroid/megakaryocytic cell lines that express tal - 1 from the promoters 1a and 1b .',\n",
       "    'DNA']]],\n",
       " ['M - TAT is a cytokine - dependent cell line with the potential to differentiate along the erythroid lineage .',\n",
       "  [['M - TAT is a cytokine - dependent cell line with the potential to differentiate along the <e1> erythroid lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These long term cultures are referred to as m - tat/epo cell, respectively .',\n",
       "  [['These long term cultures are referred to as <e1> m - tat/epo cell </e1> , respectively .',\n",
       "    'CELL_LINE']]],\n",
       " ['These long term cultures are referred to as m - tat/gm - csf cell, respectively .',\n",
       "  [['These long term cultures are referred to as <e1> m - tat/gm - csf cell </e1> , respectively .',\n",
       "    'CELL_LINE']]],\n",
       " ['These long term cultures are referred to as m - tat/scf cell, respectively .',\n",
       "  [['These long term cultures are referred to as <e1> m - tat/scf cell </e1> , respectively .',\n",
       "    'CELL_LINE']]],\n",
       " [' e2f - 1 mrnawere not detectable in granulocytes , monocytes and resting T - lymphocytes but were induced after the mitogenic stimulation of T - lymphocytes .',\n",
       "  [[' <e1> e2f - 1 mrna </e1> were not detectable in granulocytes , monocytes and resting T - lymphocytes but were induced after the mitogenic stimulation of T - lymphocytes .',\n",
       "    'RNA']]],\n",
       " ['Human immunodeficiency virus type 1 ( HIV - 1 ) replication is controlled by a complex array of virally encoded protein .',\n",
       "  [['Human immunodeficiency virus type 1 ( HIV - 1 ) replication is controlled by a complex array of <e1> virally encoded protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['A predominance of p50 NF - kappa B moieties and possibly p50 homodimers or closely related species , rather than the p50 - p56 heterodimer of NF - kappa B that is the predominant NF - kappa B species in most t lymphocytic cell, is demonstrated in the nuclei of U1 cells .',\n",
       "  [['A predominance of p50 NF - kappa B moieties and possibly p50 homodimers or closely related species , rather than the p50 - p56 heterodimer of NF - kappa B that is the predominant NF - kappa B species in most <e1> t lymphocytic cell </e1> , is demonstrated in the nuclei of U1 cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['OBJECTIVES : To analyze the effect of retinoic acids ( RA ) on HIV - 1 expression and correlate this effect with expression levels of RA receptors ( RARs ) in t - lymphoid cell line .',\n",
       "  [['OBJECTIVES : To analyze the effect of retinoic acids ( RA ) on HIV - 1 expression and correlate this effect with expression levels of RA receptors ( RARs ) in <e1> t - lymphoid cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Human RXR alpha was significantly expressed in u937 cell, weakly expressed in H9 cells and not detectable in CEM cells .',\n",
       "  [['Human RXR alpha was significantly expressed in <e1> u937 cell </e1> , weakly expressed in H9 cells and not detectable in CEM cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['CONCLUSIONS : In the present work , we show that human fetal erythroid progenitorare distinct in their transcription factors , and that the commitment to fetal or adult program occurs at a very early differentiation stage .',\n",
       "  [['CONCLUSIONS : In the present work , we show that <e1> human fetal erythroid progenitor </e1> are distinct in their transcription factors , and that the commitment to fetal or adult program occurs at a very early differentiation stage .',\n",
       "    'CELL_TYPE']]],\n",
       " ['CONCLUSIONS : In the present work , we show that human adult erythroid progenitorare distinct in their transcription factors , and that the commitment to fetal or adult program occurs at a very early differentiation stage .',\n",
       "  [['CONCLUSIONS : In the present work , we show that <e1> human adult erythroid progenitor </e1> are distinct in their transcription factors , and that the commitment to fetal or adult program occurs at a very early differentiation stage .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In contrast , stat1 proteins were not tyrosine phosphorylated after IL - 2 ligation , whereas tyrosine - phosphorylated stat1 proteins ( 91 kda protein) were translocated to the nucleus following interferon - gamma treatment of HeLa cells .',\n",
       "  [['In contrast , stat1 proteins were not tyrosine phosphorylated after IL - 2 ligation , whereas tyrosine - phosphorylated stat1 proteins ( <e1> 91 kda protein </e1> ) were translocated to the nucleus following interferon - gamma treatment of HeLa cells .',\n",
       "    'PROTEIN']]],\n",
       " ['The 160 N - terminal amino acids in the nonfinger region of clone 18 are highly homologous with similar regions of several other human sequence, defining a subfamily of Kruppel - like zinc finger proteins termed TAB ( tramtrack [ ttk ] - associated box ) here .',\n",
       "  [['The 160 N - terminal amino acids in the nonfinger region of clone 18 are highly homologous with similar regions of several other <e1> human sequence </e1> , defining a subfamily of Kruppel - like zinc finger proteins termed TAB ( tramtrack [ ttk ] - associated box ) here .',\n",
       "    'DNA']]],\n",
       " ['The 160 N - terminal amino acids in the nonfinger region of clone 18 are highly homologous with similar regions of several other human , mouse sequence, defining a subfamily of Kruppel - like zinc finger proteins termed TAB ( tramtrack [ ttk ] - associated box ) here .',\n",
       "  [['The 160 N - terminal amino acids in the nonfinger region of clone 18 are highly homologous with similar regions of several other human , <e1> mouse sequence </e1> , defining a subfamily of Kruppel - like zinc finger proteins termed TAB ( tramtrack [ ttk ] - associated box ) here .',\n",
       "    'DNA']]],\n",
       " ['This protein appears to require both the s element, which are separated by 10 bp for optimal binding .',\n",
       "  [['This protein appears to require both the <e1> s element </e1> , which are separated by 10 bp for optimal binding .',\n",
       "    'DNA']]],\n",
       " ['Furthermore , transfected c - jun expression vectorpartially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "  [['Furthermore , transfected <e1> c - jun expression vector </e1> partially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "    'DNA']]],\n",
       " ['Furthermore , transfected c - jun - b expression vectorpartially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "  [['Furthermore , transfected c - <e1> jun - b expression vector </e1> partially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "    'DNA']]],\n",
       " ['Furthermore , transfected c - jun - d expression vectorpartially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "  [['Furthermore , transfected c - <e1> jun - d expression vector </e1> partially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "    'DNA']]],\n",
       " ['Furthermore , transfected c - fos expression vectorpartially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "  [['Furthermore , transfected <e1> c - fos expression vector </e1> partially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "    'DNA']]],\n",
       " ['Furthermore , transfected c - jun , jun - B , jun - D , c - fos - b expression vectorpartially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "  [['Furthermore , transfected c - jun , jun - B , jun - D , c - <e1> fos - b expression vector </e1> partially substitute for TPA and Ca2+ and cooperate with Oct - 2 for the transactivation of the combined OAP/octamer cis - element .',\n",
       "    'DNA']]],\n",
       " ['As in XSCID , X - chromosome inactivation in obligate carriers of XCID was nonrandom in t lymphocyte .',\n",
       "  [['As in XSCID , X - chromosome inactivation in obligate carriers of XCID was nonrandom in <e1> t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " [' pma - induced il - 2r alphais inhibited by the PKC inhibitor H7 , whereas TG - but not PMA - induced IL - 2R alpha was inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "  [[' <e1> pma - induced il - 2r alpha </e1> is inhibited by the PKC inhibitor H7 , whereas TG - but not PMA - induced IL - 2R alpha was inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "    'PROTEIN']]],\n",
       " ['PMA - but not TG - induced IL - 2R alpha is inhibited by the PKC inhibitor H7 , whereas tg - induced il - 2r alphawas inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "  [['PMA - but not TG - induced IL - 2R alpha is inhibited by the PKC inhibitor H7 , whereas <e1> tg - induced il - 2r alpha </e1> was inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "    'PROTEIN']]],\n",
       " [' pma - induced il - 2r alphais inhibited by the PKC inhibitor H7 , whereas TG - but not PMA - induced IL - 2R alpha was inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "  [[' <e1> pma - induced il - 2r alpha </e1> is inhibited by the PKC inhibitor H7 , whereas TG - but not PMA - induced IL - 2R alpha was inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "    'PROTEIN']]],\n",
       " ['PMA - but not TG - induced IL - 2R alpha is inhibited by the PKC inhibitor H7 , whereas tg - induced il - 2r alphawas inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "  [['PMA - but not TG - induced IL - 2R alpha is inhibited by the PKC inhibitor H7 , whereas <e1> tg - induced il - 2r alpha </e1> was inhibited by cholera toxin , forskolin and 1 , 9 - dideoxy forskolin .',\n",
       "    'PROTEIN']]],\n",
       " ['The TCL1 gene sequence , which , to our knowledge , shows no sequence homology with other human genes , is preferentially expressed early in t - lymphocytedifferentiation .',\n",
       "  [['The TCL1 gene sequence , which , to our knowledge , shows no sequence homology with other human genes , is preferentially expressed early in <e1> t - lymphocyte </e1> differentiation .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These results indicate that the monocyte/B - cell - specific transcription factor PU.1 and the mono a protein complexact in concert to regulate monocyte - specific transcription of the CSF - 1 receptor .',\n",
       "  [['These results indicate that the monocyte/B - cell - specific transcription factor PU.1 and the <e1> mono a protein complex </e1> act in concert to regulate monocyte - specific transcription of the CSF - 1 receptor .',\n",
       "    'PROTEIN']]],\n",
       " ['These results indicate that the monocyte/B - cell - specific transcription factor PU.1 and the mono b protein complexact in concert to regulate monocyte - specific transcription of the CSF - 1 receptor .',\n",
       "  [['These results indicate that the monocyte/B - cell - specific transcription factor PU.1 and the <e1> mono b protein complex </e1> act in concert to regulate monocyte - specific transcription of the CSF - 1 receptor .',\n",
       "    'PROTEIN']]],\n",
       " ['Untreated NS - Meg cells expressed mRNA for the Epo receptor ( EpoR ) , for GATA - 1 , and for alpha 1 globin gene .',\n",
       "  [['Untreated NS - Meg cells expressed mRNA for the Epo receptor ( EpoR ) , for GATA - 1 , and for <e1> alpha 1 globin gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Untreated NS - Meg cells expressed mRNA for the Epo receptor ( EpoR ) , for GATA - 1 , and for alpha 2 globin gene .',\n",
       "  [['Untreated NS - Meg cells expressed mRNA for the Epo receptor ( EpoR ) , for GATA - 1 , and for <e1> alpha 2 globin gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['These results indicate that NS - Meg cells undergo terminal differentiation of both megakaryocytic lineage .',\n",
       "  [['These results indicate that NS - Meg cells undergo terminal differentiation of both <e1> megakaryocytic lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Previous studies have demonstrated that IL - 1 receptor ( IL - 1R ) - and TCR - initiated signals can interact synergistically to increase the rate of transcription of several lymphokine geneduring the competence phase of the activation program in T helper lymphocytes .',\n",
       "  [['Previous studies have demonstrated that IL - 1 receptor ( IL - 1R ) - and TCR - initiated signals can interact synergistically to increase the rate of transcription of several <e1> lymphokine gene </e1> during the competence phase of the activation program in T helper lymphocytes .',\n",
       "    'DNA']]],\n",
       " ['Previous studies have demonstrated that IL - 1 receptor ( IL - 1R ) - and TCR - initiated signals can interact synergistically to increase the rate of transcription of several lymphokine receptor geneduring the competence phase of the activation program in T helper lymphocytes .',\n",
       "  [['Previous studies have demonstrated that IL - 1 receptor ( IL - 1R ) - and TCR - initiated signals can interact synergistically to increase the rate of transcription of several <e1> lymphokine receptor gene </e1> during the competence phase of the activation program in T helper lymphocytes .',\n",
       "    'DNA']]],\n",
       " ['T cell hybridomas respond to activation signals by undergoing apoptotic cell death , and this is likely to represent comparable events related to tolerance induction in immature t cellin vivo .',\n",
       "  [['T cell hybridomas respond to activation signals by undergoing apoptotic cell death , and this is likely to represent comparable events related to tolerance induction in <e1> immature t cell </e1> in vivo .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Previously , an inducible enhancer between nucleotides - 299 and - 228 that contains nf - kappa b motifwas identified .',\n",
       "  [['Previously , an inducible enhancer between nucleotides - 299 and - 228 that contains <e1> nf - kappa b motif </e1> was identified .',\n",
       "    'DNA']]],\n",
       " ['Transcription from the IL - 2R alpha promoter was inhibited when either the elf - 1 binding sitewas mutated .',\n",
       "  [['Transcription from the IL - 2R alpha promoter was inhibited when either the <e1> elf - 1 binding site </e1> was mutated .',\n",
       "    'DNA']]],\n",
       " ['Elf - 1 physically associated with HMG - I and with NF - kappa B p50 and c - Rel in vitro , suggesting that protein - protein interactions might functionally coordinate the actions of the upstream positive regulatory element .',\n",
       "  [['Elf - 1 physically associated with HMG - I and with NF - kappa B p50 and c - Rel in vitro , suggesting that protein - protein interactions might functionally coordinate the actions of the <e1> upstream positive regulatory element </e1> .',\n",
       "    'DNA']]],\n",
       " ['Androgen receptor mRNA was translated in vitro , and androgen - bound receptor complexwere studied using limited proteolytic digestion by trypsin .',\n",
       "  [['Androgen receptor mRNA was translated in vitro , and <e1> androgen - bound receptor complex </e1> were studied using limited proteolytic digestion by trypsin .',\n",
       "    'PROTEIN']]],\n",
       " ['Cotransfection of nf - kappa b p50/p65 - expression vectorinto Jurkat cells with a luciferase reporter containing the GM - CSF promoter did not stimulate transcription from the GM - CSF promoter .',\n",
       "  [['Cotransfection of <e1> nf - kappa b p50/p65 - expression vector </e1> into Jurkat cells with a luciferase reporter containing the GM - CSF promoter did not stimulate transcription from the GM - CSF promoter .',\n",
       "    'DNA']]],\n",
       " ['Cotransfection of NF - kappa B ( p50/p65 ) - or ap - 1 c - jun/c - fos - expression vectorinto Jurkat cells with a luciferase reporter containing the GM - CSF promoter did not stimulate transcription from the GM - CSF promoter .',\n",
       "  [['Cotransfection of NF - kappa B ( p50/p65 ) - or <e1> ap - 1 c - jun/c - fos - expression vector </e1> into Jurkat cells with a luciferase reporter containing the GM - CSF promoter did not stimulate transcription from the GM - CSF promoter .',\n",
       "    'DNA']]],\n",
       " ['These results suggest that cooperation among nf - kappa b - binding sequenceis required for induction of the GM - CSF gene through PKC - and Ca2+ - signaling pathways downstream of T cell activation .',\n",
       "  [['These results suggest that cooperation among <e1> nf - kappa b - binding sequence </e1> is required for induction of the GM - CSF gene through PKC - and Ca2+ - signaling pathways downstream of T cell activation .',\n",
       "    'DNA']]],\n",
       " ['These results suggest that cooperation among NF - kappa B - , ap - 1 - binding sequenceis required for induction of the GM - CSF gene through PKC - and Ca2+ - signaling pathways downstream of T cell activation .',\n",
       "  [['These results suggest that cooperation among NF - kappa B - , <e1> ap - 1 - binding sequence </e1> is required for induction of the GM - CSF gene through PKC - and Ca2+ - signaling pathways downstream of T cell activation .',\n",
       "    'DNA']]],\n",
       " ['These results suggest that cooperation among nf - at - binding sequenceis required for induction of the GM - CSF gene through PKC - and Ca2+ - signaling pathways downstream of T cell activation .',\n",
       "  [['These results suggest that cooperation among <e1> nf - at - binding sequence </e1> is required for induction of the GM - CSF gene through PKC - and Ca2+ - signaling pathways downstream of T cell activation .',\n",
       "    'DNA']]],\n",
       " ['When we compared the activity of the transfected dPRL promoter in prl - secreting lymphoid cell, we found that the 3 kb 5 - flanking region of the dPRL promoter did not contain elements restricting expression to only those lymphocytes that produce PRL but allowed expression of fusion reporter genes irrespective of the status of the endogenous PRL gene .',\n",
       "  [['When we compared the activity of the transfected dPRL promoter in <e1> prl - secreting lymphoid cell </e1> , we found that the 3 kb 5 - flanking region of the dPRL promoter did not contain elements restricting expression to only those lymphocytes that produce PRL but allowed expression of fusion reporter genes irrespective of the status of the endogenous PRL gene .',\n",
       "    'CELL_TYPE']]],\n",
       " ['When we compared the activity of the transfected dPRL promoter in prl - nonsecreting lymphoid cell, we found that the 3 kb 5 - flanking region of the dPRL promoter did not contain elements restricting expression to only those lymphocytes that produce PRL but allowed expression of fusion reporter genes irrespective of the status of the endogenous PRL gene .',\n",
       "  [['When we compared the activity of the transfected dPRL promoter in <e1> prl - nonsecreting lymphoid cell </e1> , we found that the 3 kb 5 - flanking region of the dPRL promoter did not contain elements restricting expression to only those lymphocytes that produce PRL but allowed expression of fusion reporter genes irrespective of the status of the endogenous PRL gene .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Signals and nuclear factors that regulate the expression of interleukin - 4 genein helper T cells .',\n",
       "  [['Signals and nuclear factors that regulate the expression of <e1> interleukin - 4 gene </e1> in helper T cells .',\n",
       "    'DNA']]],\n",
       " ['Signals and nuclear factors that regulate the expression of interleukin - 5 genein helper T cells .',\n",
       "  [['Signals and nuclear factors that regulate the expression of <e1> interleukin - 5 gene </e1> in helper T cells .',\n",
       "    'DNA']]],\n",
       " ['The P sequence of the IL - 4 gene , defined as a responsive element for PMA and calcium ionophore ( A23187 ) , shares sequence similarity with the nf kappa b binding site .',\n",
       "  [['The P sequence of the IL - 4 gene , defined as a responsive element for PMA and calcium ionophore ( A23187 ) , shares sequence similarity with the <e1> nf kappa b binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['Taken together , these results suggest that NF - AT - like factors are involved in the regulation of il - 4 gene .',\n",
       "  [['Taken together , these results suggest that NF - AT - like factors are involved in the regulation of <e1> il - 4 gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Taken together , these results suggest that NF - AT - like factors are involved in the regulation of il - 5 gene .',\n",
       "  [['Taken together , these results suggest that NF - AT - like factors are involved in the regulation of <e1> il - 5 gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['A salient difference between solution structureis observed in the C - terminal segment of alpha - helix 3 ( the HTH recognition helix ) , which is not well ordered in solution .',\n",
       "  [['A salient difference between <e1> solution structure </e1> is observed in the C - terminal segment of alpha - helix 3 ( the HTH recognition helix ) , which is not well ordered in solution .',\n",
       "    'PROTEIN']]],\n",
       " ['Ii kappa B - 1 is a positive regulatory element in B - cell lines and in the Ii - expressing T - cell line , H9 , but acts as a negative regulatory element in myelomonocytic cell line .',\n",
       "  [['Ii kappa B - 1 is a positive regulatory element in B - cell lines and in the Ii - expressing T - cell line , H9 , but acts as a negative regulatory element in <e1> myelomonocytic cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In vivo protein - DNA contacts are detectable at Ii kappa B - 1 in cell lines in which this site is functional as either a positive regulator .',\n",
       "  [['In vivo protein - DNA contacts are detectable at Ii kappa B - 1 in cell lines in which this site is functional as either a <e1> positive regulator </e1> .',\n",
       "    'DNA']]],\n",
       " ['Ii kappa B - 2 is a site of positive regulation in B - cell lines and a site of negative regulation in H9 T cells , myelomonocytic cell line .',\n",
       "  [['Ii kappa B - 2 is a site of positive regulation in B - cell lines and a site of negative regulation in H9 T cells , <e1> myelomonocytic cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['PGE2 enhanced the complex formation with nf - at sequencewhen the cells were activated by either anti - CD3 mAb or PMA /A23187 stimulation .',\n",
       "  [['PGE2 enhanced the complex formation with <e1> nf - at sequence </e1> when the cells were activated by either anti - CD3 mAb or PMA /A23187 stimulation .',\n",
       "    'DNA']]],\n",
       " ['PGE2 enhanced the complex formation with NF - AT , ap - 1 sequencewhen the cells were activated by either anti - CD3 mAb or PMA /A23187 stimulation .',\n",
       "  [['PGE2 enhanced the complex formation with NF - AT , <e1> ap - 1 sequence </e1> when the cells were activated by either anti - CD3 mAb or PMA /A23187 stimulation .',\n",
       "    'DNA']]],\n",
       " ['Treatment of both U - 937 and TUR cells with okadaic acid , an inhibitor of serine/threonine protein phosphatase 2a, was associated with induction of monocytic differentiation and EGR - 1 transcription through the 5 - most CArG element .',\n",
       "  [['Treatment of both U - 937 and TUR cells with okadaic acid , an inhibitor of <e1> serine/threonine protein phosphatase 2a </e1> , was associated with induction of monocytic differentiation and EGR - 1 transcription through the 5 - most CArG element .',\n",
       "    'PROTEIN']]],\n",
       " ['In contrast , analysis of pp90rsk activity by phosphorylation of a peptide derived from S6 protein demonstrated stimulation of this kinase in tpa - treated u - 937 cell .',\n",
       "  [['In contrast , analysis of pp90rsk activity by phosphorylation of a peptide derived from S6 protein demonstrated stimulation of this kinase in <e1> tpa - treated u - 937 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In contrast , analysis of pp90rsk activity by phosphorylation of a peptide derived from S6 protein demonstrated stimulation of this kinase in tpa - treated tur cell .',\n",
       "  [['In contrast , analysis of pp90rsk activity by phosphorylation of a peptide derived from S6 protein demonstrated stimulation of this kinase in <e1> tpa - treated tur cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['The activity of the immunoglobulin heavy - chain ( IgH ) enhancer is restricted to B cells , although it binds both b - cell - restricted transcription factor .',\n",
       "  [['The activity of the immunoglobulin heavy - chain ( IgH ) enhancer is restricted to B cells , although it binds both <e1> b - cell - restricted transcription factor </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The transcription factor nuclear factor - kappa B ( NF - kappa B ) is critical for the inducible expression of multiple cellular geneinvolved in inflammation and infection including interleukin - 1 ( IL - 1 ) , IL - 6 , and adhesion molecules .',\n",
       "  [['The transcription factor nuclear factor - kappa B ( NF - kappa B ) is critical for the inducible expression of multiple <e1> cellular gene </e1> involved in inflammation and infection including interleukin - 1 ( IL - 1 ) , IL - 6 , and adhesion molecules .',\n",
       "    'DNA']]],\n",
       " ['Activation of a novel serine /threonine kinase that phosphorylates c - Fos upon stimulation of t lymphocytevia antigen and cytokine receptors .',\n",
       "  [['Activation of a novel serine /threonine kinase that phosphorylates c - Fos upon stimulation of <e1> t lymphocyte </e1> via antigen and cytokine receptors .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Ligation of Ag receptors in t lymphocyteinitiates signal transduction cascades which alter the expression of genes that regulate cellular proliferation and differentiation .',\n",
       "  [['Ligation of Ag receptors in <e1> t lymphocyte </e1> initiates signal transduction cascades which alter the expression of genes that regulate cellular proliferation and differentiation .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The transmission of signals from the membrane to the nucleus is mediated principally through the action of protein tyrosine kinase .',\n",
       "  [['The transmission of signals from the membrane to the nucleus is mediated principally through the action of <e1> protein tyrosine kinase </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The transmission of signals from the membrane to the nucleus is mediated principally through the action of protein serine/threonine kinase .',\n",
       "  [['The transmission of signals from the membrane to the nucleus is mediated principally through the action of <e1> protein serine/threonine kinase </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Fos kinase was rapidly activated after ligation of the cd3 receptorin Jurkat and normal human T lymphocytes and in response to IL - 6 and anti - IgM in the human B cell lines AF10 and Ramos , respectively .',\n",
       "  [['Fos kinase was rapidly activated after ligation of the <e1> cd3 receptor </e1> in Jurkat and normal human T lymphocytes and in response to IL - 6 and anti - IgM in the human B cell lines AF10 and Ramos , respectively .',\n",
       "    'PROTEIN']]],\n",
       " ['The gc2 - specific cd4+ clonehad cytotoxic activity .',\n",
       "  [['The <e1> gc2 - specific cd4+ clone </e1> had cytotoxic activity .',\n",
       "    'CELL_LINE']]],\n",
       " ['We conclude that the nf - il6 sitemediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "  [['We conclude that the <e1> nf - il6 site </e1> mediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "    'DNA']]],\n",
       " ['We conclude that the nf - kappa b sitemediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "  [['We conclude that the <e1> nf - kappa b site </e1> mediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "    'DNA']]],\n",
       " ['We conclude that the NF - IL6 and NF - kappa B sites mediate IL - 6 induction in response to both LPS and LAM , acting as bacterial response element .',\n",
       "  [['We conclude that the NF - IL6 and NF - kappa B sites mediate IL - 6 induction in response to both LPS and LAM , acting as <e1> bacterial response element </e1> .',\n",
       "    'DNA']]],\n",
       " ['We conclude that the nf - il6 sitemediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "  [['We conclude that the <e1> nf - il6 site </e1> mediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "    'DNA']]],\n",
       " ['We conclude that the nf - kappa b sitemediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "  [['We conclude that the <e1> nf - kappa b site </e1> mediate IL - 6 induction in response to both LPS and LAM , acting as bacterial or mycobacterial response elements .',\n",
       "    'DNA']]],\n",
       " ['We conclude that the NF - IL6 and NF - kappa B sites mediate IL - 6 induction in response to both LPS and LAM , acting as bacterial response element .',\n",
       "  [['We conclude that the NF - IL6 and NF - kappa B sites mediate IL - 6 induction in response to both LPS and LAM , acting as <e1> bacterial response element </e1> .',\n",
       "    'DNA']]],\n",
       " ['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in lymphoid cellon reporter genes containing HLF or C/EBP consensus binding sites .',\n",
       "  [['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in <e1> lymphoid cell </e1> on reporter genes containing HLF or C/EBP consensus binding sites .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in lymphoid and nonlymphoid cells on reporter genes containing hlf consensus binding site .',\n",
       "  [['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in lymphoid and nonlymphoid cells on reporter genes containing <e1> hlf consensus binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in lymphoid cellon reporter genes containing HLF or C/EBP consensus binding sites .',\n",
       "  [['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in <e1> lymphoid cell </e1> on reporter genes containing HLF or C/EBP consensus binding sites .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in lymphoid and nonlymphoid cells on reporter genes containing hlf consensus binding site .',\n",
       "  [['Both wild - type and chimeric HLF proteins displayed transcriptional activator properties in lymphoid and nonlymphoid cells on reporter genes containing <e1> hlf consensus binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['The glutathione - depleting agent diethyl maleate ( DEM ) prevented the development of differentiated features in response to phorbol esters , including adherence of the cells to plastic surfaces and repression of the myeloperoxidase gene .',\n",
       "  [['The glutathione - depleting agent diethyl maleate ( DEM ) prevented the development of differentiated features in response to phorbol esters , including adherence of the cells to plastic surfaces and repression of the <e1> myeloperoxidase gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['In vitro binding studies demonstrated that the TF site bound translated c - rel homodimerbut not p50/p65 heterodimers or p50 homodimers .',\n",
       "  [['In vitro binding studies demonstrated that the TF site bound translated <e1> c - rel homodimer </e1> but not p50/p65 heterodimers or p50 homodimers .',\n",
       "    'PROTEIN']]],\n",
       " ['Recent findings suggest an involvement of reactive oxygen species in the pathway of TPA and protein kinase C leading to expression of c - fos mrna .',\n",
       "  [['Recent findings suggest an involvement of reactive oxygen species in the pathway of TPA and protein kinase C leading to expression of <e1> c - fos mrna </e1> .',\n",
       "    'RNA']]],\n",
       " ['Recent findings suggest an involvement of reactive oxygen species in the pathway of TPA and protein kinase C leading to expression of c - jun mrna .',\n",
       "  [['Recent findings suggest an involvement of reactive oxygen species in the pathway of TPA and protein kinase C leading to expression of <e1> c - jun mrna </e1> .',\n",
       "    'RNA']]],\n",
       " ['Our studies reveal that inactivation of the v - abl protein tyrosine kinase triggers high - frequency rearrangement of kappa light - chain gene .',\n",
       "  [['Our studies reveal that inactivation of the v - abl protein tyrosine kinase triggers high - frequency rearrangement of <e1> kappa light - chain gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Peripheral blood mononuclear cells from seventeen patients with primary myelodysplastic syndromes ( MDS ) in advanced stage were enriched for blasts and tested for ( 1 ) karyotype , ( 2 ) genomic configuration and ( 3 ) expression of il - 3 genewhich are all located on the long arm of chromosome 5 .',\n",
       "  [['Peripheral blood mononuclear cells from seventeen patients with primary myelodysplastic syndromes ( MDS ) in advanced stage were enriched for blasts and tested for ( 1 ) karyotype , ( 2 ) genomic configuration and ( 3 ) expression of <e1> il - 3 gene </e1> which are all located on the long arm of chromosome 5 .',\n",
       "    'DNA']]],\n",
       " ['Peripheral blood mononuclear cells from seventeen patients with primary myelodysplastic syndromes ( MDS ) in advanced stage were enriched for blasts and tested for ( 1 ) karyotype , ( 2 ) genomic configuration and ( 3 ) expression of IL - 3 , gm - csf genewhich are all located on the long arm of chromosome 5 .',\n",
       "  [['Peripheral blood mononuclear cells from seventeen patients with primary myelodysplastic syndromes ( MDS ) in advanced stage were enriched for blasts and tested for ( 1 ) karyotype , ( 2 ) genomic configuration and ( 3 ) expression of IL - 3 , <e1> gm - csf gene </e1> which are all located on the long arm of chromosome 5 .',\n",
       "    'DNA']]],\n",
       " ['Peripheral blood mononuclear cells from seventeen patients with primary myelodysplastic syndromes ( MDS ) in advanced stage were enriched for blasts and tested for ( 1 ) karyotype , ( 2 ) genomic configuration and ( 3 ) expression of IL - 3 , GM - CSF , fms genewhich are all located on the long arm of chromosome 5 .',\n",
       "  [['Peripheral blood mononuclear cells from seventeen patients with primary myelodysplastic syndromes ( MDS ) in advanced stage were enriched for blasts and tested for ( 1 ) karyotype , ( 2 ) genomic configuration and ( 3 ) expression of IL - 3 , GM - CSF , <e1> fms gene </e1> which are all located on the long arm of chromosome 5 .',\n",
       "    'DNA']]],\n",
       " ['An autocrine mechanism of growth could be hypothesized only for the 2 patients whose cells expressed both the m - csf gene .',\n",
       "  [['An autocrine mechanism of growth could be hypothesized only for the 2 patients whose cells expressed both the <e1> m - csf gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['The expression of the p50 nf - kappa b subunitwas also shown to be regulated differentially after exposures to 1.0 and 2.0 Gy .',\n",
       "  [['The expression of the <e1> p50 nf - kappa b subunit </e1> was also shown to be regulated differentially after exposures to 1.0 and 2.0 Gy .',\n",
       "    'PROTEIN']]],\n",
       " ['The human gata3 transcription unitstart at a major initiation site .',\n",
       "  [['The <e1> human gata3 transcription unit </e1> start at a major initiation site .',\n",
       "    'DNA']]],\n",
       " ['CD38 is a leukocyte differentiation antigen that has been thought to be a phenotypic marker of different subpopulations of t - lymphocyte .',\n",
       "  [['CD38 is a leukocyte differentiation antigen that has been thought to be a phenotypic marker of different subpopulations of <e1> t - lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Induction of CD38 in acute promyelocytic leukemia cellwas independent of ATRA - induced cytodifferentiation .',\n",
       "  [['Induction of CD38 in <e1> acute promyelocytic leukemia cell </e1> was independent of ATRA - induced cytodifferentiation .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Induction of CD38 in acute myeloblastic leukemia cellwas independent of ATRA - induced cytodifferentiation .',\n",
       "  [['Induction of CD38 in <e1> acute myeloblastic leukemia cell </e1> was independent of ATRA - induced cytodifferentiation .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Macrophages and t lymphocyteinfiltrated the tumor in moderate numbers with occasional lymphoid aggregate formation .',\n",
       "  [['Macrophages and <e1> t lymphocyte </e1> infiltrated the tumor in moderate numbers with occasional lymphoid aggregate formation .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These parameters were examined in normal and Epstein - Barr virus - immortalized human B cells and compared with 1 alpha,25 - (OH)2D3 - responsive cells of the t lineage .',\n",
       "  [['These parameters were examined in normal and Epstein - Barr virus - immortalized human B cells and compared with 1 alpha,25 - (OH)2D3 - responsive cells of the <e1> t lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Finally , the ap - 1 - 186 to - 176 site( - 206 to - 195 ) respond to different cellular activation in EL4 cells .',\n",
       "  [['Finally , the <e1> ap - 1 - 186 to - 176 site </e1> ( - 206 to - 195 ) respond to different cellular activation in EL4 cells .',\n",
       "    'DNA']]],\n",
       " ['One major transcription start sitewere identified 101 and 130 base pairs ( bp ) upstream , respectively , from the protein translation start ATG .',\n",
       "  [['One <e1> major transcription start site </e1> were identified 101 and 130 base pairs ( bp ) upstream , respectively , from the protein translation start ATG .',\n",
       "    'DNA']]],\n",
       " ['The interleukin - 8 ap - 1 siteare genetic end targets of FK506 - sensitive pathway accompanied by calcium mobilization .',\n",
       "  [['The <e1> interleukin - 8 ap - 1 site </e1> are genetic end targets of FK506 - sensitive pathway accompanied by calcium mobilization .',\n",
       "    'DNA']]],\n",
       " ['Our results indicate that not only the reported il - 2 nf - at site, but also the IL - 8 AP - 1 and kappa B - like sites are terminals of FK506 - sensitive pathway involving Ca2+ mobilization .',\n",
       "  [['Our results indicate that not only the reported <e1> il - 2 nf - at site </e1> , but also the IL - 8 AP - 1 and kappa B - like sites are terminals of FK506 - sensitive pathway involving Ca2+ mobilization .',\n",
       "    'DNA']]],\n",
       " ['Our results indicate that not only the reported IL - 2 NF - AT and nfil - 2a site, but also the IL - 8 AP - 1 and kappa B - like sites are terminals of FK506 - sensitive pathway involving Ca2+ mobilization .',\n",
       "  [['Our results indicate that not only the reported IL - 2 NF - AT and <e1> nfil - 2a site </e1> , but also the IL - 8 AP - 1 and kappa B - like sites are terminals of FK506 - sensitive pathway involving Ca2+ mobilization .',\n",
       "    'DNA']]],\n",
       " [' corticosteroid type i receptorare inversely correlated with age .',\n",
       "  [[' <e1> corticosteroid type i receptor </e1> are inversely correlated with age .',\n",
       "    'PROTEIN']]],\n",
       " [' corticosteroid type ii receptorare inversely correlated with age .',\n",
       "  [[' <e1> corticosteroid type ii receptor </e1> are inversely correlated with age .',\n",
       "    'PROTEIN']]],\n",
       " ['Priming with 10 nM RA , or subsequent treatment with D3 ( 5 nM ) , did not alter expression of mRNAs encoding receptors for D3 ( VDR ) , RA ( RAR alpha ) or 9 - CIS RA ( rxr beta, gamma ) .',\n",
       "  [['Priming with 10 nM RA , or subsequent treatment with D3 ( 5 nM ) , did not alter expression of mRNAs encoding receptors for D3 ( VDR ) , RA ( RAR alpha ) or 9 - CIS RA ( <e1> rxr beta </e1> , gamma ) .',\n",
       "    'PROTEIN']]],\n",
       " ['Priming with 10 nM RA , or subsequent treatment with D3 ( 5 nM ) , did not alter expression of mRNAs encoding receptors for D3 ( VDR ) , RA ( RAR alpha ) or 9 - CIS RA ( rxr gamma) .',\n",
       "  [['Priming with 10 nM RA , or subsequent treatment with D3 ( 5 nM ) , did not alter expression of mRNAs encoding receptors for D3 ( VDR ) , RA ( RAR alpha ) or 9 - CIS RA ( <e1> rxr gamma </e1> ) .',\n",
       "    'PROTEIN']]],\n",
       " ['Thrombin stimulated tyrosine phosphorylation of several proteins of molecular mass 40 kda .',\n",
       "  [['Thrombin stimulated tyrosine phosphorylation of several proteins of <e1> molecular mass 40 kda </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Thrombin stimulated tyrosine phosphorylation of several proteins of molecular mass 42 kda .',\n",
       "  [['Thrombin stimulated tyrosine phosphorylation of several proteins of <e1> molecular mass 42 kda </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Thrombin stimulated tyrosine phosphorylation of several proteins of molecular mass 70 kda .',\n",
       "  [['Thrombin stimulated tyrosine phosphorylation of several proteins of <e1> molecular mass 70 kda </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Thrombin stimulated tyrosine phosphorylation of several proteins of molecular mass 120 kda .',\n",
       "  [['Thrombin stimulated tyrosine phosphorylation of several proteins of <e1> molecular mass 120 kda </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Thrombin stimulated tyrosine phosphorylation of several proteins of molecular mass 130 kda .',\n",
       "  [['Thrombin stimulated tyrosine phosphorylation of several proteins of <e1> molecular mass 130 kda </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['We have identified and characterized a previously unrecognized form of acute leukemia that shares features of both myeloid cell .',\n",
       "  [['We have identified and characterized a previously unrecognized form of acute leukemia that shares features of both <e1> myeloid cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We have identified and characterized a previously unrecognized form of acute leukemia that shares features of both myeloid and natural killer nk cell .',\n",
       "  [['We have identified and characterized a previously unrecognized form of acute leukemia that shares features of both myeloid and <e1> natural killer nk cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Multicolor flow cytometric assays confirmed the coexpression of myeloid cd33 , cd13 , cd15 antigenin each case , whereas reverse transcription polymerase chain reaction ( RT - PCR ) assays confirmed the identity of CD56 ( neural cell adhesion molecule ) in leukemic blasts .',\n",
       "  [['Multicolor flow cytometric assays confirmed the coexpression of <e1> myeloid cd33 , cd13 , cd15 antigen </e1> in each case , whereas reverse transcription polymerase chain reaction ( RT - PCR ) assays confirmed the identity of CD56 ( neural cell adhesion molecule ) in leukemic blasts .',\n",
       "    'PROTEIN']]],\n",
       " ['Multicolor flow cytometric assays confirmed the coexpression of myeloid ( CD33 , CD13 , CD15 ) and nk cell - associated cd56 antigenin each case , whereas reverse transcription polymerase chain reaction ( RT - PCR ) assays confirmed the identity of CD56 ( neural cell adhesion molecule ) in leukemic blasts .',\n",
       "  [['Multicolor flow cytometric assays confirmed the coexpression of myeloid ( CD33 , CD13 , CD15 ) and <e1> nk cell - associated cd56 antigen </e1> in each case , whereas reverse transcription polymerase chain reaction ( RT - PCR ) assays confirmed the identity of CD56 ( neural cell adhesion molecule ) in leukemic blasts .',\n",
       "    'PROTEIN']]],\n",
       " ['Although two cases expressed CD4 , no case expressed CD2 , CD3 , or CD8 and no case showed clonal rearrangement of genes encoding the T - cell receptor ( tcr gamma, delta ) .',\n",
       "  [['Although two cases expressed CD4 , no case expressed CD2 , CD3 , or CD8 and no case showed clonal rearrangement of genes encoding the T - cell receptor ( <e1> tcr gamma </e1> , delta ) .',\n",
       "    'PROTEIN']]],\n",
       " ['Although two cases expressed CD4 , no case expressed CD2 , CD3 , or CD8 and no case showed clonal rearrangement of genes encoding the T - cell receptor ( tcr delta) .',\n",
       "  [['Although two cases expressed CD4 , no case expressed CD2 , CD3 , or CD8 and no case showed clonal rearrangement of genes encoding the T - cell receptor ( <e1> tcr delta </e1> ) .',\n",
       "    'PROTEIN']]],\n",
       " ['Our studies suggest that this form of acute leukemia may arise from transformation of a precursor cell common to both the myeloid cell lineage; thus we propose the designation myeloid/NK acute leukemia .',\n",
       "  [['Our studies suggest that this form of acute leukemia may arise from transformation of a precursor cell common to both the <e1> myeloid cell lineage </e1> ; thus we propose the designation myeloid/NK acute leukemia .',\n",
       "    'CELL_TYPE']]],\n",
       " ['IL - 4 , a product of the t - helper 0 th0 subset, was originally described as a B - cell stimulatory factor and has subsequently been found to suppress IL - 1 alpha , IL - 1 beta , IL - 6 , IL - 8 , and TNF - alpha gene expression in monocytes stimulated with LPS , and to upregulate IL - 1 receptor antagonist ( IL1 - RA ) gene expression .',\n",
       "  [['IL - 4 , a product of the <e1> t - helper 0 th0 subset </e1> , was originally described as a B - cell stimulatory factor and has subsequently been found to suppress IL - 1 alpha , IL - 1 beta , IL - 6 , IL - 8 , and TNF - alpha gene expression in monocytes stimulated with LPS , and to upregulate IL - 1 receptor antagonist ( IL1 - RA ) gene expression .',\n",
       "    'CELL_TYPE']]],\n",
       " ['IL - 4 , a product of the t - helper 2 th2 subset, was originally described as a B - cell stimulatory factor and has subsequently been found to suppress IL - 1 alpha , IL - 1 beta , IL - 6 , IL - 8 , and TNF - alpha gene expression in monocytes stimulated with LPS , and to upregulate IL - 1 receptor antagonist ( IL1 - RA ) gene expression .',\n",
       "  [['IL - 4 , a product of the <e1> t - helper 2 th2 subset </e1> , was originally described as a B - cell stimulatory factor and has subsequently been found to suppress IL - 1 alpha , IL - 1 beta , IL - 6 , IL - 8 , and TNF - alpha gene expression in monocytes stimulated with LPS , and to upregulate IL - 1 receptor antagonist ( IL1 - RA ) gene expression .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The induced expression of il - 1 beta mrnain the monocytic THP - 1 cell line increased as quickly as in primary cells .',\n",
       "  [['The induced expression of <e1> il - 1 beta mrna </e1> in the monocytic THP - 1 cell line increased as quickly as in primary cells .',\n",
       "    'RNA']]],\n",
       " ['IL - 1 beta and IL - 6 genes contain AP - 1 binding sites as regulatory elements , the AP - 1 protein being composed of c - jun gene product .',\n",
       "  [['IL - 1 beta and IL - 6 genes contain AP - 1 binding sites as regulatory elements , the AP - 1 protein being composed of <e1> c - jun gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['IL - 1 beta and IL - 6 genes contain AP - 1 binding sites as regulatory elements , the AP - 1 protein being composed of c - fos gene product .',\n",
       "  [['IL - 1 beta and IL - 6 genes contain AP - 1 binding sites as regulatory elements , the AP - 1 protein being composed of <e1> c - fos gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Protein tyrosine kinases transmit signals from cytokine receptor .',\n",
       "  [['Protein tyrosine kinases transmit signals from <e1> cytokine receptor </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Two new isoforms were found that generate novel amino - terminal sequence .',\n",
       "  [['Two new isoforms were found that generate novel <e1> amino - terminal sequence </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The zinc finger transcription factor GATA - 1 is a major regulator of gene expression in erythroid cell lineage .',\n",
       "  [['The zinc finger transcription factor GATA - 1 is a major regulator of gene expression in <e1> erythroid cell lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The zinc finger transcription factor GATA - 1 is a major regulator of gene expression in erythroid , megakaryocyte cell lineage .',\n",
       "  [['The zinc finger transcription factor GATA - 1 is a major regulator of gene expression in erythroid , <e1> megakaryocyte cell lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In clonogenetic fetal calf serum - supplemented fcs+ culturetreated with saturating levels of interleukin - 3 ( IL - 3 ) granulocyte - macrophage colony - stimulating factor ( GM - CSF ) and erythropoietin ( Ep ) ( combined with c - kit ligand in FCS( - ) - culture conditions ) , RA induces a dramatic dose - dependent shift from erythroid to granulomonocytic colony formation , the latter colonies being essentially represented by granulocytic clones .',\n",
       "  [['In <e1> clonogenetic fetal calf serum - supplemented fcs+ culture </e1> treated with saturating levels of interleukin - 3 ( IL - 3 ) granulocyte - macrophage colony - stimulating factor ( GM - CSF ) and erythropoietin ( Ep ) ( combined with c - kit ligand in FCS( - ) - culture conditions ) , RA induces a dramatic dose - dependent shift from erythroid to granulomonocytic colony formation , the latter colonies being essentially represented by granulocytic clones .',\n",
       "    'CELL_LINE']]],\n",
       " ['In clonogenetic fetal calf serum - nonsupplemented fcs - culturetreated with saturating levels of interleukin - 3 ( IL - 3 ) granulocyte - macrophage colony - stimulating factor ( GM - CSF ) and erythropoietin ( Ep ) ( combined with c - kit ligand in FCS( - ) - culture conditions ) , RA induces a dramatic dose - dependent shift from erythroid to granulomonocytic colony formation , the latter colonies being essentially represented by granulocytic clones .',\n",
       "  [['In <e1> clonogenetic fetal calf serum - nonsupplemented fcs - culture </e1> treated with saturating levels of interleukin - 3 ( IL - 3 ) granulocyte - macrophage colony - stimulating factor ( GM - CSF ) and erythropoietin ( Ep ) ( combined with c - kit ligand in FCS( - ) - culture conditions ) , RA induces a dramatic dose - dependent shift from erythroid to granulomonocytic colony formation , the latter colonies being essentially represented by granulocytic clones .',\n",
       "    'CELL_LINE']]],\n",
       " ['Both sIgM - and sIgD - crosslinking with antibodies induced elevation of cAMP levels , phosphatidylinositol turnover , PKC activation and expression of egr - 1 mrna, although sIgM - crosslinking was more effective than sIgD - crosslinking , presumably due to the higher expression of sIgM than of sIgD .',\n",
       "  [['Both sIgM - and sIgD - crosslinking with antibodies induced elevation of cAMP levels , phosphatidylinositol turnover , PKC activation and expression of <e1> egr - 1 mrna </e1> , although sIgM - crosslinking was more effective than sIgD - crosslinking , presumably due to the higher expression of sIgM than of sIgD .',\n",
       "    'RNA']]],\n",
       " ['Therefore , we compared the nucleotide sequences of the cd11a gene promoterto identify common elements that might contribute to inducible expression .',\n",
       "  [['Therefore , we compared the nucleotide sequences of the <e1> cd11a gene promoter </e1> to identify common elements that might contribute to inducible expression .',\n",
       "    'DNA']]],\n",
       " ['Therefore , we compared the nucleotide sequences of the CD11a , cd11b gene promoterto identify common elements that might contribute to inducible expression .',\n",
       "  [['Therefore , we compared the nucleotide sequences of the CD11a , <e1> cd11b gene promoter </e1> to identify common elements that might contribute to inducible expression .',\n",
       "    'DNA']]],\n",
       " ['PML appears to be transcriptionally regulated by class i interferon, which raises the possibility that interferons modulate the function and growth and differentiation potential of normal myeloid cells and precursors by activating PML - dependent pathways .',\n",
       "  [['PML appears to be transcriptionally regulated by <e1> class i interferon </e1> , which raises the possibility that interferons modulate the function and growth and differentiation potential of normal myeloid cells and precursors by activating PML - dependent pathways .',\n",
       "    'PROTEIN']]],\n",
       " ['PML appears to be transcriptionally regulated by class ii interferon, which raises the possibility that interferons modulate the function and growth and differentiation potential of normal myeloid cells and precursors by activating PML - dependent pathways .',\n",
       "  [['PML appears to be transcriptionally regulated by <e1> class ii interferon </e1> , which raises the possibility that interferons modulate the function and growth and differentiation potential of normal myeloid cells and precursors by activating PML - dependent pathways .',\n",
       "    'PROTEIN']]],\n",
       " ['We report here that PML is expressed at low levels or not expressed in normal circulating human monocytes , lymphocytes , and polymorphonucleate cells , but is markedly induced by interferon ; that PML and PML/RAR alpha expression is augmented by interferon in the NB4 APL cell line , which carries the t(15; 17) , and in APL blasts from patients ; that interferon inhibits growth and survival of NB4 APL cells in cooperation with RA ; that interferons alone have minimal maturation effect on NB4 cells ; and , finally , that interferon alphaor beta , induces maturation and growth suppression of NB4 cells with de novo retinoid resistance , and partially restores RA response .',\n",
       "  [['We report here that PML is expressed at low levels or not expressed in normal circulating human monocytes , lymphocytes , and polymorphonucleate cells , but is markedly induced by interferon ; that PML and PML/RAR alpha expression is augmented by interferon in the NB4 APL cell line , which carries the t(15; 17) , and in APL blasts from patients ; that interferon inhibits growth and survival of NB4 APL cells in cooperation with RA ; that interferons alone have minimal maturation effect on NB4 cells ; and , finally , that <e1> interferon alpha </e1> or beta , induces maturation and growth suppression of NB4 cells with de novo retinoid resistance , and partially restores RA response .',\n",
       "    'PROTEIN']]],\n",
       " ['We report here that PML is expressed at low levels or not expressed in normal circulating human monocytes , lymphocytes , and polymorphonucleate cells , but is markedly induced by interferon ; that PML and PML/RAR alpha expression is augmented by interferon in the NB4 APL cell line , which carries the t(15; 17) , and in APL blasts from patients ; that interferon inhibits growth and survival of NB4 APL cells in cooperation with RA ; that interferons alone have minimal maturation effect on NB4 cells ; and , finally , that interferon beta, induces maturation and growth suppression of NB4 cells with de novo retinoid resistance , and partially restores RA response .',\n",
       "  [['We report here that PML is expressed at low levels or not expressed in normal circulating human monocytes , lymphocytes , and polymorphonucleate cells , but is markedly induced by interferon ; that PML and PML/RAR alpha expression is augmented by interferon in the NB4 APL cell line , which carries the t(15; 17) , and in APL blasts from patients ; that interferon inhibits growth and survival of NB4 APL cells in cooperation with RA ; that interferons alone have minimal maturation effect on NB4 cells ; and , finally , that <e1> interferon beta </e1> , induces maturation and growth suppression of NB4 cells with de novo retinoid resistance , and partially restores RA response .',\n",
       "    'PROTEIN']]],\n",
       " ['Effects of Ara - C on neutral sphingomyelinase and mitogen - activated protein kinasein T - lymphocyte cell lines .',\n",
       "  [['Effects of Ara - C on neutral sphingomyelinase and <e1> mitogen - activated protein kinase </e1> in T - lymphocyte cell lines .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparative analysis identifies conserved tumor necrosis factor receptor - associated factor 3 binding sites in the human epstein - barr virus oncogene lmp1 .',\n",
       "  [['Comparative analysis identifies conserved tumor necrosis factor receptor - associated factor 3 binding sites in the <e1> human epstein - barr virus oncogene lmp1 </e1> .',\n",
       "    'DNA']]],\n",
       " ['Chromosome 1 abnormalities with loss of 1p36 have been investigated in 95 breast - cancer samples by means of a dual - target fluorescence in - situ hybridization ( FISH ) technique using the pUC 1.77 and p1 - 79 probes , specific for the 1q12 region, respectively .',\n",
       "  [['Chromosome 1 abnormalities with loss of 1p36 have been investigated in 95 breast - cancer samples by means of a dual - target fluorescence in - situ hybridization ( FISH ) technique using the pUC 1.77 and p1 - 79 probes , specific for the <e1> 1q12 region </e1> , respectively .',\n",
       "    'DNA']]],\n",
       " ['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the human chromosomal band 1p21 - 22, 3q21 - 24 , 8p12 , 13q14 , 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "  [['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the <e1> human chromosomal band 1p21 - 22 </e1> , 3q21 - 24 , 8p12 , 13q14 , 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "    'DNA']]],\n",
       " ['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the human chromosomal band 3q21 - 24, 8p12 , 13q14 , 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "  [['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the <e1> human chromosomal band 3q21 - 24 </e1> , 8p12 , 13q14 , 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "    'DNA']]],\n",
       " ['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the human chromosomal band 8p12, 13q14 , 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "  [['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the <e1> human chromosomal band 8p12 </e1> , 13q14 , 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "    'DNA']]],\n",
       " ['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the human chromosomal band 13q14, 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "  [['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the <e1> human chromosomal band 13q14 </e1> , 19p13.3 and 11p14 - 15.1 , respectively .',\n",
       "    'DNA']]],\n",
       " ['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the human chromosomal band 19p13.3and 11p14 - 15.1 , respectively .',\n",
       "  [['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the <e1> human chromosomal band 19p13.3 </e1> and 11p14 - 15.1 , respectively .',\n",
       "    'DNA']]],\n",
       " ['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the human chromosomal band 11p14 - 15.1, respectively .',\n",
       "  [['Using fluorescence in situ hybridization ( FISH ) , we show here that the genes encoding the TATA - box binding protein ( TBP ) , TFIIB , TFIIE alpha , TFIIE beta , RAP30 , RAP74 and the 62 kDa subunit , of TFIIH are located at the <e1> human chromosomal band 11p14 - 15.1 </e1> , respectively .',\n",
       "    'DNA']]],\n",
       " ['Early buds are characterized by a branching morphogenesis of the ductal epithelium from which endocrine precursor cellbud to eventually form the two other compartments .',\n",
       "  [['Early buds are characterized by a branching morphogenesis of the ductal epithelium from which <e1> endocrine precursor cell </e1> bud to eventually form the two other compartments .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Deletion mutants containing 1.3 kb 5 - flanking dnawere incrementally less transcriptionally active , although 0.2 kb still mediated a 12 - fold induction by PRL .',\n",
       "  [['Deletion mutants containing <e1> 1.3 kb 5 - flanking dna </e1> were incrementally less transcriptionally active , although 0.2 kb still mediated a 12 - fold induction by PRL .',\n",
       "    'DNA']]],\n",
       " ['Deletion mutants containing 1.3 , 0.6 kb 5 - flanking dnawere incrementally less transcriptionally active , although 0.2 kb still mediated a 12 - fold induction by PRL .',\n",
       "  [['Deletion mutants containing 1.3 , <e1> 0.6 kb 5 - flanking dna </e1> were incrementally less transcriptionally active , although 0.2 kb still mediated a 12 - fold induction by PRL .',\n",
       "    'DNA']]],\n",
       " ['SCL protein was detected in erythroid cell line, as well as in several lymphoid leukemia cell lines which are known to harbor SCL gene rearrangements .',\n",
       "  [['SCL protein was detected in <e1> erythroid cell line </e1> , as well as in several lymphoid leukemia cell lines which are known to harbor SCL gene rearrangements .',\n",
       "    'CELL_LINE']]],\n",
       " ['SCL protein was detected in erythroid , megakaryocyte cell line, as well as in several lymphoid leukemia cell lines which are known to harbor SCL gene rearrangements .',\n",
       "  [['SCL protein was detected in erythroid , <e1> megakaryocyte cell line </e1> , as well as in several lymphoid leukemia cell lines which are known to harbor SCL gene rearrangements .',\n",
       "    'CELL_LINE']]],\n",
       " ['SCL protein was detected in erythroid , megakaryocyte , mast cell line, as well as in several lymphoid leukemia cell lines which are known to harbor SCL gene rearrangements .',\n",
       "  [['SCL protein was detected in erythroid , megakaryocyte , <e1> mast cell line </e1> , as well as in several lymphoid leukemia cell lines which are known to harbor SCL gene rearrangements .',\n",
       "    'CELL_LINE']]],\n",
       " ['Cytomegalovirus ( CMV ) infection is nonpermissive or persistent in many lymphoid cell typebut can be activated in differentiated macrophages .',\n",
       "  [['Cytomegalovirus ( CMV ) infection is nonpermissive or persistent in many <e1> lymphoid cell type </e1> but can be activated in differentiated macrophages .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Two multicopy basal enhancer motifs within the SCMV MIE enhancer , namely , 11 copies of the 16 - bp cyclic AMP response element ( CRE ) and 3 copies of novel 17 - bp serum response factor ( SRF ) binding sites referred to as the SNE ( SRF/NFkappaB - like element ) , as well as four classical NFkappaB sites within the HCMV version , contribute to TPA responsiveness in transient assays in monocyte type .',\n",
       "  [['Two multicopy basal enhancer motifs within the SCMV MIE enhancer , namely , 11 copies of the 16 - bp cyclic AMP response element ( CRE ) and 3 copies of novel 17 - bp serum response factor ( SRF ) binding sites referred to as the SNE ( SRF/NFkappaB - like element ) , as well as four classical NFkappaB sites within the HCMV version , contribute to TPA responsiveness in transient assays in <e1> monocyte type </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The SCMV SNE sites contain potential overlapping core recognition binding motifs for SRF , Rel/NFkappaB , ETS , and YY1 class transcription factors but fail to respond to either serum necrosis factor alpha .',\n",
       "  [['The SCMV SNE sites contain potential overlapping core recognition binding motifs for SRF , Rel/NFkappaB , ETS , and YY1 class transcription factors but fail to respond to either <e1> serum necrosis factor alpha </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the hcmv mie enhancer, we have examined the functional responses and protein binding properties of multimerized wild - type and mutant elements added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "  [['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the <e1> hcmv mie enhancer </e1> , we have examined the functional responses and protein binding properties of multimerized wild - type and mutant elements added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "    'DNA']]],\n",
       " ['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of multimerized wild - type elementadded upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "  [['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of <e1> multimerized wild - type element </e1> added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "    'DNA']]],\n",
       " ['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of multimerized mutant elementadded upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "  [['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of <e1> multimerized mutant element </e1> added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "    'DNA']]],\n",
       " ['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the hcmv mie enhancer, we have examined the functional responses and protein binding properties of multimerized wild - type and mutant elements added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "  [['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the <e1> hcmv mie enhancer </e1> , we have examined the functional responses and protein binding properties of multimerized wild - type and mutant elements added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "    'DNA']]],\n",
       " ['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of multimerized wild - type elementadded upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "  [['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of <e1> multimerized wild - type element </e1> added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "    'DNA']]],\n",
       " ['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of multimerized mutant elementadded upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "  [['Therefore , to evaluate the mechanism of TPA responsiveness of the SNE motifs and of a related 16 - bp SEE ( SRF/ ETS element ) motif found in the HCMV and chimpanzee CMV MIE enhancers , we have examined the functional responses and protein binding properties of <e1> multimerized mutant element </e1> added upstream to the SCMV MIE or simian virus 40 minimal promoter regions in the U - 937 , K - 562 , HL - 60 , THP - 1 , and Jurkat cell lines .',\n",
       "    'DNA']]],\n",
       " ['Slight differences in the sequence of the core SRF binding site compared with that of the classical c - Fos promoter serum response element , together with differences in the spacing between the srf motif, appear to account for the inability of the SCMV SNEs to respond to serum induction .',\n",
       "  [['Slight differences in the sequence of the core SRF binding site compared with that of the classical c - Fos promoter serum response element , together with differences in the spacing between the <e1> srf motif </e1> , appear to account for the inability of the SCMV SNEs to respond to serum induction .',\n",
       "    'DNA']]],\n",
       " ['Murine fra - 1 was highly homologous to rat fra - 1 .',\n",
       "  [['Murine fra - 1 was highly homologous to <e1> rat fra - 1 </e1> .',\n",
       "    'DNA']]],\n",
       " ['The PB1 sequence motif is highly conserved in both human gm - csf gene .',\n",
       "  [['The PB1 sequence motif is highly conserved in both <e1> human gm - csf gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['In vitro mutagenesis experiments demonstrated that both the ets siteare required for binding of the inducible PB1 nuclear protein complex and for the transcriptional activity of this element and the GM - CSF promoter in activated T cells .',\n",
       "  [['In vitro mutagenesis experiments demonstrated that both the <e1> ets site </e1> are required for binding of the inducible PB1 nuclear protein complex and for the transcriptional activity of this element and the GM - CSF promoter in activated T cells .',\n",
       "    'DNA']]],\n",
       " ['Surprisingly , traf - 2, or - 3 does not interact with the terminal LMP1 CT aa 333 to 386 which can independently mediate NF - kappaB activation .',\n",
       "  [['Surprisingly , <e1> traf - 2 </e1> , or - 3 does not interact with the terminal LMP1 CT aa 333 to 386 which can independently mediate NF - kappaB activation .',\n",
       "    'PROTEIN']]],\n",
       " ['Surprisingly , traf - 3does not interact with the terminal LMP1 CT aa 333 to 386 which can independently mediate NF - kappaB activation .',\n",
       "  [['Surprisingly , <e1> traf - 3 </e1> does not interact with the terminal LMP1 CT aa 333 to 386 which can independently mediate NF - kappaB activation .',\n",
       "    'PROTEIN']]],\n",
       " ['In this report we address the contribution of the regulatory domainof pp56(lck) to another function of this enzyme independent of CD4 : TCR - induced IL - 2 production .',\n",
       "  [['In this report we address the contribution of the <e1> regulatory domain </e1> of pp56(lck) to another function of this enzyme independent of CD4 : TCR - induced IL - 2 production .',\n",
       "    'PROTEIN']]],\n",
       " ['PRRI is an inducible proximal enhancer , located between nucleotides - 276 and - 244 , which contains nf - kappab motif .',\n",
       "  [['PRRI is an inducible proximal enhancer , located between nucleotides - 276 and - 244 , which contains <e1> nf - kappab motif </e1> .',\n",
       "    'DNA']]],\n",
       " ['IL - 2 induced the binding of stat 5a proteinto the human GASd element .',\n",
       "  [['IL - 2 induced the binding of <e1> stat 5a protein </e1> to the human GASd element .',\n",
       "    'PROTEIN']]],\n",
       " ['IL - 2 induced the binding of stat 5b proteinto the human GASd element .',\n",
       "  [['IL - 2 induced the binding of <e1> stat 5b protein </e1> to the human GASd element .',\n",
       "    'PROTEIN']]],\n",
       " ['p52 and RelB were detected in the nuclei in both CD1a+ and CD68+ cells from the T cell area , whereas p50 was found only in cd68 - cell .',\n",
       "  [['p52 and RelB were detected in the nuclei in both CD1a+ and CD68+ cells from the T cell area , whereas p50 was found only in <e1> cd68 - cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['When the promoter region was linked with a heterologous reporter gene , we found that the promoter region is inducible by both interferons ( interferon - gamma) and interferon regulatory factor 1 .',\n",
       "  [['When the promoter region was linked with a heterologous reporter gene , we found that the promoter region is inducible by both interferons ( <e1> interferon - gamma </e1> ) and interferon regulatory factor 1 .',\n",
       "    'PROTEIN']]],\n",
       " ['Freshly isolated cells show high levels of fosb/g0s3 mrna, which decline rapidly during incubation in culture medium .',\n",
       "  [['Freshly isolated cells show high levels of <e1> fosb/g0s3 mrna </e1> , which decline rapidly during incubation in culture medium .',\n",
       "    'RNA']]],\n",
       " ['In freshly isolated cells , both fos mrnaincrease dramatically in response to the protein synthesis inhibitor cycloheximide .',\n",
       "  [['In freshly isolated cells , both <e1> fos mrna </e1> increase dramatically in response to the protein synthesis inhibitor cycloheximide .',\n",
       "    'RNA']]],\n",
       " ['In fetal liver of PU.1 - / - mice , which lack lymphoid cell, expression of Btk was reduced two - to threefold but not abolished .',\n",
       "  [['In fetal liver of PU.1 - / - mice , which lack <e1> lymphoid cell </e1> , expression of Btk was reduced two - to threefold but not abolished .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Collectively this study shows that expression of the Btk gene is regulated by the combined action of sp1 - family member .',\n",
       "  [['Collectively this study shows that expression of the Btk gene is regulated by the combined action of <e1> sp1 - family member </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Studies of cell lines representing endocrine lineageusing polyclonal antisera to detect E2A proteins have suggested that significant E2A protein expression is restricted to B - lymphocytes .',\n",
       "  [['Studies of cell lines representing <e1> endocrine lineage </e1> using polyclonal antisera to detect E2A proteins have suggested that significant E2A protein expression is restricted to B - lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Studies of cell lines representing endocrine , fibroblast lineageusing polyclonal antisera to detect E2A proteins have suggested that significant E2A protein expression is restricted to B - lymphocytes .',\n",
       "  [['Studies of cell lines representing endocrine , <e1> fibroblast lineage </e1> using polyclonal antisera to detect E2A proteins have suggested that significant E2A protein expression is restricted to B - lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We have developed a monoclonal antibody , Yae , which is specific for Pan/ E2A proteins , and have used the Yae antibody to examine a variety of endocrine cell lineagefor differences in Pan/ E2A protein expression , subcellular localization , and heteromeric complex formation .',\n",
       "  [['We have developed a monoclonal antibody , Yae , which is specific for Pan/ E2A proteins , and have used the Yae antibody to examine a variety of <e1> endocrine cell lineage </e1> for differences in Pan/ E2A protein expression , subcellular localization , and heteromeric complex formation .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In contrast to previous results obtained using polyclonal antiseras to detect Pan/ E2A proteins , we report comparable levels of Pan proteins in gh/prl - producing lymphocyte cell .',\n",
       "  [['In contrast to previous results obtained using polyclonal antiseras to detect Pan/ E2A proteins , we report comparable levels of Pan proteins in <e1> gh/prl - producing lymphocyte cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In contrast to previous results obtained using polyclonal antiseras to detect Pan/ E2A proteins , we report comparable levels of Pan proteins in GH/PRL - and insulin - producing lymphocyte cell .',\n",
       "  [['In contrast to previous results obtained using polyclonal antiseras to detect Pan/ E2A proteins , we report comparable levels of Pan proteins in GH/PRL - and <e1> insulin - producing lymphocyte cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In contrast to previous results obtained using polyclonal antiseras to detect Pan/ E2A proteins , we report comparable levels of Pan proteins in GH/PRL - and insulin - producing , b - lymphocyte cell .',\n",
       "  [['In contrast to previous results obtained using polyclonal antiseras to detect Pan/ E2A proteins , we report comparable levels of Pan proteins in GH/PRL - and insulin - producing , <e1> b - lymphocyte cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " [' nasal nk - cell lymphomashare the same type of Epstein - Barr virus latency as nasopharyngeal carcinoma and Hodgkin s disease .',\n",
       "  [[' <e1> nasal nk - cell lymphoma </e1> share the same type of Epstein - Barr virus latency as nasopharyngeal carcinoma and Hodgkin s disease .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Both nk - cell subtypeare closely associated with Epstein - Barr virus ( EBV ) .',\n",
       "  [['Both <e1> nk - cell subtype </e1> are closely associated with Epstein - Barr virus ( EBV ) .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In addition , analysis of EBV protein expression by IH revealed a heterogeneous pattern of EBV gene expression at the single - cell level consisting of both lmp1+ tumour cell, suggesting a mixture of latency I and II .',\n",
       "  [['In addition , analysis of EBV protein expression by IH revealed a heterogeneous pattern of EBV gene expression at the single - cell level consisting of both <e1> lmp1+ tumour cell </e1> , suggesting a mixture of latency I and II .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Now we have examined whether naive t lymphocytediffer in their sensitivity to ROI - mediated signals .',\n",
       "  [['Now we have examined whether <e1> naive t lymphocyte </e1> differ in their sensitivity to ROI - mediated signals .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not jun family member, renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "  [['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not <e1> jun family member </e1> , renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "    'PROTEIN']]],\n",
       " ['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not Jun , fos family member, renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "  [['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not Jun , <e1> fos family member </e1> , renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "    'PROTEIN']]],\n",
       " ['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not Jun , Fos , nf - kappab family member, renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "  [['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not Jun , Fos , <e1> nf - kappab family member </e1> , renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "    'PROTEIN']]],\n",
       " ['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not Jun , Fos , NF - kappaB , oct family member, renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "  [['Furthermore , overexpression and constitutive nuclear localization of NF - AT , but not Jun , Fos , NF - kappaB , <e1> oct family member </e1> , renders the interleukin - 2 enhancer in Jurkat T lymphocytes resistant to FK506 and cyclosporin A .',\n",
       "    'PROTEIN']]],\n",
       " ['Subsequent studies have revealed that ( i ) cytoplasmic RelA is stably associated not only with I kappa B alpha but also with other ankyrin motif - rich proteins including the products of the nf - kappa b2 p100 genes; ( ii ) in contrast to RelA - I kappa B alpha , RelA - p100 cytoplasmic complexes are not dissociated following tumor necrosis factor alpha activation ; ( iii ) p100 functions as a potent inhibitor of RelA - mediated transcription in vivo ; ( iv ) the interaction of RelA and p100 involves the conserved Rel homology domain of both proteins but not the nuclear localization signal of RelA , which is required for I kappa B alpha binding ; ( v ) p100 inhibition of RelA function requires the C - terminal ankyrin motif domain , which mediates cytoplasmic retention of RelA ; and ( vi ) as observed with I kappa B alpha , nuclear RelA stimulates p100 mRNA and protein expression .',\n",
       "  [['Subsequent studies have revealed that ( i ) cytoplasmic RelA is stably associated not only with I kappa B alpha but also with other ankyrin motif - rich proteins including the products of the <e1> nf - kappa b2 p100 genes </e1> ; ( ii ) in contrast to RelA - I kappa B alpha , RelA - p100 cytoplasmic complexes are not dissociated following tumor necrosis factor alpha activation ; ( iii ) p100 functions as a potent inhibitor of RelA - mediated transcription in vivo ; ( iv ) the interaction of RelA and p100 involves the conserved Rel homology domain of both proteins but not the nuclear localization signal of RelA , which is required for I kappa B alpha binding ; ( v ) p100 inhibition of RelA function requires the C - terminal ankyrin motif domain , which mediates cytoplasmic retention of RelA ; and ( vi ) as observed with I kappa B alpha , nuclear RelA stimulates p100 mRNA and protein expression .',\n",
       "    'DNA']]],\n",
       " ['Subsequent studies have revealed that ( i ) cytoplasmic RelA is stably associated not only with I kappa B alpha but also with other ankyrin motif - rich proteins including the products of the nf - kappa b1 p105 genes; ( ii ) in contrast to RelA - I kappa B alpha , RelA - p100 cytoplasmic complexes are not dissociated following tumor necrosis factor alpha activation ; ( iii ) p100 functions as a potent inhibitor of RelA - mediated transcription in vivo ; ( iv ) the interaction of RelA and p100 involves the conserved Rel homology domain of both proteins but not the nuclear localization signal of RelA , which is required for I kappa B alpha binding ; ( v ) p100 inhibition of RelA function requires the C - terminal ankyrin motif domain , which mediates cytoplasmic retention of RelA ; and ( vi ) as observed with I kappa B alpha , nuclear RelA stimulates p100 mRNA and protein expression .',\n",
       "  [['Subsequent studies have revealed that ( i ) cytoplasmic RelA is stably associated not only with I kappa B alpha but also with other ankyrin motif - rich proteins including the products of the <e1> nf - kappa b1 p105 genes </e1> ; ( ii ) in contrast to RelA - I kappa B alpha , RelA - p100 cytoplasmic complexes are not dissociated following tumor necrosis factor alpha activation ; ( iii ) p100 functions as a potent inhibitor of RelA - mediated transcription in vivo ; ( iv ) the interaction of RelA and p100 involves the conserved Rel homology domain of both proteins but not the nuclear localization signal of RelA , which is required for I kappa B alpha binding ; ( v ) p100 inhibition of RelA function requires the C - terminal ankyrin motif domain , which mediates cytoplasmic retention of RelA ; and ( vi ) as observed with I kappa B alpha , nuclear RelA stimulates p100 mRNA and protein expression .',\n",
       "    'DNA']]],\n",
       " ['In this study , we obtained evidence that transfection of the cDNAs for cn a catalytic subunitalso augments transcription from the GM - CSF promoter and recovers the transcription inhibited by CsA .',\n",
       "  [['In this study , we obtained evidence that transfection of the cDNAs for <e1> cn a catalytic subunit </e1> also augments transcription from the GM - CSF promoter and recovers the transcription inhibited by CsA .',\n",
       "    'PROTEIN']]],\n",
       " ['In this study , we obtained evidence that transfection of the cDNAs for cn b regulatory subunitalso augments transcription from the GM - CSF promoter and recovers the transcription inhibited by CsA .',\n",
       "  [['In this study , we obtained evidence that transfection of the cDNAs for <e1> cn b regulatory subunit </e1> also augments transcription from the GM - CSF promoter and recovers the transcription inhibited by CsA .',\n",
       "    'PROTEIN']]],\n",
       " ['These results suggest that CN is involved in the coordinated induction of the gm - csf geneis a functional analogue of the NF - AT - binding site in the IL - 2 promoter , which mediates signals downstream of T cell activation .',\n",
       "  [['These results suggest that CN is involved in the coordinated induction of the <e1> gm - csf gene </e1> is a functional analogue of the NF - AT - binding site in the IL - 2 promoter , which mediates signals downstream of T cell activation .',\n",
       "    'DNA']]],\n",
       " ['These results suggest that CN is involved in the coordinated induction of the GM - CSF and il - 2 geneis a functional analogue of the NF - AT - binding site in the IL - 2 promoter , which mediates signals downstream of T cell activation .',\n",
       "  [['These results suggest that CN is involved in the coordinated induction of the GM - CSF and <e1> il - 2 gene </e1> is a functional analogue of the NF - AT - binding site in the IL - 2 promoter , which mediates signals downstream of T cell activation .',\n",
       "    'DNA']]],\n",
       " ['These results suggest that blasts in M7 and TMD may correspond to progenitors of both erythroid lineage .',\n",
       "  [['These results suggest that blasts in M7 and TMD may correspond to progenitors of both <e1> erythroid lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['E3 transcripts are present in the myeloid lineage, absent in nonhematopoietic cells , and encode a highly hydrophobic , potentially phosphorylated polypeptide of unknown function with significant homology to a putative protein expressed in myeloid cells .',\n",
       "  [['E3 transcripts are present in the <e1> myeloid lineage </e1> , absent in nonhematopoietic cells , and encode a highly hydrophobic , potentially phosphorylated polypeptide of unknown function with significant homology to a putative protein expressed in myeloid cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['E3 transcripts are present in the myeloid , b - lymphoid lineage, absent in nonhematopoietic cells , and encode a highly hydrophobic , potentially phosphorylated polypeptide of unknown function with significant homology to a putative protein expressed in myeloid cells .',\n",
       "  [['E3 transcripts are present in the myeloid , <e1> b - lymphoid lineage </e1> , absent in nonhematopoietic cells , and encode a highly hydrophobic , potentially phosphorylated polypeptide of unknown function with significant homology to a putative protein expressed in myeloid cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We have identified two new isoforms of the transcription factor NFAT1 ( previously termed NFATp ) that are the predominant isoforms expressed in murine t cell .',\n",
       "  [['We have identified two new isoforms of the transcription factor NFAT1 ( previously termed NFATp ) that are the predominant isoforms expressed in <e1> murine t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We previously characterized drosophila taf subunitthat make up the core TFIID complex found in all cells .',\n",
       "  [['We previously characterized <e1> drosophila taf subunit </e1> that make up the core TFIID complex found in all cells .',\n",
       "    'PROTEIN']]],\n",
       " ['No instance of microsatellite instability was observed in a total of 880 comparisons of granulocyte dna .',\n",
       "  [['No instance of microsatellite instability was observed in a total of 880 comparisons of <e1> granulocyte dna </e1> .',\n",
       "    'DNA']]],\n",
       " ['In t - cell line, the E ( gre ) motifs were of major importance for Akv MLV transcriptional activity , while the EA/S motif had some effect .',\n",
       "  [['In <e1> t - cell line </e1> , the E ( gre ) motifs were of major importance for Akv MLV transcriptional activity , while the EA/S motif had some effect .',\n",
       "    'CELL_LINE']]],\n",
       " ['The positive region can be divided into an upstream regulatory region .',\n",
       "  [['The positive region can be divided into an <e1> upstream regulatory region </e1> .',\n",
       "    'DNA']]],\n",
       " ['Northern analysis using a probe encompassing a common region of both isoforms showed two mRNA species of 2.7 kilobase pairmRNA which was preferentially expressed in the spleen .',\n",
       "  [['Northern analysis using a probe encompassing a common region of both isoforms showed two mRNA species of <e1> 2.7 kilobase pair </e1> mRNA which was preferentially expressed in the spleen .',\n",
       "    'RNA']]],\n",
       " ['Northern analysis using a probe encompassing a common region of both isoforms showed two mRNA species of 2.7 and 4.5 kilobase pairmRNA which was preferentially expressed in the spleen .',\n",
       "  [['Northern analysis using a probe encompassing a common region of both isoforms showed two mRNA species of 2.7 and <e1> 4.5 kilobase pair </e1> mRNA which was preferentially expressed in the spleen .',\n",
       "    'RNA']]],\n",
       " ['Latent membrane protein ( LMP ) 2A mRNA was detected in three of seven cases ; however , neither lmp1 mrnawas detected in any of the tumours tested .',\n",
       "  [['Latent membrane protein ( LMP ) 2A mRNA was detected in three of seven cases ; however , neither <e1> lmp1 mrna </e1> was detected in any of the tumours tested .',\n",
       "    'RNA']]],\n",
       " ['Both are located in the particulate fraction following platelet activation and are also able to bind to a GST - construct containing the sh2 domainof phospholipase C gamma 1 .',\n",
       "  [['Both are located in the particulate fraction following platelet activation and are also able to bind to a GST - construct containing the <e1> sh2 domain </e1> of phospholipase C gamma 1 .',\n",
       "    'PROTEIN']]],\n",
       " ['In this study , we characterize an essential upstream IL - 2 response element that contains both consensus gas motif, and a GATA motif , which overlaps the non - consensus GAS motif .',\n",
       "  [['In this study , we characterize an essential upstream IL - 2 response element that contains both <e1> consensus gas motif </e1> , and a GATA motif , which overlaps the non - consensus GAS motif .',\n",
       "    'DNA']]],\n",
       " ['In this study , we characterize an essential upstream IL - 2 response element that contains both consensus and non - consensus gas motif, and a GATA motif , which overlaps the non - consensus GAS motif .',\n",
       "  [['In this study , we characterize an essential upstream IL - 2 response element that contains both consensus and <e1> non - consensus gas motif </e1> , and a GATA motif , which overlaps the non - consensus GAS motif .',\n",
       "    'DNA']]],\n",
       " ['Thus , IL - 2 - induced IL - 2R alpha promoter activity requires a complex upstream element , which appears to contain binding sites for both positive regulatory factor .',\n",
       "  [['Thus , IL - 2 - induced IL - 2R alpha promoter activity requires a complex upstream element , which appears to contain binding sites for both <e1> positive regulatory factor </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Cloning and characterization of the beta subunit of human proximal sequence element - binding transcription factor and its involvement in transcription of small nuclear RNA genes by rna polymerase iii .',\n",
       "  [['Cloning and characterization of the beta subunit of human proximal sequence element - binding transcription factor and its involvement in transcription of small nuclear RNA genes by <e1> rna polymerase iii </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['We previously reported the purification of human PTF , a complex of four subunits , and the molecular cloning and characterization of ptf gamma subunit .',\n",
       "  [['We previously reported the purification of human PTF , a complex of four subunits , and the molecular cloning and characterization of <e1> ptf gamma subunit </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['We previously reported the purification of human PTF , a complex of four subunits , and the molecular cloning and characterization of ptf delta subunit .',\n",
       "  [['We previously reported the purification of human PTF , a complex of four subunits , and the molecular cloning and characterization of <e1> ptf delta subunit </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Abnormalities of p16 genein recurrent malignant astrocytomas .',\n",
       "  [['Abnormalities of <e1> p16 gene </e1> in recurrent malignant astrocytomas .',\n",
       "    'DNA']]],\n",
       " ['Abnormalities of p16 , p15 genein recurrent malignant astrocytomas .',\n",
       "  [['Abnormalities of p16 , <e1> p15 gene </e1> in recurrent malignant astrocytomas .',\n",
       "    'DNA']]],\n",
       " ['Abnormalities in the p16 genethat regulate transition through the G1 phase of the cell cycle have been implicated in the malignant progression of astrocytomas .',\n",
       "  [['Abnormalities in the <e1> p16 gene </e1> that regulate transition through the G1 phase of the cell cycle have been implicated in the malignant progression of astrocytomas .',\n",
       "    'DNA']]],\n",
       " ['Abnormalities in the p16 , p15 genethat regulate transition through the G1 phase of the cell cycle have been implicated in the malignant progression of astrocytomas .',\n",
       "  [['Abnormalities in the p16 , <e1> p15 gene </e1> that regulate transition through the G1 phase of the cell cycle have been implicated in the malignant progression of astrocytomas .',\n",
       "    'DNA']]],\n",
       " ['The primary and recurrent tumors in Group A did not show structural alterations in the p16 gene, whereas homozygous codeletion of p16 and p15 was observed in both primary and recurrent tumors in Group B .',\n",
       "  [['The primary and recurrent tumors in Group A did not show structural alterations in the <e1> p16 gene </e1> , whereas homozygous codeletion of p16 and p15 was observed in both primary and recurrent tumors in Group B .',\n",
       "    'DNA']]],\n",
       " ['The primary and recurrent tumors in Group A did not show structural alterations in the p16 , p15 gene, whereas homozygous codeletion of p16 and p15 was observed in both primary and recurrent tumors in Group B .',\n",
       "  [['The primary and recurrent tumors in Group A did not show structural alterations in the p16 , <e1> p15 gene </e1> , whereas homozygous codeletion of p16 and p15 was observed in both primary and recurrent tumors in Group B .',\n",
       "    'DNA']]],\n",
       " ['Signals leading to the activation of NF - kappa B transcription factor are stronger in neonatal t lymphocyte .',\n",
       "  [['Signals leading to the activation of NF - kappa B transcription factor are stronger in <e1> neonatal t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The data obtained demonstrate that phorbol dibutyrate/calcium ionophore A23187 ( PDBu/iono ) combination induced a clearly higher nuclear translocation of NF - kappa B in neonatal t cell .',\n",
       "  [['The data obtained demonstrate that phorbol dibutyrate/calcium ionophore A23187 ( PDBu/iono ) combination induced a clearly higher nuclear translocation of NF - kappa B in <e1> neonatal t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 protein, indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "  [['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the <e1> p50 protein </e1> , indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and rela p65 protein, indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "  [['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and <e1> rela p65 protein </e1> , indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and RelA ( p65 ) proteins , indicated that the composition of NF - kappa B was similar in neonatal cell .',\n",
       "  [['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and RelA ( p65 ) proteins , indicated that the composition of NF - kappa B was similar in <e1> neonatal cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 protein, indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "  [['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the <e1> p50 protein </e1> , indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and rela p65 protein, indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "  [['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and <e1> rela p65 protein </e1> , indicated that the composition of NF - kappa B was similar in neonatal and adult cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and RelA ( p65 ) proteins , indicated that the composition of NF - kappa B was similar in neonatal cell .',\n",
       "  [['Analysis of the nuclear extracts with antibodies directed against the major components of NF - kappa B the p50 and RelA ( p65 ) proteins , indicated that the composition of NF - kappa B was similar in <e1> neonatal cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Studies on unicellular daughter cell cultureunequivocally indicate that the shift is mediated by modulation of the HPC differentiation program to the granulopoietic pathway ( rather than RA - induced down - modulation of multipotent / erythroid /monocytic HPC growth coupled with recruitment of granulocytic HPCs ) .',\n",
       "  [['Studies on <e1> unicellular daughter cell culture </e1> unequivocally indicate that the shift is mediated by modulation of the HPC differentiation program to the granulopoietic pathway ( rather than RA - induced down - modulation of multipotent / erythroid /monocytic HPC growth coupled with recruitment of granulocytic HPCs ) .',\n",
       "    'CELL_LINE']]],\n",
       " ['In the present study , corticosteroid type i receptorin mononuclear leukocytes were measured in 10 elderly subjects and in 9 young adults , before and after overnight DST ( 1 mg ) .',\n",
       "  [['In the present study , <e1> corticosteroid type i receptor </e1> in mononuclear leukocytes were measured in 10 elderly subjects and in 9 young adults , before and after overnight DST ( 1 mg ) .',\n",
       "    'PROTEIN']]],\n",
       " ['In the present study , corticosteroid type ii receptorin mononuclear leukocytes were measured in 10 elderly subjects and in 9 young adults , before and after overnight DST ( 1 mg ) .',\n",
       "  [['In the present study , <e1> corticosteroid type ii receptor </e1> in mononuclear leukocytes were measured in 10 elderly subjects and in 9 young adults , before and after overnight DST ( 1 mg ) .',\n",
       "    'PROTEIN']]],\n",
       " ['The number of type i receptorbefore the test was lower in elderly subjects than in adults .',\n",
       "  [['The number of <e1> type i receptor </e1> before the test was lower in elderly subjects than in adults .',\n",
       "    'PROTEIN']]],\n",
       " ['The number of type ii receptorsbefore the test was lower in elderly subjects than in adults .',\n",
       "  [['The number of <e1> type ii receptors </e1> before the test was lower in elderly subjects than in adults .',\n",
       "    'PROTEIN']]],\n",
       " [' il4 receptorshare the gamma c chain and activate STAT6 , STAT3 and STAT5 proteins in normal human B cells .',\n",
       "  [[' <e1> il4 receptor </e1> share the gamma c chain and activate STAT6 , STAT3 and STAT5 proteins in normal human B cells .',\n",
       "    'PROTEIN']]],\n",
       " ['IL4 and IL13 receptors share the gamma c chain and activate stat6 proteinin normal human B cells .',\n",
       "  [['IL4 and IL13 receptors share the gamma c chain and activate <e1> stat6 protein </e1> in normal human B cells .',\n",
       "    'PROTEIN']]],\n",
       " ['IL4 and IL13 receptors share the gamma c chain and activate STAT6 , stat3 proteinin normal human B cells .',\n",
       "  [['IL4 and IL13 receptors share the gamma c chain and activate STAT6 , <e1> stat3 protein </e1> in normal human B cells .',\n",
       "    'PROTEIN']]],\n",
       " [' il4 receptorshare the gamma c chain and activate STAT6 , STAT3 and STAT5 proteins in normal human B cells .',\n",
       "  [[' <e1> il4 receptor </e1> share the gamma c chain and activate STAT6 , STAT3 and STAT5 proteins in normal human B cells .',\n",
       "    'PROTEIN']]],\n",
       " ['IL4 and IL13 receptors share the gamma c chain and activate stat6 proteinin normal human B cells .',\n",
       "  [['IL4 and IL13 receptors share the gamma c chain and activate <e1> stat6 protein </e1> in normal human B cells .',\n",
       "    'PROTEIN']]],\n",
       " ['IL4 and IL13 receptors share the gamma c chain and activate STAT6 , stat3 proteinin normal human B cells .',\n",
       "  [['IL4 and IL13 receptors share the gamma c chain and activate STAT6 , <e1> stat3 protein </e1> in normal human B cells .',\n",
       "    'PROTEIN']]],\n",
       " ['In previous studies , we identified calcineurin mutations that block binding by the cyclophilin a /csa complexand thereby render yeast cells resistant to the antifungal effects of CsA or FK506 .',\n",
       "  [['In previous studies , we identified calcineurin mutations that block binding by the <e1> cyclophilin a /csa complex </e1> and thereby render yeast cells resistant to the antifungal effects of CsA or FK506 .',\n",
       "    'PROTEIN']]],\n",
       " ['A critical role of sp1 - related transcription factorin maintaining CTL - specific expression of the mouse perforin gene .',\n",
       "  [['A critical role of <e1> sp1 - related transcription factor </e1> in maintaining CTL - specific expression of the mouse perforin gene .',\n",
       "    'PROTEIN']]],\n",
       " ['The previously defined minimal enhancer containing the mua siteis transactivated by a combination of the ETS - domain proteins PU.1 and Ets - 1 in nonlymphoid cells .',\n",
       "  [['The previously defined minimal enhancer containing the <e1> mua site </e1> is transactivated by a combination of the ETS - domain proteins PU.1 and Ets - 1 in nonlymphoid cells .',\n",
       "    'DNA']]],\n",
       " ['The previously defined minimal enhancer containing the muA , mue3 siteis transactivated by a combination of the ETS - domain proteins PU.1 and Ets - 1 in nonlymphoid cells .',\n",
       "  [['The previously defined minimal enhancer containing the muA , <e1> mue3 site </e1> is transactivated by a combination of the ETS - domain proteins PU.1 and Ets - 1 in nonlymphoid cells .',\n",
       "    'DNA']]],\n",
       " ['The core GGAAs of the mua siteare separated by 30 nucleotides , suggesting that ETS proteins bind to these sites from these same side of the DNA helix .',\n",
       "  [['The core GGAAs of the <e1> mua site </e1> are separated by 30 nucleotides , suggesting that ETS proteins bind to these sites from these same side of the DNA helix .',\n",
       "    'DNA']]],\n",
       " ['We propose that the requirement for precise spacing of the mua elementis due in part to a directed DNA bend induced by PU.1 .',\n",
       "  [['We propose that the requirement for precise spacing of the <e1> mua element </e1> is due in part to a directed DNA bend induced by PU.1 .',\n",
       "    'DNA']]],\n",
       " ['Our results , although limited , suggest that normal individuals can express both the wild - type and truncated version at the same time , whereas lupus patients only express either the wild - type er .',\n",
       "  [['Our results , although limited , suggest that normal individuals can express both the wild - type and truncated version at the same time , whereas lupus patients only express either the <e1> wild - type er </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Sialoadhesin is a macrophage - restricted member of the Ig superfamily that mediates adhesion with lymphoid cell .',\n",
       "  [['Sialoadhesin is a macrophage - restricted member of the Ig superfamily that mediates adhesion with <e1> lymphoid cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In addition , inhibition of mRNA synthesis with actinomycin D indicated that DOX induced an increased stability of pbgd mrna, whereas ACLA did not affect the half - lives of these mRNAs .',\n",
       "  [['In addition , inhibition of mRNA synthesis with actinomycin D indicated that DOX induced an increased stability of <e1> pbgd mrna </e1> , whereas ACLA did not affect the half - lives of these mRNAs .',\n",
       "    'RNA']]],\n",
       " ['Finally , GATA - 1 protein level was shown to be increased in acla - treated cell .',\n",
       "  [['Finally , GATA - 1 protein level was shown to be increased in <e1> acla - treated cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal b cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of <e1> normal b cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal t cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of <e1> normal t cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic b cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and <e1> leukemic b cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal t lymphocyte .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of <e1> normal t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic t lymphocyte .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and <e1> leukemic t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic b cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic <e1> b cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal b cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of <e1> normal b cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal t cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of <e1> normal t cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic b cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and <e1> leukemic b cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal t lymphocyte .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of <e1> normal t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic t lymphocyte .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and <e1> leukemic t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic b cell precursorand T lymphocytes .',\n",
       "  [['Interleukin - 7 ( IL - 7 ) stimulates the proliferation of normal and leukemic <e1> b cell precursor </e1> and T lymphocytes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['To determine the contribution of the cytoplasmic domain of the IL - 7 receptor alpha chain ( IL - 7R alpha ) to activation of STAT proteins , transfectants of the murine pro - B cell line BAF3 were made that express chimeric receptors consisting of the extracellular domain of human granulocyte colony - stimulating factor receptor ( G - CSF - R ) and the transmembrane domainof human IL - 7R alpha .',\n",
       "  [['To determine the contribution of the cytoplasmic domain of the IL - 7 receptor alpha chain ( IL - 7R alpha ) to activation of STAT proteins , transfectants of the murine pro - B cell line BAF3 were made that express chimeric receptors consisting of the extracellular domain of human granulocyte colony - stimulating factor receptor ( G - CSF - R ) and the <e1> transmembrane domain </e1> of human IL - 7R alpha .',\n",
       "    'PROTEIN']]],\n",
       " ['The oligo sequence I stimulated proliferation of both t cell .',\n",
       "  [['The oligo sequence I stimulated proliferation of both <e1> t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The percentage of cells expressing CD40 in t cell co - cultureincreased from 4.2 % to 13.8 % after oligo stimulation in patients , while it increased form 4.7 % to 48.6 % in healthy controls .',\n",
       "  [['The percentage of cells expressing CD40 in <e1> t cell co - culture </e1> increased from 4.2 % to 13.8 % after oligo stimulation in patients , while it increased form 4.7 % to 48.6 % in healthy controls .',\n",
       "    'CELL_LINE']]],\n",
       " ['IL - 12 - induced activation of nk celloccurs in the absence of immediate - early activation gene expression .',\n",
       "  [['IL - 12 - induced activation of <e1> nk cell </e1> occurs in the absence of immediate - early activation gene expression .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Using reverse - transcription PCR differential display , we isolated an mRNA species expressed in il - 2 - stimulated nk cell .',\n",
       "  [['Using reverse - transcription PCR differential display , we isolated an mRNA species expressed in <e1> il - 2 - stimulated nk cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Using reverse - transcription PCR differential display , we isolated an mRNA species expressed in il - 12 - stimulated nk cell .',\n",
       "  [['Using reverse - transcription PCR differential display , we isolated an mRNA species expressed in <e1> il - 12 - stimulated nk cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['This was identified as the mRNA encoding the transcription factor egr - 1 , which is expressed with fast kinetics in t cellupon IL - 2 , but not IL - 12 , stimulation .',\n",
       "  [['This was identified as the mRNA encoding the transcription factor egr - 1 , which is expressed with fast kinetics in <e1> t cell </e1> upon IL - 2 , but not IL - 12 , stimulation .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Analysis of the accumulation of mRNA - encoding members of the AP - 1 transcription factor family demonstrated that c - fos and junB are also expressed upon stimulation of nk cellwith IL - 2 , but not IL - 12 , whereas expression of c - jun and junD is not modified by either cytokine .',\n",
       "  [['Analysis of the accumulation of mRNA - encoding members of the AP - 1 transcription factor family demonstrated that c - fos and junB are also expressed upon stimulation of <e1> nk cell </e1> with IL - 2 , but not IL - 12 , whereas expression of c - jun and junD is not modified by either cytokine .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Analysis of the expression of genes reported to regulate cytokine - induced proliferation demonstrated that both IL - 2 and IL - 12 induce c - myc mRNA accumulation in nk cell, whereas only IL - 2 induces bcl - 2 expression .',\n",
       "  [['Analysis of the expression of genes reported to regulate cytokine - induced proliferation demonstrated that both IL - 2 and IL - 12 induce c - myc mRNA accumulation in <e1> nk cell </e1> , whereas only IL - 2 induces bcl - 2 expression .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Our data provide the first demonstration that IL - 12 - mediated activation of t celldoes not involve expression of members of the immediate - early activation genes family ( egr - 1 , c - fos , and junB ) , AP - 1 transcriptional activity , or bcl - 2 expression .',\n",
       "  [['Our data provide the first demonstration that IL - 12 - mediated activation of <e1> t cell </e1> does not involve expression of members of the immediate - early activation genes family ( egr - 1 , c - fos , and junB ) , AP - 1 transcriptional activity , or bcl - 2 expression .',\n",
       "    'CELL_TYPE']]],\n",
       " ['This indicates that functional differences observed in il - 2 - stimulated cellmay depend , at least in part , on differential gene regulation .',\n",
       "  [['This indicates that functional differences observed in <e1> il - 2 - stimulated cell </e1> may depend , at least in part , on differential gene regulation .',\n",
       "    'CELL_LINE']]],\n",
       " ['This indicates that functional differences observed in il - 12 - stimulated cellmay depend , at least in part , on differential gene regulation .',\n",
       "  [['This indicates that functional differences observed in <e1> il - 12 - stimulated cell </e1> may depend , at least in part , on differential gene regulation .',\n",
       "    'CELL_LINE']]],\n",
       " ['Here we demonstrate that the human gm - csf enhanceralso encompass binding sites for core - binding factor ( CBF ) .',\n",
       "  [['Here we demonstrate that the <e1> human gm - csf enhancer </e1> also encompass binding sites for core - binding factor ( CBF ) .',\n",
       "    'DNA']]],\n",
       " ['The aml1 genethat encode CBF each have the ability to influence cell growth and differentiation and have been implicated as proto - oncogenes in acute myeloid leukemia .',\n",
       "  [['The <e1> aml1 gene </e1> that encode CBF each have the ability to influence cell growth and differentiation and have been implicated as proto - oncogenes in acute myeloid leukemia .',\n",
       "    'DNA']]],\n",
       " ['However , PHA/ PMA but not oxidative signals induced the coordinate activation of reporter constructs containing the ap - 1 - tre promoter regionalong with IL - 2 mRNA expression .',\n",
       "  [['However , PHA/ PMA but not oxidative signals induced the coordinate activation of reporter constructs containing the <e1> ap - 1 - tre promoter region </e1> along with IL - 2 mRNA expression .',\n",
       "    'DNA']]],\n",
       " ['However , PHA/ PMA but not oxidative signals induced the coordinate activation of reporter constructs containing the AP - 1 - TRE , nf - at promoter regionalong with IL - 2 mRNA expression .',\n",
       "  [['However , PHA/ PMA but not oxidative signals induced the coordinate activation of reporter constructs containing the AP - 1 - TRE , <e1> nf - at promoter region </e1> along with IL - 2 mRNA expression .',\n",
       "    'DNA']]],\n",
       " ['Activation of human monocytic cells by treponema pallidum lipoproteinand synthetic lipopeptides proceeds via a pathway distinct from that of lipopolysaccharide but involves the transcriptional activator NF - kappa B .',\n",
       "  [['Activation of human monocytic cells by <e1> treponema pallidum lipoprotein </e1> and synthetic lipopeptides proceeds via a pathway distinct from that of lipopolysaccharide but involves the transcriptional activator NF - kappa B .',\n",
       "    'PROTEIN']]],\n",
       " ['Analysis of GR chimeric receptors containing the potent vp16 viral transactivation domainin place of the GR N terminus revealed that even low level expression of these receptors resulted in both enhanced steroid sensitivity and MMTV induction , thus supporting a role for transactivation in apoptosis .',\n",
       "  [['Analysis of GR chimeric receptors containing the potent <e1> vp16 viral transactivation domain </e1> in place of the GR N terminus revealed that even low level expression of these receptors resulted in both enhanced steroid sensitivity and MMTV induction , thus supporting a role for transactivation in apoptosis .',\n",
       "    'DNA']]],\n",
       " ['Finally , IL - 2 induced tyrosine phosphorylation of JAK1 and JAK3 , while IL - 12 induced phosphorylation of JAK2 and TYK2 in both preactivated primary nk cell .',\n",
       "  [['Finally , IL - 2 induced tyrosine phosphorylation of JAK1 and JAK3 , while IL - 12 induced phosphorylation of JAK2 and TYK2 in both <e1> preactivated primary nk cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Finally , IL - 2 induced tyrosine phosphorylation of JAK1 and JAK3 , while IL - 12 induced phosphorylation of JAK2 and TYK2 in both preactivated primary nk3.3 cell .',\n",
       "  [['Finally , IL - 2 induced tyrosine phosphorylation of JAK1 and JAK3 , while IL - 12 induced phosphorylation of JAK2 and TYK2 in both <e1> preactivated primary nk3.3 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['The proliferative responses of purified cd4+ cellare suppressed more strongly by TPCK when anti - CD28 rather than the phorbol ester PMA is used as the mitogenic coactivator .',\n",
       "  [['The proliferative responses of <e1> purified cd4+ cell </e1> are suppressed more strongly by TPCK when anti - CD28 rather than the phorbol ester PMA is used as the mitogenic coactivator .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The proliferative responses of purified cd8+ t cellare suppressed more strongly by TPCK when anti - CD28 rather than the phorbol ester PMA is used as the mitogenic coactivator .',\n",
       "  [['The proliferative responses of <e1> purified cd8+ t cell </e1> are suppressed more strongly by TPCK when anti - CD28 rather than the phorbol ester PMA is used as the mitogenic coactivator .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Cytokine ( IL - 2 , IL - 6 , INF - gamma ) production is inhibited 95 - 100 % by concentrations of TPCK that totally suppress the mitogenesis of cd4+ cell .',\n",
       "  [['Cytokine ( IL - 2 , IL - 6 , INF - gamma ) production is inhibited 95 - 100 % by concentrations of TPCK that totally suppress the mitogenesis of <e1> cd4+ cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Cytokines that bind to the interleukin - 2 ( IL - 2 ) receptor common gamma chain ( gamma c ) , including IL - 2 , IL - 4 , IL - 7 , IL - 9 , and IL - 15 , are important for the growth and differentiation of t lymphocyte, natural killer cells , macrophages , and monoctyes .',\n",
       "  [['Cytokines that bind to the interleukin - 2 ( IL - 2 ) receptor common gamma chain ( gamma c ) , including IL - 2 , IL - 4 , IL - 7 , IL - 9 , and IL - 15 , are important for the growth and differentiation of <e1> t lymphocyte </e1> , natural killer cells , macrophages , and monoctyes .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Yet , little is known as to how the ra receptor networkoperates in hematopoietic cells , and whether receptor interactions can explain the interplay between the RA - and VitD3 - signaling pathways during differentiation .',\n",
       "  [['Yet , little is known as to how the <e1> ra receptor network </e1> operates in hematopoietic cells , and whether receptor interactions can explain the interplay between the RA - and VitD3 - signaling pathways during differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Although the band 14q11 is a locus of t - cell receptor alpha - chain( TCR alpha/delta ) , lymphoma cells expressed B - cell , IgGk phenotype .',\n",
       "  [['Although the band 14q11 is a locus of <e1> t - cell receptor alpha - chain </e1> ( TCR alpha/delta ) , lymphoma cells expressed B - cell , IgGk phenotype .',\n",
       "    'PROTEIN']]],\n",
       " ['Although the band 14q11 is a locus of t - cell receptor delta - chain( TCR alpha/delta ) , lymphoma cells expressed B - cell , IgGk phenotype .',\n",
       "  [['Although the band 14q11 is a locus of <e1> t - cell receptor delta - chain </e1> ( TCR alpha/delta ) , lymphoma cells expressed B - cell , IgGk phenotype .',\n",
       "    'PROTEIN']]],\n",
       " ['This is achieved in part through the positive activities of the ap - 1 sitein the IL - 3 promoter .',\n",
       "  [['This is achieved in part through the positive activities of the <e1> ap - 1 site </e1> in the IL - 3 promoter .',\n",
       "    'DNA']]],\n",
       " ['Transcription factors of t lymphocyte- - basic research and clinical perspectives for gastroenterology .',\n",
       "  [['Transcription factors of <e1> t lymphocyte </e1> - - basic research and clinical perspectives for gastroenterology .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Furthermore , the most important trans - acting factors of t lymphocyte( e.g. NF - kB , NF - AT and STAT families ) and their functional importance are described .',\n",
       "  [['Furthermore , the most important trans - acting factors of <e1> t lymphocyte </e1> ( e.g. NF - kB , NF - AT and STAT families ) and their functional importance are described .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Receptors for interleukin il - 10 - type cytokineuse similar signaling mechanisms for inducing transcription through IL - 6 response elements .',\n",
       "  [['Receptors for <e1> interleukin il - 10 - type cytokine </e1> use similar signaling mechanisms for inducing transcription through IL - 6 response elements .',\n",
       "    'PROTEIN']]],\n",
       " ['Receptors for interleukin ( il - 6 - type cytokineuse similar signaling mechanisms for inducing transcription through IL - 6 response elements .',\n",
       "  [['Receptors for interleukin ( <e1> il - 6 - type cytokine </e1> use similar signaling mechanisms for inducing transcription through IL - 6 response elements .',\n",
       "    'PROTEIN']]],\n",
       " ['To determine whether the IL - 10R has signaling functions similar to IL - 6R in cells normally expressing these receptors , leukocytes of the b - cell lineagewere treated with either cytokine .',\n",
       "  [['To determine whether the IL - 10R has signaling functions similar to IL - 6R in cells normally expressing these receptors , leukocytes of the <e1> b - cell lineage </e1> were treated with either cytokine .',\n",
       "    'CELL_TYPE']]],\n",
       " ['To determine whether the IL - 10R has signaling functions similar to IL - 6R in cells normally expressing these receptors , leukocytes of the B - , t - cell lineagewere treated with either cytokine .',\n",
       "  [['To determine whether the IL - 10R has signaling functions similar to IL - 6R in cells normally expressing these receptors , leukocytes of the B - , <e1> t - cell lineage </e1> were treated with either cytokine .',\n",
       "    'CELL_TYPE']]],\n",
       " [' c - terminal activating domaindetermine the transactivation potential of BSAP ( Pax - 5 ) , Pax - 2 and Pax - 8 .',\n",
       "  [[' <e1> c - terminal activating domain </e1> determine the transactivation potential of BSAP ( Pax - 5 ) , Pax - 2 and Pax - 8 .',\n",
       "    'PROTEIN']]],\n",
       " [' c - terminal inhibitory domaindetermine the transactivation potential of BSAP ( Pax - 5 ) , Pax - 2 and Pax - 8 .',\n",
       "  [[' <e1> c - terminal inhibitory domain </e1> determine the transactivation potential of BSAP ( Pax - 5 ) , Pax - 2 and Pax - 8 .',\n",
       "    'PROTEIN']]],\n",
       " ['In vitro mutagenesis and transient transfection experiments indicate that the C - terminal serine / threonine / proline - rich region of BSAP contains a potent transactivation domain of 55 amino acids which is active from promoter position .',\n",
       "  [['In vitro mutagenesis and transient transfection experiments indicate that the C - terminal serine / threonine / proline - rich region of BSAP contains a potent transactivation domain of 55 amino acids which is active from <e1> promoter position </e1> .',\n",
       "    'DNA']]],\n",
       " ['The same arrangement of positively acting sequencehas been conserved in the mammalian Pax - 2 and Pax - 8 , the zebrafish Pax - b as well as the sea urchin Pax - 258 proteins .',\n",
       "  [['The same arrangement of <e1> positively acting sequence </e1> has been conserved in the mammalian Pax - 2 and Pax - 8 , the zebrafish Pax - b as well as the sea urchin Pax - 258 proteins .',\n",
       "    'DNA']]],\n",
       " ['These data demonstrate that the transcriptional competence of a subfamily of Pax proteins is determined by a C - terminal regulatory module composed of activating sequence .',\n",
       "  [['These data demonstrate that the transcriptional competence of a subfamily of Pax proteins is determined by a C - terminal regulatory module composed of <e1> activating sequence </e1> .',\n",
       "    'DNA']]],\n",
       " ['The ability to regulate AR gene transcription via the second messenger pathway is lost in the pc - 3 cell line .',\n",
       "  [['The ability to regulate AR gene transcription via the second messenger pathway is lost in the <e1> pc - 3 cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Absence of t - cell - specific transcription factorTCF - 1 , GATA - 3 , and BSAP in Hodgkin s Reed - Sternberg cells .',\n",
       "  [['Absence of <e1> t - cell - specific transcription factor </e1> TCF - 1 , GATA - 3 , and BSAP in Hodgkin s Reed - Sternberg cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Based on the presence of T cell receptor - beta ( TcR - beta ) gene rearrangements in l428 celllines derived from patients with Hodgkin s disease are lymphoid in nature .',\n",
       "  [['Based on the presence of T cell receptor - beta ( TcR - beta ) gene rearrangements in <e1> l428 cell </e1> lines derived from patients with Hodgkin s disease are lymphoid in nature .',\n",
       "    'CELL_LINE']]],\n",
       " ['Based on the presence of T cell receptor - beta ( TcR - beta ) gene rearrangements in L428 and hdlm - 1 celllines derived from patients with Hodgkin s disease are lymphoid in nature .',\n",
       "  [['Based on the presence of T cell receptor - beta ( TcR - beta ) gene rearrangements in L428 and <e1> hdlm - 1 cell </e1> lines derived from patients with Hodgkin s disease are lymphoid in nature .',\n",
       "    'CELL_LINE']]],\n",
       " ['The sensitivity and specificity of this assay for determination of cell lineage have been established in a large number of cultured human cell line .',\n",
       "  [['The sensitivity and specificity of this assay for determination of cell lineage have been established in a large number of <e1> cultured human cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['The sensitivity and specificity of this assay for determination of cell lineage have been established in a large number of cultured murine cell line .',\n",
       "  [['The sensitivity and specificity of this assay for determination of cell lineage have been established in a large number of <e1> cultured murine cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Triplex formation by GM3 blocked recombinant nf - kappab proteinbinding to the GM - CSF element .',\n",
       "  [['Triplex formation by GM3 blocked <e1> recombinant nf - kappab protein </e1> binding to the GM - CSF element .',\n",
       "    'PROTEIN']]],\n",
       " ['Although p45 mRNA is transcribed from two different promoters , aNF - E2 promoter and fNF - E2 promoter , in erythroid lineage cell, p45 mRNA is transcribed only from aNF - E2 promoter .',\n",
       "  [['Although p45 mRNA is transcribed from two different promoters , aNF - E2 promoter and fNF - E2 promoter , in <e1> erythroid lineage cell </e1> , p45 mRNA is transcribed only from aNF - E2 promoter .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The binding activity of transcription factor AP - 1 , which consists of dimers of fos family protein, was also studied using a gel shift assay .',\n",
       "  [['The binding activity of transcription factor AP - 1 , which consists of dimers of <e1> fos family protein </e1> , was also studied using a gel shift assay .',\n",
       "    'PROTEIN']]],\n",
       " ['We demonstrated an uncoupling of AP - 1 binding activity from p62c - fos , and p39c - jun AP - 1 activity was expressed more strongly in the G1 - and G2/M - phase enriched samples than in the S - phase enriched samples of HL60 cells , while levels of nuclear p39c - junwere constant .',\n",
       "  [['We demonstrated an uncoupling of AP - 1 binding activity from p62c - fos , and p39c - jun AP - 1 activity was expressed more strongly in the G1 - and G2/M - phase enriched samples than in the S - phase enriched samples of HL60 cells , while levels of <e1> nuclear p39c - jun </e1> were constant .',\n",
       "    'PROTEIN']]],\n",
       " ['We also observed that cytoplasmic p39c - junwere present in HL60 cells and PHA - stimulated PBL .',\n",
       "  [['We also observed that <e1> cytoplasmic p39c - jun </e1> were present in HL60 cells and PHA - stimulated PBL .',\n",
       "    'PROTEIN']]],\n",
       " ['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid celland myeloid genes , and is critical for development of both of these lineages .',\n",
       "  [['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in <e1> myeloid cell </e1> and myeloid genes , and is critical for development of both of these lineages .',\n",
       "    'CELL_TYPE']]],\n",
       " ['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and b celland myeloid genes , and is critical for development of both of these lineages .',\n",
       "  [['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and <e1> b cell </e1> and myeloid genes , and is critical for development of both of these lineages .',\n",
       "    'CELL_TYPE']]],\n",
       " ['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and B cells , activates many b cell gene, and is critical for development of both of these lineages .',\n",
       "  [['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and B cells , activates many <e1> b cell gene </e1> , and is critical for development of both of these lineages .',\n",
       "    'DNA']]],\n",
       " ['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid celland myeloid genes , and is critical for development of both of these lineages .',\n",
       "  [['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in <e1> myeloid cell </e1> and myeloid genes , and is critical for development of both of these lineages .',\n",
       "    'CELL_TYPE']]],\n",
       " ['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and b celland myeloid genes , and is critical for development of both of these lineages .',\n",
       "  [['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and <e1> b cell </e1> and myeloid genes , and is critical for development of both of these lineages .',\n",
       "    'CELL_TYPE']]],\n",
       " ['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and B cells , activates many b cell gene, and is critical for development of both of these lineages .',\n",
       "  [['PU.1 ( spi - 1 ) , a member of the Ets transcription factor family , is predominantly expressed in myeloid and B cells , activates many <e1> b cell gene </e1> , and is critical for development of both of these lineages .',\n",
       "    'DNA']]],\n",
       " ['Both the 5 untranslated regionof the TNF gene were required to elicit maximal translational suppression by CNI - 1493 .',\n",
       "  [['Both the <e1> 5 untranslated region </e1> of the TNF gene were required to elicit maximal translational suppression by CNI - 1493 .',\n",
       "    'DNA']]],\n",
       " ['The human T - cell leukemia virus - encoded tax protein is a potent activator of many viral genetranscribed by RNA polymerase II .',\n",
       "  [['The human T - cell leukemia virus - encoded tax protein is a potent activator of many <e1> viral gene </e1> transcribed by RNA polymerase II .',\n",
       "    'DNA']]],\n",
       " ['1 alpha, 25 - Dihydroxyvitamin D3 ( VD ) is a potent inducer of monocytic differentiation of both normal cell .',\n",
       "  [['1 alpha, 25 - Dihydroxyvitamin D3 ( VD ) is a potent inducer of monocytic differentiation of both <e1> normal cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Alternative splicing generates type a transcriptthat potentially encode two entirely distinct proteins ; type A transcripts code for a small mitochondrial protein , p8MTCP1 , and type B transcripts , containing an additional open reading frame , may code for 107 amino - acid protein , p13MTCP1 .',\n",
       "  [['Alternative splicing generates <e1> type a transcript </e1> that potentially encode two entirely distinct proteins ; type A transcripts code for a small mitochondrial protein , p8MTCP1 , and type B transcripts , containing an additional open reading frame , may code for 107 amino - acid protein , p13MTCP1 .',\n",
       "    'RNA']]],\n",
       " ['Alternative splicing generates type b transcriptthat potentially encode two entirely distinct proteins ; type A transcripts code for a small mitochondrial protein , p8MTCP1 , and type B transcripts , containing an additional open reading frame , may code for 107 amino - acid protein , p13MTCP1 .',\n",
       "  [['Alternative splicing generates <e1> type b transcript </e1> that potentially encode two entirely distinct proteins ; type A transcripts code for a small mitochondrial protein , p8MTCP1 , and type B transcripts , containing an additional open reading frame , may code for 107 amino - acid protein , p13MTCP1 .',\n",
       "    'RNA']]],\n",
       " ['We have generated rabbit antisera against this putative p13MTCP1 protein and screened for expression of p13MTCP1 normal lymphoid tissues and 33 cases of immature lymphoid t - cell proliferationusing a sensitive Western blot assay .',\n",
       "  [['We have generated rabbit antisera against this putative p13MTCP1 protein and screened for expression of p13MTCP1 normal lymphoid tissues and 33 cases of <e1> immature lymphoid t - cell proliferation </e1> using a sensitive Western blot assay .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Priming of human monocytes with IFN - gamma resulted in the down regulation of c - fos mrnain response to stimulation with lipopolysaccharide ( LPS ) compared to the effects of LPS alone .',\n",
       "  [['Priming of human monocytes with IFN - gamma resulted in the down regulation of <e1> c - fos mrna </e1> in response to stimulation with lipopolysaccharide ( LPS ) compared to the effects of LPS alone .',\n",
       "    'RNA']]],\n",
       " ['Priming of human monocytes with IFN - gamma resulted in the down regulation of c - jun mrnain response to stimulation with lipopolysaccharide ( LPS ) compared to the effects of LPS alone .',\n",
       "  [['Priming of human monocytes with IFN - gamma resulted in the down regulation of <e1> c - jun mrna </e1> in response to stimulation with lipopolysaccharide ( LPS ) compared to the effects of LPS alone .',\n",
       "    'RNA']]],\n",
       " ['BCR - induced increases in CD44 mRNA expression and transcription levels are shown to occur in egr1 - expressing subcloneof the B - cell line WEHI - 231 .',\n",
       "  [['BCR - induced increases in CD44 mRNA expression and transcription levels are shown to occur in <e1> egr1 - expressing subclone </e1> of the B - cell line WEHI - 231 .',\n",
       "    'CELL_LINE']]],\n",
       " ['BCR - induced increases in CD44 mRNA expression and transcription levels are shown to occur in egr1 - nonexpressing subcloneof the B - cell line WEHI - 231 .',\n",
       "  [['BCR - induced increases in CD44 mRNA expression and transcription levels are shown to occur in <e1> egr1 - nonexpressing subclone </e1> of the B - cell line WEHI - 231 .',\n",
       "    'CELL_LINE']]],\n",
       " ['Transactivation of the interleukin - 1alpha promoter by human t - cell leukemia virus type i tax protein .',\n",
       "  [['Transactivation of the interleukin - 1alpha promoter by <e1> human t - cell leukemia virus type i tax protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Transactivation of the interleukin - 1alpha promoter by human t - cell leukemia virus type ii tax protein .',\n",
       "  [['Transactivation of the interleukin - 1alpha promoter by <e1> human t - cell leukemia virus type ii tax protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Moreover , deletion of either 5 nf - kappab sitereduced IL - 1alpha promoter activity in MT - 2 cells and transactivation of the IL - 1alpha promoter by exogenous NF - kappaB and Tax in Jurkat cells .',\n",
       "  [['Moreover , deletion of either <e1> 5 nf - kappab site </e1> reduced IL - 1alpha promoter activity in MT - 2 cells and transactivation of the IL - 1alpha promoter by exogenous NF - kappaB and Tax in Jurkat cells .',\n",
       "    'DNA']]],\n",
       " ['To develop a transformation system with a conditional Epstein - Barr virus nuclear antigen 2 ( EBNA2 ) gene , we fused the hormone binding domain of the oestrogen receptor to the n terminusof EBNA2 .',\n",
       "  [['To develop a transformation system with a conditional Epstein - Barr virus nuclear antigen 2 ( EBNA2 ) gene , we fused the hormone binding domain of the oestrogen receptor to the <e1> n terminus </e1> of EBNA2 .',\n",
       "    'PROTEIN']]],\n",
       " ['In order to better characterize the human gammac promoter and define the minimal tissue - specific promoter region , progressive 5 - deletion constructs of a segment extending 1053 base pairs upstream of the major transcription start site were generated and tested for promoter activity in various hematopoietic cell type .',\n",
       "  [['In order to better characterize the human gammac promoter and define the minimal tissue - specific promoter region , progressive 5 - deletion constructs of a segment extending 1053 base pairs upstream of the major transcription start site were generated and tested for promoter activity in various <e1> hematopoietic cell type </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Coexpression of the interleukin - 13 genecluster on human chromosome 5q23 - 31 .',\n",
       "  [['Coexpression of the <e1> interleukin - 13 gene </e1> cluster on human chromosome 5q23 - 31 .',\n",
       "    'DNA']]],\n",
       " ['Coexpression of the interleukin - 4 genecluster on human chromosome 5q23 - 31 .',\n",
       "  [['Coexpression of the <e1> interleukin - 4 gene </e1> cluster on human chromosome 5q23 - 31 .',\n",
       "    'DNA']]],\n",
       " ['Similar to IL - 4 , IL - 13 mRNA expression was highest in T - cell populations enriched for cells that had previously been primed in vivo or in vitro , indicating that priming increases the expression of the il - 13 genein a coordinate manner .',\n",
       "  [['Similar to IL - 4 , IL - 13 mRNA expression was highest in T - cell populations enriched for cells that had previously been primed in vivo or in vitro , indicating that priming increases the expression of the <e1> il - 13 gene </e1> in a coordinate manner .',\n",
       "    'DNA']]],\n",
       " ['Similar to IL - 4 , IL - 13 mRNA expression was highest in T - cell populations enriched for cells that had previously been primed in vivo or in vitro , indicating that priming increases the expression of the il - 4 genein a coordinate manner .',\n",
       "  [['Similar to IL - 4 , IL - 13 mRNA expression was highest in T - cell populations enriched for cells that had previously been primed in vivo or in vitro , indicating that priming increases the expression of the <e1> il - 4 gene </e1> in a coordinate manner .',\n",
       "    'DNA']]],\n",
       " ['Because the primed T cells contain higher levels of nuclear NF - ATp , capable of binding to P elements of the il - 4 promoter, than do freshly - isolated T cells , the NF - AT - binding P elements are attractive candidates to mediate the coordinate expression of these two cytokine genes .',\n",
       "  [['Because the primed T cells contain higher levels of nuclear NF - ATp , capable of binding to P elements of the <e1> il - 4 promoter </e1> , than do freshly - isolated T cells , the NF - AT - binding P elements are attractive candidates to mediate the coordinate expression of these two cytokine genes .',\n",
       "    'DNA']]],\n",
       " ['Because the primed T cells contain higher levels of nuclear NF - ATp , capable of binding to P elements of the il - 13 promoter, than do freshly - isolated T cells , the NF - AT - binding P elements are attractive candidates to mediate the coordinate expression of these two cytokine genes .',\n",
       "  [['Because the primed T cells contain higher levels of nuclear NF - ATp , capable of binding to P elements of the <e1> il - 13 promoter </e1> , than do freshly - isolated T cells , the NF - AT - binding P elements are attractive candidates to mediate the coordinate expression of these two cytokine genes .',\n",
       "    'DNA']]],\n",
       " ['Thus , the RAR - RXR response pathway can signal growth inhibition of normal bone marrow myeloid cell .',\n",
       "  [['Thus , the RAR - RXR response pathway can signal growth inhibition of <e1> normal bone marrow myeloid cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['After treatment of APL cells with AM580 either alone or in combination with granulocyte colony - stimulating factor ( G - CSF ) , the compound induces granulocytic maturation , as assessed by determination of the levels of leukocyte alkaline phosphatase mrna, at concentrations that are 10 - to 100 - fold lower than those of ATRA necessary to produce similar effects .',\n",
       "  [['After treatment of APL cells with AM580 either alone or in combination with granulocyte colony - stimulating factor ( G - CSF ) , the compound induces granulocytic maturation , as assessed by determination of the levels of <e1> leukocyte alkaline phosphatase mrna </e1> , at concentrations that are 10 - to 100 - fold lower than those of ATRA necessary to produce similar effects .',\n",
       "    'RNA']]],\n",
       " ['After treatment of APL cells with AM580 either alone or in combination with granulocyte colony - stimulating factor ( G - CSF ) , the compound induces granulocytic maturation , as assessed by determination of the levels of leukocyte alkaline phosphatase , cd11b mrna, at concentrations that are 10 - to 100 - fold lower than those of ATRA necessary to produce similar effects .',\n",
       "  [['After treatment of APL cells with AM580 either alone or in combination with granulocyte colony - stimulating factor ( G - CSF ) , the compound induces granulocytic maturation , as assessed by determination of the levels of leukocyte alkaline phosphatase , <e1> cd11b mrna </e1> , at concentrations that are 10 - to 100 - fold lower than those of ATRA necessary to produce similar effects .',\n",
       "    'RNA']]],\n",
       " ['After treatment of APL cells with AM580 either alone or in combination with granulocyte colony - stimulating factor ( G - CSF ) , the compound induces granulocytic maturation , as assessed by determination of the levels of leukocyte alkaline phosphatase , CD11b , cd33 mrna, at concentrations that are 10 - to 100 - fold lower than those of ATRA necessary to produce similar effects .',\n",
       "  [['After treatment of APL cells with AM580 either alone or in combination with granulocyte colony - stimulating factor ( G - CSF ) , the compound induces granulocytic maturation , as assessed by determination of the levels of leukocyte alkaline phosphatase , CD11b , <e1> cd33 mrna </e1> , at concentrations that are 10 - to 100 - fold lower than those of ATRA necessary to produce similar effects .',\n",
       "    'RNA']]],\n",
       " ['The extracts were compared for their ability to retard the migration of radiolabeled double stranded oligomers representative of the VDREs of the human osteocalcin gene promoter .',\n",
       "  [['The extracts were compared for their ability to retard the migration of radiolabeled double stranded oligomers representative of the VDREs of the <e1> human osteocalcin gene promoter </e1> .',\n",
       "    'DNA']]],\n",
       " [' resting b lymphocytecontain high levels of BCL - 6 mRNA .',\n",
       "  [[' <e1> resting b lymphocyte </e1> contain high levels of BCL - 6 mRNA .',\n",
       "    'CELL_TYPE']]],\n",
       " [' resting t lymphocytecontain high levels of BCL - 6 mRNA .',\n",
       "  [[' <e1> resting t lymphocyte </e1> contain high levels of BCL - 6 mRNA .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Stimulation of mouse B cells with anti - igm antibody, bacterial lipopolysaccharide , phorbol 12 - myristate 13 - acetate plus ionomycin , or CD40 ligand led to a five - fold to 35 - fold decrease in BCL - 6 mRNA levels .',\n",
       "  [['Stimulation of mouse B cells with <e1> anti - igm antibody </e1> , bacterial lipopolysaccharide , phorbol 12 - myristate 13 - acetate plus ionomycin , or CD40 ligand led to a five - fold to 35 - fold decrease in BCL - 6 mRNA levels .',\n",
       "    'PROTEIN']]],\n",
       " ['Stimulation of mouse B cells with anti - igd antibody, bacterial lipopolysaccharide , phorbol 12 - myristate 13 - acetate plus ionomycin , or CD40 ligand led to a five - fold to 35 - fold decrease in BCL - 6 mRNA levels .',\n",
       "  [['Stimulation of mouse B cells with <e1> anti - igd antibody </e1> , bacterial lipopolysaccharide , phorbol 12 - myristate 13 - acetate plus ionomycin , or CD40 ligand led to a five - fold to 35 - fold decrease in BCL - 6 mRNA levels .',\n",
       "    'PROTEIN']]],\n",
       " ['The role of alterations of the MTS1 tumor suppressor gene on chromosome 9p21 , which encodes p16 , the inhibitor of cyclin - dependent - kinase - 6, in tumorigenesis is not yet clear .',\n",
       "  [['The role of alterations of the MTS1 tumor suppressor gene on chromosome 9p21 , which encodes p16 , the inhibitor of <e1> cyclin - dependent - kinase - 6 </e1> , in tumorigenesis is not yet clear .',\n",
       "    'PROTEIN']]],\n",
       " ['Compared to normal lymphocytes , molt - 4 cellcycle regulation of thymidine kinase , a much higher intracellular activity of this enzyme , and higher thymidine kinase mRNA expression .',\n",
       "  [['Compared to normal lymphocytes , <e1> molt - 4 cell </e1> cycle regulation of thymidine kinase , a much higher intracellular activity of this enzyme , and higher thymidine kinase mRNA expression .',\n",
       "    'CELL_LINE']]],\n",
       " ['Compared to normal lymphocytes , MOLT - 4 and cem cellcycle regulation of thymidine kinase , a much higher intracellular activity of this enzyme , and higher thymidine kinase mRNA expression .',\n",
       "  [['Compared to normal lymphocytes , MOLT - 4 and <e1> cem cell </e1> cycle regulation of thymidine kinase , a much higher intracellular activity of this enzyme , and higher thymidine kinase mRNA expression .',\n",
       "    'CELL_LINE']]],\n",
       " ['Transient expression of p16 in normal human lymphocytes caused arrest in G1 , but was without effect on the cell growth of molt - 4 cell, although all of them express functional retinoblastoma protein .',\n",
       "  [['Transient expression of p16 in normal human lymphocytes caused arrest in G1 , but was without effect on the cell growth of <e1> molt - 4 cell </e1> , although all of them express functional retinoblastoma protein .',\n",
       "    'CELL_LINE']]],\n",
       " ['Authentic nfat targetdid not compete for the META(D+) binding factor ( s ) .',\n",
       "  [['Authentic <e1> nfat target </e1> did not compete for the META(D+) binding factor ( s ) .',\n",
       "    'DNA']]],\n",
       " ['Pretreatment with a protein kinase C activator , phorbol 12 - myristate 13 - acetate , caused almost complete inhibition of histamine - induced [ Ca2+ ]i rise , but did not do so by activators of camp - dependent protein kinase .',\n",
       "  [['Pretreatment with a protein kinase C activator , phorbol 12 - myristate 13 - acetate , caused almost complete inhibition of histamine - induced [ Ca2+ ]i rise , but did not do so by activators of <e1> camp - dependent protein kinase </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['We used oligonucleotide probes from the beta - casein gene promoterand the ISRE probe to detect STAT proteins in nuclear extracts from acute leukemia cells in bandshift assays .',\n",
       "  [['We used oligonucleotide probes from the <e1> beta - casein gene promoter </e1> and the ISRE probe to detect STAT proteins in nuclear extracts from acute leukemia cells in bandshift assays .',\n",
       "    'DNA']]],\n",
       " [' stat5 - related factorwere detected in ALL and STAT1 - , STAT3 - , and STAT5 - related proteins were present in nuclear cell extracts from AML .',\n",
       "  [[' <e1> stat5 - related factor </e1> were detected in ALL and STAT1 - , STAT3 - , and STAT5 - related proteins were present in nuclear cell extracts from AML .',\n",
       "    'PROTEIN']]],\n",
       " ['STAT5 - and STAT1 - related factors were detected in ALL and stat1 - related proteinswere present in nuclear cell extracts from AML .',\n",
       "  [['STAT5 - and STAT1 - related factors were detected in ALL and <e1> stat1 - related proteins </e1> were present in nuclear cell extracts from AML .',\n",
       "    'PROTEIN']]],\n",
       " ['STAT5 - and STAT1 - related factors were detected in ALL and STAT1 - , stat3 - related proteinswere present in nuclear cell extracts from AML .',\n",
       "  [['STAT5 - and STAT1 - related factors were detected in ALL and STAT1 - , <e1> stat3 - related proteins </e1> were present in nuclear cell extracts from AML .',\n",
       "    'PROTEIN']]],\n",
       " [' stat5 - related factorwere detected in ALL and STAT1 - , STAT3 - , and STAT5 - related proteins were present in nuclear cell extracts from AML .',\n",
       "  [[' <e1> stat5 - related factor </e1> were detected in ALL and STAT1 - , STAT3 - , and STAT5 - related proteins were present in nuclear cell extracts from AML .',\n",
       "    'PROTEIN']]],\n",
       " ['STAT5 - and STAT1 - related factors were detected in ALL and stat1 - related proteinswere present in nuclear cell extracts from AML .',\n",
       "  [['STAT5 - and STAT1 - related factors were detected in ALL and <e1> stat1 - related proteins </e1> were present in nuclear cell extracts from AML .',\n",
       "    'PROTEIN']]],\n",
       " ['STAT5 - and STAT1 - related factors were detected in ALL and STAT1 - , stat3 - related proteinswere present in nuclear cell extracts from AML .',\n",
       "  [['STAT5 - and STAT1 - related factors were detected in ALL and STAT1 - , <e1> stat3 - related proteins </e1> were present in nuclear cell extracts from AML .',\n",
       "    'PROTEIN']]],\n",
       " ['Here , we describe the effect of 383 - bp delta383 sequencecontaining the PYBF - binding site on transcription from various globin and non - globin promoters , using a transient assay with the cat reporter gene in murine erythroleukemia ( MEL ) cells , a cell line with abundant PYBF activity .',\n",
       "  [['Here , we describe the effect of <e1> 383 - bp delta383 sequence </e1> containing the PYBF - binding site on transcription from various globin and non - globin promoters , using a transient assay with the cat reporter gene in murine erythroleukemia ( MEL ) cells , a cell line with abundant PYBF activity .',\n",
       "    'DNA']]],\n",
       " ['Here , we describe the effect of 383 - bp ( delta383 ) and 99 - bp delta99 sequencecontaining the PYBF - binding site on transcription from various globin and non - globin promoters , using a transient assay with the cat reporter gene in murine erythroleukemia ( MEL ) cells , a cell line with abundant PYBF activity .',\n",
       "  [['Here , we describe the effect of 383 - bp ( delta383 ) and <e1> 99 - bp delta99 sequence </e1> containing the PYBF - binding site on transcription from various globin and non - globin promoters , using a transient assay with the cat reporter gene in murine erythroleukemia ( MEL ) cells , a cell line with abundant PYBF activity .',\n",
       "    'DNA']]],\n",
       " ['We show that both delta383 and delta99 specifically enhance expression of cat for plasmids containing a human adult globin ( HBB ) promoter , whereas expression of similar constructs using human fetal a gamma - globin hbg1 promoteris not enhanced .',\n",
       "  [['We show that both delta383 and delta99 specifically enhance expression of cat for plasmids containing a human adult globin ( HBB ) promoter , whereas expression of similar constructs using <e1> human fetal a gamma - globin hbg1 promoter </e1> is not enhanced .',\n",
       "    'DNA']]],\n",
       " ['We show that both delta383 and delta99 specifically enhance expression of cat for plasmids containing a human adult globin ( HBB ) promoter , whereas expression of similar constructs using human fetal ( A gamma - ) globin ( HBG1 ) or simian virus 40 sv40 promoteris not enhanced .',\n",
       "  [['We show that both delta383 and delta99 specifically enhance expression of cat for plasmids containing a human adult globin ( HBB ) promoter , whereas expression of similar constructs using human fetal ( A gamma - ) globin ( HBG1 ) or <e1> simian virus 40 sv40 promoter </e1> is not enhanced .',\n",
       "    'DNA']]],\n",
       " ['In B lymphoid cells , deltaspi - b mrnawere present simultaneously in a ratio of around 10 % .',\n",
       "  [['In B lymphoid cells , <e1> deltaspi - b mrna </e1> were present simultaneously in a ratio of around 10 % .',\n",
       "    'RNA']]],\n",
       " ['Efficient transcription and replication of simian immunodeficiency virus in the absence of nf - kappab binding element .',\n",
       "  [['Efficient transcription and replication of simian immunodeficiency virus in the absence of <e1> nf - kappab binding element </e1> .',\n",
       "    'DNA']]],\n",
       " ['All mutants , including one extensively mutagenized strain entirely missing the nf - kappab binding element, replicated with wild - type kinetics and to a wild - type level in peripheral blood mononuclear cell cultures in 50 to 100 % of the experiments .',\n",
       "  [['All mutants , including one extensively mutagenized strain entirely missing the <e1> nf - kappab binding element </e1> , replicated with wild - type kinetics and to a wild - type level in peripheral blood mononuclear cell cultures in 50 to 100 % of the experiments .',\n",
       "    'DNA']]],\n",
       " ['Reversions or additional mutations were not detected in the u3 regionof proviral DNA from CEMxl74 cells infected with the SIVmac239 mutants .',\n",
       "  [['Reversions or additional mutations were not detected in the <e1> u3 region </e1> of proviral DNA from CEMxl74 cells infected with the SIVmac239 mutants .',\n",
       "    'DNA']]],\n",
       " ['Analysis of long terminal repeat - driven secreted alkaline phosphatase activity in transient assays showed that , unlike human immunodeficiency virus type 1 , the SIV long terminal repeat possesses an enhancer region just upstream of the NF - kappaB element which maintains significant levels of basal transcription in the absence of nf - kappab site .',\n",
       "  [['Analysis of long terminal repeat - driven secreted alkaline phosphatase activity in transient assays showed that , unlike human immunodeficiency virus type 1 , the SIV long terminal repeat possesses an enhancer region just upstream of the NF - kappaB element which maintains significant levels of basal transcription in the absence of <e1> nf - kappab site </e1> .',\n",
       "    'DNA']]],\n",
       " ['Using human b - cell linetransfected with ZpCat reporter gene constructs , we demonstrate that a region designated the ZII domain of Zp is the target of HHV - 6 transactivation .',\n",
       "  [['Using <e1> human b - cell line </e1> transfected with ZpCat reporter gene constructs , we demonstrate that a region designated the ZII domain of Zp is the target of HHV - 6 transactivation .',\n",
       "    'CELL_LINE']]],\n",
       " ['Using human t - cell linetransfected with ZpCat reporter gene constructs , we demonstrate that a region designated the ZII domain of Zp is the target of HHV - 6 transactivation .',\n",
       "  [['Using <e1> human t - cell line </e1> transfected with ZpCat reporter gene constructs , we demonstrate that a region designated the ZII domain of Zp is the target of HHV - 6 transactivation .',\n",
       "    'CELL_LINE']]],\n",
       " ['Structure/function analysis of various Oct - 2a effector regions in the context of the GAL4 DNA - binding domain revealed that Oct - 2a contains two functionally different activation domains at the n termini .',\n",
       "  [['Structure/function analysis of various Oct - 2a effector regions in the context of the GAL4 DNA - binding domain revealed that Oct - 2a contains two functionally different activation domains at the <e1> n termini </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Opposing effects of glucocorticoids on the rate of apoptosis in neutrophilic granulocyte .',\n",
       "  [['Opposing effects of glucocorticoids on the rate of apoptosis in <e1> neutrophilic granulocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Mice lacking Rel are defective in mitogenic activation of b lymphocyteand display impaired humoral immunity .',\n",
       "  [['Mice lacking Rel are defective in mitogenic activation of <e1> b lymphocyte </e1> and display impaired humoral immunity .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Exogenous IL - 2 , which restitutes the proliferative response of the anti - cd3 - treated rel - / - t cell, restores production of IL - 5 , TNF - alpha , and IFN - gamma , but not IL - 3 and GM - CSF expression to approximately normal levels .',\n",
       "  [['Exogenous IL - 2 , which restitutes the proliferative response of the <e1> anti - cd3 - treated rel - / - t cell </e1> , restores production of IL - 5 , TNF - alpha , and IFN - gamma , but not IL - 3 and GM - CSF expression to approximately normal levels .',\n",
       "    'CELL_LINE']]],\n",
       " ['Exogenous IL - 2 , which restitutes the proliferative response of the anti - cd28 - treated rel - / - t cell, restores production of IL - 5 , TNF - alpha , and IFN - gamma , but not IL - 3 and GM - CSF expression to approximately normal levels .',\n",
       "  [['Exogenous IL - 2 , which restitutes the proliferative response of the <e1> anti - cd28 - treated rel - / - t cell </e1> , restores production of IL - 5 , TNF - alpha , and IFN - gamma , but not IL - 3 and GM - CSF expression to approximately normal levels .',\n",
       "    'CELL_LINE']]],\n",
       " ['Further , in normal and cd45 - deficient cell line, Fas stimulation results in activation of Jun kinase ( JNK ) , a proposed mediator of stress activation pathways .',\n",
       "  [['Further , in normal and <e1> cd45 - deficient cell line </e1> , Fas stimulation results in activation of Jun kinase ( JNK ) , a proposed mediator of stress activation pathways .',\n",
       "    'CELL_LINE']]],\n",
       " ['In promoter regions , the actively transcribed genes demonstrated extensive and consistent footprints over the canonical elements , such as caccc motif .',\n",
       "  [['In promoter regions , the actively transcribed genes demonstrated extensive and consistent footprints over the canonical elements , such as <e1> caccc motif </e1> .',\n",
       "    'DNA']]],\n",
       " ['The adult globin expressing cells displayed more extensive footprints than the fetal globin expressing cells in the 3 regulatory sequences of both the agamma - globin geneexpression .',\n",
       "  [['The adult globin expressing cells displayed more extensive footprints than the fetal globin expressing cells in the 3 regulatory sequences of both the <e1> agamma - globin gene </e1> expression .',\n",
       "    'DNA']]],\n",
       " ['The adult globin expressing cells displayed more extensive footprints than the fetal globin expressing cells in the 3 regulatory sequences of both the Agamma - and the beta - globin geneexpression .',\n",
       "  [['The adult globin expressing cells displayed more extensive footprints than the fetal globin expressing cells in the 3 regulatory sequences of both the Agamma - and the <e1> beta - globin gene </e1> expression .',\n",
       "    'DNA']]],\n",
       " ['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the gamma - globin gene promoter, and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the fetal or the adult globin gene .',\n",
       "  [['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the <e1> gamma - globin gene promoter </e1> , and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the fetal or the adult globin gene .',\n",
       "    'DNA']]],\n",
       " ['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the gamma - and beta - globin gene promoters , and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the fetal globin gene .',\n",
       "  [['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the gamma - and beta - globin gene promoters , and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the <e1> fetal globin gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the gamma - globin gene promoter, and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the fetal or the adult globin gene .',\n",
       "  [['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the <e1> gamma - globin gene promoter </e1> , and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the fetal or the adult globin gene .',\n",
       "    'DNA']]],\n",
       " ['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the gamma - and beta - globin gene promoters , and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the fetal globin gene .',\n",
       "  [['Our results suggest that the bulk of protein - DNA interactions that underlies the developmental control of globin genes takes place in the gamma - and beta - globin gene promoters , and that GT motifs of the beta - globin locus LCR may play a role in the developmental regulation of human beta - globin gene expression , perhaps by increasing the probability of interaction of the LCR holocomplex with the <e1> fetal globin gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Galectin - 3 is widely distributed in various tissues and cell types and is expressed in many leukocytes , with the notable exception of b lymphocyte .',\n",
       "  [['Galectin - 3 is widely distributed in various tissues and cell types and is expressed in many leukocytes , with the notable exception of <e1> b lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The up - regulation of galectin - 3 expression appeared to correlate well with HTLV - I gene expression , as undetectable or very low levels of galectin - 3 were found in the s1t cell line, which are nonproductively infected with HTLV - I .',\n",
       "  [['The up - regulation of galectin - 3 expression appeared to correlate well with HTLV - I gene expression , as undetectable or very low levels of galectin - 3 were found in the <e1> s1t cell line </e1> , which are nonproductively infected with HTLV - I .',\n",
       "    'CELL_LINE']]],\n",
       " ['Interstitial collagenase ( MMP - 1 ) , a metalloproteinase produced by resident cellduring connective tissue turnover , cleaves type I collagen fibrils .',\n",
       "  [['Interstitial collagenase ( MMP - 1 ) , a metalloproteinase produced by <e1> resident cell </e1> during connective tissue turnover , cleaves type I collagen fibrils .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Since promoter elements controlling collagenase transcription in monocytic cells have not been previously defined , we sought to delineate responsive cis - acting elements of the collagenase promoter in transiently transfected human u937 monocytic cell line .',\n",
       "  [['Since promoter elements controlling collagenase transcription in monocytic cells have not been previously defined , we sought to delineate responsive cis - acting elements of the collagenase promoter in transiently transfected <e1> human u937 monocytic cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Since promoter elements controlling collagenase transcription in monocytic cells have not been previously defined , we sought to delineate responsive cis - acting elements of the collagenase promoter in transiently transfected human ( U937 ) and murine j774 monocytic cell line .',\n",
       "  [['Since promoter elements controlling collagenase transcription in monocytic cells have not been previously defined , we sought to delineate responsive cis - acting elements of the collagenase promoter in transiently transfected human ( U937 ) and <e1> murine j774 monocytic cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Nuclear extracts from LPS - and zymosan - treated cells showed strong AP - 1 activity by gel - shift analysis , and supershift analysis showed the AP - 1 complexes contained specific members of both the jun gene family .',\n",
       "  [['Nuclear extracts from LPS - and zymosan - treated cells showed strong AP - 1 activity by gel - shift analysis , and supershift analysis showed the AP - 1 complexes contained specific members of both the <e1> jun gene family </e1> .',\n",
       "    'DNA']]],\n",
       " ['LPS treatment of RAW 264.7 cells , murine bone marrow - derived macrophages , and the human monocyte cell line THP - 1 resulted in rapid activation of the p46 isoformof JNK .',\n",
       "  [['LPS treatment of RAW 264.7 cells , murine bone marrow - derived macrophages , and the human monocyte cell line THP - 1 resulted in rapid activation of the <e1> p46 isoform </e1> of JNK .',\n",
       "    'PROTEIN']]],\n",
       " [' truncated c - jun mutant proteinincluding wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "  [[' <e1> truncated c - jun mutant protein </e1> including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "    'PROTEIN']]],\n",
       " [' truncated c - fos mutant proteinincluding wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "  [[' <e1> truncated c - fos mutant protein </e1> including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "    'PROTEIN']]],\n",
       " ['Truncated c - Jun and c - Fos mutant proteins including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization domainof c - Jun and c - Fos .',\n",
       "  [['Truncated c - Jun and c - Fos mutant proteins including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the <e1> dimerization domain </e1> of c - Jun and c - Fos .',\n",
       "    'PROTEIN']]],\n",
       " [' truncated c - jun mutant proteinincluding wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "  [[' <e1> truncated c - jun mutant protein </e1> including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "    'PROTEIN']]],\n",
       " [' truncated c - fos mutant proteinincluding wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "  [[' <e1> truncated c - fos mutant protein </e1> including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization and DNA binding domains of c - Jun and c - Fos .',\n",
       "    'PROTEIN']]],\n",
       " ['Truncated c - Jun and c - Fos mutant proteins including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the dimerization domainof c - Jun and c - Fos .',\n",
       "  [['Truncated c - Jun and c - Fos mutant proteins including wbJun and wbFos were also ubiquitinylated under the same conditions , suggesting the sites of ubiquitinylation are located within the <e1> dimerization domain </e1> of c - Jun and c - Fos .',\n",
       "    'PROTEIN']]],\n",
       " ['IL - 10 cooperates with TNF - alpha to activate HIV - 1 from latently infected cellof monocyte/macrophage lineage .',\n",
       "  [['IL - 10 cooperates with TNF - alpha to activate HIV - 1 from <e1> latently infected cell </e1> of monocyte/macrophage lineage .',\n",
       "    'CELL_TYPE']]],\n",
       " ['MHC class II density on the APC was modified using dq2 homo zygous b - lclas APCs , however this variation of MHC concentration had no effect on T cell proliferation .',\n",
       "  [['MHC class II density on the APC was modified using <e1> dq2 homo zygous b - lcl </e1> as APCs , however this variation of MHC concentration had no effect on T cell proliferation .',\n",
       "    'CELL_LINE']]],\n",
       " ['MHC class II density on the APC was modified using dq2 hetero zygous b - lclas APCs , however this variation of MHC concentration had no effect on T cell proliferation .',\n",
       "  [['MHC class II density on the APC was modified using <e1> dq2 hetero zygous b - lcl </e1> as APCs , however this variation of MHC concentration had no effect on T cell proliferation .',\n",
       "    'CELL_LINE']]],\n",
       " ['One - hybrid assays in yeast isolated NF - ATp as an E1 binding protein , and transfection of NF - ATp into t cell linestrongly enhanced the activation - dependent SCM - 1 promoter activity .',\n",
       "  [['One - hybrid assays in yeast isolated NF - ATp as an E1 binding protein , and transfection of NF - ATp into <e1> t cell line </e1> strongly enhanced the activation - dependent SCM - 1 promoter activity .',\n",
       "    'CELL_LINE']]],\n",
       " ['The importance of NF - kappaB sites was confirmed by using vectors containing wild - type kappab sitein a heterologous promoter .',\n",
       "  [['The importance of NF - kappaB sites was confirmed by using vectors containing <e1> wild - type kappab site </e1> in a heterologous promoter .',\n",
       "    'DNA']]],\n",
       " ['In this study , we analyzed the binding activity of il - 10 - activated stat moleculeto two kinds of GAS - motif sequences .',\n",
       "  [['In this study , we analyzed the binding activity of <e1> il - 10 - activated stat molecule </e1> to two kinds of GAS - motif sequences .',\n",
       "    'PROTEIN']]],\n",
       " [' mouse cellthat were rendered cytolytic susceptible by E1A were also sensitized to CL - induced and chemically induced apoptosis .',\n",
       "  [[' <e1> mouse cell </e1> that were rendered cytolytic susceptible by E1A were also sensitized to CL - induced and chemically induced apoptosis .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Mouse , rat cellthat were rendered cytolytic susceptible by E1A were also sensitized to CL - induced and chemically induced apoptosis .',\n",
       "  [['Mouse , <e1> rat cell </e1> that were rendered cytolytic susceptible by E1A were also sensitized to CL - induced and chemically induced apoptosis .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Mouse , rat , hamster cellthat were rendered cytolytic susceptible by E1A were also sensitized to CL - induced and chemically induced apoptosis .',\n",
       "  [['Mouse , rat , <e1> hamster cell </e1> that were rendered cytolytic susceptible by E1A were also sensitized to CL - induced and chemically induced apoptosis .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Consecutive il - 1beta - mrnawere elevated ( up to 1.9 +/ - 0.58 - fold ) .',\n",
       "  [['Consecutive <e1> il - 1beta - mrna </e1> were elevated ( up to 1.9 +/ - 0.58 - fold ) .',\n",
       "    'RNA']]],\n",
       " ['Consecutive il - 6 - mrnawere elevated ( up to 1.9 +/ - 0.58 - fold ) .',\n",
       "  [['Consecutive <e1> il - 6 - mrna </e1> were elevated ( up to 1.9 +/ - 0.58 - fold ) .',\n",
       "    'RNA']]],\n",
       " ['Mutagenesis of nf kappa b elementwithin the LTR resulted in no change in the ability of CD8+ T cell supernatants to inhibit Tat - or mitogen - mediated LTR transcription .',\n",
       "  [['Mutagenesis of <e1> nf kappa b element </e1> within the LTR resulted in no change in the ability of CD8+ T cell supernatants to inhibit Tat - or mitogen - mediated LTR transcription .',\n",
       "    'DNA']]],\n",
       " ['The resulting diminished phosphorylation of c - Jun and JunD at their NH(2) - termini decreases the ability of these nuclear proteins to autostimulate the expression of the c - jun gene, thus leading to lower production of c - Jun and JunD .',\n",
       "  [['The resulting diminished phosphorylation of c - Jun and JunD at their NH(2) - termini decreases the ability of these nuclear proteins to autostimulate the expression of the <e1> c - jun gene </e1> , thus leading to lower production of c - Jun and JunD .',\n",
       "    'DNA']]],\n",
       " ['Consistent with these data , CD40 signals up - regulate c - jun mrnaand alter the transcription factor ATF2 but not the Raf - 1 protein .',\n",
       "  [['Consistent with these data , CD40 signals up - regulate <e1> c - jun mrna </e1> and alter the transcription factor ATF2 but not the Raf - 1 protein .',\n",
       "    'RNA']]],\n",
       " ['Consistent with these data , CD40 signals up - regulate c - fos mrnaand alter the transcription factor ATF2 but not the Raf - 1 protein .',\n",
       "  [['Consistent with these data , CD40 signals up - regulate <e1> c - fos mrna </e1> and alter the transcription factor ATF2 but not the Raf - 1 protein .',\n",
       "    'RNA']]],\n",
       " ['Using cd45 - negative variantof the Jurkat leukemic T - cell line we show that the different biochemical events induced by pervanadate appeared to be dependent on the presence at the cell surface of either CD45 or CD3 .',\n",
       "  [['Using <e1> cd45 - negative variant </e1> of the Jurkat leukemic T - cell line we show that the different biochemical events induced by pervanadate appeared to be dependent on the presence at the cell surface of either CD45 or CD3 .',\n",
       "    'CELL_LINE']]],\n",
       " ['We also show that expression of the B1 transcript from a second gene , MTCP1 , occurred at a relatively high level only in two T - PLL tumours from A - T patients with t(X;14) translocations whereas the MTCP1 /A1 transcript is much more widely expressed in both tumour cellof A - T and non - A - T individuals .',\n",
       "  [['We also show that expression of the B1 transcript from a second gene , MTCP1 , occurred at a relatively high level only in two T - PLL tumours from A - T patients with t(X;14) translocations whereas the MTCP1 /A1 transcript is much more widely expressed in both <e1> tumour cell </e1> of A - T and non - A - T individuals .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Sequencing of BRCA1 amplified from genomic DNA of lymphocytes and microdissected ovarian tumor cells of a familial ovarian cancer patient revealed three , rare heterozygous DNA variations ( 2418delA , 233G - - > A , and IVS1 - 10T - - > C ) in both tumor dna .',\n",
       "  [['Sequencing of BRCA1 amplified from genomic DNA of lymphocytes and microdissected ovarian tumor cells of a familial ovarian cancer patient revealed three , rare heterozygous DNA variations ( 2418delA , 233G - - > A , and IVS1 - 10T - - > C ) in both <e1> tumor dna </e1> .',\n",
       "    'DNA']]],\n",
       " ['Sequencing of BRCA1 amplified from genomic DNA of lymphocytes and microdissected ovarian tumor cells of a familial ovarian cancer patient revealed three , rare heterozygous DNA variations ( 2418delA , 233G - - > A , and IVS1 - 10T - - > C ) in both tumor and constitutional lymphocyte dna .',\n",
       "  [['Sequencing of BRCA1 amplified from genomic DNA of lymphocytes and microdissected ovarian tumor cells of a familial ovarian cancer patient revealed three , rare heterozygous DNA variations ( 2418delA , 233G - - > A , and IVS1 - 10T - - > C ) in both tumor and <e1> constitutional lymphocyte dna </e1> .',\n",
       "    'DNA']]],\n",
       " ['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal protein, whereas the DC - specific cDNAs are homologous to signaling , chemokine , and IFN - gamma - inducible genes .',\n",
       "  [['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding <e1> cytoskeletal protein </e1> , whereas the DC - specific cDNAs are homologous to signaling , chemokine , and IFN - gamma - inducible genes .',\n",
       "    'PROTEIN']]],\n",
       " ['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to signaling gene .',\n",
       "  [['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to <e1> signaling gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to signaling , chemokine gene .',\n",
       "  [['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to signaling , <e1> chemokine gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal protein, whereas the DC - specific cDNAs are homologous to signaling , chemokine , and IFN - gamma - inducible genes .',\n",
       "  [['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding <e1> cytoskeletal protein </e1> , whereas the DC - specific cDNAs are homologous to signaling , chemokine , and IFN - gamma - inducible genes .',\n",
       "    'PROTEIN']]],\n",
       " ['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to signaling gene .',\n",
       "  [['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to <e1> signaling gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to signaling , chemokine gene .',\n",
       "  [['A distinctive pattern of differentially expressed cDNAs is evident where macrophage - specific cDNAs are homologous to genes encoding cytoskeletal and cell - surface proteins , whereas the DC - specific cDNAs are homologous to signaling , <e1> chemokine gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Although the mutant strain was able to kill hl - 60 - derived macrophage, it could not replicate within a protozoan host , Acanthamoeba castellanii .',\n",
       "  [['Although the mutant strain was able to kill <e1> hl - 60 - derived macrophage </e1> , it could not replicate within a protozoan host , Acanthamoeba castellanii .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Pax5 is predominantly transcribed from only one of its two alleles in early B - lymphoid progenitors and mature B cells , while it transiently switches to a biallelic mode of transcription in pre - b cell .',\n",
       "  [['Pax5 is predominantly transcribed from only one of its two alleles in early B - lymphoid progenitors and mature B cells , while it transiently switches to a biallelic mode of transcription in <e1> pre - b cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These data suggest the potential utility of agents that disrupt calcineurin - mediated signal transduction pathways by blocking formation of the catalytically active dimer of calcineurin a subunit .',\n",
       "  [['These data suggest the potential utility of agents that disrupt calcineurin - mediated signal transduction pathways by blocking formation of the catalytically active dimer of <e1> calcineurin a subunit </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['These data suggest the potential utility of agents that disrupt calcineurin - mediated signal transduction pathways by blocking formation of the catalytically active dimer of calcineurin b subunit .',\n",
       "  [['These data suggest the potential utility of agents that disrupt calcineurin - mediated signal transduction pathways by blocking formation of the catalytically active dimer of <e1> calcineurin b subunit </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The hematopoietic cell - specific pattern of MNDA expression was elucidated through a comprehensive analysis of normal and neoplastic tissues , and the results provide further evidence of the coexpression of b - cell markerin neoplastic B cells and identify a normal B cell population that might be related to the cell of origin of a subset of B cell neoplasms .',\n",
       "  [['The hematopoietic cell - specific pattern of MNDA expression was elucidated through a comprehensive analysis of normal and neoplastic tissues , and the results provide further evidence of the coexpression of <e1> b - cell marker </e1> in neoplastic B cells and identify a normal B cell population that might be related to the cell of origin of a subset of B cell neoplasms .',\n",
       "    'PROTEIN']]],\n",
       " ['To determine the functionality of both the c/ebp - binding site, we analyzed whether C/EBPbeta and GATA - 1 can stimulate the MBP promoter in the C/EBPbeta and GATA - 1 negative Jurkat T - cell line .',\n",
       "  [['To determine the functionality of both the <e1> c/ebp - binding site </e1> , we analyzed whether C/EBPbeta and GATA - 1 can stimulate the MBP promoter in the C/EBPbeta and GATA - 1 negative Jurkat T - cell line .',\n",
       "    'DNA']]],\n",
       " ['Cotransfection with c/ebpbeta expression vectorproduced a 5 - fold increase compared with cotransfection with the C/EBPbeta or GATA - 1 expression vectors individually .',\n",
       "  [['Cotransfection with <e1> c/ebpbeta expression vector </e1> produced a 5 - fold increase compared with cotransfection with the C/EBPbeta or GATA - 1 expression vectors individually .',\n",
       "    'DNA']]],\n",
       " ['Our results provide the first evidence that both GATA - 1 and C/EBPbeta synergistically transactivate the promoter of an eosinophil - specific granule protein gene and that FOG may act as a negative cofactor for the eosinophil lineage , unlike its positively regulatory function for the erythroid lineage .',\n",
       "  [['Our results provide the first evidence that both GATA - 1 and C/EBPbeta synergistically transactivate the promoter of an eosinophil - specific granule protein gene and that FOG may act as a negative cofactor for the eosinophil lineage , unlike its positively regulatory function for the <e1> erythroid lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['DNase I footprinting identified several protected areas including sp1 binding siteswithin the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including <e1> sp1 binding sites </e1> within the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , sp1/ap - 2 binding siteswithin the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , <e1> sp1/ap - 2 binding sites </e1> within the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and camp response element cre binding siteswithin the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and <e1> camp response element cre binding sites </e1> within the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and sp1 binding sitein the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and <e1> sp1 binding site </e1> in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , nre - 2a binding sitein the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , <e1> nre - 2a binding site </e1> in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , NRE - 2a , tcf - 1/lef - 1 binding sitein the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , NRE - 2a , <e1> tcf - 1/lef - 1 binding site </e1> in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including sp1 binding siteswithin the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including <e1> sp1 binding sites </e1> within the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , sp1/ap - 2 binding siteswithin the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , <e1> sp1/ap - 2 binding sites </e1> within the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and camp response element cre binding siteswithin the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and <e1> camp response element cre binding sites </e1> within the 201 - bp core promoter region and Sp1 , NRE - 2a , TCF - 1/LEF - 1 , and Sp1/NF - AT binding sites in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and sp1 binding sitein the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and <e1> sp1 binding site </e1> in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , nre - 2a binding sitein the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , <e1> nre - 2a binding site </e1> in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , NRE - 2a , tcf - 1/lef - 1 binding sitein the upstream regulatory region .',\n",
       "  [['DNase I footprinting identified several protected areas including Sp1 , Sp1/AP - 2 , and cAMP response element ( CRE ) binding sites within the 201 - bp core promoter region and Sp1 , NRE - 2a , <e1> tcf - 1/lef - 1 binding site </e1> in the upstream regulatory region .',\n",
       "    'DNA']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 1 gene product .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 1 gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 2 gene product .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 2 gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 2or GATA - 1 and - 3 gene products .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 2 </e1> or GATA - 1 and - 3 gene products .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing GATA - 1 and - 2 or gata - 3gene products .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing GATA - 1 and - 2 or <e1> gata - 3 </e1> gene products .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 1 gene product .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 1 gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 2 gene product .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 2 gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 2or GATA - 1 and - 3 gene products .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 2 </e1> or GATA - 1 and - 3 gene products .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing GATA - 1 and - 2 or gata - 3gene products .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing GATA - 1 and - 2 or <e1> gata - 3 </e1> gene products .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 1 gene product .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 1 gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 2 gene product .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 2 gene product </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing gata - 2or GATA - 1 and - 3 gene products .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing <e1> gata - 2 </e1> or GATA - 1 and - 3 gene products .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing GATA - 1 and - 2 or gata - 3gene products .',\n",
       "  [['Interestingly , GATA - 2 and GATA - 3 proteins also localized to the same nuclear bodies in cell lines co - expressing GATA - 1 and - 2 or <e1> gata - 3 </e1> gene products .',\n",
       "    'PROTEIN']]],\n",
       " ['After incubation of purified mbp - hbdwith rabbit reticulocyte lysate , known to be rich in heat shock proteins , we obtained saturable binding of [3H]aldosterone .',\n",
       "  [['After incubation of purified <e1> mbp - hbd </e1> with rabbit reticulocyte lysate , known to be rich in heat shock proteins , we obtained saturable binding of [3H]aldosterone .',\n",
       "    'PROTEIN']]],\n",
       " ['Estrogen receptors are found on vascular endothelial cell; their expression is influenced by exposure to the hormone .',\n",
       "  [['Estrogen receptors are found on <e1> vascular endothelial cell </e1> ; their expression is influenced by exposure to the hormone .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Overexpression of dominant negative forms of ikk - betademonstrates that only IKKbeta is the target for PKC and calcineurin .',\n",
       "  [['Overexpression of dominant negative forms of <e1> ikk - beta </e1> demonstrates that only IKKbeta is the target for PKC and calcineurin .',\n",
       "    'PROTEIN']]],\n",
       " ['The product of the B - cell - specific B29 gene ( B29 , Ig beta , CD79b ) is essential for Ig - mediated B - cell activation via the B - cell antigen receptor complex ( BCR ) on human b lymphocyte .',\n",
       "  [['The product of the B - cell - specific B29 gene ( B29 , Ig beta , CD79b ) is essential for Ig - mediated B - cell activation via the B - cell antigen receptor complex ( BCR ) on <e1> human b lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The human B29 gene lacks either a tata boxand transcription is initiated at multiple sites .',\n",
       "  [['The human B29 gene lacks either a <e1> tata box </e1> and transcription is initiated at multiple sites .',\n",
       "    'DNA']]],\n",
       " ['This minimal promoter exhibits B - cell - specific activity and contains sp1 transcription factor motif .',\n",
       "  [['This minimal promoter exhibits B - cell - specific activity and contains <e1> sp1 transcription factor motif </e1> .',\n",
       "    'DNA']]],\n",
       " ['This minimal promoter exhibits B - cell - specific activity and contains SP1 , ets transcription factor motif .',\n",
       "  [['This minimal promoter exhibits B - cell - specific activity and contains SP1 , <e1> ets transcription factor motif </e1> .',\n",
       "    'DNA']]],\n",
       " ['This minimal promoter exhibits B - cell - specific activity and contains SP1 , ETS , oct transcription factor motif .',\n",
       "  [['This minimal promoter exhibits B - cell - specific activity and contains SP1 , ETS , <e1> oct transcription factor motif </e1> .',\n",
       "    'DNA']]],\n",
       " ['The transcriptional mechanisms that drive colony - forming unit granulocyte - macrophage ( CFU - GM ) myeloid progenitors to differentiate into cells of either the granulocytic lineageare not fully understood .',\n",
       "  [['The transcriptional mechanisms that drive colony - forming unit granulocyte - macrophage ( CFU - GM ) myeloid progenitors to differentiate into cells of either the <e1> granulocytic lineage </e1> are not fully understood .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We have recently demonstrated that hydrocortisone and other glucocorticoids inhibit reactive oxygen species ( ROS ) generation by mononuclear mnc leucocyte( PMNL ) .',\n",
       "  [['We have recently demonstrated that hydrocortisone and other glucocorticoids inhibit reactive oxygen species ( ROS ) generation by <e1> mononuclear mnc leucocyte </e1> ( PMNL ) .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Finally , we investigated whether the modulation of corticosensitivity in tnfalpha - pretreated u937 cellwas related to a change of the glucocorticoid receptor concentration and affinity .',\n",
       "  [['Finally , we investigated whether the modulation of corticosensitivity in <e1> tnfalpha - pretreated u937 cell </e1> was related to a change of the glucocorticoid receptor concentration and affinity .',\n",
       "    'CELL_LINE']]],\n",
       " ['The intensity of C/EBP beta staining was greater ( P < 0.001 ) in synovial fluid monocytes than in those from normal peripheral blood .',\n",
       "  [['The intensity of C/EBP beta staining was greater ( P < 0.001 ) in synovial fluid monocytes than in those from <e1> normal peripheral blood </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Direct interaction of hematopoietic transcription factor gata - 1: functional antagonism in erythroid cells .',\n",
       "  [['Direct interaction of <e1> hematopoietic transcription factor gata - 1 </e1> : functional antagonism in erythroid cells .',\n",
       "    'PROTEIN']]],\n",
       " ['PU.1 is a hematopoietic - specific Ets family transcription factor that is required for development of some lymphoid lineage .',\n",
       "  [['PU.1 is a hematopoietic - specific Ets family transcription factor that is required for development of some <e1> lymphoid lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['To determine whether it is possible to artificially enhance the opening of the chromatin structure of a minimal beta - globin promoter , we placed a 101bp , erythroid - specific DNase 1 hypersensitive site - forming element ( HSFE ) immediately upstream of the beta - globin gene .',\n",
       "  [['To determine whether it is possible to artificially enhance the opening of the chromatin structure of a minimal beta - globin promoter , we placed a 101bp , erythroid - specific DNase 1 hypersensitive site - forming element ( HSFE ) immediately upstream of the <e1> beta - globin gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Despite constitutive association with TRADD or RIP , LMP1 does not induce apoptosis in ebv - negative burkitt lymphoma cell .',\n",
       "  [['Despite constitutive association with TRADD or RIP , LMP1 does not induce apoptosis in <e1> ebv - negative burkitt lymphoma cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Inhibition of NF - kappa B activity in human T lymphocytes induces caspase - dependent apoptosis without detectable activation of caspase - 3 .',\n",
       "  [['Inhibition of NF - kappa B activity in human T lymphocytes induces caspase - dependent apoptosis without detectable activation of <e1> caspase - 3 </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['We identified this 92 - kDa protein as STAT5 , but not as stats 3, and 6 nor c - fes and vav protooncogene products , and demonstrated its translocation to the nucleus , enhancement of specific DNA binding capacity , and potentiation of trancriptional activity by GM - CSF .',\n",
       "  [['We identified this 92 - kDa protein as STAT5 , but not as <e1> stats 3 </e1> , and 6 nor c - fes and vav protooncogene products , and demonstrated its translocation to the nucleus , enhancement of specific DNA binding capacity , and potentiation of trancriptional activity by GM - CSF .',\n",
       "    'PROTEIN']]],\n",
       " ['We identified this 92 - kDa protein as STAT5 , but not as stats 6nor c - fes and vav protooncogene products , and demonstrated its translocation to the nucleus , enhancement of specific DNA binding capacity , and potentiation of trancriptional activity by GM - CSF .',\n",
       "  [['We identified this 92 - kDa protein as STAT5 , but not as <e1> stats 6 </e1> nor c - fes and vav protooncogene products , and demonstrated its translocation to the nucleus , enhancement of specific DNA binding capacity , and potentiation of trancriptional activity by GM - CSF .',\n",
       "    'PROTEIN']]],\n",
       " ['N - formyl - methionyl - leucyl - phenylalanine ( FMLP ) and phorbol myristate acetate ( PMA ) induced tyrosine phosphorylation of 42 - kda protein, which were identified as extracellular signal - regulated kinase ( ERK ) , in human monocytes .',\n",
       "  [['N - formyl - methionyl - leucyl - phenylalanine ( FMLP ) and phorbol myristate acetate ( PMA ) induced tyrosine phosphorylation of <e1> 42 - kda protein </e1> , which were identified as extracellular signal - regulated kinase ( ERK ) , in human monocytes .',\n",
       "    'PROTEIN']]],\n",
       " ['We searched for proteins that interact with Ras in interleukin il - 2 - stimulated cell, and found that the transcription factor Aiolos interacts with Ras .',\n",
       "  [['We searched for proteins that interact with Ras in <e1> interleukin il - 2 - stimulated cell </e1> , and found that the transcription factor Aiolos interacts with Ras .',\n",
       "    'CELL_LINE']]],\n",
       " ['We searched for proteins that interact with Ras in interleukin ( il - 2 - deprived cell, and found that the transcription factor Aiolos interacts with Ras .',\n",
       "  [['We searched for proteins that interact with Ras in interleukin ( <e1> il - 2 - deprived cell </e1> , and found that the transcription factor Aiolos interacts with Ras .',\n",
       "    'CELL_LINE']]],\n",
       " ['In addition , we show that PNU156804 does not block significantly the induction of either IL - 2 or il - 2r alpha - chainbut inhibits IL - 2 - dependent T cell proliferation .',\n",
       "  [['In addition , we show that PNU156804 does not block significantly the induction of either IL - 2 or <e1> il - 2r alpha - chain </e1> but inhibits IL - 2 - dependent T cell proliferation .',\n",
       "    'PROTEIN']]],\n",
       " ['In addition , we show that PNU156804 does not block significantly the induction of either IL - 2 or il - 2r gamma - chainbut inhibits IL - 2 - dependent T cell proliferation .',\n",
       "  [['In addition , we show that PNU156804 does not block significantly the induction of either IL - 2 or <e1> il - 2r gamma - chain </e1> but inhibits IL - 2 - dependent T cell proliferation .',\n",
       "    'PROTEIN']]],\n",
       " ['Human T - cell leukemia virus type 1 Tax protein induces the expression of stat1 genein T - cells .',\n",
       "  [['Human T - cell leukemia virus type 1 Tax protein induces the expression of <e1> stat1 gene </e1> in T - cells .',\n",
       "    'DNA']]],\n",
       " ['HTLV - 1 - transformed T - cell lines expressed higher amounts of stat1 rnaand proteins than virus - negative T cells .',\n",
       "  [['HTLV - 1 - transformed T - cell lines expressed higher amounts of <e1> stat1 rna </e1> and proteins than virus - negative T cells .',\n",
       "    'RNA']]],\n",
       " ['HTLV - 1 - transformed T - cell lines expressed higher amounts of STAT1 , stat3 rnaand proteins than virus - negative T cells .',\n",
       "  [['HTLV - 1 - transformed T - cell lines expressed higher amounts of STAT1 , <e1> stat3 rna </e1> and proteins than virus - negative T cells .',\n",
       "    'RNA']]],\n",
       " ['The expression of stat1 mrnawere also induced by a T - cell mitogen in normal human peripheral blood mononuclear cells .',\n",
       "  [['The expression of <e1> stat1 mrna </e1> were also induced by a T - cell mitogen in normal human peripheral blood mononuclear cells .',\n",
       "    'RNA']]],\n",
       " ['The expression of STAT1 , stat3 mrnawere also induced by a T - cell mitogen in normal human peripheral blood mononuclear cells .',\n",
       "  [['The expression of STAT1 , <e1> stat3 mrna </e1> were also induced by a T - cell mitogen in normal human peripheral blood mononuclear cells .',\n",
       "    'RNA']]],\n",
       " ['A unique transcriptional mechanism distinct from those regulating the il - 2 gene .',\n",
       "  [['A unique transcriptional mechanism distinct from those regulating the <e1> il - 2 gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['A unique transcriptional mechanism distinct from those regulating the il - 4 gene .',\n",
       "  [['A unique transcriptional mechanism distinct from those regulating the <e1> il - 4 gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['We found that the spontaneous production of tumor necrosis factor alpha and other pro - inflammatory cytokines is NF - kappaB - dependent in rheumatoid synovial tissue , in contrast to the main anti - inflammatory mediators , like il - 11, and the IL - 1 receptor antagonist .',\n",
       "  [['We found that the spontaneous production of tumor necrosis factor alpha and other pro - inflammatory cytokines is NF - kappaB - dependent in rheumatoid synovial tissue , in contrast to the main anti - inflammatory mediators , like <e1> il - 11 </e1> , and the IL - 1 receptor antagonist .',\n",
       "    'PROTEIN']]],\n",
       " ['Of even more interest , IkappaBalpha overexpression inhibited the production of matrix metalloproteinase 3while not affecting their tissue inhibitor .',\n",
       "  [['Of even more interest , IkappaBalpha overexpression inhibited the production of <e1> matrix metalloproteinase 3 </e1> while not affecting their tissue inhibitor .',\n",
       "    'PROTEIN']]],\n",
       " ['To find out if the induction by UV light is caused by the formation of an Ah receptor ligand , hepa - 1 wild - type cell linewere applied .',\n",
       "  [['To find out if the induction by UV light is caused by the formation of an Ah receptor ligand , <e1> hepa - 1 wild - type cell line </e1> were applied .',\n",
       "    'CELL_LINE']]],\n",
       " ['Enforced expression of Id3 , which has the capacity to inhibit many basic helix - loop - helix ( bHLH ) transcription factors , in human CD34(+) hematopoietic progenitor cells that have not undergone T cell receptor ( TCR ) gene rearrangements inhibits development of the transduced cells into tcr alpha beta cellin a fetal thymic organ culture ( FTOC ) .',\n",
       "  [['Enforced expression of Id3 , which has the capacity to inhibit many basic helix - loop - helix ( bHLH ) transcription factors , in human CD34(+) hematopoietic progenitor cells that have not undergone T cell receptor ( TCR ) gene rearrangements inhibits development of the transduced cells into <e1> tcr alpha beta cell </e1> in a fetal thymic organ culture ( FTOC ) .',\n",
       "    'CELL_LINE']]],\n",
       " ['Thus , cosignaling via the lt - beta receptoris probably involved in the modulation of HIV - 1 replication and the subsequent determination of HIV - 1 viral burden in monocytes .',\n",
       "  [['Thus , cosignaling via the <e1> lt - beta receptor </e1> is probably involved in the modulation of HIV - 1 replication and the subsequent determination of HIV - 1 viral burden in monocytes .',\n",
       "    'PROTEIN']]],\n",
       " ['However NF - ATs appear in the nuclei of il - 4 - stimulated eosinophil .',\n",
       "  [['However NF - ATs appear in the nuclei of <e1> il - 4 - stimulated eosinophil </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['However NF - ATs appear in the nuclei of il - 5 - stimulated eosinophil .',\n",
       "  [['However NF - ATs appear in the nuclei of <e1> il - 5 - stimulated eosinophil </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Only NF - AT1 and NF - AT4 , but not NF - AT2 and NF - AT3 , have translocated into the nuclei in il - 4 - stimulated eosinophil .',\n",
       "  [['Only NF - AT1 and NF - AT4 , but not NF - AT2 and NF - AT3 , have translocated into the nuclei in <e1> il - 4 - stimulated eosinophil </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Only NF - AT1 and NF - AT4 , but not NF - AT2 and NF - AT3 , have translocated into the nuclei in il - 5 - stimulated eosinophil .',\n",
       "  [['Only NF - AT1 and NF - AT4 , but not NF - AT2 and NF - AT3 , have translocated into the nuclei in <e1> il - 5 - stimulated eosinophil </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['A well - studied example is nuclear factors of activated T - cell ( NFAT ) sites which are composite elements of a nfatp/c binding site .',\n",
       "  [['A well - studied example is nuclear factors of activated T - cell ( NFAT ) sites which are composite elements of a <e1> nfatp/c binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['A well - studied example is nuclear factors of activated T - cell ( NFAT ) sites which are composite elements of a NFATp/c and an activating protein 1 ap - 1 binding site .',\n",
       "  [['A well - studied example is nuclear factors of activated T - cell ( NFAT ) sites which are composite elements of a NFATp/c and <e1> an activating protein 1 ap - 1 binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['All three steps progressively helpX?ed to discriminate T - cell - specific promoter sequences against other functional regions ( coding sequence) of the same genes , against promoters of muscle - specific genes or against random sequences .',\n",
       "  [['All three steps progressively helpX?ed to discriminate T - cell - specific promoter sequences against other functional regions ( <e1> coding sequence </e1> ) of the same genes , against promoters of muscle - specific genes or against random sequences .',\n",
       "    'DNA']]],\n",
       " ['A silent mutation in codon 10 of exon 1 , detected in er - negative human breast cancer cell line, in breast tumors and blood DNA from breast cancer patients , has been recognized as a polymorphic site .',\n",
       "  [['A silent mutation in codon 10 of exon 1 , detected in <e1> er - negative human breast cancer cell line </e1> , in breast tumors and blood DNA from breast cancer patients , has been recognized as a polymorphic site .',\n",
       "    'CELL_LINE']]],\n",
       " ['A silent mutation in codon 10 of exon 1 , detected in er - positive human breast cancer cell line, in breast tumors and blood DNA from breast cancer patients , has been recognized as a polymorphic site .',\n",
       "  [['A silent mutation in codon 10 of exon 1 , detected in <e1> er - positive human breast cancer cell line </e1> , in breast tumors and blood DNA from breast cancer patients , has been recognized as a polymorphic site .',\n",
       "    'CELL_LINE']]],\n",
       " ['The c - Myc oncoprotein is important in control of cellular proliferation and differentiation , while Yin - Yang 1 ( YY1 ) has been shown to control the expression of a number of cellular gene .',\n",
       "  [['The c - Myc oncoprotein is important in control of cellular proliferation and differentiation , while Yin - Yang 1 ( YY1 ) has been shown to control the expression of a number of <e1> cellular gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Expression and role of PML gene in normal adult hematopoiesis : functional interaction between pml proteinin erythropoiesis .',\n",
       "  [['Expression and role of PML gene in normal adult hematopoiesis : functional interaction between <e1> pml protein </e1> in erythropoiesis .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , early treatment ( day 0 HPCs ) with alpha - PML reduced the number of both erythroid colony, whereas late treatment ( day 5 culture ) reduced erythroid , but not granulocytic , clonogenesis .',\n",
       "  [['Interestingly , early treatment ( day 0 HPCs ) with alpha - PML reduced the number of both <e1> erythroid colony </e1> , whereas late treatment ( day 5 culture ) reduced erythroid , but not granulocytic , clonogenesis .',\n",
       "    'CELL_LINE']]],\n",
       " ['Different sequence requirements for expression in erythroid cellwithin a regulatory element upstream of the GATA - 1 gene .',\n",
       "  [['Different sequence requirements for expression in <e1> erythroid cell </e1> within a regulatory element upstream of the GATA - 1 gene .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The lineage - restricted transcription factor GATA - 1 is required for differentiation of erythroid cell .',\n",
       "  [['The lineage - restricted transcription factor GATA - 1 is required for differentiation of <e1> erythroid cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['HS I directs high - level expression of reporter GATA - 1/lacZ genes to primitive erythroid celland megakaryocytes in transgenic mice .',\n",
       "  [['HS I directs high - level expression of reporter GATA - 1/lacZ genes to <e1> primitive erythroid cell </e1> and megakaryocytes in transgenic mice .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Immortalization of cd4+ t lymphocyteby human T - cell leukemia virus type 1 Tax mutants expressed in a functional molecular clone .',\n",
       "  [['Immortalization of <e1> cd4+ t lymphocyte </e1> by human T - cell leukemia virus type 1 Tax mutants expressed in a functional molecular clone .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In contrast , dominant negative rac proteindid not affect monocyte adhesion or spreading .',\n",
       "  [['In contrast , <e1> dominant negative rac protein </e1> did not affect monocyte adhesion or spreading .',\n",
       "    'PROTEIN']]],\n",
       " ['In contrast , dominant negative cdc42 proteindid not affect monocyte adhesion or spreading .',\n",
       "  [['In contrast , <e1> dominant negative cdc42 protein </e1> did not affect monocyte adhesion or spreading .',\n",
       "    'PROTEIN']]],\n",
       " ['Finally , receptor clusters colocalized with ezrin protein .',\n",
       "  [['Finally , receptor clusters colocalized with <e1> ezrin protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Finally , receptor clusters colocalized with ezrin / moesin protein .',\n",
       "  [['Finally , receptor clusters colocalized with ezrin / <e1> moesin protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Expression of th1 type cytokineresponding to HBsAg and HBxAg in chronic hepatitis B patients .',\n",
       "  [['Expression of <e1> th1 type cytokine </e1> responding to HBsAg and HBxAg in chronic hepatitis B patients .',\n",
       "    'PROTEIN']]],\n",
       " ['METHODS : Eight malignant lymphocytic cell lines and primary cultures of lymphocytic leukemia cellwere treated with As2O3 , with or without dithiothreitol ( DTT ) or buthionine sulfoximine ( BSO ) ( an inhibitor of glutathione synthesis ) .',\n",
       "  [['METHODS : Eight malignant lymphocytic cell lines and primary cultures of <e1> lymphocytic leukemia cell </e1> were treated with As2O3 , with or without dithiothreitol ( DTT ) or buthionine sulfoximine ( BSO ) ( an inhibitor of glutathione synthesis ) .',\n",
       "    'CELL_TYPE']]],\n",
       " ['NF - kappaB functions as both a proapoptotic regulatory factorwithin a single cell type .',\n",
       "  [['NF - kappaB functions as both a <e1> proapoptotic regulatory factor </e1> within a single cell type .',\n",
       "    'PROTEIN']]],\n",
       " ['To determine why alveolar macrophages do not express AP - 1 DNA binding activity , we first showed that there was not a decrease in expression of the fos proteinthat make up the AP - 1 complex .',\n",
       "  [['To determine why alveolar macrophages do not express AP - 1 DNA binding activity , we first showed that there was not a decrease in expression of the <e1> fos protein </e1> that make up the AP - 1 complex .',\n",
       "    'PROTEIN']]],\n",
       " ['The secondary lymphoid tissues are located at strategic sites where foreign antigens can be efficiently brought together with immune system regulatory cell .',\n",
       "  [['The secondary lymphoid tissues are located at strategic sites where foreign antigens can be efficiently brought together with <e1> immune system regulatory cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The secondary lymphoid tissues are located at strategic sites where foreign antigens can be efficiently brought together with immune system effector cell .',\n",
       "  [['The secondary lymphoid tissues are located at strategic sites where foreign antigens can be efficiently brought together with <e1> immune system effector cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['This correlates with changes of nuclear protein - binding activities to two motifs , site beta, within the CD28 minimal promoter .',\n",
       "  [['This correlates with changes of nuclear protein - binding activities to two motifs , <e1> site beta </e1> , within the CD28 minimal promoter .',\n",
       "    'PROTEIN']]],\n",
       " ['In vivo expanded cd4+cd28null t celluniformly lack alpha - and beta - bound complexes , resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "  [['In vivo expanded <e1> cd4+cd28null t cell </e1> uniformly lack alpha - and beta - bound complexes , resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['In vivo expanded CD4+CD28null and CD8+CD28null T cells uniformly lack alpha - bound complex, resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "  [['In vivo expanded CD4+CD28null and CD8+CD28null T cells uniformly lack <e1> alpha - bound complex </e1> , resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "    'PROTEIN']]],\n",
       " ['In vivo expanded cd4+cd28null t celluniformly lack alpha - and beta - bound complexes , resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "  [['In vivo expanded <e1> cd4+cd28null t cell </e1> uniformly lack alpha - and beta - bound complexes , resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['In vivo expanded CD4+CD28null and CD8+CD28null T cells uniformly lack alpha - bound complex, resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "  [['In vivo expanded CD4+CD28null and CD8+CD28null T cells uniformly lack <e1> alpha - bound complex </e1> , resembling the pattern seen in chronically activated cells and not of senescent cells .',\n",
       "    'PROTEIN']]],\n",
       " ['The contact of natural killer ( NK ) cells with foreign cells and with certain virus - infected celltriggers the cytolytic machinery of NK cells .',\n",
       "  [['The contact of natural killer ( NK ) cells with foreign cells and with certain <e1> virus - infected cell </e1> triggers the cytolytic machinery of NK cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Differences in phosphorylation of the IL - 2R associated JAK/STAT proteins between htlv - i+ cell lineand uncultured leukemic cells from patients with adult T - cell lymphoma/leukemia .',\n",
       "  [['Differences in phosphorylation of the IL - 2R associated JAK/STAT proteins between <e1> htlv - i+ cell line </e1> and uncultured leukemic cells from patients with adult T - cell lymphoma/leukemia .',\n",
       "    'CELL_LINE']]],\n",
       " ['Differences in phosphorylation of the IL - 2R associated JAK/STAT proteins between HTLV - I(+) , il - 2 - independent cell lineand uncultured leukemic cells from patients with adult T - cell lymphoma/leukemia .',\n",
       "  [['Differences in phosphorylation of the IL - 2R associated JAK/STAT proteins between HTLV - I(+) , <e1> il - 2 - independent cell line </e1> and uncultured leukemic cells from patients with adult T - cell lymphoma/leukemia .',\n",
       "    'CELL_LINE']]],\n",
       " ['Differences in phosphorylation of the IL - 2R associated JAK/STAT proteins between HTLV - I(+) , il - 2 - dependent cell lineand uncultured leukemic cells from patients with adult T - cell lymphoma/leukemia .',\n",
       "  [['Differences in phosphorylation of the IL - 2R associated JAK/STAT proteins between HTLV - I(+) , <e1> il - 2 - dependent cell line </e1> and uncultured leukemic cells from patients with adult T - cell lymphoma/leukemia .',\n",
       "    'CELL_LINE']]],\n",
       " ['IL - 10 is a well - known immunosuppressive cytokine .',\n",
       "  [['IL - 10 is a well - known <e1> immunosuppressive cytokine </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Cell growth - regulated expression of mammalian mcm5 genemediated by the transcription factor E2F .',\n",
       "  [['Cell growth - regulated expression of <e1> mammalian mcm5 gene </e1> mediated by the transcription factor E2F .',\n",
       "    'DNA']]],\n",
       " ['Cell growth - regulated expression of mammalian mcm6 genemediated by the transcription factor E2F .',\n",
       "  [['Cell growth - regulated expression of <e1> mammalian mcm6 gene </e1> mediated by the transcription factor E2F .',\n",
       "    'DNA']]],\n",
       " [' human mcm5 promoterregulation following serum stimulation and exogenous E2F expression .',\n",
       "  [[' <e1> human mcm5 promoter </e1> regulation following serum stimulation and exogenous E2F expression .',\n",
       "    'DNA']]],\n",
       " [' human mcm6 promoterregulation following serum stimulation and exogenous E2F expression .',\n",
       "  [[' <e1> human mcm6 promoter </e1> regulation following serum stimulation and exogenous E2F expression .',\n",
       "    'DNA']]],\n",
       " ['Our results demonstrated that the growth - regulated expression of mammalian mcm5 gene, and presumably other MCM members , is primarily regulated by E2F through binding to multiple E2F sites in the promoters .',\n",
       "  [['Our results demonstrated that the growth - regulated expression of <e1> mammalian mcm5 gene </e1> , and presumably other MCM members , is primarily regulated by E2F through binding to multiple E2F sites in the promoters .',\n",
       "    'DNA']]],\n",
       " ['Our results demonstrated that the growth - regulated expression of mammalian mcm6 gene, and presumably other MCM members , is primarily regulated by E2F through binding to multiple E2F sites in the promoters .',\n",
       "  [['Our results demonstrated that the growth - regulated expression of <e1> mammalian mcm6 gene </e1> , and presumably other MCM members , is primarily regulated by E2F through binding to multiple E2F sites in the promoters .',\n",
       "    'DNA']]],\n",
       " ['These results offer insight into a new role for NF - kappaB in maintaining survival and function in hematopoietic stem celland suggest that proposed strategies involving inhibition of NF - kappaB activation as an adjunct to cancer chemotherapy should be approached with caution .',\n",
       "  [['These results offer insight into a new role for NF - kappaB in maintaining survival and function in <e1> hematopoietic stem cell </e1> and suggest that proposed strategies involving inhibition of NF - kappaB activation as an adjunct to cancer chemotherapy should be approached with caution .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These results offer insight into a new role for NF - kappaB in maintaining survival and function in hematopoietic progenitor celland suggest that proposed strategies involving inhibition of NF - kappaB activation as an adjunct to cancer chemotherapy should be approached with caution .',\n",
       "  [['These results offer insight into a new role for NF - kappaB in maintaining survival and function in <e1> hematopoietic progenitor cell </e1> and suggest that proposed strategies involving inhibition of NF - kappaB activation as an adjunct to cancer chemotherapy should be approached with caution .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The effects of GDSP on pancreatic weight , serum lipase, morphologic changes in the pancreas , neutrophil infiltration , and nuclear factor KB ( NF - KB ) activation were measured in rats infused with supramaximal dose of cerulein ( 5 ( g/kg/h ) for 6 h .',\n",
       "  [['The effects of GDSP on pancreatic weight , <e1> serum lipase </e1> , morphologic changes in the pancreas , neutrophil infiltration , and nuclear factor KB ( NF - KB ) activation were measured in rats infused with supramaximal dose of cerulein ( 5 ( g/kg/h ) for 6 h .',\n",
       "    'PROTEIN']]],\n",
       " ['Capacitance would simply involve expression of il - 12r beta 1 subunit, regulated by TCR , IL - 4 and IFNs .',\n",
       "  [['Capacitance would simply involve expression of <e1> il - 12r beta 1 subunit </e1> , regulated by TCR , IL - 4 and IFNs .',\n",
       "    'PROTEIN']]],\n",
       " ['Capacitance would simply involve expression of il - 12r beta 2 subunit, regulated by TCR , IL - 4 and IFNs .',\n",
       "  [['Capacitance would simply involve expression of <e1> il - 12r beta 2 subunit </e1> , regulated by TCR , IL - 4 and IFNs .',\n",
       "    'PROTEIN']]],\n",
       " ['This effect can be associated with the activation of specific hormone receptors , since we observed the expression of the estrogen receptor alpha er - alpha mrna; the ER - alpha protein expression was confirmed by immunocytochemical analysis .',\n",
       "  [['This effect can be associated with the activation of specific hormone receptors , since we observed the expression of the <e1> estrogen receptor alpha er - alpha mrna </e1> ; the ER - alpha protein expression was confirmed by immunocytochemical analysis .',\n",
       "    'RNA']]],\n",
       " ['This effect can be associated with the activation of specific hormone receptors , since we observed the expression of the estrogen receptor alpha ( er - beta mrna; the ER - alpha protein expression was confirmed by immunocytochemical analysis .',\n",
       "  [['This effect can be associated with the activation of specific hormone receptors , since we observed the expression of the estrogen receptor alpha ( <e1> er - beta mrna </e1> ; the ER - alpha protein expression was confirmed by immunocytochemical analysis .',\n",
       "    'RNA']]],\n",
       " ['This effect can be associated with the activation of specific hormone receptors , since we observed the expression of the estrogen receptor alpha ( ER - alpha ) , ER - beta , and progesterone receptor pr mrna; the ER - alpha protein expression was confirmed by immunocytochemical analysis .',\n",
       "  [['This effect can be associated with the activation of specific hormone receptors , since we observed the expression of the estrogen receptor alpha ( ER - alpha ) , ER - beta , and <e1> progesterone receptor pr mrna </e1> ; the ER - alpha protein expression was confirmed by immunocytochemical analysis .',\n",
       "    'RNA']]],\n",
       " ['Efforts to pinpoint the exact site of action and specificity of HePTP in the signaling cascade revealed that HePTP acts directly on the mitogen - activated protein ( MAP ) kinases erk 2and consequently reduces the magnitude and duration of their catalytic activation in intact T cells .',\n",
       "  [['Efforts to pinpoint the exact site of action and specificity of HePTP in the signaling cascade revealed that HePTP acts directly on the mitogen - activated protein ( MAP ) kinases <e1> erk 2 </e1> and consequently reduces the magnitude and duration of their catalytic activation in intact T cells .',\n",
       "    'PROTEIN']]],\n",
       " ['T cell proliferation and production of IL - 2 , IL - 4 , and IFN - gamma induced by both CD3 and CD3 /CD28 ligation and the nuclear expression of the c - jun proteinare each blocked by the p38 MAPK inhibitor SB203580 .',\n",
       "  [['T cell proliferation and production of IL - 2 , IL - 4 , and IFN - gamma induced by both CD3 and CD3 /CD28 ligation and the nuclear expression of the <e1> c - jun protein </e1> are each blocked by the p38 MAPK inhibitor SB203580 .',\n",
       "    'PROTEIN']]],\n",
       " ['Epo caused an increase in the proportion of cells expressing cell surface glycophorin A ( GPA ) and up - regulated beta - globinby several fold .',\n",
       "  [['Epo caused an increase in the proportion of cells expressing cell surface glycophorin A ( GPA ) and up - regulated <e1> beta - globin </e1> by several fold .',\n",
       "    'PROTEIN']]],\n",
       " [' interferon - betapromote the reversion of blast Tcells to a resting G0/G1 configuration with all the characteristic features of stromal cell rescue ; such as high Bcl - XL expression and low Bcl - 2 .',\n",
       "  [[' <e1> interferon - beta </e1> promote the reversion of blast Tcells to a resting G0/G1 configuration with all the characteristic features of stromal cell rescue ; such as high Bcl - XL expression and low Bcl - 2 .',\n",
       "    'PROTEIN']]],\n",
       " ['Here we show that both a fasl - reporter constructare inefficiently induced in these cells compared to wild - type cells .',\n",
       "  [['Here we show that both a <e1> fasl - reporter construct </e1> are inefficiently induced in these cells compared to wild - type cells .',\n",
       "    'DNA']]],\n",
       " ['Inhibition of IL - 4 - inducible gene expression in human monocytes by type i interferon .',\n",
       "  [['Inhibition of IL - 4 - inducible gene expression in human monocytes by <e1> type i interferon </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Inhibition of IL - 4 - inducible gene expression in human monocytes by type ii interferon .',\n",
       "  [['Inhibition of IL - 4 - inducible gene expression in human monocytes by <e1> type ii interferon </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The Th2 - type cytokines , interleukin - 4 ( IL - 4 ) and interleukin - 13 ( IL - 13 ) , induce expression of a distinct subset of genes in human monocytes , including FcepsilonRIIb ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and type i il - 1 receptor( IL - 1R ) .',\n",
       "  [['The Th2 - type cytokines , interleukin - 4 ( IL - 4 ) and interleukin - 13 ( IL - 13 ) , induce expression of a distinct subset of genes in human monocytes , including FcepsilonRIIb ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and <e1> type i il - 1 receptor </e1> ( IL - 1R ) .',\n",
       "    'PROTEIN']]],\n",
       " ['The Th2 - type cytokines , interleukin - 4 ( IL - 4 ) and interleukin - 13 ( IL - 13 ) , induce expression of a distinct subset of genes in human monocytes , including FcepsilonRIIb ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and type ii il - 1 receptor( IL - 1R ) .',\n",
       "  [['The Th2 - type cytokines , interleukin - 4 ( IL - 4 ) and interleukin - 13 ( IL - 13 ) , induce expression of a distinct subset of genes in human monocytes , including FcepsilonRIIb ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and <e1> type ii il - 1 receptor </e1> ( IL - 1R ) .',\n",
       "    'PROTEIN']]],\n",
       " ['We also discuss our recent findings that type i ifnsuppress IL - 4 / IL - 13 - inducible gene expression by inhibiting tyrosine phosphorylation and nuclear translocation of STAT6 .',\n",
       "  [['We also discuss our recent findings that <e1> type i ifn </e1> suppress IL - 4 / IL - 13 - inducible gene expression by inhibiting tyrosine phosphorylation and nuclear translocation of STAT6 .',\n",
       "    'PROTEIN']]],\n",
       " ['We also discuss our recent findings that type ii ifnsuppress IL - 4 / IL - 13 - inducible gene expression by inhibiting tyrosine phosphorylation and nuclear translocation of STAT6 .',\n",
       "  [['We also discuss our recent findings that <e1> type ii ifn </e1> suppress IL - 4 / IL - 13 - inducible gene expression by inhibiting tyrosine phosphorylation and nuclear translocation of STAT6 .',\n",
       "    'PROTEIN']]],\n",
       " ['The ability of type i ifnto inhibit IL - 4 / IL - 13 - induced STAT6 activity is dose - and time - dependent , and is not unique to monocytes because IFNs induce the same effects in fibroblasts .',\n",
       "  [['The ability of <e1> type i ifn </e1> to inhibit IL - 4 / IL - 13 - induced STAT6 activity is dose - and time - dependent , and is not unique to monocytes because IFNs induce the same effects in fibroblasts .',\n",
       "    'PROTEIN']]],\n",
       " ['The ability of type ii ifnto inhibit IL - 4 / IL - 13 - induced STAT6 activity is dose - and time - dependent , and is not unique to monocytes because IFNs induce the same effects in fibroblasts .',\n",
       "  [['The ability of <e1> type ii ifn </e1> to inhibit IL - 4 / IL - 13 - induced STAT6 activity is dose - and time - dependent , and is not unique to monocytes because IFNs induce the same effects in fibroblasts .',\n",
       "    'PROTEIN']]],\n",
       " ['A cdk2 inhibitor , Olomoucine , as well as a dominant - negative cdk2 construct prevented HLA class I - mediated inactivation of Rb ; in contrast , dominant - negative cdk4 constructhad no effect .',\n",
       "  [['A cdk2 inhibitor , Olomoucine , as well as a dominant - negative cdk2 construct prevented HLA class I - mediated inactivation of Rb ; in contrast , <e1> dominant - negative cdk4 construct </e1> had no effect .',\n",
       "    'DNA']]],\n",
       " ['A cdk2 inhibitor , Olomoucine , as well as a dominant - negative cdk2 construct prevented HLA class I - mediated inactivation of Rb ; in contrast , dominant - negative cdk6 constructhad no effect .',\n",
       "  [['A cdk2 inhibitor , Olomoucine , as well as a dominant - negative cdk2 construct prevented HLA class I - mediated inactivation of Rb ; in contrast , <e1> dominant - negative cdk6 construct </e1> had no effect .',\n",
       "    'DNA']]],\n",
       " ['PG490 can completely inhibit transcriptional activation at the purine - box/arre/nf - at target dna sequencetriggered by all stimuli examined ( PMA , PMA /Iono , tumor necrosis factor - alpha ) .',\n",
       "  [['PG490 can completely inhibit transcriptional activation at the <e1> purine - box/arre/nf - at target dna sequence </e1> triggered by all stimuli examined ( PMA , PMA /Iono , tumor necrosis factor - alpha ) .',\n",
       "    'DNA']]],\n",
       " ['Most erythroleukemic cell lines established in vitro coexpress erythrocytic markerthat often are associated with expression of Spi - 1 and/or Fli - 1 transcription factors known as transactivators of megakaryocyte - specific promoters .',\n",
       "  [['Most erythroleukemic cell lines established in vitro coexpress <e1> erythrocytic marker </e1> that often are associated with expression of Spi - 1 and/or Fli - 1 transcription factors known as transactivators of megakaryocyte - specific promoters .',\n",
       "    'PROTEIN']]],\n",
       " ['Despite this drastic effect in vivo , the in vitro immortalization of erythropoietin - dependent erythroid progenitors unexpectedly occurred at low frequency , and all four cell lines established expressed both erythrocytic globins marker( glycoprotein IIb , platelet factor 4 ) as well as Spi - 1 and Fli - 1 transcripts at permissive temperature .',\n",
       "  [['Despite this drastic effect in vivo , the in vitro immortalization of erythropoietin - dependent erythroid progenitors unexpectedly occurred at low frequency , and all four cell lines established expressed both <e1> erythrocytic globins marker </e1> ( glycoprotein IIb , platelet factor 4 ) as well as Spi - 1 and Fli - 1 transcripts at permissive temperature .',\n",
       "    'PROTEIN']]],\n",
       " ['Interestingly , enhanced expression of spi - 1 genealready was detected in the Ter 119 positive cell population of transgenic mice spleen in vivo .',\n",
       "  [['Interestingly , enhanced expression of <e1> spi - 1 gene </e1> already was detected in the Ter 119 positive cell population of transgenic mice spleen in vivo .',\n",
       "    'DNA']]],\n",
       " ['Taken together , these data indicate that the unexpected expression of megakaryocytic genes is a specific property of immortalized cells that can not be explained only by enhanced expression of spi - 1 gene .',\n",
       "  [['Taken together , these data indicate that the unexpected expression of megakaryocytic genes is a specific property of immortalized cells that can not be explained only by enhanced expression of <e1> spi - 1 gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Here we show that fludarabine , but not the immunosuppressant cyclosporine A , inhibits the cytokine - induced activation of STAT1 and STAT1 - dependent gene transcription in normal resting lymphocyte .',\n",
       "  [['Here we show that fludarabine , but not the immunosuppressant cyclosporine A , inhibits the cytokine - induced activation of STAT1 and STAT1 - dependent gene transcription in <e1> normal resting lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Furthermore , pre - b cell linewere Spi - C - negative , suggesting that Spi - C might be a regulatory molecule during a specific phase of B lymphoid development .',\n",
       "  [['Furthermore , <e1> pre - b cell line </e1> were Spi - C - negative , suggesting that Spi - C might be a regulatory molecule during a specific phase of B lymphoid development .',\n",
       "    'CELL_LINE']]],\n",
       " ['Whereas the NF - kappaB binding activity in Tax - expressing T - cell lines consisted mostly of p50/c - Rel , fresh ATL samples contained p50/p50 heterodimer .',\n",
       "  [['Whereas the NF - kappaB binding activity in Tax - expressing T - cell lines consisted mostly of p50/c - Rel , fresh ATL samples contained <e1> p50/p50 heterodimer </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Interferon - alpha induction of stats 3DNA binding and growth arrest is independent of Lck and active mitogen - activated kinase in T cells .',\n",
       "  [['Interferon - alpha induction of <e1> stats 3 </e1> DNA binding and growth arrest is independent of Lck and active mitogen - activated kinase in T cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Furthermore , stats 3are effectively phosphorylated and activated to bind DNA in the absence of Lck expression in IFN - alpha - treated cells .',\n",
       "  [['Furthermore , <e1> stats 3 </e1> are effectively phosphorylated and activated to bind DNA in the absence of Lck expression in IFN - alpha - treated cells .',\n",
       "    'PROTEIN']]],\n",
       " ['The evolutionarily conserved sequence upstream of the human Ig heavy chain S gamma 3 region is an inducible promoter : synergistic activation by CD40 ligand and IL - 4 via cooperative nf - kappa b binding site .',\n",
       "  [['The evolutionarily conserved sequence upstream of the human Ig heavy chain S gamma 3 region is an inducible promoter : synergistic activation by CD40 ligand and IL - 4 via cooperative <e1> nf - kappa b binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['These partially overlapping cd40l responsive elementare functionally cooperative as the disruption of one of them prevents synergistic promoter activation .',\n",
       "  [['These partially overlapping <e1> cd40l responsive element </e1> are functionally cooperative as the disruption of one of them prevents synergistic promoter activation .',\n",
       "    'DNA']]],\n",
       " ['The STP oncoproteins of the herpesvirus saimiri ( HVS ) subgroup A strain 11 and subgroup C strain 488 are now found to be stably associated with tumor necrosis factor receptor - associated factor traf 1, 2 , or 3 .',\n",
       "  [['The STP oncoproteins of the herpesvirus saimiri ( HVS ) subgroup A strain 11 and subgroup C strain 488 are now found to be stably associated with <e1> tumor necrosis factor receptor - associated factor traf 1 </e1> , 2 , or 3 .',\n",
       "    'PROTEIN']]],\n",
       " ['The STP oncoproteins of the herpesvirus saimiri ( HVS ) subgroup A strain 11 and subgroup C strain 488 are now found to be stably associated with tumor necrosis factor receptor - associated factor traf 2, or 3 .',\n",
       "  [['The STP oncoproteins of the herpesvirus saimiri ( HVS ) subgroup A strain 11 and subgroup C strain 488 are now found to be stably associated with <e1> tumor necrosis factor receptor - associated factor traf 2 </e1> , or 3 .',\n",
       "    'PROTEIN']]],\n",
       " ['The STP oncoproteins of the herpesvirus saimiri ( HVS ) subgroup A strain 11 and subgroup C strain 488 are now found to be stably associated with tumor necrosis factor receptor - associated factor traf 3 .',\n",
       "  [['The STP oncoproteins of the herpesvirus saimiri ( HVS ) subgroup A strain 11 and subgroup C strain 488 are now found to be stably associated with <e1> tumor necrosis factor receptor - associated factor traf 3 </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['CD40 ligation on MM by CD40L - transfected murine L - cells or by a soluble CD40L fusion protein up - regulated their expression of intercellular adhesion molecule - 1 and mhc class i moleculeand their secretion of IL - 6 , IL - 8 , tumor necrosis factor - a , and granulocyte macrophage colony - stimulating factor and also induced a rapid activation of the transcription factor nuclear factor kappaB .',\n",
       "  [['CD40 ligation on MM by CD40L - transfected murine L - cells or by a soluble CD40L fusion protein up - regulated their expression of intercellular adhesion molecule - 1 and <e1> mhc class i molecule </e1> and their secretion of IL - 6 , IL - 8 , tumor necrosis factor - a , and granulocyte macrophage colony - stimulating factor and also induced a rapid activation of the transcription factor nuclear factor kappaB .',\n",
       "    'PROTEIN']]],\n",
       " ['CD40 ligation on MM by CD40L - transfected murine L - cells or by a soluble CD40L fusion protein up - regulated their expression of intercellular adhesion molecule - 1 and mhc class ii moleculeand their secretion of IL - 6 , IL - 8 , tumor necrosis factor - a , and granulocyte macrophage colony - stimulating factor and also induced a rapid activation of the transcription factor nuclear factor kappaB .',\n",
       "  [['CD40 ligation on MM by CD40L - transfected murine L - cells or by a soluble CD40L fusion protein up - regulated their expression of intercellular adhesion molecule - 1 and <e1> mhc class ii molecule </e1> and their secretion of IL - 6 , IL - 8 , tumor necrosis factor - a , and granulocyte macrophage colony - stimulating factor and also induced a rapid activation of the transcription factor nuclear factor kappaB .',\n",
       "    'PROTEIN']]],\n",
       " ['We evaluated the immunoreactivity of the regulatory subunitof cAMP - dependent protein kinase ( protein kinase A ) and 1 of its substrates , Rap1 , in platelets from untreated euthymic , manic , and depressed patients with bipolar disorder and healthy subjects .',\n",
       "  [['We evaluated the immunoreactivity of the <e1> regulatory subunit </e1> of cAMP - dependent protein kinase ( protein kinase A ) and 1 of its substrates , Rap1 , in platelets from untreated euthymic , manic , and depressed patients with bipolar disorder and healthy subjects .',\n",
       "    'PROTEIN']]],\n",
       " ['Freshly isolated monocytes treated with the protein phosphatase inhibitor okadaic acid secreted high levels of IL - 6 protein , which coincided with enhanced binding activity of NF - kappa B as well as with phosphorylation and activation of the erk1/2 protein .',\n",
       "  [['Freshly isolated monocytes treated with the protein phosphatase inhibitor okadaic acid secreted high levels of IL - 6 protein , which coincided with enhanced binding activity of NF - kappa B as well as with phosphorylation and activation of the <e1> erk1/2 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as bcl - 2+ cell, and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "  [['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as <e1> bcl - 2+ cell </e1> , and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as BCL - 2+ , cd56+ cell, and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "  [['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as BCL - 2+ , <e1> cd56+ cell </e1> , and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as BCL - 2+ , CD56+ , cd3+ cell, and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "  [['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as BCL - 2+ , CD56+ , <e1> cd3+ cell </e1> , and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as BCL - 2+ , CD56+ , CD3+ , cd8+ cell, and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "  [['Tissues from women undergoing elective termination of uterine pregnancy and from women with ectopic pregnancy were used to detect the presence of cells important for the maintenance of pregnancy , such as BCL - 2+ , CD56+ , CD3+ , <e1> cd8+ cell </e1> , and the presence of oestrogen ( ER ) and progesterone receptors ( PR ) by immunohistochemistry .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These effects were blocked by mAbs against endothelial adhesion molecule( beta1 and beta2 integrins ) .',\n",
       "  [['These effects were blocked by mAbs against <e1> endothelial adhesion molecule </e1> ( beta1 and beta2 integrins ) .',\n",
       "    'PROTEIN']]],\n",
       " ['These effects were blocked by mAbs against endothelial and/or leukocyte adhesion molecules ( beta1 integrin) .',\n",
       "  [['These effects were blocked by mAbs against endothelial and/or leukocyte adhesion molecules ( <e1> beta1 integrin </e1> ) .',\n",
       "    'PROTEIN']]],\n",
       " ['These effects were blocked by mAbs against endothelial adhesion molecule( beta1 and beta2 integrins ) .',\n",
       "  [['These effects were blocked by mAbs against <e1> endothelial adhesion molecule </e1> ( beta1 and beta2 integrins ) .',\n",
       "    'PROTEIN']]],\n",
       " ['These effects were blocked by mAbs against endothelial and/or leukocyte adhesion molecules ( beta1 integrin) .',\n",
       "  [['These effects were blocked by mAbs against endothelial and/or leukocyte adhesion molecules ( <e1> beta1 integrin </e1> ) .',\n",
       "    'PROTEIN']]],\n",
       " ['The E2A - HLF fusion gene , generated by t(17;19)(q22;p13) in acute lymphoblastic leukemia ( ALL ) , encodes a chimeric transcription factor in which the trans - activating domains of E2A are fused to the dna - binding domainof hepatic leukemic factor ( HLF ) .',\n",
       "  [['The E2A - HLF fusion gene , generated by t(17;19)(q22;p13) in acute lymphoblastic leukemia ( ALL ) , encodes a chimeric transcription factor in which the trans - activating domains of E2A are fused to the <e1> dna - binding domain </e1> of hepatic leukemic factor ( HLF ) .',\n",
       "    'PROTEIN']]],\n",
       " ['The E2A - HLF fusion gene , generated by t(17;19)(q22;p13) in acute lymphoblastic leukemia ( ALL ) , encodes a chimeric transcription factor in which the trans - activating domains of E2A are fused to the dna - dimerization domainof hepatic leukemic factor ( HLF ) .',\n",
       "  [['The E2A - HLF fusion gene , generated by t(17;19)(q22;p13) in acute lymphoblastic leukemia ( ALL ) , encodes a chimeric transcription factor in which the trans - activating domains of E2A are fused to the <e1> dna - dimerization domain </e1> of hepatic leukemic factor ( HLF ) .',\n",
       "    'PROTEIN']]],\n",
       " ['To investigate its biological role , we generated transgenic mice expressing E2A - HLF using ig promoter, which direct transgene expression in cells committed to the lymphoid lineage .',\n",
       "  [['To investigate its biological role , we generated transgenic mice expressing E2A - HLF using <e1> ig promoter </e1> , which direct transgene expression in cells committed to the lymphoid lineage .',\n",
       "    'DNA']]],\n",
       " ['IL - 12 and IL - 2 can stimulate mitogen - activated t cellto proliferate , produce IFN - gamma , and kill tumor cells .',\n",
       "  [['IL - 12 and IL - 2 can stimulate <e1> mitogen - activated t cell </e1> to proliferate , produce IFN - gamma , and kill tumor cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Consistent with previous data from human fibroblastand T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas E2F 1 and - 3 are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "  [['Consistent with previous data from <e1> human fibroblast </e1> and T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas E2F 1 and - 3 are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Consistent with previous data from human and mouse fibroblasts and T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas e2f - 3are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "  [['Consistent with previous data from human and mouse fibroblasts and T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas <e1> e2f - 3 </e1> are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "    'PROTEIN']]],\n",
       " ['Consistent with previous data from human fibroblastand T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas E2F 1 and - 3 are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "  [['Consistent with previous data from <e1> human fibroblast </e1> and T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas E2F 1 and - 3 are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Consistent with previous data from human and mouse fibroblasts and T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas e2f - 3are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "  [['Consistent with previous data from human and mouse fibroblasts and T - lymphocytes , E2F4 and DP1 form the predominant E2F heterodimers both in G0 and G1 phases of the human B - lymphocyte cell cycle , whereas <e1> e2f - 3 </e1> are first detected in late G1 , and their expression levels increase towards S phase .',\n",
       "    'PROTEIN']]],\n",
       " ['Though the levels of dp - 2increase when cells progress from G0 to S , the proportion of DP1 to DP2 remains relatively constant during the cell cycle .',\n",
       "  [['Though the levels of <e1> dp - 2 </e1> increase when cells progress from G0 to S , the proportion of DP1 to DP2 remains relatively constant during the cell cycle .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of tnf receptor 2; PKC isoforms alpha , beta I , beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of <e1> tnf receptor 2 </e1> ; PKC isoforms alpha , beta I , beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; pkc isoform beta i, beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; <e1> pkc isoform beta i </e1> , beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; pkc isoform beta ii, and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; <e1> pkc isoform beta ii </e1> , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; pkc isoform mu; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; <e1> pkc isoform mu </e1> ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of tnf receptor 2; PKC isoforms alpha , beta I , beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of <e1> tnf receptor 2 </e1> ; PKC isoforms alpha , beta I , beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; pkc isoform beta i, beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; <e1> pkc isoform beta i </e1> , beta II , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; pkc isoform beta ii, and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; <e1> pkc isoform beta ii </e1> , and mu ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; pkc isoform mu; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "  [['Comparison of KG1 to the PMA - unresponsive subline KG1a reveals differences in expression of TNF receptors 1 and 2 ; <e1> pkc isoform mu </e1> ; and RelB , suggesting that these components/pathways are important for DC differentiation .',\n",
       "    'PROTEIN']]],\n",
       " ['NF - ATc isoforms are differentially expressed and regulated in murine t cell .',\n",
       "  [['NF - ATc isoforms are differentially expressed and regulated in <e1> murine t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['NF - ATc isoforms are differentially expressed and regulated in murine mast cell .',\n",
       "  [['NF - ATc isoforms are differentially expressed and regulated in <e1> murine mast cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Detection of NF - ATc.alpha mRNA is strictly dependent on cell activation signals in both t cell line .',\n",
       "  [['Detection of NF - ATc.alpha mRNA is strictly dependent on cell activation signals in both <e1> t cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " [' aml proteinregulate the I alpha1 germ - line promoter .',\n",
       "  [[' <e1> aml protein </e1> regulate the I alpha1 germ - line promoter .',\n",
       "    'PROTEIN']]],\n",
       " ['Here we show that the I alpha1 promoter contains several binding sites for the AML /PEBP2/CBF family of transcription factors and that aml proteinare major regulators of the basal and TGF - beta - inducible promoter activity .',\n",
       "  [['Here we show that the I alpha1 promoter contains several binding sites for the AML /PEBP2/CBF family of transcription factors and that <e1> aml protein </e1> are major regulators of the basal and TGF - beta - inducible promoter activity .',\n",
       "    'PROTEIN']]],\n",
       " ['Role of Egr - 2 in up - regulation of Fas ligand in normal T cells and aberrant double - negative lpr t cell .',\n",
       "  [['Role of Egr - 2 in up - regulation of Fas ligand in normal T cells and <e1> aberrant double - negative lpr t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Role of Egr - 2 in up - regulation of Fas ligand in normal T cells and aberrant double - negative gld t cell .',\n",
       "  [['Role of Egr - 2 in up - regulation of Fas ligand in normal T cells and <e1> aberrant double - negative gld t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Unstimulated mrl - lpr/lpr t cellconstitutively contained high levels of two proteins that bound to the FLRE .',\n",
       "  [['Unstimulated <e1> mrl - lpr/lpr t cell </e1> constitutively contained high levels of two proteins that bound to the FLRE .',\n",
       "    'CELL_LINE']]],\n",
       " ['Unstimulated MRL - lpr/lpr and c3h - gld/gld cd4 - cd8 - t cellconstitutively contained high levels of two proteins that bound to the FLRE .',\n",
       "  [['Unstimulated MRL - lpr/lpr and <e1> c3h - gld/gld cd4 - cd8 - t cell </e1> constitutively contained high levels of two proteins that bound to the FLRE .',\n",
       "    'CELL_LINE']]],\n",
       " ['Activation of normal lymph node cells resulted in increased expression of egr - 2, and - 3 .',\n",
       "  [['Activation of normal lymph node cells resulted in increased expression of <e1> egr - 2 </e1> , and - 3 .',\n",
       "    'PROTEIN']]],\n",
       " ['Activation of normal lymph node cells resulted in increased expression of egr - 3 .',\n",
       "  [['Activation of normal lymph node cells resulted in increased expression of <e1> egr - 3 </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['FasL - dependent reporter constructs in mrl - lpr/lpr t cellwere constitutively active , and this activity was largely prevented by mutation of the critical Egr family binding element .',\n",
       "  [['FasL - dependent reporter constructs in <e1> mrl - lpr/lpr t cell </e1> were constitutively active , and this activity was largely prevented by mutation of the critical Egr family binding element .',\n",
       "    'CELL_LINE']]],\n",
       " ['FasL - dependent reporter constructs in MRL - lpr/lpr and c3h - gld/gld cd4 - cd8 - t cellwere constitutively active , and this activity was largely prevented by mutation of the critical Egr family binding element .',\n",
       "  [['FasL - dependent reporter constructs in MRL - lpr/lpr and <e1> c3h - gld/gld cd4 - cd8 - t cell </e1> were constitutively active , and this activity was largely prevented by mutation of the critical Egr family binding element .',\n",
       "    'CELL_LINE']]],\n",
       " ['Thus , Egr - 2 , in addition to Egr - 3 , regulates FasL expression in activated normal T cells , and Egr - 2 is likely to play a direct role in aberrant fasL up - regulation in lpr/lpr t cell .',\n",
       "  [['Thus , Egr - 2 , in addition to Egr - 3 , regulates FasL expression in activated normal T cells , and Egr - 2 is likely to play a direct role in aberrant fasL up - regulation in <e1> lpr/lpr t cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Thus , Egr - 2 , in addition to Egr - 3 , regulates FasL expression in activated normal T cells , and Egr - 2 is likely to play a direct role in aberrant fasL up - regulation in lpr/lpr and gld/gld cd4 - cd8 - t cell .',\n",
       "  [['Thus , Egr - 2 , in addition to Egr - 3 , regulates FasL expression in activated normal T cells , and Egr - 2 is likely to play a direct role in aberrant fasL up - regulation in lpr/lpr and <e1> gld/gld cd4 - cd8 - t cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Recent molecular evidence points to transient and/or stage - specific expression of delta - opioid receptorby thymic and peripheral T lymphocytes .',\n",
       "  [['Recent molecular evidence points to transient and/or stage - specific expression of <e1> delta - opioid receptor </e1> by thymic and peripheral T lymphocytes .',\n",
       "    'PROTEIN']]],\n",
       " ['Recent molecular evidence points to transient and/or stage - specific expression of delta - and kappa - opioid receptors by thymic t lymphocyte .',\n",
       "  [['Recent molecular evidence points to transient and/or stage - specific expression of delta - and kappa - opioid receptors by <e1> thymic t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Recent molecular evidence points to transient and/or stage - specific expression of delta - opioid receptorby thymic and peripheral T lymphocytes .',\n",
       "  [['Recent molecular evidence points to transient and/or stage - specific expression of <e1> delta - opioid receptor </e1> by thymic and peripheral T lymphocytes .',\n",
       "    'PROTEIN']]],\n",
       " ['Recent molecular evidence points to transient and/or stage - specific expression of delta - and kappa - opioid receptors by thymic t lymphocyte .',\n",
       "  [['Recent molecular evidence points to transient and/or stage - specific expression of delta - and kappa - opioid receptors by <e1> thymic t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Reporter gene constructs were used to map this effect of deltorphin to the ap - 1 - binding siteof the IL - 2 promoter .',\n",
       "  [['Reporter gene constructs were used to map this effect of deltorphin to the <e1> ap - 1 - binding site </e1> of the IL - 2 promoter .',\n",
       "    'DNA']]],\n",
       " ['Electrophoretic mobility shift assay revealed that the potential NF - kappaB binding sequence which is located around 120 bp upstream of the transcription initiation site in murine p40 geneformed an NF - kappaB complex with nuclear extract from Daudi cells stimulated by CD40 ligation .',\n",
       "  [['Electrophoretic mobility shift assay revealed that the potential NF - kappaB binding sequence which is located around 120 bp upstream of the transcription initiation site in <e1> murine p40 gene </e1> formed an NF - kappaB complex with nuclear extract from Daudi cells stimulated by CD40 ligation .',\n",
       "    'DNA']]],\n",
       " ['These results suggest that the NF - kappaB binding site located around 120 bp upstream of the transcription initiation site in murine p40 promoter regioncould be important for the p40 induction by CD40 ligation via activation of NF - kappaB .',\n",
       "  [['These results suggest that the NF - kappaB binding site located around 120 bp upstream of the transcription initiation site in <e1> murine p40 promoter region </e1> could be important for the p40 induction by CD40 ligation via activation of NF - kappaB .',\n",
       "    'DNA']]],\n",
       " ['HS I - III , located 5 from the transcriptional initiation site , were found in hematopoietic cell, whereas HS IV - VII , located 3 from the transcriptional start site , were exclusively observed in T - cells .',\n",
       "  [['HS I - III , located 5 from the transcriptional initiation site , were found in <e1> hematopoietic cell </e1> , whereas HS IV - VII , located 3 from the transcriptional start site , were exclusively observed in T - cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Two CAGGTG E - boxes , located at the 5 - endof the silencer , were necessary for this silencer activity .',\n",
       "  [['Two CAGGTG E - boxes , located at the <e1> 5 - end </e1> of the silencer , were necessary for this silencer activity .',\n",
       "    'DNA']]],\n",
       " ['5 - LOX and 5 - LOX activating protein ( FLAP ) are coexpressed in lymphoid cells but not in monocytic cell .',\n",
       "  [['5 - LOX and 5 - LOX activating protein ( FLAP ) are coexpressed in lymphoid cells but not in <e1> monocytic cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The molecular basis for restricted cytokine expression by t helper 1 th1 cellis unclear .',\n",
       "  [['The molecular basis for restricted cytokine expression by <e1> t helper 1 th1 cell </e1> is unclear .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The molecular basis for restricted cytokine expression by t helper 2 th2 cellis unclear .',\n",
       "  [['The molecular basis for restricted cytokine expression by <e1> t helper 2 th2 cell </e1> is unclear .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Here we demonstrate that fusion of HeLa cells with either diploid b cell- encoded genes that are expressed in a B cell - specific manner .',\n",
       "  [['Here we demonstrate that fusion of HeLa cells with either <e1> diploid b cell </e1> - encoded genes that are expressed in a B cell - specific manner .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Here we demonstrate that fusion of HeLa cells with either diploid or tetraploid b cell- encoded genes that are expressed in a B cell - specific manner .',\n",
       "  [['Here we demonstrate that fusion of HeLa cells with either diploid or <e1> tetraploid b cell </e1> - encoded genes that are expressed in a B cell - specific manner .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In contrast , both in th1 cloneNFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between Th1 and Th2 clones .',\n",
       "  [['In contrast , both in <e1> th1 clone </e1> NFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between Th1 and Th2 clones .',\n",
       "    'CELL_LINE']]],\n",
       " ['In contrast , both in Th1 and Th2 clones NFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between th1 clone .',\n",
       "  [['In contrast , both in Th1 and Th2 clones NFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between <e1> th1 clone </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In contrast , both in th1 cloneNFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between Th1 and Th2 clones .',\n",
       "  [['In contrast , both in <e1> th1 clone </e1> NFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between Th1 and Th2 clones .',\n",
       "    'CELL_LINE']]],\n",
       " ['In contrast , both in Th1 and Th2 clones NFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between th1 clone .',\n",
       "  [['In contrast , both in Th1 and Th2 clones NFAT - containing complexes were strongly inducible at the P1 site , consisting of NFAT2 and a P0 - compatible NFAT activity , without apparent differences between <e1> th1 clone </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['To characterize the human elk - 1 promoter , we mapped the transcriptional start site and isolated elk - 1 - specific genomic phage clones that contained extensive upstream sequence .',\n",
       "  [['To characterize the human elk - 1 promoter , we mapped the transcriptional start site and isolated elk - 1 - specific genomic phage clones that contained extensive <e1> upstream sequence </e1> .',\n",
       "    'DNA']]],\n",
       " ['Anti - rheumatic compound aurothioglucose inhibits tumor necrosis factor - alpha - induced HIV - 1 replication in latently infected om10.1 cell .',\n",
       "  [['Anti - rheumatic compound aurothioglucose inhibits tumor necrosis factor - alpha - induced HIV - 1 replication in latently infected <e1> om10.1 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['In this report we have examined the efficacy of the gold compound AuTG as an inhibitor of HIV replication in latently infected om10.1 cell .',\n",
       "  [['In this report we have examined the efficacy of the gold compound AuTG as an inhibitor of HIV replication in latently infected <e1> om10.1 cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Methylation of hpaii sitenear the polymorphic trinucleotide repeat of the human androgen receptor gene ( HUMARA ) has been shown to correlate with X - inactivation .',\n",
       "  [['Methylation of <e1> hpaii site </e1> near the polymorphic trinucleotide repeat of the human androgen receptor gene ( HUMARA ) has been shown to correlate with X - inactivation .',\n",
       "    'DNA']]],\n",
       " ['Its effects are exerted by interaction with the vitamin D receptor , which is present on human monocytes and activated t lymphocyte .',\n",
       "  [['Its effects are exerted by interaction with the vitamin D receptor , which is present on human monocytes and <e1> activated t lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Its effects are exerted by interaction with the vitamin D receptor , which is present on human monocytes and activated b lymphocyte .',\n",
       "  [['Its effects are exerted by interaction with the vitamin D receptor , which is present on human monocytes and <e1> activated b lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and bp - 14 kd) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and <e1> bp - 14 kd </e1> ) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "    'PROTEIN']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and bp - 18 kd) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and <e1> bp - 18 kd </e1> ) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "    'PROTEIN']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and bp - 19 kd) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and <e1> bp - 19 kd </e1> ) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "    'PROTEIN']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and BP - 14 , 18 , 19 kDs ) isolated from splenic cellof immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and BP - 14 , 18 , 19 kDs ) isolated from <e1> splenic cell </e1> of immunized rats were studied .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and bp - 14 kd) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and <e1> bp - 14 kd </e1> ) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "    'PROTEIN']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and bp - 18 kd) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and <e1> bp - 18 kd </e1> ) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "    'PROTEIN']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and bp - 19 kd) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and <e1> bp - 19 kd </e1> ) isolated from splenic and brain cells of immunized rats were studied .',\n",
       "    'PROTEIN']]],\n",
       " ['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and BP - 14 , 18 , 19 kDs ) isolated from splenic cellof immunized rats were studied .',\n",
       "  [['The physico - chemical and functional parameters of the low molecular nuclear proteins ( SP and BP - 14 , 18 , 19 kDs ) isolated from <e1> splenic cell </e1> of immunized rats were studied .',\n",
       "    'CELL_TYPE']]],\n",
       " ['When we analyzed the nature of STAT proteins capable of binding to IL - 2Ralpha , pim - 1 elementafter cytokine stimulation , we observed IFN - alpha - induced binding of STAT1 , STAT3 , and STAT4 , but not STAT5 to all of these elements .',\n",
       "  [['When we analyzed the nature of STAT proteins capable of binding to IL - 2Ralpha , <e1> pim - 1 element </e1> after cytokine stimulation , we observed IFN - alpha - induced binding of STAT1 , STAT3 , and STAT4 , but not STAT5 to all of these elements .',\n",
       "    'DNA']]],\n",
       " ['Binding sites for testosterone on the surface of both cd4+ subsetof T cells are directly revealed with the impeded ligand testosterone - BSA - FITC by confocal laser scanning microscopy ( CLSM ) and flow cytometry , respectively .',\n",
       "  [['Binding sites for testosterone on the surface of both <e1> cd4+ subset </e1> of T cells are directly revealed with the impeded ligand testosterone - BSA - FITC by confocal laser scanning microscopy ( CLSM ) and flow cytometry , respectively .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Using the wild - type human retinoic acid receptor alphaas a model , kDa values determined by classical competition binding assays using tritiated ligands are in agreement with those measured by the proteolytic assay .',\n",
       "  [['Using the <e1> wild - type human retinoic acid receptor alpha </e1> as a model , kDa values determined by classical competition binding assays using tritiated ligands are in agreement with those measured by the proteolytic assay .',\n",
       "    'PROTEIN']]],\n",
       " ['Interleukin - 10 inhibits expression of both interferon alpha - induced geneby suppressing tyrosine phosphorylation of STAT1 .',\n",
       "  [['Interleukin - 10 inhibits expression of both <e1> interferon alpha - induced gene </e1> by suppressing tyrosine phosphorylation of STAT1 .',\n",
       "    'DNA']]],\n",
       " ['Interleukin - 10 inhibits expression of both interferon gamma - induced geneby suppressing tyrosine phosphorylation of STAT1 .',\n",
       "  [['Interleukin - 10 inhibits expression of both <e1> interferon gamma - induced gene </e1> by suppressing tyrosine phosphorylation of STAT1 .',\n",
       "    'DNA']]],\n",
       " ['The reduction in gene expression resulted from the ability of IL - 10 to suppress IFN - induced assembly of signal transducer and activator of transcription ( STAT ) factors to specific promoter motifs on ifnalpha - inducible gene .',\n",
       "  [['The reduction in gene expression resulted from the ability of IL - 10 to suppress IFN - induced assembly of signal transducer and activator of transcription ( STAT ) factors to specific promoter motifs on <e1> ifnalpha - inducible gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['This was accomplished by preventing the IFN - induced tyrosine phosphorylation of STAT1 , a component of both ifnalpha - induced dna binding complex .',\n",
       "  [['This was accomplished by preventing the IFN - induced tyrosine phosphorylation of STAT1 , a component of both <e1> ifnalpha - induced dna binding complex </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['These include Fc epsilonRII ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and type i il - 1 receptor( IL - 1R ) .',\n",
       "  [['These include Fc epsilonRII ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and <e1> type i il - 1 receptor </e1> ( IL - 1R ) .',\n",
       "    'PROTEIN']]],\n",
       " ['These include Fc epsilonRII ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and type ii il - 1 receptor( IL - 1R ) .',\n",
       "  [['These include Fc epsilonRII ( CD23 ) , 15 - lipoxygenase , IL - 1 receptor antagonist ( IL - 1ra ) , and <e1> type ii il - 1 receptor </e1> ( IL - 1R ) .',\n",
       "    'PROTEIN']]],\n",
       " ['In both cell types , the cytokine that inhibits the induction of IkappaBalpha by DEX , also rescues these cells from DEX - induced apoptosis , although the rescue cytokine is different in th1 cell .',\n",
       "  [['In both cell types , the cytokine that inhibits the induction of IkappaBalpha by DEX , also rescues these cells from DEX - induced apoptosis , although the rescue cytokine is different in <e1> th1 cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['We identified a cell - type - specific differential response : creb genewere strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "  [['We identified a cell - type - specific differential response : <e1> creb gene </e1> were strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "    'DNA']]],\n",
       " ['We identified a cell - type - specific differential response : CREB , ctf genewere strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "  [['We identified a cell - type - specific differential response : CREB , <e1> ctf gene </e1> were strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "    'DNA']]],\n",
       " ['We identified a cell - type - specific differential response : CREB , CTF , otf - 1 genewere strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "  [['We identified a cell - type - specific differential response : CREB , CTF , <e1> otf - 1 gene </e1> were strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "    'DNA']]],\n",
       " ['We identified a cell - type - specific differential response : CREB , CTF , OTF - 1 , oft - 2 genewere strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "  [['We identified a cell - type - specific differential response : CREB , CTF , OTF - 1 , <e1> oft - 2 gene </e1> were strongly induced 1 to 4 hours after influenza A virus infection in the monocytic cell line Mono Mac 6 , while in freshly prepared human monocytes no significant changes were detected .',\n",
       "    'DNA']]],\n",
       " ['Late gene expression from the Epstein - Barr virus bclf1 promoterdoes not require DNA replication in cis .',\n",
       "  [['Late gene expression from the Epstein - Barr virus <e1> bclf1 promoter </e1> does not require DNA replication in cis .',\n",
       "    'DNA']]],\n",
       " ['LPS core - specific mabinhibited mCD14 - mediated LPS uptake by both cell types in the presence of serum .',\n",
       "  [['LPS <e1> core - specific mab </e1> inhibited mCD14 - mediated LPS uptake by both cell types in the presence of serum .',\n",
       "    'PROTEIN']]],\n",
       " ['Cross - competition assays among the P - like elements revealed element - specific complex .',\n",
       "  [['Cross - competition assays among the P - like elements revealed <e1> element - specific complex </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Constitutive expression c - fos mrnais in nucleated fetal blood cells and up - regulation of c - fos and c - jun with anti - CD3 stimulation .',\n",
       "  [['Constitutive expression <e1> c - fos mrna </e1> is in nucleated fetal blood cells and up - regulation of c - fos and c - jun with anti - CD3 stimulation .',\n",
       "    'RNA']]],\n",
       " ['Constitutive expression c - jun mrnais in nucleated fetal blood cells and up - regulation of c - fos and c - jun with anti - CD3 stimulation .',\n",
       "  [['Constitutive expression <e1> c - jun mrna </e1> is in nucleated fetal blood cells and up - regulation of c - fos and c - jun with anti - CD3 stimulation .',\n",
       "    'RNA']]],\n",
       " [' fetal lymphocyteare relatively resistant to activation and cytokine production when stimulated either via their T - cell antigen receptors or lectins .',\n",
       "  [[' <e1> fetal lymphocyte </e1> are relatively resistant to activation and cytokine production when stimulated either via their T - cell antigen receptors or lectins .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Furthermore , PDTC differentially targeted both nfatp family member, inhibiting the transactivation functions of NFATp and mRNA induction of NFATc .',\n",
       "  [['Furthermore , PDTC differentially targeted both <e1> nfatp family member </e1> , inhibiting the transactivation functions of NFATp and mRNA induction of NFATc .',\n",
       "    'PROTEIN']]],\n",
       " ['The A - myb transcription factor in neoplastic b cell .',\n",
       "  [['The A - myb transcription factor in <e1> neoplastic b cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In previous experiments , the - 70 to - 65 regionof the CD11b promoter were shown to be essential for Sp1 - mediated activation of these genes .',\n",
       "  [['In previous experiments , the <e1> - 70 to - 65 region </e1> of the CD11b promoter were shown to be essential for Sp1 - mediated activation of these genes .',\n",
       "    'DNA']]],\n",
       " ['In previous experiments , the - 121 to - 103 regionof the CD11b promoter were shown to be essential for Sp1 - mediated activation of these genes .',\n",
       "  [['In previous experiments , the <e1> - 121 to - 103 region </e1> of the CD11b promoter were shown to be essential for Sp1 - mediated activation of these genes .',\n",
       "    'DNA']]],\n",
       " ['Deletion of both the - 70 to - 65 regionof the CD11c promoter resulted in the loss of activation by Sp3 .',\n",
       "  [['Deletion of both the <e1> - 70 to - 65 region </e1> of the CD11c promoter resulted in the loss of activation by Sp3 .',\n",
       "    'DNA']]],\n",
       " ['Deletion of both the - 121 to - 103 regionof the CD11c promoter resulted in the loss of activation by Sp3 .',\n",
       "  [['Deletion of both the <e1> - 121 to - 103 region </e1> of the CD11c promoter resulted in the loss of activation by Sp3 .',\n",
       "    'DNA']]],\n",
       " ['Antisense experiments that knocked out endogenous Sp3 expression in the myelomocytic cell line , HL60 , revealed that Sp3 participates in activation of the cd11c promoterin vivo .',\n",
       "  [['Antisense experiments that knocked out endogenous Sp3 expression in the myelomocytic cell line , HL60 , revealed that Sp3 participates in activation of the <e1> cd11c promoter </e1> in vivo .',\n",
       "    'DNA']]],\n",
       " ['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to gata sitein the H5 promoter and enhancer .',\n",
       "  [['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to <e1> gata site </e1> in the H5 promoter and enhancer .',\n",
       "    'DNA']]],\n",
       " ['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to GATA and CACCC sites in the h5 enhancer .',\n",
       "  [['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to GATA and CACCC sites in the <e1> h5 enhancer </e1> .',\n",
       "    'DNA']]],\n",
       " ['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to gata sitein the H5 promoter and enhancer .',\n",
       "  [['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to <e1> gata site </e1> in the H5 promoter and enhancer .',\n",
       "    'DNA']]],\n",
       " ['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to GATA and CACCC sites in the h5 enhancer .',\n",
       "  [['Using electrophoretic mobility shift assays , we found that in the presence of rabbit reticulocyte lysate , human ER reduced the binding activities of chicken immature erythrocyte nuclear extracted proteins to GATA and CACCC sites in the <e1> h5 enhancer </e1> .',\n",
       "    'DNA']]],\n",
       " ['Since the DSE and TRE sites can not bind CREB/ATF homodimers , we propose a mechanism in which the HIV - 1 DSE bind heterodimers composed of both ap - 1 protein .',\n",
       "  [['Since the DSE and TRE sites can not bind CREB/ATF homodimers , we propose a mechanism in which the HIV - 1 DSE bind heterodimers composed of both <e1> ap - 1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " [' c - rel subunitbind to an upstream NF - kappaB site in human granulocyte macrophage - colony stimulating factor promoter involved in phorbol ester response in 5637 cells .',\n",
       "  [[' <e1> c - rel subunit </e1> bind to an upstream NF - kappaB site in human granulocyte macrophage - colony stimulating factor promoter involved in phorbol ester response in 5637 cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Of these , the xpb geneencode proteins that are subunits of a general transcription factor , TFIIH , involved in both nucleotide excision repair ( NER ) and initiation of mRNA transcription by RNA polymerase II .',\n",
       "  [['Of these , the <e1> xpb gene </e1> encode proteins that are subunits of a general transcription factor , TFIIH , involved in both nucleotide excision repair ( NER ) and initiation of mRNA transcription by RNA polymerase II .',\n",
       "    'DNA']]],\n",
       " ['In humans , mutation of the xpb geneimpairs NER , resulting in hyper - sensitivity to sunlight and greatly increased skin tumor formation .',\n",
       "  [['In humans , mutation of the <e1> xpb gene </e1> impairs NER , resulting in hyper - sensitivity to sunlight and greatly increased skin tumor formation .',\n",
       "    'DNA']]],\n",
       " ['Although the growth rate was normal , the xp - b cellcontained reduced amounts of TFIIH .',\n",
       "  [['Although the growth rate was normal , the <e1> xp - b cell </e1> contained reduced amounts of TFIIH .',\n",
       "    'CELL_LINE']]],\n",
       " ['Although the growth rate was normal , the xp - d cellcontained reduced amounts of TFIIH .',\n",
       "  [['Although the growth rate was normal , the <e1> xp - d cell </e1> contained reduced amounts of TFIIH .',\n",
       "    'CELL_LINE']]],\n",
       " ['Extracts prepared from xp - b lymphoblastoid cellexhibited similar transcription activity from the adenovirus major late promoter when compared to that in extracts from normal cells .',\n",
       "  [['Extracts prepared from <e1> xp - b lymphoblastoid cell </e1> exhibited similar transcription activity from the adenovirus major late promoter when compared to that in extracts from normal cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['Extracts prepared from xp - d lymphoblastoid cellexhibited similar transcription activity from the adenovirus major late promoter when compared to that in extracts from normal cells .',\n",
       "  [['Extracts prepared from <e1> xp - d lymphoblastoid cell </e1> exhibited similar transcription activity from the adenovirus major late promoter when compared to that in extracts from normal cells .',\n",
       "    'CELL_LINE']]],\n",
       " ['Thus , we conclude that the xp - b lymphoblastoid celldo not have impaired RNA transcription activity .',\n",
       "  [['Thus , we conclude that the <e1> xp - b lymphoblastoid cell </e1> do not have impaired RNA transcription activity .',\n",
       "    'CELL_LINE']]],\n",
       " ['Thus , we conclude that the xp - d lymphoblastoid celldo not have impaired RNA transcription activity .',\n",
       "  [['Thus , we conclude that the <e1> xp - d lymphoblastoid cell </e1> do not have impaired RNA transcription activity .',\n",
       "    'CELL_LINE']]],\n",
       " ['RESULTS : Here , we have isolated the murine cdnawhich encode a novel form of MZF protein ( MZF - 2 ) .',\n",
       "  [['RESULTS : Here , we have isolated the <e1> murine cdna </e1> which encode a novel form of MZF protein ( MZF - 2 ) .',\n",
       "    'DNA']]],\n",
       " [' murine mzf - 2 proteinconsisted of 814 and 775 amino acids , respectively , and have identity of 75.3 % between them .',\n",
       "  [[' <e1> murine mzf - 2 protein </e1> consisted of 814 and 775 amino acids , respectively , and have identity of 75.3 % between them .',\n",
       "    'PROTEIN']]],\n",
       " ['CONCLUSIONS : mzf - 1 mrnaseem to be produced by the alternative use of two different transcription initiation sites .',\n",
       "  [['CONCLUSIONS : <e1> mzf - 1 mrna </e1> seem to be produced by the alternative use of two different transcription initiation sites .',\n",
       "    'RNA']]],\n",
       " ['CONCLUSIONS : mzf - 2 mrnaseem to be produced by the alternative use of two different transcription initiation sites .',\n",
       "  [['CONCLUSIONS : <e1> mzf - 2 mrna </e1> seem to be produced by the alternative use of two different transcription initiation sites .',\n",
       "    'RNA']]],\n",
       " ['In mammalian cells , I kappa B alpha and I kappa B beta proteins have been purified and shown to be the inhibitors of NF - kappa B through their association with the p65 subunit .',\n",
       "  [['In mammalian cells , I kappa B alpha and I kappa B beta proteins have been purified and shown to be the inhibitors of NF - kappa B through their association with the <e1> p65 subunit </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Aberrant splicing of the tsg101 geneoccurs frequently in multiple malignancies and in normal tissues and mimics alterations previously described in tumours .',\n",
       "  [['Aberrant splicing of the <e1> tsg101 gene </e1> occurs frequently in multiple malignancies and in normal tissues and mimics alterations previously described in tumours .',\n",
       "    'DNA']]],\n",
       " ['Our findings demonstrate that truncated tsg101 transcriptare commonly detected in both normal and malignant tissues and that a significant fraction of these are likely to be the result of aberrant splicing .',\n",
       "  [['Our findings demonstrate that <e1> truncated tsg101 transcript </e1> are commonly detected in both normal and malignant tissues and that a significant fraction of these are likely to be the result of aberrant splicing .',\n",
       "    'RNA']]],\n",
       " ['Our findings demonstrate that truncated fhit transcriptare commonly detected in both normal and malignant tissues and that a significant fraction of these are likely to be the result of aberrant splicing .',\n",
       "  [['Our findings demonstrate that <e1> truncated fhit transcript </e1> are commonly detected in both normal and malignant tissues and that a significant fraction of these are likely to be the result of aberrant splicing .',\n",
       "    'RNA']]],\n",
       " ['Inhibition of proliferation and apoptosis of human t lymphocyteby curcumin , a curry pigment .',\n",
       "  [['Inhibition of proliferation and apoptosis of <e1> human t lymphocyte </e1> by curcumin , a curry pigment .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Since AP - 1 is part of the NFAT complex , we conclude that the IL - 7 - signaling pathway is involved in the activation of the fos proteinof which AP - 1 consists .',\n",
       "  [['Since AP - 1 is part of the NFAT complex , we conclude that the IL - 7 - signaling pathway is involved in the activation of the <e1> fos protein </e1> of which AP - 1 consists .',\n",
       "    'PROTEIN']]],\n",
       " ['The visualization of heavy chain genein B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the gamma and lambda genes consistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "  [['The visualization of <e1> heavy chain gene </e1> in B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the gamma and lambda genes consistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "    'DNA']]],\n",
       " ['The visualization of heavy and light chain genes in B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the gamma geneconsistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "  [['The visualization of heavy and light chain genes in B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the <e1> gamma gene </e1> consistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "    'DNA']]],\n",
       " ['The visualization of heavy chain genein B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the gamma and lambda genes consistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "  [['The visualization of <e1> heavy chain gene </e1> in B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the gamma and lambda genes consistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "    'DNA']]],\n",
       " ['The visualization of heavy and light chain genes in B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the gamma geneconsistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "  [['The visualization of heavy and light chain genes in B - lymphoid cells showed that the three Ig genes are differentially and nonrandomly distributed in different nuclear subvolumes : the kappa genes were found to be preferentially confined to an outer nuclear volume , whereas the <e1> gamma gene </e1> consistently occupied more central positions within the nucleus , the lambda genes being more interior when compared with the gamma genes .',\n",
       "    'DNA']]],\n",
       " ['The activity of A - MYB in b cellwas observed when either an artificial construct or the c - MYC promoter was used as a reporter .',\n",
       "  [['The activity of A - MYB in <e1> b cell </e1> was observed when either an artificial construct or the c - MYC promoter was used as a reporter .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Finally , a 110 - kDa protein has been identified in the nuclei of all the b cell linethat specifically binds to this A - MYB N - terminal domain .',\n",
       "  [['Finally , a 110 - kDa protein has been identified in the nuclei of all the <e1> b cell line </e1> that specifically binds to this A - MYB N - terminal domain .',\n",
       "    'CELL_LINE']]],\n",
       " ['Finally , a 110 - kDa protein has been identified in the nuclei of all the B , but not t cell linethat specifically binds to this A - MYB N - terminal domain .',\n",
       "  [['Finally , a 110 - kDa protein has been identified in the nuclei of all the B , but not <e1> t cell line </e1> that specifically binds to this A - MYB N - terminal domain .',\n",
       "    'CELL_LINE']]],\n",
       " [' polyclonal ctlthat were generated by stimulating peripheral blood mononuclear cells with an HLA B8+ homozygous LCL lysed T cell blasts pulsed with the peptide , RAKFKQLLQ ; lysis of certain HLA B8+ LCL targets was associated with the abundance of BZLF1 transcripts .',\n",
       "  [[' <e1> polyclonal ctl </e1> that were generated by stimulating peripheral blood mononuclear cells with an HLA B8+ homozygous LCL lysed T cell blasts pulsed with the peptide , RAKFKQLLQ ; lysis of certain HLA B8+ LCL targets was associated with the abundance of BZLF1 transcripts .',\n",
       "    'CELL_LINE']]],\n",
       " ['To define the CD40 signaling components upstream of NF - kappa B activation and the functional consequences downstream of NF - kappa B activation , we examined mouse B cell transfectants expressing wild - type cd40 .',\n",
       "  [['To define the CD40 signaling components upstream of NF - kappa B activation and the functional consequences downstream of NF - kappa B activation , we examined mouse B cell transfectants expressing <e1> wild - type cd40 </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Messenger RNA ( mRNA ) levels for the granulocyte - colony stimulating factor ( G - CSF ) receptor , granulocyte - macrophage ( GM ) - CSF receptor alpha subunit and tumor necrosis factor tnf receptor i p55) and II ( p75 ) were also detected in this subset in addition to c - kit and flt - 3 , receptors known to be expressed on progenitor cells .',\n",
       "  [['Messenger RNA ( mRNA ) levels for the granulocyte - colony stimulating factor ( G - CSF ) receptor , granulocyte - macrophage ( GM ) - CSF receptor alpha subunit and <e1> tumor necrosis factor tnf receptor i p55 </e1> ) and II ( p75 ) were also detected in this subset in addition to c - kit and flt - 3 , receptors known to be expressed on progenitor cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Messenger RNA ( mRNA ) levels for the granulocyte - colony stimulating factor ( G - CSF ) receptor , granulocyte - macrophage ( GM ) - CSF receptor alpha subunit and tumor necrosis factor tnf receptor ii p75) were also detected in this subset in addition to c - kit and flt - 3 , receptors known to be expressed on progenitor cells .',\n",
       "  [['Messenger RNA ( mRNA ) levels for the granulocyte - colony stimulating factor ( G - CSF ) receptor , granulocyte - macrophage ( GM ) - CSF receptor alpha subunit and <e1> tumor necrosis factor tnf receptor ii p75 </e1> ) were also detected in this subset in addition to c - kit and flt - 3 , receptors known to be expressed on progenitor cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Both cytokines induced assembly of STAT5A and STAT5B containing complexes capable of binding to the interferon - gamma activation sequence ( GAS ) , and these complexes rapidly translocated ( within 1 min ) into the nucleus of il - 2 - treated cell .',\n",
       "  [['Both cytokines induced assembly of STAT5A and STAT5B containing complexes capable of binding to the interferon - gamma activation sequence ( GAS ) , and these complexes rapidly translocated ( within 1 min ) into the nucleus of <e1> il - 2 - treated cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Both cytokines induced assembly of STAT5A and STAT5B containing complexes capable of binding to the interferon - gamma activation sequence ( GAS ) , and these complexes rapidly translocated ( within 1 min ) into the nucleus of il - 7 - treated cell .',\n",
       "  [['Both cytokines induced assembly of STAT5A and STAT5B containing complexes capable of binding to the interferon - gamma activation sequence ( GAS ) , and these complexes rapidly translocated ( within 1 min ) into the nucleus of <e1> il - 7 - treated cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Thus , our data establish that ATF/CREB factors functionally modulate histone H4 gene transcription at distal promoter element .',\n",
       "  [['Thus , our data establish that ATF/CREB factors functionally modulate histone H4 gene transcription at <e1> distal promoter element </e1> .',\n",
       "    'DNA']]],\n",
       " ['Finally , we speculate that molecular mimicry between one or more gliadin peptides and some , as yet unidentified , bacterial superantigenplays a role in disease pathogenesis .',\n",
       "  [['Finally , we speculate that molecular mimicry between one or more gliadin peptides and some , as yet unidentified , <e1> bacterial superantigen </e1> plays a role in disease pathogenesis .',\n",
       "    'PROTEIN']]],\n",
       " ['SR 31747A , defined as a sigma ligand , is a novel immunosuppressive agent that blocks proliferation of human lymphocyte .',\n",
       "  [['SR 31747A , defined as a sigma ligand , is a novel immunosuppressive agent that blocks proliferation of <e1> human lymphocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['In this study , we used a RT - PCR based approach to compare the stability of the big t allelein normal heterozygous lymphocytes and the heterozygous cell lines NB4 ( myeloid leukemia ) and PC - 3 and DU 145 ( prostate cancers ) .',\n",
       "  [['In this study , we used a RT - PCR based approach to compare the stability of the <e1> big t allele </e1> in normal heterozygous lymphocytes and the heterozygous cell lines NB4 ( myeloid leukemia ) and PC - 3 and DU 145 ( prostate cancers ) .',\n",
       "    'DNA']]],\n",
       " ['Differanisole A and 9 - cis retinoic acid ( 9cisRA ) synergistically inhibited the growth and induced functional and morphologic differentiation of hl - 60 cell, whereas the combined treatment with differanisole A and all - trans retinoic acid or 1alpha,25 - dihydroxyvitamin D3 was less effective .',\n",
       "  [['Differanisole A and 9 - cis retinoic acid ( 9cisRA ) synergistically inhibited the growth and induced functional and morphologic differentiation of <e1> hl - 60 cell </e1> , whereas the combined treatment with differanisole A and all - trans retinoic acid or 1alpha,25 - dihydroxyvitamin D3 was less effective .',\n",
       "    'CELL_LINE']]],\n",
       " ['In contrast , long - term treatment with oxLDL prevented the lipopolysaccharide - induced depletion of I kappa B - alpha , accompanied by an inhibition of both NF - kappa B activation and the expression of tumor necrosis factor - alpha gene .',\n",
       "  [['In contrast , long - term treatment with oxLDL prevented the lipopolysaccharide - induced depletion of I kappa B - alpha , accompanied by an inhibition of both NF - kappa B activation and the expression of <e1> tumor necrosis factor - alpha gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Among the regulatory promoter elements potentially regulating eotaxin gene expression and/or mediating the effects of anti - inflammatory drugs we identified consensus sequences known to interact with nuclear factors like NF - IL6 , AP - 1 , a NF - kappa - B like consensus sequence and gamma - interferon - response element .',\n",
       "  [['Among the regulatory promoter elements potentially regulating eotaxin gene expression and/or mediating the effects of anti - inflammatory drugs we identified consensus sequences known to interact with nuclear factors like NF - IL6 , AP - 1 , a NF - kappa - B like consensus sequence and <e1> gamma - interferon - response element </e1> .',\n",
       "    'DNA']]],\n",
       " ['Differential interaction of nuclear factors with the leukocyte - specific pp52 promoter in b cell .',\n",
       "  [['Differential interaction of nuclear factors with the leukocyte - specific pp52 promoter in <e1> b cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The leukocyte - specific , cytoskeleton - binding pp52 ( LSP - 1 , WP - 34 ) protein is widely expressed in multiple leukocyte lineages , including b lymphocyte, granulocytes , and macrophages .',\n",
       "  [['The leukocyte - specific , cytoskeleton - binding pp52 ( LSP - 1 , WP - 34 ) protein is widely expressed in multiple leukocyte lineages , including <e1> b lymphocyte </e1> , granulocytes , and macrophages .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Here we describe the functional characterization of this promoter and identification of the factors in b cellthat regulate its activity .',\n",
       "  [['Here we describe the functional characterization of this promoter and identification of the factors in <e1> b cell </e1> that regulate its activity .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The pp52 promoter contains an initiator specifying the unique 5 terminus of pp52 mRNA , tandem pairs of ets motif .',\n",
       "  [['The pp52 promoter contains an initiator specifying the unique 5 terminus of pp52 mRNA , tandem pairs of <e1> ets motif </e1> .',\n",
       "    'DNA']]],\n",
       " ['The pp52 promoter contains an initiator specifying the unique 5 terminus of pp52 mRNA , tandem pairs of Ets and sp1 motif .',\n",
       "  [['The pp52 promoter contains an initiator specifying the unique 5 terminus of pp52 mRNA , tandem pairs of Ets and <e1> sp1 motif </e1> .',\n",
       "    'DNA']]],\n",
       " ['These findings reveal the features controlling the pp52 promoter in b celland provide the foundation for determining the regulation of this promoter in other leukocyte lineages .',\n",
       "  [['These findings reveal the features controlling the pp52 promoter in <e1> b cell </e1> and provide the foundation for determining the regulation of this promoter in other leukocyte lineages .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The immediate - early gene product Egr - 1 regulates the human interleukin - 2 receptor beta - chain promoter through noncanonical egr binding site .',\n",
       "  [['The immediate - early gene product Egr - 1 regulates the human interleukin - 2 receptor beta - chain promoter through <e1> noncanonical egr binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['The immediate - early gene product Egr - 1 regulates the human interleukin - 2 receptor beta - chain promoter through noncanonical sp1 binding site .',\n",
       "  [['The immediate - early gene product Egr - 1 regulates the human interleukin - 2 receptor beta - chain promoter through <e1> noncanonical sp1 binding site </e1> .',\n",
       "    'DNA']]],\n",
       " ['The tcr alpha beta chainbind the peptide ligand , whereas the associated CD3 delta epsilon gamma and TCR zeta subunits couple the TCR to intracellular signal transduction components .',\n",
       "  [['The <e1> tcr alpha beta chain </e1> bind the peptide ligand , whereas the associated CD3 delta epsilon gamma and TCR zeta subunits couple the TCR to intracellular signal transduction components .',\n",
       "    'PROTEIN']]],\n",
       " ['The tcr gamma delta chainbind the peptide ligand , whereas the associated CD3 delta epsilon gamma and TCR zeta subunits couple the TCR to intracellular signal transduction components .',\n",
       "  [['The <e1> tcr gamma delta chain </e1> bind the peptide ligand , whereas the associated CD3 delta epsilon gamma and TCR zeta subunits couple the TCR to intracellular signal transduction components .',\n",
       "    'PROTEIN']]],\n",
       " ['In addition , in vivo footprint analysis showed the putative x boxto be occupied by transcription factors in wild - type B cells , but not in RFX - deficient B cells .',\n",
       "  [['In addition , in vivo footprint analysis showed the putative <e1> x box </e1> to be occupied by transcription factors in wild - type B cells , but not in RFX - deficient B cells .',\n",
       "    'DNA']]],\n",
       " ['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding hepatocyte nuclear factor - 3beta, - 3gamma , - 4gamma , and - 6 .',\n",
       "  [['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding <e1> hepatocyte nuclear factor - 3beta </e1> , - 3gamma , - 4gamma , and - 6 .',\n",
       "    'PROTEIN']]],\n",
       " ['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding hepatocyte nuclear factor - 3gamma, - 4gamma , and - 6 .',\n",
       "  [['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding <e1> hepatocyte nuclear factor - 3gamma </e1> , - 4gamma , and - 6 .',\n",
       "    'PROTEIN']]],\n",
       " ['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding hepatocyte nuclear factor - 4gamma, and - 6 .',\n",
       "  [['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding <e1> hepatocyte nuclear factor - 4gamma </e1> , and - 6 .',\n",
       "    'PROTEIN']]],\n",
       " ['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding hepatocyte nuclear factor - 6 .',\n",
       "  [['Pancreatic islet expression studies and polymorphic DNA markers in the genes encoding <e1> hepatocyte nuclear factor - 6 </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['In this study , we show that hnf - 3beta, - 3gamma , - 4gamma , and - 6 are expressed in pancreatic beta - cells .',\n",
       "  [['In this study , we show that <e1> hnf - 3beta </e1> , - 3gamma , - 4gamma , and - 6 are expressed in pancreatic beta - cells .',\n",
       "    'PROTEIN']]],\n",
       " ['In this study , we show that hnf - 3gamma, - 4gamma , and - 6 are expressed in pancreatic beta - cells .',\n",
       "  [['In this study , we show that <e1> hnf - 3gamma </e1> , - 4gamma , and - 6 are expressed in pancreatic beta - cells .',\n",
       "    'PROTEIN']]],\n",
       " ['In this study , we show that hnf - 4gamma, and - 6 are expressed in pancreatic beta - cells .',\n",
       "  [['In this study , we show that <e1> hnf - 4gamma </e1> , and - 6 are expressed in pancreatic beta - cells .',\n",
       "    'PROTEIN']]],\n",
       " ['In this study , we show that hnf - 6are expressed in pancreatic beta - cells .',\n",
       "  [['In this study , we show that <e1> hnf - 6 </e1> are expressed in pancreatic beta - cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Our previous studies showed that the exposure of human monocytes to E. chaffeensis induces the expression of interleukin - 1beta ( IL - 1beta ) , IL - 8 , and IL - 10 genes in vitro but not the expression of tumor necrosis factor alpha tnf - alpha mrna .',\n",
       "  [['Our previous studies showed that the exposure of human monocytes to E. chaffeensis induces the expression of interleukin - 1beta ( IL - 1beta ) , IL - 8 , and IL - 10 genes in vitro but not the expression of <e1> tumor necrosis factor alpha tnf - alpha mrna </e1> .',\n",
       "    'RNA']]],\n",
       " ['The expression of tnf - alpha mrnawas also induced .',\n",
       "  [['The expression of <e1> tnf - alpha mrna </e1> was also induced .',\n",
       "    'RNA']]],\n",
       " ['During myelopoiesis , which includes the development of granulocytic lineage, transcription factors from several families are active , including AML1/CBF beta , C/EBP , Ets , c - Myb , HOX , and MZF - 1 .',\n",
       "  [['During myelopoiesis , which includes the development of <e1> granulocytic lineage </e1> , transcription factors from several families are active , including AML1/CBF beta , C/EBP , Ets , c - Myb , HOX , and MZF - 1 .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Stat ( signal transducers and activators of transcription ) and Jak ( Janus kinases ) proteins are central components in the signal transduction events in hematopoietic cell .',\n",
       "  [['Stat ( signal transducers and activators of transcription ) and Jak ( Janus kinases ) proteins are central components in the signal transduction events in <e1> hematopoietic cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Cis - acting elements with nfkappab - like binding motifhave been identified in the promoter region of the GM - CSF gene , and are important or essential for transcriptional activity following T cell activation .',\n",
       "  [['Cis - acting elements with <e1> nfkappab - like binding motif </e1> have been identified in the promoter region of the GM - CSF gene , and are important or essential for transcriptional activity following T cell activation .',\n",
       "    'DNA']]],\n",
       " ['Cis - acting elements with NFkappaB , ap1 - like binding motifhave been identified in the promoter region of the GM - CSF gene , and are important or essential for transcriptional activity following T cell activation .',\n",
       "  [['Cis - acting elements with NFkappaB , <e1> ap1 - like binding motif </e1> have been identified in the promoter region of the GM - CSF gene , and are important or essential for transcriptional activity following T cell activation .',\n",
       "    'DNA']]],\n",
       " ['The tandem cre siteare of major functional importance as judged by mutational and electrophoretic mobility shift analyses .',\n",
       "  [['The tandem <e1> cre site </e1> are of major functional importance as judged by mutational and electrophoretic mobility shift analyses .',\n",
       "    'DNA']]],\n",
       " ['NAC had no effect on the half - life of e - selectin mrna .',\n",
       "  [['NAC had no effect on the half - life of <e1> e - selectin mrna </e1> .',\n",
       "    'RNA']]],\n",
       " ['Human cytomegalovirus induces interleukin - 8 production by a human monocytic cell line , THP - 1 , through acting concurrently on ap - 1 - binding siteof the interleukin - 8 gene .',\n",
       "  [['Human cytomegalovirus induces interleukin - 8 production by a human monocytic cell line , THP - 1 , through acting concurrently on <e1> ap - 1 - binding site </e1> of the interleukin - 8 gene .',\n",
       "    'DNA']]],\n",
       " ['The functional analysis of the IL - 8 gene revealed that both ap - 1 - binding elementwere involved in conferring the responsiveness to CMV .',\n",
       "  [['The functional analysis of the IL - 8 gene revealed that both <e1> ap - 1 - binding element </e1> were involved in conferring the responsiveness to CMV .',\n",
       "    'DNA']]],\n",
       " ['By transfecting tumor cells with both il - 12 cdna, it may be possible to use these modified targets as vaccines .',\n",
       "  [['By transfecting tumor cells with both <e1> il - 12 cdna </e1> , it may be possible to use these modified targets as vaccines .',\n",
       "    'DNA']]],\n",
       " ['The chimeric oncoprotein E2a - Pbx1 results from fusion of the e2a genefollowing t(1;19) chromosomal translocations in B cell precursor acute leukemias .',\n",
       "  [['The chimeric oncoprotein E2a - Pbx1 results from fusion of the <e1> e2a gene </e1> following t(1;19) chromosomal translocations in B cell precursor acute leukemias .',\n",
       "    'DNA']]],\n",
       " ['In DNA - protein binding studies , we detected the binding of transcription factors gabp - betato a dyad symmetry element ( DSE ) of the distal enhancer , which is formed by palindromic binding sites of Ets - like factors .',\n",
       "  [['In DNA - protein binding studies , we detected the binding of transcription factors <e1> gabp - beta </e1> to a dyad symmetry element ( DSE ) of the distal enhancer , which is formed by palindromic binding sites of Ets - like factors .',\n",
       "    'PROTEIN']]],\n",
       " ['In addition , LPS stimulation induced the binding of cognate nuclear factors to the egr - 1 site, which were identified as Egr - 1 and p50/p65 , respectively .',\n",
       "  [['In addition , LPS stimulation induced the binding of cognate nuclear factors to the <e1> egr - 1 site </e1> , which were identified as Egr - 1 and p50/p65 , respectively .',\n",
       "    'DNA']]],\n",
       " ['The cre sitein region II together conferred strong LPS responsiveness to a heterologous promoter , whereas individually they failed to provide transcriptional activation .',\n",
       "  [['The <e1> cre site </e1> in region II together conferred strong LPS responsiveness to a heterologous promoter , whereas individually they failed to provide transcriptional activation .',\n",
       "    'DNA']]],\n",
       " ['We found that IL - 2 , interferon - alpha ( IFN - alpha ) , and IFN - beta inhibited IL - 2 deprivation apoptosis in th0 clone .',\n",
       "  [['We found that IL - 2 , interferon - alpha ( IFN - alpha ) , and IFN - beta inhibited IL - 2 deprivation apoptosis in <e1> th0 clone </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['We found that IL - 2 , interferon - alpha ( IFN - alpha ) , and IFN - beta inhibited IL - 2 deprivation apoptosis in Th0 , th1 clone .',\n",
       "  [['We found that IL - 2 , interferon - alpha ( IFN - alpha ) , and IFN - beta inhibited IL - 2 deprivation apoptosis in Th0 , <e1> th1 clone </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Our present study shows that ifn - betamediate anti - apoptotic effect through other pathways than that of IL - 2 in growth factor deprivation apoptosis .',\n",
       "  [['Our present study shows that <e1> ifn - beta </e1> mediate anti - apoptotic effect through other pathways than that of IL - 2 in growth factor deprivation apoptosis .',\n",
       "    'PROTEIN']]],\n",
       " ['Analysis of phosphotyrosine - containing proteins bound to truncated p85 constructs revealed cooperative binding of the two SH2 domains but no apparent differences between the n - terminal sh2 domain .',\n",
       "  [['Analysis of phosphotyrosine - containing proteins bound to truncated p85 constructs revealed cooperative binding of the two SH2 domains but no apparent differences between the <e1> n - terminal sh2 domain </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Constitutive and inducible protein/DNA interactions of the interferon - gamma promoter in vivo in [ corrected ] cd45ra t helper subset[ published erratum appears in Eur J Immunol 1997 Jul ; 27 ( 7 ) : 1830 .',\n",
       "  [['Constitutive and inducible protein/DNA interactions of the interferon - gamma promoter in vivo in [ corrected ] <e1> cd45ra t helper subset </e1> [ published erratum appears in Eur J Immunol 1997 Jul ; 27 ( 7 ) : 1830 .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Accordingly , Jurkat T cells and primary cd45ra cd4+ t cell subsetisolated from peripheral blood using immunomagnetic beads were cultured and analyzed by LM - PCR .',\n",
       "  [['Accordingly , Jurkat T cells and <e1> primary cd45ra cd4+ t cell subset </e1> isolated from peripheral blood using immunomagnetic beads were cultured and analyzed by LM - PCR .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Accordingly , Jurkat T cells and primary cd45r0 cd4+ t cell subsetisolated from peripheral blood using immunomagnetic beads were cultured and analyzed by LM - PCR .',\n",
       "  [['Accordingly , Jurkat T cells and <e1> primary cd45r0 cd4+ t cell subset </e1> isolated from peripheral blood using immunomagnetic beads were cultured and analyzed by LM - PCR .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Interestingly , an inducible footprint between - 183 and - 196 was consistently observed in Jurkat T cells and cd45ra t helper subsetupon stimulation with phorbol 12 - myristate 13 - acetate + phytohemagglutinin ( PMA + PHA ) that was highly sensitive to treatment with corticosteroids .',\n",
       "  [['Interestingly , an inducible footprint between - 183 and - 196 was consistently observed in Jurkat T cells and <e1> cd45ra t helper subset </e1> upon stimulation with phorbol 12 - myristate 13 - acetate + phytohemagglutinin ( PMA + PHA ) that was highly sensitive to treatment with corticosteroids .',\n",
       "    'CELL_TYPE']]],\n",
       " ['ZEBRA activation can be replaced by a heterologous acidic activation domain .',\n",
       "  [['ZEBRA activation can be replaced by a <e1> heterologous acidic activation domain </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['ZEBRA activation can be replaced by a heterologous proline - rich activation domain .',\n",
       "  [['ZEBRA activation can be replaced by a <e1> heterologous proline - rich activation domain </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['ZEBRA activation can be replaced by a heterologous glutamine - rich activation domain .',\n",
       "  [['ZEBRA activation can be replaced by a <e1> heterologous glutamine - rich activation domain </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['To better understand the AP - 1 - dependent luciferase expression in Jurkat T cells , we tested two specific inhibitors of serine/threonine phosphatase - 2a: okadaic acid and calyculin A .',\n",
       "  [['To better understand the AP - 1 - dependent luciferase expression in Jurkat T cells , we tested two specific inhibitors of <e1> serine/threonine phosphatase - 2a </e1> : okadaic acid and calyculin A .',\n",
       "    'PROTEIN']]],\n",
       " ['Moderate ubiquitous activation is further induced by creb/atf factorthrough proximal upstream elements .',\n",
       "  [['Moderate ubiquitous activation is further induced by <e1> creb/atf factor </e1> through proximal upstream elements .',\n",
       "    'PROTEIN']]],\n",
       " ['The biologic actions of GM - CSF are mediated by binding to a specific receptor consisting of two chains designated as alpha subunit .',\n",
       "  [['The biologic actions of GM - CSF are mediated by binding to a specific receptor consisting of two chains designated as <e1> alpha subunit </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the human gm - csf receptor alpha subunit( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "  [['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the <e1> human gm - csf receptor alpha subunit </e1> ( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "    'PROTEIN']]],\n",
       " ['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the human gm - csf receptor beta subunit( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "  [['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the <e1> human gm - csf receptor beta subunit </e1> ( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "    'PROTEIN']]],\n",
       " ['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the human gm - csf receptor alpha subunit( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "  [['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the <e1> human gm - csf receptor alpha subunit </e1> ( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "    'PROTEIN']]],\n",
       " ['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the human gm - csf receptor beta subunit( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "  [['We have demonstrated that the murine FDC - P1 - derived cell line WT - 19 transfected with the <e1> human gm - csf receptor beta subunit </e1> ( GM - CSFR alpha and beta ) can be induced to differentiate by the addition of human GM - CSF ( hGM - CSF ) .',\n",
       "    'PROTEIN']]],\n",
       " ['An enhancer - blocking element between alpha gene segmentwithin the human T cell receptor alpha/delta locus .',\n",
       "  [['An enhancer - blocking element between <e1> alpha gene segment </e1> within the human T cell receptor alpha/delta locus .',\n",
       "    'DNA']]],\n",
       " ['Involvement of rel proteinin binding activity to the IL - 2 promoter CD28 response element/ AP - 1 sequence in human T cells .',\n",
       "  [['Involvement of <e1> rel protein </e1> in binding activity to the IL - 2 promoter CD28 response element/ AP - 1 sequence in human T cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Involvement of Rel , fos proteinin binding activity to the IL - 2 promoter CD28 response element/ AP - 1 sequence in human T cells .',\n",
       "  [['Involvement of Rel , <e1> fos protein </e1> in binding activity to the IL - 2 promoter CD28 response element/ AP - 1 sequence in human T cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Patients with one type of major histocompatibility complex class II combined immunodeficiency have mutations in a gene termed class II transactivator ( CIITA ) , which coordinately controls the transcription of the three major human class II genes , hla - dq, and - DP .',\n",
       "  [['Patients with one type of major histocompatibility complex class II combined immunodeficiency have mutations in a gene termed class II transactivator ( CIITA ) , which coordinately controls the transcription of the three major human class II genes , <e1> hla - dq </e1> , and - DP .',\n",
       "    'DNA']]],\n",
       " ['Patients with one type of major histocompatibility complex class II combined immunodeficiency have mutations in a gene termed class II transactivator ( CIITA ) , which coordinately controls the transcription of the three major human class II genes , hla - dp .',\n",
       "  [['Patients with one type of major histocompatibility complex class II combined immunodeficiency have mutations in a gene termed class II transactivator ( CIITA ) , which coordinately controls the transcription of the three major human class II genes , <e1> hla - dp </e1> .',\n",
       "    'DNA']]],\n",
       " ['We show that ligand - induced homodimerization of chimeric surface receptors consisting of the extracellular domainof IL - 4Ralpha induces Janus kinase 1 ( Jak1 ) activation , STAT6 activation , and Cepsilon germline transcripts in human B cell line BJAB .',\n",
       "  [['We show that ligand - induced homodimerization of chimeric surface receptors consisting of the <e1> extracellular domain </e1> of IL - 4Ralpha induces Janus kinase 1 ( Jak1 ) activation , STAT6 activation , and Cepsilon germline transcripts in human B cell line BJAB .',\n",
       "    'PROTEIN']]],\n",
       " ['We show that ligand - induced homodimerization of chimeric surface receptors consisting of the extracellular and transmembrane domainof IL - 4Ralpha induces Janus kinase 1 ( Jak1 ) activation , STAT6 activation , and Cepsilon germline transcripts in human B cell line BJAB .',\n",
       "  [['We show that ligand - induced homodimerization of chimeric surface receptors consisting of the extracellular and <e1> transmembrane domain </e1> of IL - 4Ralpha induces Janus kinase 1 ( Jak1 ) activation , STAT6 activation , and Cepsilon germline transcripts in human B cell line BJAB .',\n",
       "    'PROTEIN']]],\n",
       " ['Gel shift assays demonstrated SP1 and Egr - 1 binding to the CD19 GC box , while unknown nuclear proteins bound the pyg box .',\n",
       "  [['Gel shift assays demonstrated SP1 and Egr - 1 binding to the CD19 GC box , while unknown nuclear proteins bound the <e1> pyg box </e1> .',\n",
       "    'DNA']]],\n",
       " ['BSAP failed to trans - activate CD19 promoter constructs in B cells or non - B cells , suggesting that cis elements such as the pyg boxare also necessary for high level CD19 promoter expression .',\n",
       "  [['BSAP failed to trans - activate CD19 promoter constructs in B cells or non - B cells , suggesting that cis elements such as the <e1> pyg box </e1> are also necessary for high level CD19 promoter expression .',\n",
       "    'DNA']]],\n",
       " ['Multiple functionally important ebf binding sitewere identified in the lambda5 promoter/enhancer region , indicating that lambda5 is a direct genetic target for these transcription factors .',\n",
       "  [['Multiple functionally important <e1> ebf binding site </e1> were identified in the lambda5 promoter/enhancer region , indicating that lambda5 is a direct genetic target for these transcription factors .',\n",
       "    'DNA']]],\n",
       " ['That the signal transduction pathways used by the cytokines IL - 2 and IL - 15 are identical would suggest that these cytokines have redundant roles in lymphoid development ; instead , IL - 2 is the guardian of thymus - derived T - cell homeostasis , while interleukin - 15 promotes extrathymic development of t cell .',\n",
       "  [['That the signal transduction pathways used by the cytokines IL - 2 and IL - 15 are identical would suggest that these cytokines have redundant roles in lymphoid development ; instead , IL - 2 is the guardian of thymus - derived T - cell homeostasis , while interleukin - 15 promotes extrathymic development of <e1> t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['To test this latter possibility , a chimeric receptor containing the extracellular domain of c - kit ( the stem cell factor ( SCF ) receptor ) and the cytoplasmic domainof the IL - 4R alpha chain was generated .',\n",
       "  [['To test this latter possibility , a chimeric receptor containing the extracellular domain of c - kit ( the stem cell factor ( SCF ) receptor ) and the <e1> cytoplasmic domain </e1> of the IL - 4R alpha chain was generated .',\n",
       "    'PROTEIN']]],\n",
       " ['We examined the effects of histamine and its agonists on the expression of the c - fos proto - oncogeneat the transcriptional and translational levels in the human promonocytic U937 cell line .',\n",
       "  [['We examined the effects of histamine and its agonists on the expression of the <e1> c - fos proto - oncogene </e1> at the transcriptional and translational levels in the human promonocytic U937 cell line .',\n",
       "    'DNA']]],\n",
       " ['We examined the effects of histamine and its agonists on the expression of the c - myc proto - oncogeneat the transcriptional and translational levels in the human promonocytic U937 cell line .',\n",
       "  [['We examined the effects of histamine and its agonists on the expression of the <e1> c - myc proto - oncogene </e1> at the transcriptional and translational levels in the human promonocytic U937 cell line .',\n",
       "    'DNA']]],\n",
       " ['The work strongly suggests that EBV - replicative lesions are subject to direct CTL control in vivo and that immediate early proteinare frequently the immunodominant targets .',\n",
       "  [['The work strongly suggests that EBV - replicative lesions are subject to direct CTL control in vivo and that <e1> immediate early protein </e1> are frequently the immunodominant targets .',\n",
       "    'PROTEIN']]],\n",
       " ['The work strongly suggests that EBV - replicative lesions are subject to direct CTL control in vivo and that immediate early proteinare frequently the immunodominant targets .',\n",
       "  [['The work strongly suggests that EBV - replicative lesions are subject to direct CTL control in vivo and that immediate <e1> early protein </e1> are frequently the immunodominant targets .',\n",
       "    'PROTEIN']]],\n",
       " ['This contrasts with findings in alpha - and beta - herpesvirus systems ( herpes simplex , cytomegalovirus ) where viral interference with the antigen - processing pathway during lytic infection renders immediate early proteinmuch less immunogenic .',\n",
       "  [['This contrasts with findings in alpha - and beta - herpesvirus systems ( herpes simplex , cytomegalovirus ) where viral interference with the antigen - processing pathway during lytic infection renders <e1> immediate early protein </e1> much less immunogenic .',\n",
       "    'PROTEIN']]],\n",
       " ['This contrasts with findings in alpha - and beta - herpesvirus systems ( herpes simplex , cytomegalovirus ) where viral interference with the antigen - processing pathway during lytic infection renders immediate early proteinmuch less immunogenic .',\n",
       "  [['This contrasts with findings in alpha - and beta - herpesvirus systems ( herpes simplex , cytomegalovirus ) where viral interference with the antigen - processing pathway during lytic infection renders immediate <e1> early protein </e1> much less immunogenic .',\n",
       "    'PROTEIN']]],\n",
       " ['Transcription factor GATA - 3 is differentially expressed in murine th1 celland controls Th2 - specific expression of the interleukin - 5 gene .',\n",
       "  [['Transcription factor GATA - 3 is differentially expressed in <e1> murine th1 cell </e1> and controls Th2 - specific expression of the interleukin - 5 gene .',\n",
       "    'CELL_LINE']]],\n",
       " ['Transcription factor GATA - 3 is differentially expressed in murine th2 celland controls Th2 - specific expression of the interleukin - 5 gene .',\n",
       "  [['Transcription factor GATA - 3 is differentially expressed in <e1> murine th2 cell </e1> and controls Th2 - specific expression of the interleukin - 5 gene .',\n",
       "    'CELL_LINE']]],\n",
       " ['We previously established a critical role for the transcription factor GATA - 3 in IL - 5 promoter activation in EL - 4 cells , which express both th1 - type cytokine .',\n",
       "  [['We previously established a critical role for the transcription factor GATA - 3 in IL - 5 promoter activation in EL - 4 cells , which express both <e1> th1 - type cytokine </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Cross - linking of the cd3 moleculeon T lymphocytes represents one of the most effective signals for T lymphocyte activation and triggering of their cytotoxic effector function .',\n",
       "  [['Cross - linking of the <e1> cd3 molecule </e1> on T lymphocytes represents one of the most effective signals for T lymphocyte activation and triggering of their cytotoxic effector function .',\n",
       "    'PROTEIN']]],\n",
       " ['We show for human Jurkat T leukemia cells , as well as human primary t lymphocyte, that this inhibitory effect is accompanied by an impaired nuclear translocation of the Rel proteins c - Rel , RelA/p65 and NF - kappa B1/p50 , whereas the nuclear appearance of RelB remains unaffected .',\n",
       "  [['We show for human Jurkat T leukemia cells , as well as <e1> human primary t lymphocyte </e1> , that this inhibitory effect is accompanied by an impaired nuclear translocation of the Rel proteins c - Rel , RelA/p65 and NF - kappa B1/p50 , whereas the nuclear appearance of RelB remains unaffected .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Administration of dexamethasone to temporal artery - SCID chimeras for 1 wk induced a partial suppression of T cell and macrophage function as indicated by the reduced tissue concentrations of il - 2 mrna, and by the diminished expression of inducible NO synthase .',\n",
       "  [['Administration of dexamethasone to temporal artery - SCID chimeras for 1 wk induced a partial suppression of T cell and macrophage function as indicated by the reduced tissue concentrations of <e1> il - 2 mrna </e1> , and by the diminished expression of inducible NO synthase .',\n",
       "    'RNA']]],\n",
       " ['Administration of dexamethasone to temporal artery - SCID chimeras for 1 wk induced a partial suppression of T cell and macrophage function as indicated by the reduced tissue concentrations of il - 1beta mrna, and by the diminished expression of inducible NO synthase .',\n",
       "  [['Administration of dexamethasone to temporal artery - SCID chimeras for 1 wk induced a partial suppression of T cell and macrophage function as indicated by the reduced tissue concentrations of <e1> il - 1beta mrna </e1> , and by the diminished expression of inducible NO synthase .',\n",
       "    'RNA']]],\n",
       " ['Administration of dexamethasone to temporal artery - SCID chimeras for 1 wk induced a partial suppression of T cell and macrophage function as indicated by the reduced tissue concentrations of il - 6 mrna, and by the diminished expression of inducible NO synthase .',\n",
       "  [['Administration of dexamethasone to temporal artery - SCID chimeras for 1 wk induced a partial suppression of T cell and macrophage function as indicated by the reduced tissue concentrations of <e1> il - 6 mrna </e1> , and by the diminished expression of inducible NO synthase .',\n",
       "    'RNA']]],\n",
       " ['Western and supershift analyses indicated that functional jun proteinwere present in nuclear extracts of PMA - differentiated U937 cells .',\n",
       "  [['Western and supershift analyses indicated that <e1> functional jun protein </e1> were present in nuclear extracts of PMA - differentiated U937 cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Western and supershift analyses indicated that functional fos proteinwere present in nuclear extracts of PMA - differentiated U937 cells .',\n",
       "  [['Western and supershift analyses indicated that <e1> functional fos protein </e1> were present in nuclear extracts of PMA - differentiated U937 cells .',\n",
       "    'PROTEIN']]],\n",
       " ['Both truncated forms also bound to the transcription factor RBP - Jkappa in extracts prepared from human t - all cell line .',\n",
       "  [['Both truncated forms also bound to the transcription factor RBP - Jkappa in extracts prepared from <e1> human t - all cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Biochemical analysis by Western blotting of fetal hematopoietic populationshows several runt isoforms .',\n",
       "  [['Biochemical analysis by Western blotting of <e1> fetal hematopoietic population </e1> shows several runt isoforms .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Upregulation of c - Fos in activated t lymphoid cellby human immunodeficiency virus - 1 Tat protein .',\n",
       "  [['Upregulation of c - Fos in <e1> activated t lymphoid cell </e1> by human immunodeficiency virus - 1 Tat protein .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Upregulation of c - Fos in activated monocytic cellby human immunodeficiency virus - 1 Tat protein .',\n",
       "  [['Upregulation of c - Fos in <e1> activated monocytic cell </e1> by human immunodeficiency virus - 1 Tat protein .',\n",
       "    'CELL_TYPE']]],\n",
       " ['To get further insights into the molecular mechanisms underlying the biological activity of Tat , we investigated the effect of endogenous and exogenous Tat protein on c - fos gene expression in t lymphoblastoid jurkat cell line, as well as in primary peripheral blood mononuclear cells ( PBMC ) .',\n",
       "  [['To get further insights into the molecular mechanisms underlying the biological activity of Tat , we investigated the effect of endogenous and exogenous Tat protein on c - fos gene expression in <e1> t lymphoblastoid jurkat cell line </e1> , as well as in primary peripheral blood mononuclear cells ( PBMC ) .',\n",
       "    'CELL_LINE']]],\n",
       " ['To get further insights into the molecular mechanisms underlying the biological activity of Tat , we investigated the effect of endogenous and exogenous Tat protein on c - fos gene expression in T lymphoblastoid ( Jurkat ) and monocytic u937 cell line, as well as in primary peripheral blood mononuclear cells ( PBMC ) .',\n",
       "  [['To get further insights into the molecular mechanisms underlying the biological activity of Tat , we investigated the effect of endogenous and exogenous Tat protein on c - fos gene expression in T lymphoblastoid ( Jurkat ) and <e1> monocytic u937 cell line </e1> , as well as in primary peripheral blood mononuclear cells ( PBMC ) .',\n",
       "    'CELL_LINE']]],\n",
       " ['Cytokines control growth , differentiation , death , and function of cells of lymphocytic system, and together with nerve cells provide a pertinent model to study intercellular communications and intercellular signal networks .',\n",
       "  [['Cytokines control growth , differentiation , death , and function of cells of <e1> lymphocytic system </e1> , and together with nerve cells provide a pertinent model to study intercellular communications and intercellular signal networks .',\n",
       "    'CELL_TYPE']]],\n",
       " ['NF - kappa B/Rel proteins have been shown to physically interact with both ap - 1 protein .',\n",
       "  [['NF - kappa B/Rel proteins have been shown to physically interact with both <e1> ap - 1 protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The two AP - 1 sites bound c - Fos /c - Jun heterodimers in both unstimulated cell .',\n",
       "  [['The two AP - 1 sites bound c - Fos /c - Jun heterodimers in both <e1> unstimulated cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Electrophoretic mobility shift ( EMS ) analyses utilizing oligonucleotides homologous to each of the 21 bp repeat elements reacted with nuclear extracts derived from cell lines of lymphocytic cell originhave demonstrated differential binding of cellular factors to the three 21 bp repeats ( 1 - 4 ) .',\n",
       "  [['Electrophoretic mobility shift ( EMS ) analyses utilizing oligonucleotides homologous to each of the 21 bp repeat elements reacted with nuclear extracts derived from cell lines of <e1> lymphocytic cell origin </e1> have demonstrated differential binding of cellular factors to the three 21 bp repeats ( 1 - 4 ) .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Electrophoretic mobility shift ( EMS ) analyses utilizing oligonucleotides homologous to each of the 21 bp repeat elements reacted with nuclear extracts derived from cell lines of lymphocytic , monocytic cell originhave demonstrated differential binding of cellular factors to the three 21 bp repeats ( 1 - 4 ) .',\n",
       "  [['Electrophoretic mobility shift ( EMS ) analyses utilizing oligonucleotides homologous to each of the 21 bp repeat elements reacted with nuclear extracts derived from cell lines of lymphocytic , <e1> monocytic cell origin </e1> have demonstrated differential binding of cellular factors to the three 21 bp repeats ( 1 - 4 ) .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Electrophoretic mobility shift ( EMS ) analyses utilizing oligonucleotides homologous to each of the 21 bp repeat elements reacted with nuclear extracts derived from cell lines of lymphocytic , monocytic , neuronal cell originhave demonstrated differential binding of cellular factors to the three 21 bp repeats ( 1 - 4 ) .',\n",
       "  [['Electrophoretic mobility shift ( EMS ) analyses utilizing oligonucleotides homologous to each of the 21 bp repeat elements reacted with nuclear extracts derived from cell lines of lymphocytic , monocytic , <e1> neuronal cell origin </e1> have demonstrated differential binding of cellular factors to the three 21 bp repeats ( 1 - 4 ) .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Unlike this physiologically relevant stimulation , PHA stimulates productive LAI replication in both naive t cell .',\n",
       "  [['Unlike this physiologically relevant stimulation , PHA stimulates productive LAI replication in both <e1> naive t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Transcription factor E2F binds to cellular promoters of certain growth - controlling geneand forms distinct heteromeric complexes with other nuclear proteins .',\n",
       "  [['Transcription factor E2F binds to cellular promoters of certain <e1> growth - controlling gene </e1> and forms distinct heteromeric complexes with other nuclear proteins .',\n",
       "    'DNA']]],\n",
       " ['In addition , RNase protection assays revealed that anti - CD3/ PMA - stimulated T cells from a substantial proportion of elderly subjects exhibited decreased levels of c - fos mrnacompared to T cells from young subjects .',\n",
       "  [['In addition , RNase protection assays revealed that anti - CD3/ PMA - stimulated T cells from a substantial proportion of elderly subjects exhibited decreased levels of <e1> c - fos mrna </e1> compared to T cells from young subjects .',\n",
       "    'RNA']]],\n",
       " ['In addition , RNase protection assays revealed that anti - CD3/ PMA - stimulated T cells from a substantial proportion of elderly subjects exhibited decreased levels of c - jun mrnacompared to T cells from young subjects .',\n",
       "  [['In addition , RNase protection assays revealed that anti - CD3/ PMA - stimulated T cells from a substantial proportion of elderly subjects exhibited decreased levels of <e1> c - jun mrna </e1> compared to T cells from young subjects .',\n",
       "    'RNA']]],\n",
       " ['Using electrophoretic mobility shift assays , the levels of nuclear regulatory proteins recognizing the AP - 1 consensus TRE motif , the proximal c - jun TRE - like promoter element , and the c - fos serum response element ( SRE ) were determined in resting t cell .',\n",
       "  [['Using electrophoretic mobility shift assays , the levels of nuclear regulatory proteins recognizing the AP - 1 consensus TRE motif , the proximal c - jun TRE - like promoter element , and the c - fos serum response element ( SRE ) were determined in <e1> resting t cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP - 1 TRE , c - jun TRE , and c - fos SRE DNA sequence motifs , age - related reductions in the activation of AP - 1 were accompanied by decreased levels of c - jun tre binding complex .',\n",
       "  [['Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP - 1 TRE , c - jun TRE , and c - fos SRE DNA sequence motifs , age - related reductions in the activation of AP - 1 were accompanied by decreased levels of <e1> c - jun tre binding complex </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP - 1 TRE , c - jun TRE , and c - fos SRE DNA sequence motifs , age - related reductions in the activation of AP - 1 were accompanied by decreased levels of c - fos sre binding complex .',\n",
       "  [['Although the stimulation of T cells from young subjects resulted in coordinated increases of nuclear protein complexes binding the AP - 1 TRE , c - jun TRE , and c - fos SRE DNA sequence motifs , age - related reductions in the activation of AP - 1 were accompanied by decreased levels of <e1> c - fos sre binding complex </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['BACKGROUND : The immediate early genes ( IE ) of human cytomegalovirus ( CMV ) can be expressed in monocytic cells and are known to regulate viral gene .',\n",
       "  [['BACKGROUND : The immediate early genes ( IE ) of human cytomegalovirus ( CMV ) can be expressed in monocytic cells and are known to regulate <e1> viral gene </e1> .',\n",
       "    'DNA']]],\n",
       " ['Cotransfection of CMV IE genes and il - 6 chloramphenicol acetyl transferase cat constructwere used to study IL - 6 promoter activity .',\n",
       "  [['Cotransfection of CMV IE genes and <e1> il - 6 chloramphenicol acetyl transferase cat construct </e1> were used to study IL - 6 promoter activity .',\n",
       "    'DNA']]],\n",
       " ['Cotransfection of CMV IE genes and il - 6 luciferase constructwere used to study IL - 6 promoter activity .',\n",
       "  [['Cotransfection of CMV IE genes and <e1> il - 6 luciferase construct </e1> were used to study IL - 6 promoter activity .',\n",
       "    'DNA']]],\n",
       " [' nuclear rel - a protein complexare differentially distributed within human thymocytes .',\n",
       "  [[' <e1> nuclear rel - a protein complex </e1> are differentially distributed within human thymocytes .',\n",
       "    'PROTEIN']]],\n",
       " [' nuclear c - rel protein complexare differentially distributed within human thymocytes .',\n",
       "  [[' <e1> nuclear c - rel protein complex </e1> are differentially distributed within human thymocytes .',\n",
       "    'PROTEIN']]],\n",
       " ['In the present report , we identified NF - kappa B proteins extracted from human thymocyte nuclei as being p50/p65 complex .',\n",
       "  [['In the present report , we identified NF - kappa B proteins extracted from human thymocyte nuclei as being <e1> p50/p65 complex </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['p65 , p50 , and c - Rel activities were found in both cd4 - positive thymocyte .',\n",
       "  [['p65 , p50 , and c - Rel activities were found in both <e1> cd4 - positive thymocyte </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These observations suggest that p65 complexplay distinct roles in gene expression and that both forms of NF - kappa B play critical roles during late stages of the intrathymic maturation of T cells .',\n",
       "  [['These observations suggest that <e1> p65 complex </e1> play distinct roles in gene expression and that both forms of NF - kappa B play critical roles during late stages of the intrathymic maturation of T cells .',\n",
       "    'PROTEIN']]],\n",
       " ['AML1 may play a role in growth and differentiation of cells along erythroid lineage, because a significant level of the AML1 gene is expressed in these cells .',\n",
       "  [['AML1 may play a role in growth and differentiation of cells along <e1> erythroid lineage </e1> , because a significant level of the AML1 gene is expressed in these cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['The influence of the anti - allergy agent azelastine hydrochloride ( Azeptin ) on NF - kappa B activation associated with the generation of cytokines and nitric oxide ( NO ) was investigated in various kinds of human cell .',\n",
       "  [['The influence of the anti - allergy agent azelastine hydrochloride ( Azeptin ) on NF - kappa B activation associated with the generation of cytokines and nitric oxide ( NO ) was investigated in various kinds of <e1> human cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['To investigate the mechanisms responsible for tissue - specific expression of the IL - 4 gene , we analyzed nuclear factors binding to the PRE - I site and compared the binding activities of these factors to the IL - 4 promoter of th1 cell .',\n",
       "  [['To investigate the mechanisms responsible for tissue - specific expression of the IL - 4 gene , we analyzed nuclear factors binding to the PRE - I site and compared the binding activities of these factors to the IL - 4 promoter of <e1> th1 cell </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Gene inactivation studies on the pu.1 genehave revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "  [['Gene inactivation studies on the <e1> pu.1 gene </e1> have revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "    'DNA']]],\n",
       " ['Gene inactivation studies on the PU.1 , ikaros genehave revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "  [['Gene inactivation studies on the PU.1 , <e1> ikaros gene </e1> have revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "    'DNA']]],\n",
       " ['Gene inactivation studies on the PU.1 , Ikaros and GATA - 3 genes have revealed that their encoded factors are essential for the earliest commitment step into the b lymphoid lineage .',\n",
       "  [['Gene inactivation studies on the PU.1 , Ikaros and GATA - 3 genes have revealed that their encoded factors are essential for the earliest commitment step into the <e1> b lymphoid lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['Gene inactivation studies on the pu.1 genehave revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "  [['Gene inactivation studies on the <e1> pu.1 gene </e1> have revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "    'DNA']]],\n",
       " ['Gene inactivation studies on the PU.1 , ikaros genehave revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "  [['Gene inactivation studies on the PU.1 , <e1> ikaros gene </e1> have revealed that their encoded factors are essential for the earliest commitment step into the B and T lymphoid lineages .',\n",
       "    'DNA']]],\n",
       " ['Gene inactivation studies on the PU.1 , Ikaros and GATA - 3 genes have revealed that their encoded factors are essential for the earliest commitment step into the b lymphoid lineage .',\n",
       "  [['Gene inactivation studies on the PU.1 , Ikaros and GATA - 3 genes have revealed that their encoded factors are essential for the earliest commitment step into the <e1> b lymphoid lineage </e1> .',\n",
       "    'CELL_TYPE']]],\n",
       " ['HIV - 1 LTR transcription is controlled by a complex array of virus - encoded protein .',\n",
       "  [['HIV - 1 LTR transcription is controlled by a complex array of <e1> virus - encoded protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Under these conditions , etoposide produced a biphasic activation of protein kinase C , which consisted in an early transient activation ( from hours 1 to 6 ) of the membrane - bound enzyme followed by a later activation ( hour 48 ) of the total , membrane - bound enzyme .',\n",
       "  [['Under these conditions , etoposide produced a biphasic activation of protein kinase C , which consisted in an early transient activation ( from hours 1 to 6 ) of the membrane - bound enzyme followed by a later activation ( hour 48 ) of the total , <e1> membrane - bound enzyme </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['The jak1 tyrosine kinasehave been demonstrated to undergo tyrosine phosphorylation in response to interleukin ( IL ) - 4 stimulation in different cell systems .',\n",
       "  [['The <e1> jak1 tyrosine kinase </e1> have been demonstrated to undergo tyrosine phosphorylation in response to interleukin ( IL ) - 4 stimulation in different cell systems .',\n",
       "    'PROTEIN']]],\n",
       " ['The Jak1 , jak2 tyrosine kinasehave been demonstrated to undergo tyrosine phosphorylation in response to interleukin ( IL ) - 4 stimulation in different cell systems .',\n",
       "  [['The Jak1 , <e1> jak2 tyrosine kinase </e1> have been demonstrated to undergo tyrosine phosphorylation in response to interleukin ( IL ) - 4 stimulation in different cell systems .',\n",
       "    'PROTEIN']]],\n",
       " ['The Jak1 , Jak2 , jak3 tyrosine kinasehave been demonstrated to undergo tyrosine phosphorylation in response to interleukin ( IL ) - 4 stimulation in different cell systems .',\n",
       "  [['The Jak1 , Jak2 , <e1> jak3 tyrosine kinase </e1> have been demonstrated to undergo tyrosine phosphorylation in response to interleukin ( IL ) - 4 stimulation in different cell systems .',\n",
       "    'PROTEIN']]],\n",
       " ['Infection of human peripheral blood lymphocytes with HIV - 1 in vitro resulted in increased interleukin - 2 ( IL - 2 ) secretion in response to T cell activation via the cd3 receptor .',\n",
       "  [['Infection of human peripheral blood lymphocytes with HIV - 1 in vitro resulted in increased interleukin - 2 ( IL - 2 ) secretion in response to T cell activation via the <e1> cd3 receptor </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Freshly isolated monocytes constitutively expressed high levels of transcriptionally inactive p50 homodimer which decreased with time in culture in favor of the transcriptionally active p50/p65 heterodimer .',\n",
       "  [['Freshly isolated monocytes constitutively expressed high levels of transcriptionally inactive p50 homodimer which decreased with time in culture in favor of the transcriptionally active <e1> p50/p65 heterodimer </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['A T cell - specific enhancer in the interleukin - 3 locus is activated cooperatively by oct elementwithin a DNase I - hypersensitive site .',\n",
       "  [['A T cell - specific enhancer in the interleukin - 3 locus is activated cooperatively by <e1> oct element </e1> within a DNase I - hypersensitive site .',\n",
       "    'DNA']]],\n",
       " ['One of these NFAT - like elements encompassed overlapping oct - binding site, which functioned in a highly synergistic manner .',\n",
       "  [['One of these NFAT - like elements encompassed overlapping <e1> oct - binding site </e1> , which functioned in a highly synergistic manner .',\n",
       "    'DNA']]],\n",
       " ['We suggest that the T cell - specific expression of the IL - 3 gene is partly controlled through the enhancer by cooperation between oct family protein .',\n",
       "  [['We suggest that the T cell - specific expression of the IL - 3 gene is partly controlled through the enhancer by cooperation between <e1> oct family protein </e1> .',\n",
       "    'PROTEIN']]],\n",
       " ['Early viral RNA splicing patterns also differ between wild - type cell .',\n",
       "  [['Early viral RNA splicing patterns also differ between <e1> wild - type cell </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p4, P6 , P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p4 </e1> , P6 , P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p6, P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p6 </e1> , P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p7, and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p7 </e1> , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p9) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p9 </e1> ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p4, P6 , P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p4 </e1> , P6 , P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p6, P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p6 </e1> , P7 , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p7, and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p7 </e1> , and P9 ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Residues 432 , 435 , 437 , 438 , and 440 ( position p9) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "  [['Residues 432 , 435 , 437 , 438 , and 440 ( <e1> position p9 </e1> ) contributed to DQ2 binding , whereas residues 431 , 433 , 434 , and 436 ( positions P 1 , P2 , P3 , and P5 ) contributed to TCR contact .',\n",
       "    'DNA']]],\n",
       " ['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding componentof the IL - 12R , is expressed on human Th1 but not Th2 clones and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "  [['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned <e1> binding component </e1> of the IL - 12R , is expressed on human Th1 but not Th2 clones and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "    'PROTEIN']]],\n",
       " ['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on human th1 cloneand is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "  [['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on <e1> human th1 clone </e1> and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "    'CELL_LINE']]],\n",
       " ['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on human th2 cloneand is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "  [['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on <e1> human th2 clone </e1> and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "    'CELL_LINE']]],\n",
       " ['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding componentof the IL - 12R , is expressed on human Th1 but not Th2 clones and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "  [['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned <e1> binding component </e1> of the IL - 12R , is expressed on human Th1 but not Th2 clones and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "    'PROTEIN']]],\n",
       " ['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on human th1 cloneand is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "  [['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on <e1> human th1 clone </e1> and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "    'CELL_LINE']]],\n",
       " ['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on human th2 cloneand is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "  [['Here we show that the IL - 12 receptor ( IL - 12R ) beta 2 subunit , a recently cloned binding and signal transducing component of the IL - 12R , is expressed on <e1> human th2 clone </e1> and is induced during differentiation of human naive cells along the Th1 but not the Th2 pathway .',\n",
       "    'CELL_LINE']]],\n",
       " ['IL - 12 and type i interferoninduce expression of the IL - 12R beta 2 chain during in vitro T cell differentiation after antigen receptor triggering .',\n",
       "  [['IL - 12 and <e1> type i interferon </e1> induce expression of the IL - 12R beta 2 chain during in vitro T cell differentiation after antigen receptor triggering .',\n",
       "    'PROTEIN']]],\n",
       " ['IL - 12 and type ii interferoninduce expression of the IL - 12R beta 2 chain during in vitro T cell differentiation after antigen receptor triggering .',\n",
       "  [['IL - 12 and <e1> type ii interferon </e1> induce expression of the IL - 12R beta 2 chain during in vitro T cell differentiation after antigen receptor triggering .',\n",
       "    'PROTEIN']]],\n",
       " ['No degradation of p105 inhibitorwas observed under these conditions .',\n",
       "  [['No degradation of <e1> p105 inhibitor </e1> was observed under these conditions .',\n",
       "    'PROTEIN']]],\n",
       " ['Pax - 5a was detectable in pro - b - cell line, but not in pro - B - cell or T - cell lines .',\n",
       "  [['Pax - 5a was detectable in <e1> pro - b - cell line </e1> , but not in pro - B - cell or T - cell lines .',\n",
       "    'CELL_LINE']]],\n",
       " ['Pax - 5a was detectable in pro - , pre - b - cell line, but not in pro - B - cell or T - cell lines .',\n",
       "  [['Pax - 5a was detectable in pro - , <e1> pre - b - cell line </e1> , but not in pro - B - cell or T - cell lines .',\n",
       "    'CELL_LINE']]],\n",
       " ['Pax - 5a was detectable in pro - , pre - , and mature b - cell line, but not in pro - B - cell or T - cell lines .',\n",
       "  [['Pax - 5a was detectable in pro - , pre - , and <e1> mature b - cell line </e1> , but not in pro - B - cell or T - cell lines .',\n",
       "    'CELL_LINE']]],\n",
       " ['Pax - 5a was detectable in pro - , pre - , and mature B - cell lines , but not in two plasmacytomas ; Pax - 5b was shown to be present at low levels in mature B - cell lines and , unexpectedly , in one plasma cell line , but not in pro - b - cell line .',\n",
       "  [['Pax - 5a was detectable in pro - , pre - , and mature B - cell lines , but not in two plasmacytomas ; Pax - 5b was shown to be present at low levels in mature B - cell lines and , unexpectedly , in one plasma cell line , but not in <e1> pro - b - cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Pax - 5a was detectable in pro - b - cell line, but not in pro - B - cell or T - cell lines .',\n",
       "  [['Pax - 5a was detectable in <e1> pro - b - cell line </e1> , but not in pro - B - cell or T - cell lines .',\n",
       "    'CELL_LINE']]],\n",
       " ['Pax - 5a was detectable in pro - , pre - b - cell line, but not in pro - B - cell or T - cell lines .',\n",
       "  [['Pax - 5a was detectable in pro - , <e1> pre - b - cell line </e1> , but not in pro - B - cell or T - cell lines .',\n",
       "    'CELL_LINE']]],\n",
       " ['Pax - 5a was detectable in pro - , pre - , and mature b - cell line, but not in pro - B - cell or T - cell lines .',\n",
       "  [['Pax - 5a was detectable in pro - , pre - , and <e1> mature b - cell line </e1> , but not in pro - B - cell or T - cell lines .',\n",
       "    'CELL_LINE']]],\n",
       " ['Pax - 5a was detectable in pro - , pre - , and mature B - cell lines , but not in two plasmacytomas ; Pax - 5b was shown to be present at low levels in mature B - cell lines and , unexpectedly , in one plasma cell line , but not in pro - b - cell line .',\n",
       "  [['Pax - 5a was detectable in pro - , pre - , and mature B - cell lines , but not in two plasmacytomas ; Pax - 5b was shown to be present at low levels in mature B - cell lines and , unexpectedly , in one plasma cell line , but not in <e1> pro - b - cell line </e1> .',\n",
       "    'CELL_LINE']]],\n",
       " ['Constitutive dephosphorylation and activation of a member of the nuclear factor of activated T cells , NF - AT1 , in tax - expressing human t cellleukemia virus - infected human T cells .',\n",
       "  [['Constitutive dephosphorylation and activation of a member of the nuclear factor of activated T cells , NF - AT1 , in <e1> tax - expressing human t cell </e1> leukemia virus - infected human T cells .',\n",
       "    'CELL_TYPE']]],\n",
       " ['These results suggest that activation of NF - AT1 in tax - expressing t cellresults from its dephosphorylation , which in turn may be due to deregulation of .',\n",
       "  [['These results suggest that activation of NF - AT1 in <e1> tax - expressing t cell </e1> results from its dephosphorylation , which in turn may be due to deregulation of .',\n",
       "    'CELL_LINE']]],\n",
       " ['Physical interactions between ets proteinplay an important role in their cooperative activation of the human immunodeficiency virus enhancer in T cells .',\n",
       "  [['Physical interactions between <e1> ets protein </e1> play an important role in their cooperative activation of the human immunodeficiency virus enhancer in T cells .',\n",
       "    'PROTEIN']]],\n",
       " ['The transcriptional regulatory elements of many inducible T - cell genes contain adjacent or overlapping binding sites for the ets familyof transcription factors .',\n",
       "  [['The transcriptional regulatory elements of many inducible T - cell genes contain adjacent or overlapping binding sites for the <e1> ets family </e1> of transcription factors .',\n",
       "    'PROTEIN']]],\n",
       " ...]"
      ]
     },
     "execution_count": 836,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combinacaoEntidades_desc = list()\n",
    "combinacaoEntidadesAll_desc = list()\n",
    "palavrasDescontinua=['AND NOT', 'AND/OR', 'AS WELL AS', 'AND', 'OR', 'BUT NOT', 'NEITHER NOR', 'THAN']\n",
    "pulando_termos_postagger=[]\n",
    "# agora, tratar descontinuas\n",
    "for key in frasesComDescontinuas:\n",
    "    entidades = dicSentencesComLabel[key][1].copy()\n",
    "    for entidade in entidades:\n",
    "        indices = entidade[1]\n",
    "        if not areConsecutive(indices): # é descontinua\n",
    "            tipo_entidade = entidade[2]\n",
    "            texto_entidade = ' '.join(entidade[0])\n",
    "            #print(texto_entidade)\n",
    "            inicio=indices[0]\n",
    "            fim=indices[-1]\n",
    "            tokens = dicSentencesComLabel[key][0].copy()\n",
    "            so_tokens = [t[0] for t in tokens]\n",
    "            #print('so_tokens:', so_tokens)\n",
    "            inicio_frase = so_tokens[:inicio]\n",
    "            fim_frase = so_tokens[fim+1:-1] \n",
    "            #print('inicio_frase:', inicio_frase)\n",
    "            #print('fim_frase:', fim_frase)\n",
    "            meio_frase = '<e1> '+texto_entidade + ' </e1> '\n",
    "            frase_final = ' '.join(inicio_frase)+' '+meio_frase+' '.join(fim_frase)+' .'\n",
    "            frase_final_texto = ' '.join(inicio_frase)+' '+texto_entidade+' '.join(fim_frase)+' .'\n",
    "            frase_final = replaceWhiteSpaces(frase_final)\n",
    "            frase_final_texto = replaceWhiteSpaces(frase_final_texto)\n",
    "            #print('frase_final_texto:', frase_final_texto)\n",
    "            #print('frase_final:', frase_final)\n",
    "            combinacaoEntidades_desc.append([frase_final, tipo_entidade])\n",
    "            if len(combinacaoEntidades_desc)>0:\n",
    "                combinacaoEntidadesAll_desc.append([frase_final_texto, combinacaoEntidades_desc])\n",
    "            combinacaoEntidades_desc = list()\n",
    "            '''\n",
    "            #agora, as combinações possiveis\n",
    "            # primeiro, gerar o termo todo\n",
    "            menorIndice = entidade[1][0]\n",
    "            maiorIndice = entidade[1][-1]\n",
    "            novoIndice=[n for n in range(menorIndice, maiorIndice+1)]\n",
    "            print(menorIndice)\n",
    "            print(maiorIndice)\n",
    "            print('novoIndice:', novoIndice)\n",
    "            novoTexto=[]\n",
    "            for token in tokens:\n",
    "                for indice in novoIndice:\n",
    "                    if token[1] == indice:\n",
    "                        novoTexto.append(token[0]) # novo termo com o CC\n",
    "                        \n",
    "            novoTextoCC = ' '.join(novoTexto).lower()\n",
    "            novoTextoCC = novoTextoCC.replace('and not', \"and_not\").replace('as well as', \"as_well_as\").replace('neither nor', \"neither_no\")\n",
    "            #indices_cc = [palavra.lower() in novoTexto.split() for palavra in palavrasDescontinua]\n",
    "            #indices2 = [txt.lower().split().index(palavra.lower()) for palavra in palavrasDescontinua]\n",
    "\n",
    "            # segundo, ver onde está o CC (AND, OR, etc)\n",
    "            indice_cc=0\n",
    "            len_cc=0\n",
    "            for cc in palavrasDescontinua:\n",
    "                try:\n",
    "                    indice_cc = novoTextoCC.split().index(cc.lower()) # pega o primero indice da primeira palavra CC\n",
    "                    len_cc = len(cc.split())\n",
    "                    break\n",
    "                except:\n",
    "                    continue\n",
    "            \n",
    "            # terceiro, fazer a combinação com as descontinuas:\n",
    "            # tudo antes do CC + <pula um token> + tokens (a partir daqui, combinar)\n",
    "            print('novoTexto[0:-2]:', novoTexto[0:len(novoTexto)])\n",
    "            termos_antes_cc = novoTexto[:indice_cc]\n",
    "            #termos_apos_cc = novoTexto[len_cc+3:-1] # nao pega o primeiro apos\n",
    "            termos_apos_cc = novoTexto[len_cc+3:len(novoTexto)] # nao pega o primeiro apos\n",
    "            #termos_ultimo_cc = novoTexto[-1] # nao pega o primeiro apos\n",
    "            print('termos_antes_cc:', termos_antes_cc)\n",
    "            print('termos_apos_cc:', termos_apos_cc)\n",
    "            #print('termos_ultimo_cc:', termos_ultimo_cc)\n",
    "            \n",
    "            # concatena o termo antes, com os termos apos, mais a ultima palavra          \n",
    "            # a primeira e ultima palavra tbm é um termo\n",
    "            #nova_entidade = termos_antes_cc\n",
    "            nova_entidade = []\n",
    "            #for i, termo_apos in enumerate(termos_apos_cc):\n",
    "            for i in range(len(termos_apos_cc), 0, -1):\n",
    "                #nova_entidade = termos_antes_cc\n",
    "                #for j in range(i, len(termos_apos_cc)):\n",
    "                #nova_entidade = nova_entidade +[termos_apos_cc[j]]\n",
    "                nova_entidade = [termos_apos_cc[i-1]]+nova_entidade\n",
    "                #for j in range(i, 0, -1):\n",
    "                    #print('j:',j)\n",
    "                #    print('add ',termos_apos_cc[j-1])\n",
    "                #    nova_entidade = [termos_apos_cc[j-1]]+nova_entidade\n",
    "                #nova_entidade = nova_entidade\n",
    "                #print('nova_entidade:', termos_antes_cc+nova_entidade)\n",
    "                #nova_entidade=[]\n",
    "                frase_final = ' '.join(inicio_frase)+' '+' '.join(termos_antes_cc+nova_entidade)+' '.join(fim_frase)\n",
    "                frase_final_texto = ' '.join(inicio_frase)+' <e1> '+' '.join(termos_antes_cc+nova_entidade)+' </e1>'+' '.join(fim_frase)\n",
    "                print('frase_final:', frase_final)\n",
    "                print('frase_final_texto:', frase_final_texto)\n",
    "                # só salva se nao tem ainda\n",
    "                # e ver se está no padrão para salvar\n",
    "                devePular = 0\n",
    "                if '. </e1>' in frase_final_texto or ', </e1>' in frase_final_texto  or '; </e1>' in frase_final_texto or '- </e1>' in frase_final_texto  or ': </e1>' in frase_final_texto  or '= </e1>' in frase_final_texto  or '/ </e1>' in frase_final_texto  or '( </e1>' in frase_final_texto  or ') </e1>' in frase_final_texto  or '[ </e1>' in frase_final_texto  or '] </e1>' in frase_final_texto  or ': </e1>' in frase_final_texto or 'and </e1>' in frase_final_texto or 'or </e1>' in frase_final_texto:\n",
    "                    devePular=1\n",
    "                print('devePular:', devePular)\n",
    "                if '<e1> .' in frase_final_texto or '<e1> ,' in frase_final_texto  or '<e1> ;' in frase_final_texto or '<e1> -' in frase_final_texto  or '<e1> :' in frase_final_texto  or '<e1> =' in frase_final_texto  or '<e1> /' in frase_final_texto  or '<e1> (' in frase_final_texto  or '<e1> )' in frase_final_texto  or '<e1> [' in frase_final_texto  or '<e1> ]' in frase_final_texto  or '<e1> :' in frase_final_texto  or '<e1> and' in frase_final_texto  or '<e1> or' in frase_final_texto:\n",
    "                    devePular=1\n",
    "                print('devePular:', devePular)\n",
    "                if re.search(\"<e1> [0-9]* </e1>\", frase_final_texto):\n",
    "                    devePular=1\n",
    "                print('devePular:', devePular)\n",
    "                pos_tagger_termo = tipoPostaggerTokens(termo, dicPosTagger)\n",
    "                if pos_tagger_termo not in lista_postaggers_entidades:\n",
    "                    pulando_termos_postagger.append([termo, pos_tagger_termo])\n",
    "                    devePular=1\n",
    "\n",
    "                tem_frase = 0\n",
    "                for frase in combinacaoEntidades:\n",
    "                    if frase[0] == frase_string:\n",
    "                        tem_frase=''\n",
    "                        break\n",
    "                print('tem_frase:', tem_frase)\n",
    "                print('devePular:', devePular)\n",
    "                if tem_frase==0 and devePular==0:\n",
    "                    combinacaoEntidades_desc.append([frase_final, 'O'])\n",
    "                    combinacaoEntidadesAll_desc.append([frase_final_texto, combinacaoEntidades_desc])\n",
    "                combinacaoEntidades_desc= list()\n",
    "            '''\n",
    "            \n",
    "    #break\n",
    "    \n",
    "combinacaoEntidadesAll_desc\n",
    "#pulando_termos_postagger"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 837,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1568"
      ]
     },
     "execution_count": 837,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(combinacaoEntidadesAll_desc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 838,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "16614"
      ]
     },
     "execution_count": 838,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(combinacaoEntidadesAll)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 840,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "numTreinamento: 13291\n",
      "numTeste: 1661\n",
      "numTotalEntidades: 165571\n",
      "numTotalEntidadesTrain: 131376\n",
      "numTotalEntidadesDev: 16986\n",
      "numTotalEntidadesTest: 17209\n"
     ]
    }
   ],
   "source": [
    "# já ir gravando arquivos treinamento, test e dev...\n",
    "# pra fazer um teste sem descontinuas\n",
    "numTreinamento = len(combinacaoEntidadesAll)*0.8\n",
    "numTreinamento = int(numTreinamento)\n",
    "numTeste = (len(combinacaoEntidadesAll) - numTreinamento)/2\n",
    "numTeste = int(numTeste)\n",
    "numTotalEntidades=0\n",
    "numTotalEntidadesTrain=0\n",
    "numTotalEntidadesDev=0\n",
    "numTotalEntidadesTest=0\n",
    "\n",
    "print('numTreinamento:', numTreinamento)\n",
    "print('numTeste:', numTeste)\n",
    "\n",
    "f_train = open('genia.train', 'w')\n",
    "f_dev = open('genia.dev', 'w')\n",
    "f_test = open('genia.test', 'w')\n",
    "f_dev_entidades = open('genia_entidades.dev', 'w')\n",
    "f_test_entidades = open('genia_entidades.test', 'w')\n",
    "\n",
    "for i, combinacaoEntidades in enumerate(combinacaoEntidadesAll):\n",
    "    #print(dicSentences[i])\n",
    "    frase = combinacaoEntidades[0]\n",
    "    frases_entidade = combinacaoEntidades[1]\n",
    "    if i<=numTreinamento:\n",
    "        f_train.write(frase+'\\n')\n",
    "        for frase_entidade in frases_entidade:\n",
    "            f_train.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "            numTotalEntidades=numTotalEntidades+1\n",
    "            numTotalEntidadesTrain=numTotalEntidadesTrain+1\n",
    "    elif i <= numTreinamento + numTeste:\n",
    "        f_dev.write(frase+'\\n')\n",
    "        for frase_entidade in frases_entidade:\n",
    "            f_dev.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "            f_dev_entidades.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "            numTotalEntidades=numTotalEntidades+1\n",
    "            numTotalEntidadesDev=numTotalEntidadesDev+1\n",
    "    else:\n",
    "        f_test.write(frase+'\\n')\n",
    "        for frase_entidade in frases_entidade:\n",
    "            f_test.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "            f_test_entidades.write(frase_entidade[1]+'\\t'+frase_entidade[0]+'\\n')\n",
    "            numTotalEntidades=numTotalEntidades+1\n",
    "            numTotalEntidadesTest=numTotalEntidadesTest+1\n",
    "\n",
    "f_train.close()\n",
    "f_test.close()\n",
    "f_dev.close()\n",
    "f_test_entidades.close()\n",
    "f_dev_entidades.close()\n",
    "\n",
    "print('numTotalEntidades:', numTotalEntidades)\n",
    "print('numTotalEntidadesTrain:', numTotalEntidadesTrain)\n",
    "print('numTotalEntidadesDev:', numTotalEntidadesDev)\n",
    "print('numTotalEntidadesTest:', numTotalEntidadesTest)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "'''\n",
    "combinacaoEntidades_desc = list()\n",
    "combinacaoEntidadesAll_desc = list()\n",
    "palavrasDescontinua=['AND NOT', 'AND/OR', 'AS WELL AS', 'AND', 'OR', 'BUT NOT', 'NEITHER NOR', 'THAN']\n",
    "\n",
    "# agora, tratar descontinuas\n",
    "for key in frasesComDescontinuas:\n",
    "    entidades = dicSentencesComLabel[key][1].copy()\n",
    "    for entidade in entidades:\n",
    "        indices = entidade[1]\n",
    "        if not areConsecutive(indices): # é descontinua\n",
    "            tipo_entidade = entidade[2]\n",
    "            inicio=indices[0]\n",
    "            fim=indices[-1]\n",
    "            tokens = dicSentencesComLabel[key][0].copy()\n",
    "            so_tokens = [t[0] for t in tokens]\n",
    "            inicio_frase = so_tokens[:inicio]\n",
    "            fim_frase = so_tokens[fim+2:-1] # TODO - mais 2 ou 3?\n",
    "            print('inicio_frase:', inicio_frase)\n",
    "            print('fim_frase:', fim_frase)\n",
    "            texto_entidade = ' '.join(entidade[0])\n",
    "            meio_frase = '<e1> '+texto_entidade + ' </e1>'\n",
    "            frase_final = inicio_frase+meio_frase+fim_frase\n",
    "            frase_final_texto = inicio_frase+texto_entidade+fim_frase\n",
    "            print('frase_final_texto:', frase_final_texto)\n",
    "            print('frase_final:', frase_final)\n",
    "            combinacaoEntidades_desc.append([frase_final, tipo_entidade])\n",
    "            combinacaoEntidadesAll_desc.append([frase_final_texto, combinacaoEntidades_desc])\n",
    "            #agora, as combinações possiveis\n",
    "            # primeiro, gerar o termo todo\n",
    "            menorIndice = entidade[1][0]\n",
    "            maiorIndice = entidade[1][-1]\n",
    "            novoIndice=[n for n in range(menorIndice, maiorIndice+1)]\n",
    "            #print(menorIndice)\n",
    "            #print(maiorIndice)\n",
    "            #print('novoIndice:', novoIndice)\n",
    "            novoTexto=[]\n",
    "            for token in tokens:\n",
    "                for indice in novoIndice:\n",
    "                    if token[1] == indice:\n",
    "                        novoTexto.append(token[0]) # novo termo com o CC\n",
    "                        \n",
    "            novoTextoCC = ' '.join(novoTexto).lower()\n",
    "            novoTextoCC = novoTexto.replace('and not', \"and_not\").replace('as well as', \"as_well_as\").replace('neither nor', \"neither_no\")\n",
    "            #indices_cc = [palavra.lower() in novoTexto.split() for palavra in palavrasDescontinua]\n",
    "            #indices2 = [txt.lower().split().index(palavra.lower()) for palavra in palavrasDescontinua]\n",
    "\n",
    "            # segundo, ver onde está o CC (AND, OR, etc)\n",
    "            indice_cc=0\n",
    "            len_cc=0\n",
    "            for cc in palavrasDescontinua:\n",
    "                try:\n",
    "                    indice_cc = novoTextoCC.split().index(cc.lower()) # pega o primero indice da primeira palavra CC\n",
    "                    len_cc = len(cc.split())\n",
    "                    break\n",
    "                except:\n",
    "                    continue\n",
    "            \n",
    "            # terceiro, fazer a combinação com as descontinuas:\n",
    "            # tudo antes do CC + <pula um token> + tokens (a partir daqui, combinar)\n",
    "            termos_antes_cc = novoTexto[:indice_cc]\n",
    "            termos_apos_cc = novoTexto[len_cc+1:-1]# nao pega o primeiro apos\n",
    "            print('termos_antes_cc:', termos_antes_cc)\n",
    "            print('termos_apos_cc:', termos_apos_cc)\n",
    "            for i, termo_apos in enumerate(termos_apos_cc):\n",
    "                nova_entidade = termos_antes_cc\n",
    "                for j in range(i, len(termos_apos_cc)):                \n",
    "                    nova_entidade += nova_entidade +' ' + termos_apos_cc[j]\n",
    "                    print('nova_entidade:', nova_entidade)\n",
    "            # só salva se nao tem ainda\n",
    "            \n",
    "            # quarto: gerar nova frase com a nova combinação (replace das tags especiaid)\n",
    "\n",
    "    \n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "'''\n",
    "#dicSentencesComLabel_tratandoDesc = {}\n",
    "\n",
    "# soma as descontinuas numa entidade só\n",
    "# ex: alfa and beta globina -> ENT, ENT, ENT, ENT (o 'AND' vira ENT)\n",
    "# de 17 a 22\n",
    "for fraseDescontinua in frasesComDescontinuas:\n",
    "    entidades = dicSentencesComLabel[fraseDescontinua][1]\n",
    "    tokens = dicSentencesComLabel[fraseDescontinua][0]\n",
    "    #print(entidades)\n",
    "    novasEnt = []\n",
    "    for entidade in entidades:\n",
    "        #print('entidade:', entidade)\n",
    "        menorIndice = entidade[1][0]\n",
    "        maiorIndice = entidade[1][-1]\n",
    "        novoIndice=[n for n in range(menorIndice, maiorIndice+1)]\n",
    "        #print(menorIndice)\n",
    "        #print(maiorIndice)\n",
    "        #print('novoIndice:', novoIndice)\n",
    "        novoTexto=[]\n",
    "        for token in tokens:\n",
    "            for indice in novoIndice:\n",
    "                if token[1] == indice:\n",
    "                    novoTexto.append(token[0])\n",
    "        #novoTexto = [token[0] for token in tokens where token[1] in novoIndice]\n",
    "        #print('novoTexto:', novoTexto)\n",
    "        novaEnt = [novoTexto, novoIndice]\n",
    "        novasEnt.append(novaEnt)\n",
    "    dicSentencesComLabel[fraseDescontinua][1] = novasEnt\n",
    "'''"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
